diff --speed-large-files --minimal -Nru rapidsvn-0.9.1.orig/configure.in rapidsvn-0.9.1/configure.in
--- rapidsvn-0.9.1.orig/configure.in	2006-02-12 17:20:59.569594000 -0500
+++ rapidsvn-0.9.1/configure.in	2006-02-12 17:20:51.478193000 -0500
@@ -7,8 +7,8 @@
 AC_PROG_LIBTOOL
 
 abs_srcdir="`cd $srcdir && pwd`"
-CPPFLAGS="$CPPFLAGS -Wall -g -fexceptions -I$abs_srcdir/include"
-CXXFLAGS="$CXXFLAGS -Wall -g -fexceptions"
+CPPFLAGS="$CPPFLAGS -I$abs_srcdir/include"
+CXXFLAGS="$CXXFLAGS"
 
 dnl
 dnl APR
@@ -204,7 +204,7 @@
                             when determining the cpp Unit configuration; 
                             defaults to ""]],
 [
-  if test "$withval" == "no"; then
+  if test "$withval" = "no"; then
     CPPUNITCONFIG=
   elif test "$withval" != "yes"; then
     CPPUNITCONFIG=$withval
@@ -230,7 +230,7 @@
 [[  --with-xsltproc=FILE     Use the given path to xsltproc; 
                             defaults to "xsltproc"]],
 [
-  if test "$withval" == "no"; then
+  if test "$withval" = "no"; then
     XSLTPROC=
   elif test "$withval" != "yes"; then
     XSLTPROC=$withval
@@ -262,7 +262,7 @@
                                      otherwise you are using the wrong stylesheet
                                      resulting in an invalid manpage]],
 [
-  if test "$withval" == "no"; then
+  if test "$withval" = "no"; then
     DOCBOOK_MANPAGES=
   else
     DOCBOOK_MANPAGES=$withval
@@ -274,7 +274,7 @@
 dnl         a for .. do .. done loop since the line
 dnl         got too long (file not found error while
 dnl         executing)
-if test "$DOCBOOK_MANPAGES" == "yes"; then
+if test "$DOCBOOK_MANPAGES" = "yes"; then
   X1=/usr/share/xml/docbook/stylesheet/nwalsh/manpages/docbook.xsl
   X2=/usr/share/sgml/docbook/stylesheet/xsl/nwalsh/manpages/docbook/docbook.xsl
   X3=/usr/share/sgml/docbook/yelp/docbook/manpages/docbook.xsl
@@ -329,7 +329,7 @@
 TARGET_MANPAGE=manpage-no
 MAN_MANS=
 AC_MSG_CHECKING([manpage creation])
-if test "$MANPAGE"=="no"; then
+if test "$MANPAGE" = "no"; then
   AC_MSG_RESULT([no])
 elif test "$XSLTPROC" -a "$DOCBOOK_MANPAGES" ; then
   AC_MSG_RESULT([yes])
@@ -348,7 +348,7 @@
                           Used for creating API documentation; 
                           defaults to "doxygen"]],
 [
-  if test "$withval" == "no"; then
+  if test "$withval" = "no"; then
     DOXYGEN=
   elif test "$withval" != "yes"; then
     DOXYGEN=$withval
@@ -375,7 +375,7 @@
                       This tool is needed by doxygen; 
                       defaults to "dot"]],
 [
-  if test "$withval" == "no"; then
+  if test "$withval" = "no"; then
     DOT=
   elif test "$withval" != "yes"; then
     DOT=$withval
