From ac6973fc238837f1618bef33603b82118a0f963d Mon Sep 17 00:00:00 2001
From: Maciej Blizinski <maciej@opencsw.org>
Date: Mon, 5 Jul 2010 00:43:50 +0200
Subject: [PATCH] Allowing CFLAGS from GAR.

---
 Makefile |    3 +--
 1 files changed, 1 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 3324dd3..ac03ef9 100644
--- a/Makefile
+++ b/Makefile
@@ -30,8 +30,7 @@ MANDIR=${prefix}/man/man1
 
 # Uncomment for Solaris:
 #CC=cc
-CFLAGS=-xO0 -v
-LDFLAGS=
+CFLAGS ?= -xO0 -v
 XOBJS=strverscmp.o
 MANDIR=${prefix}/share/man/man1
 INSTALL=ginstall
-- 
1.7.1

