--- tree-1.5.3/Makefile.orig	2010-02-02 09:04:08.804909345 +0100
+++ tree-1.5.3/Makefile	2010-02-02 09:04:12.780237318 +0100
@@ -6,9 +6,9 @@
 # warranties, including, without limitation, the implied warranties
 # of merchant-ability and fitness for a particular purpose.
 
-prefix = /usr
+prefix = /opt/csw
 
-CC=gcc
+# CC=gcc
 
 VERSION=1.5.3
 TREE_DEST=tree
@@ -19,7 +19,7 @@
 # Uncomment options below for your particular OS:
 
 # Linux defaults:
-CFLAGS=-ggdb -Wall -DLINUX -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64
+# CFLAGS=-ggdb -Wall -DLINUX -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64
 #CFLAGS=-O2 -Wall -fomit-frame-pointer -DLINUX -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64
 #LDFLAGS=-s
 
@@ -30,10 +30,11 @@
 
 # Uncomment for Solaris:
 #CC=cc
-#CFLAGS=-xO0 -v
-#LDFLAGS=
-#XOBJS=strverscmp.o
-#MANDIR=${prefix}/share/man/man1
+CFLAGS=-xO0 -v
+LDFLAGS=
+XOBJS=strverscmp.o
+MANDIR=${prefix}/share/man/man1
+INSTALL=ginstall
 
 # Uncomment for Cygwin:
 #CFLAGS=-O2 -Wall -fomit-frame-pointer -DCYGWIN
@@ -83,12 +84,12 @@
 	rm -f *~
 
 install: tree
-	install -d $(BINDIR)
-	install -d $(MANDIR)
-	if [ -e $(TREE_DEST) ]; then \
-		install -s $(TREE_DEST) $(BINDIR)/$(TREE_DEST); \
-	fi
-	install man/$(MAN) $(MANDIR)/$(MAN)
+	$(INSTALL) -d $(DESTDIR)$(BINDIR)
+	$(INSTALL) -d $(DESTDIR)$(MANDIR)
+	if [ -r $(TREE_DEST) ]; then \
+		$(INSTALL) -s $(TREE_DEST) $(DESTDIR)$(BINDIR)/$(TREE_DEST); \
+ 	fi
+	$(INSTALL) man/$(MAN) $(DESTDIR)$(MANDIR)/$(MAN)
 
 distclean:
 	if [ -f tree.o ]; then rm *.o; fi
