diff --speed-large-files --minimal -Nru subversion-1.2.0-rc1.orig/Makefile.in subversion-1.2.0-rc1/Makefile.in
--- subversion-1.2.0-rc1.orig/Makefile.in	2005-04-11 09:21:10.150005000 -0700
+++ subversion-1.2.0-rc1/Makefile.in	2005-04-11 09:18:19.860026000 -0700
@@ -66,14 +66,14 @@
 swig_rb_libdir = @libdir@
 
 ### these possibly need further discussion
-swig_pydir = @libdir@/svn-python/libsvn
-swig_pydir_extra = @libdir@/svn-python/svn
-swig_pldir = @libdir@/svn-perl
+swig_pydir = @prefix@/lib/python/site-packages/libsvn
+swig_pydir_extra = @prefix@/lib/python/site-packages/svn
+#swig_pldir = @libdir@/svn-perl
 swig_rbdir = $(SWIG_RB_SITE_ARCH_DIR)/svn/ext
 
-javahl_javadir = @libdir@/svn-javahl
-javahl_javahdir = @libdir@/svn-javahl/include
-javahl_libdir = @libdir@
+javahl_javadir = @libdir@
+javahl_javahdir = @libdir@/include
+javahl_libdir = @libdir@
 
 ### should search for these...
 MAKEINFO = @MAKEINFO@
