diff --speed-large-files --minimal -Nru MPlayer-1.0rc1.orig/libavcodec/vp56.h MPlayer-1.0rc1/liba
vcodec/vp56.h
--- MPlayer-1.0rc1.orig/libavcodec/vp56.h       2006-10-22 18:32:34.000000000 -0400
+++ MPlayer-1.0rc1/libavcodec/vp56.h    2007-07-13 08:59:08.157679000 -0400
@@ -24,7 +24,11 @@
 #ifndef VP56_H
 #define VP56_H

+#if defined(sun)
+#include <inttypes.h>
+#else
 #include <stdint.h>
+#endif

 #include "vp56data.h"
 #include "dsputil.h"
diff --speed-large-files --minimal -Nru MPlayer-1.0rc1.orig/libmpcodecs/vf_ass.c MPlayer-1.0rc1/l
ibmpcodecs/vf_ass.c
--- MPlayer-1.0rc1.orig/libmpcodecs/vf_ass.c    2006-10-22 18:32:24.000000000 -0400
+++ MPlayer-1.0rc1/libmpcodecs/vf_ass.c 2007-07-13 09:00:32.339044000 -0400
@@ -23,7 +23,13 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+
+#if defined(sun)
+#include <inttypes.h>
+#else
 #include <stdint.h>
+#endif
+
 #include <assert.h>

 #include "config.h"
