From e044f36f4e7c77202278f1dc24a5f42ceffbfeb5 Mon Sep 17 00:00:00 2001
From: Ihsan Dogan <ihsan@opencsw.org>
Date: Mon, 11 Jun 2012 11:28:48 +0200
Subject: [PATCH] amavisd-nanny

---
 amavisd-nanny |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/amavisd-nanny b/amavisd-nanny
index ddb95de..392841f 100755
--- a/amavisd-nanny
+++ b/amavisd-nanny
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -T
+#!/opt/csw/bin/perl -T
 
 #------------------------------------------------------------------------------
 # This is amavisd-nanny, a program to show the status
@@ -58,7 +58,7 @@ my($activettl) = 10*60; # stuck active children are sent a SIGTERM
 
 my($dbfile) = 'nanny.db';
 my($db_home) =  # DB databases directory
-  defined $ENV{'AMAVISD_DB_HOME'} ? $ENV{'AMAVISD_DB_HOME'} : '/var/amavis/db';
+  defined $ENV{'AMAVISD_DB_HOME'} ? $ENV{'AMAVISD_DB_HOME'} : '/var/opt/csw/amavisn/db';
 my($wakeuptime) = 2;  # -w, sleep time in seconds, may be fractional
 my($repeatcount);     # -c, repeat count (when defined)
 
-- 
1.7.10.3

