From 9990a716a025efad8d53a6788b8614c9bfd89075 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Wed, 19 Jan 2011 14:50:32 +0100
Subject: [PATCH 2/2] Take endianness into account on alignment check

---
 configure.ac |    8 +++++++-
 1 files changed, 7 insertions(+), 1 deletions(-)

diff --git a/configure.ac b/configure.ac
index 1fc9ebc..98b6812 100644
--- a/configure.ac
+++ b/configure.ac
@@ -425,7 +425,13 @@ AC_DEFUN([AC_C_ALIGNMENT],
        // catch unaligned word access (ARM cpus)
        *buf =  1; *(buf +1) = 2; *(buf + 2) = 3; *(buf + 3) = 4; *(buf + 4) = 5;
        int* i = (int*)(buf+1);
-       return (84148994 == i) ? 0 : 1;
+       return (
+#ifdef ENDIAN_BIG
+                0x02030405
+#else
+                0x05040302
+#endif
+                        == *i) ? 0 : 1;
     ])
   ],[
     ac_cv_c_alignment=none
-- 
1.7.3.2

