From c9367a8a9f1356e77a81d16cbb2586ffd904ca7e Mon Sep 17 00:00:00 2001
From: Geoff Davis <gadavis@opencsw.org>
Date: Fri, 14 Jan 2011 22:27:41 +0100
Subject: [PATCH] Add LDFLAGS to shared object linking commands

---
 src/Makefile       |    6 +++---
 src/x2sys/Makefile |    2 +-
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/Makefile b/src/Makefile
index 128b4fb..422df20 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -195,13 +195,13 @@ libgmtps.a:	$(LIBPS_O)
 		$(RANLIB) $@
 
 libpsl.$(SL):	libpsl.a
-		$(LD) $(LD_OPT) pslib.o $(LIBS) -o $@
+		$(LD) $(LD_OPT) $(LDFLAGS) pslib.o $(LIBS) -o $@
 
 libgmt.$(SL):	libgmt.a
-		$(LD) $(LD_OPT) $(LIB_O) $(TRIANGLE_O) $(ALPHA_SINCOS_O) $(NETCDF_LIB) $(GDAL_LIB) $(LIBS) -o $@
+		$(LD) $(LD_OPT) $(LDFLAGS) $(LIB_O) $(TRIANGLE_O) $(ALPHA_SINCOS_O) $(NETCDF_LIB) $(GDAL_LIB) $(LIBS) -o $@
 
 libgmtps.$(SL):	libgmtps.a libgmt.$(SL) libpsl.$(SL)
-		$(LD) $(LD_OPT) $(LIBPS_O) libgmt.$(SL) libpsl.$(SL) $(LIBS) -o $@
+		$(LD) $(LD_OPT) $(LDFLAGS) $(LIBPS_O) libgmt.$(SL) libpsl.$(SL) $(LIBS) -o $@
 
 #-------------------------------------------------------------------------------
 #	program rules
diff --git a/src/x2sys/Makefile b/src/x2sys/Makefile
index 23732ef..2ccf224 100644
--- a/src/x2sys/Makefile
+++ b/src/x2sys/Makefile
@@ -83,7 +83,7 @@ libx2sys.a:	$(LIB_O)
 		$(RANLIB) $@
 
 libx2sys.$(SL):	libx2sys.a
-		$(LD) $(LD_OPT) x2sys.o $(ALLLIB) -o $@
+		$(LD) $(LD_OPT) $(LDFLAGS) x2sys.o $(ALLLIB) -o $@
 
 #-------------------------------------------------------------------------------
 #	program rules
-- 
1.7.3.2

