index b77a8a8..cb4872a 100755
--- a/configure
+++ b/configure
@@ -267,8 +267,8 @@ else
              LDSHARED=${LDSHARED-"cc -dy -KPIC -G"} ;;
   SunOS\ 5* | solaris*)
          LDSHARED=${LDSHARED-"cc -G -h libz$shared_ext.$VER1"}
-         SFLAGS=${CFLAGS-"-fast -KPIC"}
-         CFLAGS=${CFLAGS-"-fast"}
+         SFLAGS=${CFLAGS-"-KPIC"}
+         CFLAGS=${CFLAGS-""}
          if test $build64 -eq 1; then
              # old versions of SunPRO/Workshop/Studio don't support -m64,
              # but newer ones do.  Check for it.
