--- zlib-1.2.5/Makefile.in.orig	2010-05-26 12:54:35.283218000 +0200
+++ zlib-1.2.5/Makefile.in	2010-05-26 13:13:37.622441685 +0200
@@ -137,7 +137,7 @@
 	-@mv objs/$*.o $@
 
 $(SHAREDLIBV): $(PIC_OBJS)
-	$(LDSHARED) $(SFLAGS) -o $@ $(PIC_OBJS) $(LDSHAREDLIBC) $(LDFLAGS)
+	$(LDSHARED) $(SFLAGS) -h $(SHAREDLIBM) -o $@ $(PIC_OBJS) $(LDSHAREDLIBC) $(LDFLAGS)
 	rm -f $(SHAREDLIB) $(SHAREDLIBM)
 	ln -s $@ $(SHAREDLIB)
 	ln -s $@ $(SHAREDLIBM)
