diff --speed-large-files --minimal -Nru libxml-perl-0.07.orig/t/stream.t libxml-perl-0.07/t/stream.t
--- libxml-perl-0.07.orig/t/stream.t	1999-08-16 09:04:03.000000000 -0700
+++ libxml-perl-0.07/t/stream.t	2003-08-17 18:14:40.540196000 -0700
@@ -41,7 +41,7 @@
     3rd line in bar <!-- Isn't this a doozy -->
   </bar>
   <zap ref="zing" />
-  This, '\240', would be a bad character in UTF-8.
+  This, '\302\240', would be a bad character in UTF-8.
 </foo>
 EOF;
 
