diff --speed-large-files --minimal -Nru firefox.orig/browser/app/firefox firefox/browser/app/firefox
--- firefox.orig/browser/app/firefox    2009-08-27 23:43:31.266507053 +0200
+++ firefox/browser/app/firefox     	2009-08-27 23:44:53.015182925 +0200
@@ -54,6 +54,11 @@

 moz_libdir=/opt/csw/mozilla/firefox/lib

+if [ "`uname -r`" = "5.8" ] ; then
+       LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/lib/lwp
+       LD_LIBRARY_PATH_64=$LD_LIBRARY_PATH_64:/usr/lib/lwp/64
+fi
+
 # Use run-mozilla.sh in the current dir if it exists
 # If not, then start resolving symlinks until we find run-mozilla.sh
 found=0
