diff --speed-large-files --minimal -Nru firefox.orig/toolkit/crashreporter/google-breakpad/src/common/md5.h firefox/toolkit/crashreporter/google-breakpad/src/common/md5.h
--- firefox.orig/toolkit/crashreporter/google-breakpad/src/common/md5.h    2007-10-21 15:36:05.000000000 +0000
+++ firefox/toolkit/crashreporter/google-breakpad/src/common/md5.h       2008-11-24 21:59:04.857036600 +0000
@@ -3,7 +3,7 @@
 #ifndef COMMON_MD5_H__
 #define COMMON_MD5_H__
 
-#include <stdint.h>
+#include <inttypes.h>
 
 typedef uint32_t u32;
 typedef uint8_t u8;
