From 3da2e2e562fc914c6f908cebf4e6c29210615136 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Mon, 2 Nov 2009 09:41:10 +0100
Subject: [PATCH 1/2] Fix types for 64 bit

---
 inc/private.h |   10 ++++++----
 1 files changed, 6 insertions(+), 4 deletions(-)

diff --git a/inc/private.h b/inc/private.h
index 0c94255..316c328 100644
--- a/inc/private.h
+++ b/inc/private.h
@@ -9,11 +9,13 @@
 #ifndef	PRIVATE_H
 #define	PRIVATE_H
 
-typedef short			word;		/* 16 bit signed int	*/
-typedef long			longword;	/* 32 bit signed int	*/
+#include <sys/int_types.h>
 
-typedef unsigned short		uword;		/* unsigned word	*/
-typedef unsigned long		ulongword;	/* unsigned longword	*/
+typedef int16_t			word;		/* 16 bit signed int	*/
+typedef int32_t			longword;	/* 32 bit signed int	*/
+
+typedef uint16_t		uword;		/* unsigned word	*/
+typedef uint32_t		ulongword;	/* unsigned longword	*/
 
 struct gsm_state {
 
-- 
1.6.5

