--- openssh-5.4p1/servconf.c.orig	Wed Mar 24 21:56:53 2010
+++ openssh-5.4p1/servconf.c	Wed Mar 24 21:56:29 2010
@@ -492,7 +492,7 @@
 	expanded = tilde_expand_filename(path, getuid());
 	if (*expanded == '/')
 		return expanded;
-	if ((cwd = getcwd(NULL, 0)) == NULL)
+	if ((cwd = getcwd(NULL, PATH_MAX)) == NULL)
 		fatal("%s: getcwd: %s", __func__, strerror(errno));
 	xasprintf(&ret, "%s/%s", cwd, expanded);
 	xfree(cwd);
