diff --speed-large-files --minimal -Nru sshpty.c.orig openssh-4.7p1/sshpty.c
--- openssh-5.2p1/sshpty.c.orig	2006-08-04 22:39:41.000000000 -0400
+++ openssh-5.2p1.new/sshpty.c	2007-11-02 17:56:19.136541000 -0400
@@ -139,7 +139,10 @@
 		close(fd);
 	}
 	/* Make it our controlling tty. */
-#ifdef TIOCSCTTY
+/* Blastwave package modification: TIOCSCTTY doesn't exist under solaris 10
+   although it is defined in /usr/include/sys/termios.h (??), so we make
+   sure it is not used. */
+#ifdef 0
 	debug("Setting controlling tty using TIOCSCTTY.");
 	if (ioctl(*ttyfd, TIOCSCTTY, NULL) < 0)
 		error("ioctl(TIOCSCTTY): %.100s", strerror(errno));
