From 0165c44eebad6a97e5dfbd476df9988136c40bc0 Mon Sep 17 00:00:00 2001
From: Toni Schmidbauer <toni@stderr.at>
Date: Wed, 5 Nov 2014 10:22:08 +0100
Subject: [PATCH] added /opt/csw/bin to find_exectuable0

---
 lib/mkmf.rb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/mkmf.rb b/lib/mkmf.rb
index 8b19eb6..c54e9cf 100644
--- a/lib/mkmf.rb
+++ b/lib/mkmf.rb
@@ -1500,7 +1500,7 @@ SRC
     if path ||= ENV['PATH']
       path = path.split(File::PATH_SEPARATOR)
     else
-      path = %w[/usr/local/bin /usr/ucb /usr/bin /bin]
+      path = %w[/opt/csw/bin /usr/local/bin /usr/ucb /usr/bin /bin]
     end
     file = nil
     path.each do |dir|
-- 
2.0.3

