diff --speed-large-files --minimal -Nru libgpod-0.3.2.orig/libgpod-1.0.pc.in libgpod-0.3.2/libgpod-1.0.pc.in
--- libgpod-0.3.2.orig/libgpod-1.0.pc.in	2005-09-11 05:10:12.000000000 -0400
+++ libgpod-0.3.2/libgpod-1.0.pc.in	2006-09-23 01:43:53.556584000 -0400
@@ -8,4 +8,4 @@
 Version: @VERSION@
 Requires: glib-2.0 
 Libs: -L${libdir} -lgpod
-Cflags: -I${includedir}/gpod-1.0
+Cflags: -I${includedir}
diff --speed-large-files --minimal -Nru libgpod-0.3.2.orig/src/Makefile.in libgpod-0.3.2/src/Makefile.in
--- libgpod-0.3.2.orig/src/Makefile.in	2006-03-04 08:15:50.000000000 -0500
+++ libgpod-0.3.2/src/Makefile.in	2006-09-23 01:44:18.169781000 -0400
@@ -225,7 +225,7 @@
         -no-undefined
 
 
-libgpodincludedir = $(includedir)/gpod-1.0/gpod
+libgpodincludedir = $(includedir)/gpod
 libgpodinclude_HEADERS = $(libgpod_la_headers)
 
 INCLUDES = $(LIBGPOD_CFLAGS)
