diff --speed-large-files --minimal -Nru libgpod-0.4.2.orig/src/Makefile.in libgpod-0.4.2/src/Makefile.in
--- libgpod-0.4.2.orig/src/Makefile.in	2007-01-16 04:46:06.000000000 +1300
+++ libgpod-0.4.2/src/Makefile.in	2007-10-07 21:51:37.681516496 +1300
@@ -295,7 +295,7 @@
 libgpod_la_headers = itdb.h
 libgpod_la_noinst_headers = itdb_private.h hal-common.h
 libgpod_la_LDFLAGS = -version-info $(LIBGPOD_SO_VERSION) -no-undefined
-libgpodincludedir = $(includedir)/gpod-1.0/gpod
+libgpodincludedir = $(includedir)/gpod
 libgpodinclude_HEADERS = $(libgpod_la_headers)
 INCLUDES = $(LIBGPOD_CFLAGS)
 all: all-am
diff --speed-large-files --minimal -Nru libgpod-0.4.2.orig/libgpod-1.0.pc.in libgpod-0.4.2/libgpod-1.0.pc.in
--- libgpod-0.4.2.orig/libgpod-1.0.pc.in        2005-09-11 21:10:12.000000000 +1200
+++ libgpod-0.4.2/libgpod-1.0.pc.in     2007-10-16 23:22:31.985717435 +1300
@@ -8,4 +8,4 @@
 Version: @VERSION@
 Requires: glib-2.0
 Libs: -L${libdir} -lgpod
-Cflags: -I${includedir}/gpod-1.0
+Cflags: -I${includedir}/gpod
