From ea3087bf5e1a881624e84233f9b1d6f48e141c76 Mon Sep 17 00:00:00 2001
From: Rafael Ostertag <rafisol@opencsw.org>
Date: Sun, 1 Jan 2012 12:41:52 +0100
Subject: [PATCH] Use plain rpath

---
 src/makefiles/Makefile.solaris |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/makefiles/Makefile.solaris b/src/makefiles/Makefile.solaris
index 79c92bc..00358f1 100644
--- a/src/makefiles/Makefile.solaris
+++ b/src/makefiles/Makefile.solaris
@@ -6,7 +6,7 @@ ifeq ($(with_gnu_ld), yes)
 export_dynamic = -Wl,-E
 rpath = -Wl,-rpath,'$(rpathdir)'
 else
-rpath = -Wl,-R'$(rpathdir)'
+rpath = -R$(rpathdir)
 endif
 
 DLSUFFIX = .so
-- 
1.7.6.1

