From dbf9d0c63719b114bd938e5d34dd08f08c0c20d4 Mon Sep 17 00:00:00 2001
From: Romeo Theriault <romeotheriault@opencsw.org>
Date: Fri, 24 Aug 2012 09:42:20 +0200
Subject: [PATCH] Dont use case range

---
 msgpack/unpack_template.h | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/msgpack/unpack_template.h b/msgpack/unpack_template.h
index b844a24..1add413 100644
--- a/msgpack/unpack_template.h
+++ b/msgpack/unpack_template.h
@@ -40,11 +40,6 @@
 #error msgpack_unpack_user type is not defined
 #endif
 
-#ifndef USE_CASE_RANGE
-#if !defined(_MSC_VER)
-#define USE_CASE_RANGE
-#endif
-#endif
 
 msgpack_unpack_struct_decl(_stack) {
 	msgpack_unpack_object obj;
-- 
1.7.11.3

