diff --speed-large-files --minimal -Nru OOF640_m14.orig/vcl/util/makefile.mk OOF640_m14/vcl/util/makefile.mk
--- OOF640_m14.orig/vcl/util/makefile.mk	Fri Jan 19 07:56:23 2007
+++ OOF640_m14/vcl/util/makefile.mk	Mon Jun 11 16:21:22 2007
@@ -290,7 +290,7 @@
 .ENDIF
 .IF "$(OS)"=="SOLARIS"
 # needed by libaudio.a
-SHL2STDLIBS += -ldl -lnsl -lsocket
+SHL2STDLIBS += -ldl -lnsl -lsocket -lXt
 .ENDIF # SOLARIS
 .ENDIF
 
