From 3cca02001f6e24683cfe890f642dd240d171656b Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Fri, 14 May 2010 10:30:11 +0200
Subject: [PATCH 2/2] Add /etc/opt/csw as config location

---
 pca |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/pca b/pca
index 7788971..512dc4d 100644
--- a/pca
+++ b/pca
@@ -2432,11 +2432,13 @@ sub parse_args {
   if ($o{proxy}) {
     push (@conf, dirname($0)."/../etc/pca-proxy.conf");
     push (@conf, "/etc/pca-proxy.conf");
+    push (@conf, "/etc/opt/csw/pca-proxy.conf");
     push (@conf, "./pca-proxy.conf");
   } else {
     push (@conf, dirname($0)."/pca.conf");
     push (@conf, dirname($0)."/../etc/pca.conf");
     push (@conf, "/etc/pca.conf");
+    push (@conf, "/etc/opt/csw/pca.conf");
     $ENV{HOME} && push (@conf, $ENV{HOME}."/.pca");
     push (@conf, "./pca.conf");
   }
-- 
1.7.0

