diff -ru ../build-global/wput-0.6.2/src/Makefile.in wput-0.6.2/src/Makefile.in
--- ../build-global/wput-0.6.2/src/Makefile.in	2008-09-27 12:31:23.000000000 +0200
+++ wput-0.6.2/src/Makefile.in	2009-03-29 18:24:32.132443546 +0200
@@ -6,7 +6,7 @@
 localedir=$(prefix)/share/locale
 CC=@CC@
 CFLAGS= @CFLAGS@ -DLOCALEDIR=\"$(localedir)\" @GNUTLS_CFLAGS@
-LIBS=@LIBINTL@ @LIBS@ @GNUTLS_LIBS@
+LIBS=@LIBINTL@ @LIBS@ @GNUTLS_LIBS@ -lsocket -lnsl
 EXE=../wput
 GETOPT=@GETOPT@
 MEMDBG=@MEMDBG@
