diff -Naur flac-1.1.4.orig/ltmain.sh flac-1.1.4.patched/ltmain.sh
--- flac-1.1.4.orig/ltmain.sh	2005-04-26 20:23:39.000000000 +0200
+++ flac-1.1.4.patched/ltmain.sh	2009-04-09 13:59:57.521039544 +0200
@@ -1545,9 +1545,7 @@
 	esac
         compile_command="$compile_command $arg"
         finalize_command="$finalize_command $arg"
-        if test "$with_gcc" = "yes" ; then
           compiler_flags="$compiler_flags $arg"
-        fi
         continue
         ;;
 
