From 1b0107dad51359fac86ab204c2331894b67be54c Mon Sep 17 00:00:00 2001
From: Maciej Blizinski <maciej@opencsw.org>
Date: Fri, 4 Oct 2013 23:49:15 +0200
Subject: [PATCH] Remove GCC-incompatible compilation flags.

---
 setup_posix.py |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup_posix.py b/setup_posix.py
index f7cb588..86e9a1b 100644
--- a/setup_posix.py
+++ b/setup_posix.py
@@ -52,7 +52,7 @@ def get_config():
     library_dirs = [ dequote(i[2:]) for i in libs if i.startswith(compiler_flag("L")) ]
     libraries = [ dequote(i[2:]) for i in libs if i.startswith(compiler_flag("l")) ]
 
-    removable_compile_args = [ compiler_flag(f) for f in "ILl" ]
+    removable_compile_args = [ compiler_flag(f) for f in "ILlKmxfn" ]
     extra_compile_args = [ i.replace("%", "%%") for i in mysql_config("cflags")
                            if i[:2] not in removable_compile_args ]
 
-- 
1.7.10.3

