diff -ur tcl8.4.19/unix/Makefile.in tcl8.4.19-solfix/unix/Makefile.in
--- tcl8.4.19/unix/Makefile.in	2008-04-17 22:29:49.000000000 +0200
+++ tcl8.4.19-solfix/unix/Makefile.in	2011-09-09 20:20:12.621101557 +0200
@@ -207,7 +207,7 @@
 SHLIB_LD		= @SHLIB_LD@
 SHLIB_CFLAGS		= @SHLIB_CFLAGS@
 SHLIB_LD_LIBS		= @SHLIB_LD_LIBS@
-TCL_SHLIB_LD_EXTRAS	= @TCL_SHLIB_LD_EXTRAS@
+TCL_SHLIB_LD_EXTRAS     = @TCL_SHLIB_LD_EXTRAS@
 
 SHLIB_SUFFIX		= @SHLIB_SUFFIX@
 #SHLIB_SUFFIX		=
@@ -634,7 +634,9 @@
 	    fi
 	@echo "Installing $(LIB_FILE) to $(LIB_INSTALL_DIR)/"
 	@@INSTALL_LIB@
-	@chmod 555 $(LIB_INSTALL_DIR)/$(LIB_FILE)
+	mv "$(LIB_INSTALL_DIR)"/$(LIB_FILE) "$(LIB_INSTALL_DIR)"/$(LIB_FILE).0
+	ln -sf $(LIB_FILE).0 "$(LIB_INSTALL_DIR)"/$(LIB_FILE)
+	@chmod 555 "$(LIB_INSTALL_DIR)"/$(LIB_FILE).0
 	@if test "$(TCL_BUILD_EXP_FILE)" != ""; then \
 	    echo "Installing $(TCL_EXP_FILE) to $(LIB_INSTALL_DIR)/"; \
 	    $(INSTALL_DATA) $(TCL_BUILD_EXP_FILE) \
diff -ur tcl8.4.19/unix/tcl.m4 tcl8.4.19-solfix/unix/tcl.m4
--- tcl8.4.19/unix/tcl.m4	2008-04-17 22:29:49.000000000 +0200
+++ tcl8.4.19-solfix/unix/tcl.m4	2011-09-09 20:21:56.672149153 +0200
@@ -1843,6 +1843,7 @@
 	    # symbols when dynamically loaded into tclsh.
 
 	    SHLIB_LD_LIBS='${LIBS}'
+	    TCL_SHLIB_LD_EXTRAS="-h \${TCL_LIB_FILE}.0"
 	    SHLIB_SUFFIX=".so"
 	    DL_OBJS="tclLoadDl.o"
 	    DL_LIBS="-ldl"
