index 1b2d361..89a885a 100644
--- a/src/bin/jp2/convert.c
+++ b/src/bin/jp2/convert.c
@@ -191,8 +191,8 @@ static int tga_readheader(FILE *fp, unsigned int *bits_per_pixel,
 
 static INLINE int16_t swap16(int16_t x)
 {
-    return((((u_int16_t)x & 0x00ffU) <<  8) |
-           (((u_int16_t)x & 0xff00U) >>  8));
+    return((((uint16_t)x & 0x00ffU) <<  8) |
+           (((uint16_t)x & 0xff00U) >>  8));
 }
 
 #endif
