diff --speed-large-files --minimal -Nru clearsilver-0.10.2.orig/dso/Makefile clearsilver-0.10.2/dso/Makefile
--- clearsilver-0.10.2.orig/dso/Makefile	2006-01-11 01:32:45.201111000 -0500
+++ clearsilver-0.10.2/dso/Makefile	2006-01-11 01:32:09.442125000 -0500
@@ -17,7 +17,7 @@
 all: $(TARGETS)
 
 $(NEO_SO): $(NEO_STATIC_LIBS) $(DEP_LIBS) Makefile 
-	 /usr/bin/ld  -rpath ../libs -shared -o libneo.so -whole-archive ../libs/libneo_cgi.a ../libs/libneo_cs.a ../libs/libneo_utl.a -no-whole-archive $(LIBS)
+	 ld  -rpath ../libs -shared -o libneo.so -whole-archive ../libs/libneo_cgi.a ../libs/libneo_cs.a ../libs/libneo_utl.a -no-whole-archive $(LIBS)
 
 #	$(LDSHARED) -o $@ $(LDFLAGS) -Wl,-whole-archive $(DLIBS)
 
@@ -26,7 +26,7 @@
 	$(INSTALL) $(TARGETS) $(DESTDIR)$(PYTHON_SITE)
 
 dsotest: dsotest.c $(NEO_SO)
-	gcc -o dsotest dsotest.c -lneo -L. -I..
+	$(CC) -o dsotest dsotest.c -lneo -L. -I..
 
 clean:
 	$(RM) *.o 
diff --speed-large-files --minimal -Nru clearsilver-0.10.2.orig/rules.mk.in clearsilver-0.10.2/rules.mk.in
--- clearsilver-0.10.2.orig/rules.mk.in	2006-01-11 01:32:35.894992000 -0500
+++ clearsilver-0.10.2/rules.mk.in	2006-01-11 01:32:29.237180000 -0500
@@ -50,7 +50,7 @@
 
 @USE_MINGW32@
 
-PICFLG = -fPIC
+PICFLG = -KPIC
 ifeq ($(OSTYPE),OSF1)
 PICFLG =
 endif
@@ -58,7 +58,6 @@
 PICFLG =
 endif
 
-
 ## -------------- base (Linux/Neotonic) options
 
 PYTHON_INC = @PYTHON_INC@
@@ -84,12 +83,12 @@
 RUBY       = @RUBY@
 
 CPPFLAGS   = -I$(NEOTONIC_ROOT) @CPPFLAGS@
-CFLAGS     = @CFLAGS@ -Wall $(CPPFLAGS) $(PICFLG)
+CFLAGS     = @CFLAGS@ $(CPPFLAGS) $(PICFLG)
 OUTPUT_OPTION = -o $@
 LD         = $(CC) -o
 LDFLAGS    = -L$(LIB_DIR) @LDFLAGS@
-LDSHARED   = $(CC) -shared $(PICFLG)
-CPPLDSHARED   = $(CPP) -shared $(PICFLG)
+LDSHARED   = $(CC) -G $(PICFLG)
+CPPLDSHARED   = $(CPP) -G $(PICFLG)
 AR         = @AR@ cr
 RANLIB     = @RANLIB@
 DEP_LIBS   = $(DLIBS:-l%=$(LIB_DIR)lib%.a)
@@ -103,7 +102,7 @@
 
 ## I don't really feel like writing a configure thing for this yet
 ifeq ($(OSNAME),SunOS)
-LDSHARED   = ld -G -fPIC
+LDSHARED   = ld -G $(PICFLG)
 endif
 
 ## --------------win32 options
@@ -154,7 +153,7 @@
 	@touch Makefile.depends
 	@if test "x" != "x$(SOURCE_FILES)"; then \
 	  for II in "$(SOURCE_FILES)"; do \
-		gcc -M -MG ${CFLAGS} $$II >> Makefile.depends; \
+		$(CC) -xM ${CFLAGS} $$II >> Makefile.depends; \
 	  done; \
 	 fi
 	@echo "** (done) "
