diff --speed-large-files --minimal -Nru firefox.orig/toolkit/mozapps/installer/packager.mk firefox/toolkit/mozapps/installer/packager.mk
--- firefox.orig/toolkit/mozapps/installer/packager.mk	2008-04-28 19:14:12.000000000 +0000
+++ firefox/toolkit/mozapps/installer/packager.mk	2008-11-15 20:28:41.679741300 +0000
@@ -363,7 +363,7 @@
 endif # MOZ_PKG_MANIFEST
 ifndef PKG_SKIP_STRIP
 	@echo "Stripping package directory..."
-	@cd $(DIST)/$(STAGEPATH)$(MOZ_PKG_APPNAME); find . ! -type d \
+	@cd $(DIST)/$(STAGEPATH)$(MOZ_PKG_APPNAME); gfind . ! -type d \
 			! -name "*.js" \
 			! -name "*.xpt" \
 			! -name "*.gif" \
@@ -426,7 +426,7 @@
 	(cd $(DIST)/sdk/include && tar $(TAR_CREATE_FLAGS) - .) | \
 	  (cd $(DESTDIR)$(includedir)/stable && tar -xf -)
 # The dist/include has module subdirectories that we need to flatten
-	find $(DIST)/include -xtype f -exec $(SYSINSTALL) $(IFLAGS1) {} $(DESTDIR)$(includedir)/unstable \;
+	gfind $(DIST)/include -xtype f -exec $(SYSINSTALL) $(IFLAGS1) {} $(DESTDIR)$(includedir)/unstable \;
 # IDL directory is stable (dist/sdk/idl) and unstable (dist/idl)
 	$(NSINSTALL) -D $(DESTDIR)$(idldir)/stable 
 	$(NSINSTALL) -D $(DESTDIR)$(idldir)/unstable
