diff --speed-large-files --minimal -Nru  firefox.orig/toolkit/crashreporter/google-breakpad/src/client/solaris/handler/solaris_lwp.h firefox/toolkit/crashreporter/google-breakpad/src/client/solaris/handler/solaris_lwp.h
--- firefox.orig/toolkit/crashreporter/google-breakpad/src/client/solaris/handler/solaris_lwp.h  2008-03-19 01:48:21.000000000 +0000
+++ firefox/toolkit/crashreporter/google-breakpad/src/client/solaris/handler/solaris_lwp.h       2008-11-24 21:18:58.900019700 +0000
@@ -41,7 +41,7 @@
 #endif
 
 #include <signal.h>
-#include <stdint.h>
+#include <inttypes.h>
 #include <sys/user.h>
 #include <ucontext.h>

