From 9c1bc0a961436ef420f1b379f58309db7ce43d60 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Fri, 14 May 2010 12:01:53 +0200
Subject: [PATCH] Check against gids 0 and 1 for root on Solaris

---
 cpio/test/test_owner_parse.c |    6 ++++++
 1 files changed, 6 insertions(+), 0 deletions(-)

diff --git a/cpio/test/test_owner_parse.c b/cpio/test/test_owner_parse.c
index 9875e91..a3ebccd 100644
--- a/cpio/test/test_owner_parse.c
+++ b/cpio/test/test_owner_parse.c
@@ -30,8 +30,14 @@ __FBSDID("$FreeBSD$");
 
 #if !defined(_WIN32)
 #define ROOT "root"
+#if defined(__sun)
+static int root_uids[] = { 0 };
+/* Solaris 9 has gid 1 (=other) */
+static int root_gids[] = { 0, 1 };
+#else
 static int root_uids[] = { 0 };
 static int root_gids[] = { 0 };
+#endif
 #elif defined(__CYGWIN__)
 /* On cygwin, the Administrator user most likely exists (unless
  * it has been renamed or is in a non-English localization), but
-- 
1.7.0

