diff --git a/sudosh-replay.8.in b/sudosh-replay.8.in
index e4b93c0..b2159e0 100644
--- a/sudosh-replay.8.in
+++ b/sudosh-replay.8.in
@@ -1,7 +1,6 @@
 .\" Process this file with
 .\" groff -man -Tascii sudosh-replay.1
 .\"
-
 .TH SUDOSH-REPLAY 8 "December 20 2007" @VERSION@ "User Manuals"
 .SH NAME
 sudosh-replay \- view and replay sudosh sessions
diff --git a/sudosh.1.in b/sudosh.1.in
index de95301..cd60622 100644
--- a/sudosh.1.in
+++ b/sudosh.1.in
@@ -1,7 +1,6 @@
 .\" Process this file with
 .\" groff -man -Tascii sudosh.1
 .\"
-
 .TH SUDOSH 1 "December 21th 2007" @VERSION@ "User Manuals"
 .SH NAME
 @PACKAGE_NAME@ \- works like a VCR and records shell sessions.
diff --git a/sudosh.conf.5.in b/sudosh.conf.5.in
index 9a42617..1d115fe 100644
--- a/sudosh.conf.5.in
+++ b/sudosh.conf.5.in
@@ -1,7 +1,6 @@
 .\" Process this file with
 .\" groff -man -Tascii sudosh.1
 .\"
-
 .TH SUDOSH.CONF 5 "December 21th 2007" @VERSION@ "User Manuals"
 .SH NAME
 sudosh.conf \- configuration file for sudosh
