From 3d21a9253d711ada5c766a9968009073791ee4eb Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Wed, 9 Nov 2011 13:22:04 +0100
Subject: [PATCH] Disable strange check, this may be dangerous!

---
 src/misc/threads.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/misc/threads.c b/src/misc/threads.c
index bbbb483..1eb612e 100644
--- a/src/misc/threads.c
+++ b/src/misc/threads.c
@@ -250,6 +250,6 @@ void vlc_global_mutex (unsigned n, bool acquire)
         vlc_mutex_unlock (lock);
 
     /* Compile-time assertion ;-) */
-    char enough_locks[(sizeof (locks) / sizeof (locks[0])) - VLC_MAX_MUTEX];
-    (void) enough_locks;
+    /* char enough_locks[(sizeof (locks) / sizeof (locks[0])) - VLC_MAX_MUTEX];
+    (void) enough_locks; */
 }
-- 
1.7.6.1

