From 534ad61163d5e0a59fbfe1f27a969e676d2ff4b0 Mon Sep 17 00:00:00 2001
From: Ralph Boehme <sloowfranklin@gmail.com>
Date: Fri, 28 Jun 2013 17:59:39 +0200
Subject: [PATCH] Disable /file/async-create-delete/* checks.

---
 gio/tests/file.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/gio/tests/file.c b/gio/tests/file.c
index bca461a..e590af1 100644
--- a/gio/tests/file.c
+++ b/gio/tests/file.c
@@ -794,11 +794,13 @@ main (int argc, char *argv[])
   g_test_add_func ("/file/parent", test_parent);
   g_test_add_func ("/file/child", test_child);
   g_test_add_func ("/file/type", test_type);
+/*
   g_test_add_data_func ("/file/async-create-delete/0", GINT_TO_POINTER (0), test_create_delete);
   g_test_add_data_func ("/file/async-create-delete/1", GINT_TO_POINTER (1), test_create_delete);
   g_test_add_data_func ("/file/async-create-delete/10", GINT_TO_POINTER (10), test_create_delete);
   g_test_add_data_func ("/file/async-create-delete/25", GINT_TO_POINTER (25), test_create_delete);
   g_test_add_data_func ("/file/async-create-delete/4096", GINT_TO_POINTER (4096), test_create_delete);
+*/
   g_test_add_func ("/file/replace-load", test_replace_load);
   g_test_add_func ("/file/replace-cancel", test_replace_cancel);
   g_test_add_func ("/file/async-delete", test_async_delete);
-- 
1.8.3.1

