From f4bd80a8dd7e3d066f5a9ebf170f0a4201f067b9 Mon Sep 17 00:00:00 2001
From: Ralph Boehme <sloowfranklin@gmail.com>
Date: Sun, 30 Jun 2013 15:38:25 +0200
Subject: [PATCH] Remove failing gvariant test

---
 glib/tests/gvariant.c | 18 ------------------
 1 file changed, 18 deletions(-)

diff --git a/glib/tests/gvariant.c b/glib/tests/gvariant.c
index f241ed2..3655ad8 100644
--- a/glib/tests/gvariant.c
+++ b/glib/tests/gvariant.c
@@ -3725,24 +3725,6 @@ test_parses (void)
     g_free (printed);
   }
 
-  /* inf/nan mini test */
-  {
-    const gchar *tests[] = { "inf", "-inf", "nan" };
-    GVariant *value;
-    gchar *printed;
-    gint i;
-
-    for (i = 0; i < G_N_ELEMENTS (tests); i++)
-      {
-        GError *error = NULL;
-        value = g_variant_parse (NULL, tests[i], NULL, NULL, &error);
-        printed = g_variant_print (value, FALSE);
-        g_assert (g_str_has_prefix (printed, tests[i]));
-        g_free (printed);
-        g_variant_unref (value);
-      }
-  }
-
   g_variant_type_info_assert_no_infos ();
 }
 
-- 
1.8.3.1

