From 0b407db28cdb9658e9c40194b59f21d58773379f Mon Sep 17 00:00:00 2001
From: Ralph Boehme <sloowfranklin@gmail.com>
Date: Tue, 2 Jul 2013 07:24:51 +0200
Subject: [PATCH] Remove failing tests, linking is broken

---
 tests/gobject/Makefile.in | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/tests/gobject/Makefile.in b/tests/gobject/Makefile.in
index b109161..3bd60bc 100644
--- a/tests/gobject/Makefile.in
+++ b/tests/gobject/Makefile.in
@@ -116,7 +116,6 @@ am__v_lt_1 =
 am__EXEEXT_1 = deftype$(EXEEXT) gvalue-test$(EXEEXT) \
 	paramspec-test$(EXEEXT) accumulator$(EXEEXT) \
 	defaultiface$(EXEEXT) dynamictype$(EXEEXT) override$(EXEEXT) \
-	performance$(EXEEXT) performance-threaded$(EXEEXT) \
 	singleton$(EXEEXT) references$(EXEEXT)
 accumulator_SOURCES = accumulator.c
 accumulator_OBJECTS = accumulator.$(OBJEXT)
@@ -719,8 +718,6 @@ test_programs = \
 	defaultiface				\
 	dynamictype				\
 	override				\
-	performance				\
-	performance-threaded			\
 	singleton				\
 	references
 
-- 
1.8.3.1

