From 26f9938de29f65324acaedf376801f2e40f04b27 Mon Sep 17 00:00:00 2001
From: =?utf-8?q?Maciej=20Blizi=C5=84ski?= <blizinski@google.com>
Date: Tue, 8 Sep 2009 20:02:35 +0100
Subject: [PATCH 1/2] remove __extension__ from util.c

---
 secmem/util.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/secmem/util.c b/secmem/util.c
index 580fd34..c836ad0 100644
--- a/secmem/util.c
+++ b/secmem/util.c
@@ -34,7 +34,7 @@
 
 #ifndef TEMP_FAILURE_RETRY
 #define TEMP_FAILURE_RETRY(expression) \
-  (__extension__							      \
+  (             							      \
     ({ long int __result;						      \
        do __result = (long int) (expression);				      \
        while (__result == -1L && errno == EINTR);			      \
-- 
1.6.3.2

