diff --speed-large-files --minimal -Nru thunderbird.orig/config/autoconf.mk.in thunderbird/config/autoconf.mk.in
--- thunderbird.orig/config/autoconf.mk.in	2006-09-14 14:07:03.000000000 -0400
+++ thunderbird/config/autoconf.mk.in	2007-02-25 07:21:36.245899000 -0500
@@ -57,14 +57,14 @@
 prefix         = @prefix@
 exec_prefix    = @exec_prefix@
 bindir         = @bindir@
-includedir     = @includedir@/$(MOZ_APP_NAME)-$(MOZ_APP_VERSION)
+includedir     = @includedir@
 libdir         = @libdir@
 datadir                = @datadir@
 mandir         = @mandir@
-idldir         = @datadir@/idl/$(MOZ_APP_NAME)-$(MOZ_APP_VERSION)
+idldir         = @datadir@/idl

-mozappdir      = $(libdir)/$(MOZ_APP_NAME)-$(MOZ_APP_VERSION)
-mredir         = $(libdir)/mre/mre-$(MOZ_APP_VERSION)
+mozappdir      = $(libdir)
+mredir         = $(libdir)/mre
 mrelibdir      = $(mredir)/lib

 DIST           = $(DEPTH)/dist
