diff --speed-large-files --minimal -Nru thunderbird.orig/directory/c-sdk/ldap/libraries/libldap/ldap-int.h thunderbird/directory/c-sdk/ldap/libraries/libldap/ldap-int.h
--- thunderbird.orig/directory/c-sdk/ldap/libraries/libldap/ldap-int.h	2009-05-27 09:59:24.939222864 +0200
+++ thunderbird/directory/c-sdk/ldap/libraries/libldap/ldap-int.h	2009-05-27 10:00:08.738847653 +0200
@@ -82,10 +82,10 @@
 #include "../liblber/lber-int.h"
 #endif /* macintosh */
 
-#include "ldap.h"
-#include "ldaprot.h"
-#include "ldaplog.h"
-#include "portable.h"
+#include "../../include/ldap.h"
+#include "../../include/ldaprot.h"
+#include "../../include/ldaplog.h"
+#include "../../include/portable.h"
 
 #ifdef LDAP_ASYNC_IO
 #ifdef NEED_FILIO
