From 7aa639db0238c894c12d8d9728e684b3ddbdc5dc Mon Sep 17 00:00:00 2001
From: Maciej Blizinski <maciej@opencsw.org>
Date: Fri, 5 Feb 2010 10:31:20 +0100
Subject: [PATCH 2/4] libintl for kproplog

---
 src/slave/Makefile.in |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/slave/Makefile.in b/src/slave/Makefile.in
index 6ff3c71..31f6844 100644
--- a/src/slave/Makefile.in
+++ b/src/slave/Makefile.in
@@ -27,7 +27,7 @@ kpropd: $(SERVEROBJS) $(KADMCLNT_DEPLIBS) $(KDB5_DEPLIB) $(KRB5_BASE_DEPLIBS) $(
 	$(CC_LINK) -o kpropd $(SERVEROBJS) $(KADMCLNT_LIBS) $(KDB5_LIB) $(KRB5_BASE_LIBS) $(APPUTILS_LIB) @LIBUTIL@
 
 kproplog: $(LOGOBJS)
-	$(CC_LINK) -o kproplog $(LOGOBJS) $(KADMSRV_LIBS) $(KRB5_BASE_LIBS)
+	$(CC_LINK) -o kproplog $(LOGOBJS) $(KADMSRV_LIBS) $(KRB5_BASE_LIBS) -lintl
 
 install::
 	for f in kprop kpropd kproplog; do \
-- 
1.6.6

