From dd7dcff4a05a4dfdd2e2db961c633e15fff4f8eb Mon Sep 17 00:00:00 2001
From: Roger Hakansson <hson@current9x.(none)>
Date: Thu, 23 Sep 2010 21:19:05 +0200
Subject: [PATCH] Solaris 9 doesn't have stdint.h

---
 epan/wslua/lua_bitop.c |    4 ++
 1 files changed, 4 insertions(+)

diff --git a/epan/wslua/lua_bitop.c b/epan/wslua/lua_bitop.c
index e358e7a..6603eb6 100644
--- a/epan/wslua/lua_bitop.c
+++ b/epan/wslua/lua_bitop.c
@@ -40,8 +40,12 @@ typedef __int32 int32_t;
 typedef unsigned __int32 uint32_t;
 typedef unsigned __int64 uint64_t;
 #else
+#if defined(__SUNPRO_C) || defined (__SUNPRO_CC)
+#include <sys/inttypes.h>
+#else
 #include <stdint.h>
 #endif
+#endif
 
 typedef int32_t SBits;
 typedef uint32_t UBits;
