--- a/configure
+++ b/configure
@@ -1383,13 +1383,13 @@ if test "$x11_include" = "not found"; then
        test -f $dir/libX11.dylib || \
        test -f $dir/libX11.sa; then
       if test $dir = /usr/lib; then
-        x11_link="-lX11"
+        x11_link="-lX11 -lXext"
       else
         x11_libs="-L$dir"
         case "$host" in
           *-kfreebsd*-gnu) x11_link="-L$dir -lX11";;
           *-*-*bsd*) x11_link="-R$dir -L$dir -lX11";;
-          *) x11_link="-L$dir -lX11";;
+          *) x11_link="-L$dir -lX11 -lXext";;
         esac
       fi
       break
@@ -1506,16 +1506,16 @@ if test $has_tk = true; then
   elif sh ./hasgot $tk_libs -ltcl$tclmaj$tclmin $tkauxlibs Tcl_DoOneEvent
   then
     tk_libs="$tk_libs -ltk$tkmaj$tkmin -ltcl$tclmaj$tclmin $dllib"
-  elif sh ./hasgot -L/sw/lib $tk_libs -ltcl$tclmaj.$tclmin $tkauxlibs \
+  elif sh ./hasgot $tk_libs -ltcl$tclmaj.$tclmin $tkauxlibs \
                    Tcl_DoOneEvent
-  then tk_libs="-L/sw/lib -ltk$tkmaj.$tkmin -ltcl$tclmaj.$tclmin $dllib"
-  elif sh ./hasgot -L/usr/pkg/lib $tk_libs $tk_x11_libs \
+  then tk_libs="-ltk$tkmaj.$tkmin -ltcl$tclmaj.$tclmin $dllib"
+  elif sh ./hasgot $tk_libs $tk_x11_libs \
         -ltk$tkmaj$tkmin -ltcl$tclmaj$tclmin -lpthread $tkauxlibs \
                    Tcl_DoOneEvent
   then
     case "$host" in
       *-*-*bsd*) tk_libs="-R/usr/pkg/lib -L/usr/pkg/lib $tk_libs $tk_x11_libs -ltk$tkmaj$tkmin -ltcl$tclmaj$tclmin -lpthread $tkauxlibs";;
-      *) tk_libs="-L/usr/pkg/lib $tk_libs $tk_x11_libs -ltk$tkmaj$tkmin -ltcl$tclmaj$tclmin -lpthread $tkauxlibs";;
+      *) tk_libs="$tk_libs $tk_x11_libs -ltk$tkmaj$tkmin -ltcl$tclmaj$tclmin -lpthread $tkauxlibs";;
     esac
   else
     echo "Tcl library not found."
@@ -1526,17 +1526,17 @@ fi
 if test $has_tk = true; then
   if sh ./hasgot $tk_libs $tk_x11_libs $tkauxlibs Tk_SetGrid; then
     echo "Tcl/Tk libraries found."
-  elif sh ./hasgot -L/sw/lib $tk_libs $tk_x11_libs $tkauxlibs Tk_SetGrid; then
+  elif sh ./hasgot $tk_libs $tk_x11_libs $tkauxlibs Tk_SetGrid; then
     case "$host" in
       *-*-*bsd*) tk_libs="-R/sw/lib -L/sw/lib $tk_libs";;
-      *) tk_libs="-L/sw/lib $tk_libs";;
+      *) tk_libs="$tk_libs";;
     esac
     echo "Tcl/Tk libraries found."
-  elif sh ./hasgot -L/usr/pkg/lib $tk_libs $tk_x11_libs $tkauxlibs \
+  elif sh ./hasgot $tk_libs $tk_x11_libs $tkauxlibs \
                    Tk_SetGrid; then
     case "$host" in
       *-*-*bsd*) tk_libs="-R/usr/pkg/lib -L/usr/pkg/lib $tk_libs";;
-      *) tk_libs="-L/usr/pkg/lib $tk_libs";;
+      *) tk_libs="$tk_libs";;
     esac
     echo "Tcl/Tk libraries found."
   else
