diff --speed-large-files --minimal -Nru nagios-plugins-1.4.13.orig/plugins-root/Makefile.am nagios-plugins-1.4.13/plugins-root/Makefile.am
--- nagios-plugins-1.4.13.orig/plugins-root/Makefile.am	2008-07-02 22:00:08.000000000 +0200
+++ nagios-plugins-1.4.13/plugins-root/Makefile.am	2009-02-02 15:04:58.121712871 +0100
@@ -41,26 +41,18 @@
 	chmod $(setuid_root_mode) $(DESTDIR)$(libexecdir)/$$p; \
 	done
 
-install-root: $(noinst_PROGRAMS)
-	@$(INSTALL_SUID)
+install-root:
+	$(INSTALL) -m 755 -d $(DESTDIR)$(libexecdir)
+	$(INSTALL) -m 744 check_dhcp $(DESTDIR)$(libexecdir)
+	$(INSTALL) -m 744 check_icmp $(DESTDIR)$(libexecdir)
+	$(INSTALL) -m 744 pst3 $(DESTDIR)$(libexecdir)
 
-install-exec-local: $(noinst_PROGRAMS)
-	@TMPFILE=$(DESTDIR)$(libexecdir)/.setuid-$$$$; \
-	rm -f $$TMPFILE; \
-	echo > $$TMPFILE; \
-	## See if we can create a setuid root executable in $(libexecdir).
-	## If not, then don't even try to install setuid plugins.
-	can_create_suid_root_executable=no; \
-	chown root $$TMPFILE > /dev/null 2>&1 \
-	  && chmod $(setuid_root_mode) $$TMPFILE > /dev/null 2>&1 \
-	  && can_create_suid_root_executable=yes; \
-	rm -f $$TMPFILE; \
-	if test $$can_create_suid_root_executable = yes; then \
-	  $(INSTALL_SUID); \
-	else \
-	  echo "WARNING: insufficient access; not installing setuid plugins"; \
-	  echo "NOTE: to install setuid plugins, run 'make install-root' as root"; \
-	fi
+
+install-exec-local:
+	$(INSTALL) -m 755 -d $(DESTDIR)$(libexecdir)
+	$(INSTALL) -m 744 check_dhcp $(DESTDIR)$(libexecdir)
+	$(INSTALL) -m 744 check_icmp $(DESTDIR)$(libexecdir)
+	$(INSTALL) -m 744 pst3 $(DESTDIR)$(libexecdir)
 
 # /* Author Coreutils team sub-citation */
 
diff --speed-large-files --minimal -Nru nagios-plugins-1.4.13.orig/plugins-root/Makefile.in nagios-plugins-1.4.13/plugins-root/Makefile.in
--- nagios-plugins-1.4.13.orig/plugins-root/Makefile.in	2008-09-25 10:33:20.000000000 +0200
+++ nagios-plugins-1.4.13/plugins-root/Makefile.in	2009-02-02 15:04:14.144038426 +0100
@@ -952,24 +952,19 @@
 test:
 	perl -I $(top_builddir) -I $(top_srcdir) ../test.pl
 
-install-root: $(noinst_PROGRAMS)
-	@$(INSTALL_SUID)
+install-root:
+	$(INSTALL) -m 755 -d $(DESTDIR)$(libexecdir)
+	$(INSTALL) -m 644 check_dhcp $(DESTDIR)$(libexecdir)
+	$(INSTALL) -m 644 check_icmp $(DESTDIR)$(libexecdir)
+	$(INSTALL) -m 644 pst3 $(DESTDIR)$(libexecdir)
 
-install-exec-local: $(noinst_PROGRAMS)
-	@TMPFILE=$(DESTDIR)$(libexecdir)/.setuid-$$$$; \
-	rm -f $$TMPFILE; \
-	echo > $$TMPFILE; \
-	can_create_suid_root_executable=no; \
-	chown root $$TMPFILE > /dev/null 2>&1 \
-	  && chmod $(setuid_root_mode) $$TMPFILE > /dev/null 2>&1 \
-	  && can_create_suid_root_executable=yes; \
-	rm -f $$TMPFILE; \
-	if test $$can_create_suid_root_executable = yes; then \
-	  $(INSTALL_SUID); \
-	else \
-	  echo "WARNING: insufficient access; not installing setuid plugins"; \
-	  echo "NOTE: to install setuid plugins, run 'make install-root' as root"; \
-	fi
+
+install-exec-local:
+	$(INSTALL) -m 755 -d $(DESTDIR)$(libexecdir)
+	$(INSTALL) -m 644 check_dhcp $(DESTDIR)$(libexecdir)
+	$(INSTALL) -m 644 check_icmp $(DESTDIR)$(libexecdir)
+	$(INSTALL) -m 644 pst3 $(DESTDIR)$(libexecdir)
+#
 # Tell versions [3.59,3.63) of GNU make to not export all variables.
 # Otherwise a system limit (for SysV at least) may be exceeded.
 .NOEXPORT:
