--- libtool.gcc.o	2009-03-11 21:48:23.343791369 -0500
+++ libtool.gcc	2009-03-11 21:48:50.846784450 -0500
@@ -82,13 +82,13 @@
 AR_FLAGS="cru"
 
 # A C compiler.
-LTCC="/opt/studio/SOS11/SUNWspro/bin/cc"
+LTCC="/opt/csw/gcc3/bin/gcc"
 
 # LTCC compiler flags.
-LTCFLAGS="-xO3 -xarch=386 -I/opt/csw/bdb44/include -I/opt/csw/apache2/include -I/usr/jdk1.5.0_15/include -I/opt/csw/include  -mt  -D_LARGEFILE64_SOURCE -DNE_LFS"
+LTCFLAGS="-O2 -pipe -m32 -march=386 -I/opt/csw/bdb44/include -I/opt/csw/apache2/include -I/usr/jdk1.5.0_15/include -I/opt/csw/include  -D_LARGEFILE64_SOURCE -DNE_LFS"
 
 # A language-specific compiler.
-CC="/opt/studio/SOS11/SUNWspro/bin/cc"
+CC="/opt/csw/gcc3/bin/gcc"
 
 # Is the compiler the GNU C compiler?
 with_gcc=
@@ -143,7 +143,7 @@
 exeext=""
 
 # Additional compiler flags for building library objects.
-pic_flag=" -KPIC -DPIC"
+pic_flag=" -fPIC -DPIC"
 pic_mode=default
 
 # What is the maximum length of a command?
@@ -1989,7 +1989,7 @@
 	continue
 	;;
 
-     -mt|-mthreads|-kthread|-Kthread|-pthread|-pthreads|--thread-safe|-threads)
+     -mthreads|-kthread|-Kthread|-pthread|-pthreads|--thread-safe|-threads)
 	compiler_flags="$compiler_flags $arg"
 	compile_command="$compile_command $arg"
 	finalize_command="$finalize_command $arg"
@@ -2003,7 +2003,7 @@
 
       # -64, -mips[0-9] enable 64-bit mode on the SGI compiler
       # -r[0-9][0-9]* specifies the processor on the SGI compiler
-      # -xarch=*, -xtarget=* enable 64-bit mode on the Sun compiler
+      # -march=*, -xtarget=* enable 64-bit mode on the Sun compiler
       # +DA*, +DD* enable 64-bit mode on the HP compiler
       # -q* pass through compiler args for the IBM compiler
       # -m* pass through architecture-specific compiler args for GCC
@@ -2012,7 +2012,7 @@
       # -p, -pg, --coverage, -fprofile-* pass through profiling flag for GCC
       # -F/path gives path to uninstalled frameworks, gcc on darwin
       # @file GCC response files
-      -64|-mips[0-9]|-r[0-9][0-9]*|-xarch=*|-xtarget=*|+DA*|+DD*|-q*|-m*| \
+      -64|-mips[0-9]|-r[0-9][0-9]*|-march=*|-xtarget=*|+DA*|+DD*|-q*|-m*| \
       -t[45]*|-txscale*|-p|-pg|--coverage|-fprofile-*|-F*|@*)
 
 	# Unknown arguments in both finalize_command and compile_command need
@@ -2484,7 +2484,7 @@
 	lib=
 	found=no
 	case $deplib in
-	-mt|-mthreads|-kthread|-Kthread|-pthread|-pthreads|--thread-safe|-threads)
+	-mthreads|-kthread|-Kthread|-pthread|-pthreads|--thread-safe|-threads)
 	  if test "$linkmode,$pass" = "prog,link"; then
 	    compile_deplibs="$deplib $compile_deplibs"
 	    finalize_deplibs="$deplib $finalize_deplibs"
@@ -7318,13 +7318,13 @@
 AR_FLAGS="cru"
 
 # A C compiler.
-LTCC="/opt/studio/SOS11/SUNWspro/bin/cc"
+LTCC="/opt/csw/gcc3/bin/gcc"
 
 # LTCC compiler flags.
-LTCFLAGS="-xO3 -xarch=386 -I/opt/csw/bdb44/include -I/opt/csw/apache2/include -I/usr/jdk1.5.0_15/include -I/opt/csw/include  -mt  -D_LARGEFILE64_SOURCE -DNE_LFS"
+LTCFLAGS="-O2 -pipe -m32 -march=386 -I/opt/csw/bdb44/include -I/opt/csw/apache2/include -I/usr/jdk1.5.0_15/include -I/opt/csw/include  -D_LARGEFILE64_SOURCE -DNE_LFS"
 
 # A language-specific compiler.
-CC="/opt/studio/SOS11/SUNWspro/bin/CC"
+CC="/opt/csw/gcc3/bin/g++"
 
 # Is the compiler the GNU C compiler?
 with_gcc=no
@@ -7379,7 +7379,7 @@
 exeext=""
 
 # Additional compiler flags for building library objects.
-pic_flag=" -KPIC -DPIC"
+pic_flag=" -fPIC -DPIC"
 pic_mode=default
 
 # What is the maximum length of a command?
@@ -7475,7 +7475,7 @@
 
 # Dependencies to place after the objects being linked to create a
 # shared library.
-postdeps="-library=Cstd -library=Crun"
+postdeps=""
 
 # The library search path used internally by the compiler when linking
 # a shared library.
@@ -7624,10 +7624,10 @@
 AR_FLAGS="cru"
 
 # A C compiler.
-LTCC="/opt/studio/SOS11/SUNWspro/bin/cc"
+LTCC="/opt/csw/gcc3/bin/gcc"
 
 # LTCC compiler flags.
-LTCFLAGS="-xO3 -xarch=386 -I/opt/csw/bdb44/include -I/opt/csw/apache2/include -I/usr/jdk1.5.0_15/include -I/opt/csw/include  -mt  -D_LARGEFILE64_SOURCE -DNE_LFS"
+LTCFLAGS="-O2 -pipe -m32 -march=386 -I/opt/csw/bdb44/include -I/opt/csw/apache2/include -I/usr/jdk1.5.0_15/include -I/opt/csw/include  -D_LARGEFILE64_SOURCE -DNE_LFS"
 
 # A language-specific compiler.
 CC="f77"
@@ -7685,7 +7685,7 @@
 exeext=""
 
 # Additional compiler flags for building library objects.
-pic_flag=" -KPIC"
+pic_flag=" -fPIC"
 pic_mode=default
 
 # What is the maximum length of a command?
--- Makefile.gcc.o	2009-03-11 21:48:17.524971502 -0500
+++ Makefile.gcc	2009-03-11 21:49:06.704433498 -0500
@@ -89,12 +89,12 @@
 PACKAGE_NAME=subversion
 PACKAGE_VERSION=1.5.6
 
-CC = /opt/studio/SOS11/SUNWspro/bin/cc
-CXX = /opt/studio/SOS11/SUNWspro/bin/CC
+CC = /opt/csw/gcc3/bin/gc
+CXX = /opt/csw/gcc3/bin/g++
 EXEEXT = 
 
 SHELL = /bin/bash
-LIBTOOL = $(SHELL) /home/mwatters/mgar/pkg/subversion/trunk/work/build-isa-i386/subversion-1.5.6/libtool
+LIBTOOL = $(SHELL) /home/mwatters/mgar/pkg/subversion/trunk/work/build-isa-i386/subversion-1.5.6/libtool.gcc
 LTFLAGS = --tag=CC --silent
 LTCXXFLAGS = --tag=CXX --silent
 LT_LDFLAGS = 
@@ -113,10 +113,10 @@
 
 SWIG = none
 SWIG_PY_INCLUDES = $(SWIG_INCLUDES) -I/opt/csw/include/python2.6 -I$(SWIG_SRC_DIR)/python/libsvn_swig_py
-SWIG_PY_COMPILE = /opt/studio/SOS11/SUNWspro/bin/cc -DNDEBUG -O -Kpic
-SWIG_PY_LINK = /opt/studio/SOS11/SUNWspro/bin/cc -G
+SWIG_PY_COMPILE = /opt/csw/gcc3/bin/gc -DNDEBUG -O -fpic
+SWIG_PY_LINK = /opt/csw/gcc3/bin/gc -G
 SWIG_PY_LIBS = 
-SWIG_PL_INCLUDES = $(SWIG_INCLUDES)  -D_REENTRANT -xO3 -xtarget=generic -xarch=generic -I/opt/csw/bdb44/include -I/opt/csw/include -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64  -I/opt/csw/lib/perl/5.8.8/CORE 
+SWIG_PL_INCLUDES = $(SWIG_INCLUDES)  -D_REENTRANT -O2 -pipe -m32 -xtarget=generic -march=generic -I/opt/csw/bdb44/include -I/opt/csw/include -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64  -I/opt/csw/lib/perl/5.8.8/CORE 
 SWIG_RB_INCLUDES = $(SWIG_INCLUDES) -I. -I/opt/csw/lib/ruby/1.8/i386-solaris2.8 -I$(SWIG_SRC_DIR)/ruby/libsvn_swig_ruby
 SWIG_RB_COMPILE = /opt/csw/gcc4/bin/gcc -mtune=i686 -O2 -pipe -m32 -march=i386 -I/opt/csw/include  -fPIC  
 SWIG_RB_LINK = /opt/csw/gcc4/bin/gcc -shared -shrext .so
@@ -142,14 +142,14 @@
 MKDIR = /opt/csw/bin/ginstall -c -d
 
 # The EXTRA_ parameters can be used to pass extra flags at 'make' time.
-CFLAGS = -xO3 -xarch=386 -I/opt/csw/bdb44/include -I/opt/csw/apache2/include -I/usr/jdk1.5.0_15/include -I/opt/csw/include  -mt  -D_LARGEFILE64_SOURCE -DNE_LFS $(EXTRA_CFLAGS)
-CXXFLAGS = -xO3 -xarch=386 -I/opt/csw/bdb44/include -I/opt/csw/apache2/include -I/usr/jdk1.5.0_15/include -I/opt/csw/include $(EXTRA_CXXFLAGS)
+CFLAGS = -O2 -pipe -m32 -march=i386 -I/opt/csw/bdb44/include -I/opt/csw/apache2/include -I/usr/jdk1.5.0_15/include -I/opt/csw/include  -D_LARGEFILE64_SOURCE -DNE_LFS $(EXTRA_CFLAGS)
+CXXFLAGS = -O2 -pipe -m32 -march=i386 -I/opt/csw/bdb44/include -I/opt/csw/apache2/include -I/usr/jdk1.5.0_15/include -I/opt/csw/include $(EXTRA_CXXFLAGS)
 ### A few of the CFLAGS (e.g. -Wmissing-prototypes, -Wstrict-prototypes,
 ### -Wmissing-declarations) are not valid for C++, and should be somehow
 ### suppressed (but they may come from httpd or APR).
 CPPFLAGS = -I/opt/csw/bdb44/include -I/opt/csw/apache2/include -I/usr/jdk1.5.0_15/include -I/opt/csw/include  -I/opt/csw/bdb44/include -I/opt/csw/include -DSOLARIS2=8 -D_POSIX_PTHREAD_SEMANTICS -D_REENTRANT -D_LARGEFILE64_SOURCE $(EXTRA_CPPFLAGS)
-LDFLAGS = -xarch=386 -L/opt/csw/lib -R/opt/csw/lib/\\\$$ISALIST -R/opt/csw/lib -L/opt/csw/bdb44/lib -R/opt/csw/bdb44/lib/\\\$$ISALIST -R/opt/csw/bdb44/lib -L/opt/csw/lib/svn -R/opt/csw/lib/svn/\\\$$ISALIST -R/opt/csw/lib/svn -L/opt/csw/apache2/lib -R/opt/csw/apache2/lib/\\\$$ISALIST -R/opt/csw/apache2/lib -lintl -liconv    -L/opt/csw/bdb44/lib -L/opt/csw/lib -L/opt/csw/lib -L/opt/csw/lib $(EXTRA_LDFLAGS)
-SWIG_LDFLAGS = -xarch=386 -L/opt/csw/lib -R/opt/csw/lib/\\\$$ISALIST -R/opt/csw/lib -L/opt/csw/bdb44/lib -R/opt/csw/bdb44/lib/\\\$$ISALIST -R/opt/csw/bdb44/lib -L/opt/csw/lib/svn -R/opt/csw/lib/svn/\\\$$ISALIST -R/opt/csw/lib/svn -L/opt/csw/apache2/lib -R/opt/csw/apache2/lib/\\\$$ISALIST -R/opt/csw/apache2/lib -lintl -liconv $(EXTRA_SWIG_LDFLAGS)
+LDFLAGS = -march=i386 -L/opt/csw/lib -R/opt/csw/lib/\\\$$ISALIST -R/opt/csw/lib -L/opt/csw/bdb44/lib -R/opt/csw/bdb44/lib/\\\$$ISALIST -R/opt/csw/bdb44/lib -L/opt/csw/lib/svn -R/opt/csw/lib/svn/\\\$$ISALIST -R/opt/csw/lib/svn -L/opt/csw/apache2/lib -R/opt/csw/apache2/lib/\\\$$ISALIST -R/opt/csw/apache2/lib -lintl -liconv    -L/opt/csw/bdb44/lib -L/opt/csw/lib -L/opt/csw/lib -L/opt/csw/lib $(EXTRA_LDFLAGS)
+SWIG_LDFLAGS = -march=i386 -L/opt/csw/lib -R/opt/csw/lib/\\\$$ISALIST -R/opt/csw/lib -L/opt/csw/bdb44/lib -R/opt/csw/bdb44/lib/\\\$$ISALIST -R/opt/csw/bdb44/lib -L/opt/csw/lib/svn -R/opt/csw/lib/svn/\\\$$ISALIST -R/opt/csw/lib/svn -L/opt/csw/apache2/lib -R/opt/csw/apache2/lib/\\\$$ISALIST -R/opt/csw/apache2/lib -lintl -liconv $(EXTRA_SWIG_LDFLAGS)
 
 COMPILE = $(CC) $(CPPFLAGS) $(CFLAGS) $(INCLUDES)
 COMPILE_CXX = $(CXX) $(CPPFLAGS) $(CXXFLAGS) $(INCLUDES)
