From b648830934c1b709ee5b02a41c5a579ef4306c0d Mon Sep 17 00:00:00 2001
From: Maciej Blizinski <maciej@opencsw.org>
Date: Sun, 14 Aug 2011 21:23:14 +0200
Subject: [PATCH] clear AM_CFLAGS in lto-plugin

---
 lto-plugin/Makefile.in |    6 +++++-
 1 files changed, 5 insertions(+), 1 deletions(-)

diff --git a/lto-plugin/Makefile.in b/lto-plugin/Makefile.in
index 86edb99..704d182 100644
--- a/lto-plugin/Makefile.in
+++ b/lto-plugin/Makefile.in
@@ -227,7 +227,11 @@ AUTOMAKE_OPTIONS = no-dependencies
 gcc_version := $(shell cat $(top_srcdir)/../gcc/BASE-VER)
 libexecsubdir := $(libexecdir)/gcc/$(target_noncanonical)/$(gcc_version)
 AM_CPPFLAGS = -I$(top_srcdir)/../include $(DEFS)
-AM_CFLAGS = -Wall -Werror
+# AM_CFLAGS = -Wall -Werror
+# The above AM_CFLAGS were breaking the bootstrap with Sun Studio, because
+# they were passed unconditionally to the compiler, which doesn't support
+# -Wall nor -Werror.
+AM_CFLAGS =
 AM_LIBTOOLFLAGS = --tag=disable-static
 libexecsub_LTLIBRARIES = liblto_plugin.la
 gcc_build_dir = ../$(host_subdir)/gcc
-- 
1.7.6

