From 280c3806583428f2d20718839129ea0758e8cdbc Mon Sep 17 00:00:00 2001
From: Sebastian Kayser <skayser@opencsw.org>
Date: Tue, 5 Jan 2010 23:19:33 +0100
Subject: [PATCH] Makefile: put background jobs in same shell as the wait call

---
 Makefile.in |   20 ++++++++++----------
 1 files changed, 10 insertions(+), 10 deletions(-)

diff --git a/Makefile.in b/Makefile.in
index b381ade..bf9953f 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -73,11 +73,11 @@ test1: test.md5
 	touch $@
 
 test2: test.md5
-	./mbuffer -q -I :8000 -o $@.tar -o - | openssl md5 > $@.md5 &
-	sleep 1
-	-rm -f $@.tar
-	./mbuffer -i test.tar -o /dev/null -O localhost:8000 -H
-	wait
+	./mbuffer -q -I :8000 -o $@.tar -o - | openssl md5 > $@.md5 & \
+		sleep 1; \
+		rm -f $@.tar; \
+		./mbuffer -i test.tar -o /dev/null -O localhost:8000 -H; \
+		wait
 	sync
 	diff $@.md5 test.md5
 	rm $@.md5
@@ -88,11 +88,11 @@ test2: test.md5
 	touch $@
 
 test3: test.md5
-	./mbuffer -q -6 -I :8000 | openssl md5 > $@.md5 &
-	sleep 1
-	-rm -f $@.tar
-	./mbuffer -i test.tar -o /dev/null -6 -O localhost:8000 -o $@.tar -H
-	wait
+	./mbuffer -q -6 -I :8000 | openssl md5 > $@.md5 & \
+		sleep 1; \
+		rm -f $@.tar; \
+		./mbuffer -i test.tar -o /dev/null -6 -O localhost:8000 -o $@.tar -H; \
+		wait
 	sync
 	diff $@.md5 test.md5
 	rm $@.md5
-- 
1.6.5.1

