diff --speed-large-files --minimal -Nru cadaver-0.22.3.orig/Makefile.in cadaver-0.22.3/Makefile.in
--- cadaver-0.22.3.orig/Makefile.in	2005-12-31 07:57:37.000000000 -0500
+++ cadaver-0.22.3/Makefile.in	2006-07-11 05:14:56.576413000 -0400
@@ -71,12 +71,12 @@
 
 install: $(TARGET) install-nls
 	@echo "Creating directories..."
-	$(INSTALL) -d $(bindir)
-	$(INSTALL) -d $(man1dir)
+	$(INSTALL) -d $(DESTDIR)/$(bindir)
+	$(INSTALL) -d $(DESTDIR)/$(man1dir)
 	@echo "Installing $(TARGET) executable..."
-	$(INSTALL_PROGRAM) $(TARGET) $(bindir)/$(TARGET)
+	$(INSTALL_PROGRAM) $(TARGET) $(DESTDIR)/$(bindir)/$(TARGET)
 	@echo "Installing man page..."
-	$(INSTALL_DATA) $(top_srcdir)/doc/cadaver.1 $(man1dir)/cadaver.1
+	$(INSTALL_DATA) $(top_srcdir)/doc/cadaver.1 $(DESTDIR)/$(man1dir)/cadaver.1
 
 install-nls:
 	@cd po && $(MAKE) install
