From 68db3657c8f086798bd01a878717d785c16c5b05 Mon Sep 17 00:00:00 2001
From: Juraj Lutter <wilbury@unstable10x.(none)>
Date: Wed, 7 Sep 2011 18:16:03 +0200
Subject: [PATCH] Do not use -Wstrict-prototypes warning.

---
 configure              |    2 +-
 m4/pandora_warnings.m4 |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure b/configure
index b6d6df9..82db435 100755
--- a/configure
+++ b/configure
@@ -18502,7 +18502,7 @@ fi
 
 fi
 
-      CC_WARNINGS="${BASE_WARNINGS} -Wstrict-prototypes -Wmissing-prototypes -Wredundant-decls -Wmissing-declarations -Wcast-align ${CC_WARNINGS_FULL}"
+      CC_WARNINGS="${BASE_WARNINGS} -Wmissing-prototypes -Wredundant-decls -Wmissing-declarations -Wcast-align ${CC_WARNINGS_FULL}"
       CXX_WARNINGS="${BASE_WARNINGS} -Woverloaded-virtual -Wnon-virtual-dtor -Wctor-dtor-privacy -Wno-long-long ${CXX_WARNINGS_FULL}"
 
       { $as_echo "$as_me:${as_lineno-$LINENO}: checking whether it is safe to use -Wmissing-declarations from C++" >&5
diff --git a/m4/pandora_warnings.m4 b/m4/pandora_warnings.m4
index 0f6d47a..372e719 100644
--- a/m4/pandora_warnings.m4
+++ b/m4/pandora_warnings.m4
@@ -237,7 +237,7 @@ uint16_t x= htons(80);
               [NO_CONVERSION="-Wno-conversion"])
       ])
 
-      CC_WARNINGS="${BASE_WARNINGS} -Wstrict-prototypes -Wmissing-prototypes -Wredundant-decls -Wmissing-declarations -Wcast-align ${CC_WARNINGS_FULL}"
+      CC_WARNINGS="${BASE_WARNINGS} -Wmissing-prototypes -Wredundant-decls -Wmissing-declarations -Wcast-align ${CC_WARNINGS_FULL}"
       CXX_WARNINGS="${BASE_WARNINGS} -Woverloaded-virtual -Wnon-virtual-dtor -Wctor-dtor-privacy -Wno-long-long ${CXX_WARNINGS_FULL}"
 
       AC_CACHE_CHECK([whether it is safe to use -Wmissing-declarations from C++],
-- 
1.7.6

