From 9bc60cdd312557c994f079597453de0eaa8dd4ea Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Wed, 2 Feb 2011 16:53:18 +0100
Subject: [PATCH] Include string.h if available

---
 src/FtpConnection.cc |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/src/FtpConnection.cc b/src/FtpConnection.cc
index debde95..d08aad7 100644
--- a/src/FtpConnection.cc
+++ b/src/FtpConnection.cc
@@ -38,6 +38,10 @@
 #include <cstdio>
 #include <cassert>
 
+#ifdef HAVE_STRING_H
+# include <string.h>
+#endif // HAVE_STRING_H
+
 #include "Request.h"
 #include "Segment.h"
 #include "Option.h"
-- 
1.7.3.2

