diff --speed-large-files --minimal -Nru asciidoc-8.2.5.orig/a2x asciidoc-8.2.5/a2x
--- asciidoc-8.2.5.orig/a2x	2007-10-29 04:50:53.000000000 +0100
+++ asciidoc-8.2.5/a2x	2008-02-24 05:04:45.829595000 +0100
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/opt/csw/bin/bash
 #
 # a2x - convert Asciidoc text file to PDF, XHTML, HTML Help, manpage
 #                  or plain text
@@ -14,7 +14,7 @@
     REALNAME=$(which "$REALNAME")
 fi
 REALNAME="$(readlink -f "$REALNAME")"
-CONF_DIR=/etc/asciidoc
+CONF_DIR=/opt/csw/etc/asciidoc
 
 #--------------------------------------------------------------------
 # Constants.
diff --speed-large-files --minimal -Nru asciidoc-8.2.5.orig/asciidoc.py asciidoc-8.2.5/asciidoc.py
--- asciidoc-8.2.5.orig/asciidoc.py	2007-11-18 06:19:31.000000000 +0100
+++ asciidoc-8.2.5/asciidoc.py	2008-02-24 05:05:04.549688000 +0100
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/opt/csw/bin/python
 '''
 asciidoc -  converts an AsciiDoc text file to DocBook, HTML or LinuxDoc
 
@@ -3986,7 +3986,7 @@
 # ---------
 APP_DIR = None              # This file's directory.
 USER_DIR = None             # ~/.asciidoc
-CONF_DIR = "/etc/asciidoc"  # Global configuration files directory.
+CONF_DIR = "/opt/csw/etc/asciidoc"  # Global configuration files directory.
 HELP_FILE = 'help.conf'     # Default (English) help file.
 
 # Globals
diff --speed-large-files --minimal -Nru asciidoc-8.2.5.orig/install.sh asciidoc-8.2.5/install.sh
--- asciidoc-8.2.5.orig/install.sh	2007-10-29 04:50:54.000000000 +0100
+++ asciidoc-8.2.5/install.sh	2008-02-24 05:05:24.204826000 +0100
@@ -8,10 +8,10 @@
 # directory) then you will also need to change the CONF_DIR constant definition
 # values in both a2x and asciiidoc.py scripts.
 
-CONFDIR=/etc/asciidoc
-BINDIR=/usr/local/bin
-MANDIR=/usr/local/man
-VIM_CONFDIR=/etc/vim
+#CONFDIR=/etc/asciidoc
+#BINDIR=/usr/local/bin
+#MANDIR=/usr/local/man
+#VIM_CONFDIR=/etc/vim
 
 if [ `basename $0` = uninstall.sh ]; then
     rm $BINDIR/asciidoc
