--- fontconfig-2.6.0/fontconfig/fontconfig.h_orig	2008-06-01 04:24:34.000000000 +0200
+++ fontconfig-2.6.0/fontconfig/fontconfig.h	2009-04-24 12:14:14.404366000 +0200
@@ -113,9 +113,9 @@
 #define FC_DECORATIVE	    "decorative"	/* Bool - true if style is a decorative variant */
 #define FC_LCD_FILTER	    "lcdfilter"		/* Int */
 
-#define FC_CACHE_SUFFIX		    ".cache-"FC_CACHE_VERSION
-#define FC_DIR_CACHE_FILE	    "fonts.cache-"FC_CACHE_VERSION
-#define FC_USER_CACHE_FILE	    ".fonts.cache-"FC_CACHE_VERSION
+#define FC_CACHE_SUFFIX		    ".cache-csw-"FC_CACHE_VERSION
+#define FC_DIR_CACHE_FILE	    "fonts.cache-csw-"FC_CACHE_VERSION
+#define FC_USER_CACHE_FILE	    ".fonts.cache-csw-"FC_CACHE_VERSION
 
 /* Adjust outline rasterizer */
 #define FC_CHAR_WIDTH	    "charwidth"	/* Int */
