From a13b7d40da0c25087c2054d8fd02a27a334bfd2b Mon Sep 17 00:00:00 2001
From: Maciej Blizinski <maciej@opencsw.org>
Date: Tue, 19 Oct 2010 01:33:35 +0200
Subject: [PATCH] Disabling the libstdc++ check

---
 SConstruct |   12 +++++++-----
 1 files changed, 7 insertions(+), 5 deletions(-)

diff --git a/SConstruct b/SConstruct
index e3046ff..0feedc0 100644
--- a/SConstruct
+++ b/SConstruct
@@ -309,7 +309,8 @@ usePCH = not GetOption( "usePCH" ) is None
 
 justClientLib = (COMMAND_LINE_TARGETS == ['mongoclient'])
 
-env = Environment( MSVS_ARCH=msarch , tools = ["default", "gch"], toolpath = '.' )
+env = Environment( MSVS_ARCH=msarch , tools = ["default", "gch"], toolpath = '.',
+    CPPPATH = ['/opt/csw/include', '/opt/csw/include/boost'])
 if GetOption( "cxx" ) is not None:
     env["CC"] = GetOption( "cxx" )
     env["CXX"] = GetOption( "cxx" )
@@ -858,10 +859,11 @@ def doConfigure( myenv , needPcre=True , shell=False ):
             print( "c++ compiler not installed!" )
             Exit(1)
 
-    if nix and not shell:
-        if not conf.CheckLib( "stdc++" ):
-            print( "can't find stdc++ library which is needed" );
-            Exit(1)
+    # libstdc++ is not necessary when compiling with Sun Studio
+    # if nix and not shell:
+    #     if not conf.CheckLib( "stdc++" ):
+    #         print( "can't find stdc++ library which is needed" );
+    #         Exit(1)
 
     def myCheckLib( poss , failIfNotFound=False , staticOnly=False):
 
-- 
1.7.3

