From 66bde8fc0a71ada36fe80f44a998b22f83fd34af Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Sat, 26 Jan 2013 11:40:51 +0100
Subject: [PATCH] Use correct struct fields in vfs_commit.c

---
 source3/modules/vfs_commit.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/source3/modules/vfs_commit.c b/source3/modules/vfs_commit.c
index 865250a..c9c0a05 100644
--- a/source3/modules/vfs_commit.c
+++ b/source3/modules/vfs_commit.c
@@ -307,10 +307,10 @@ static int commit_ftruncate(
 static struct vfs_fn_pointers vfs_commit_fns = {
         .open_fn = commit_open,
         .close_fn = commit_close,
-        .write = commit_write,
-        .pwrite = commit_pwrite,
+        .write_fn = commit_write,
+        .pwrite_fn = commit_pwrite,
         .connect_fn = commit_connect,
-        .ftruncate = commit_ftruncate
+        .ftruncate_fn = commit_ftruncate
 };
 
 NTSTATUS vfs_commit_init(void);
-- 
1.8.0

