From 2ba13b722abacec5bf8b60130a38368045460067 Mon Sep 17 00:00:00 2001
From: Jan Holzhueter <jh@opencsw.org>
Date: Tue, 29 May 2012 11:11:58 +0200
Subject: [PATCH] change-linker-flags

---
 configure |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure b/configure
index 4bd5d89..5bc68de 100755
--- a/configure
+++ b/configure
@@ -1896,7 +1896,7 @@ enable static
 enable swscale_alpha
 
 # build settings
-SHFLAGS='-shared -Wl,-soname,$$(@F)'
+SHFLAGS='-G -Wl,-h,$$(@F)'
 FFSERVERLDFLAGS=-Wl,-E
 LIBPREF="lib"
 LIBSUF=".a"
@@ -2661,7 +2661,7 @@ case $target_os in
         ;;
     sunos)
         FFSERVERLDFLAGS=""
-        SHFLAGS='-shared -Wl,-h,$$(@F)'
+        SHFLAGS='-G -Wl,-h,$$(@F)'
         enabled x86 && SHFLAGS="-mimpure-text $SHFLAGS"
         network_extralibs="-lsocket -lnsl"
         add_cppflags -D__EXTENSIONS__ -D_XOPEN_SOURCE=600
-- 
1.7.10

