From 38f1e7dee2711117bc6f9cde25ab2a31531cb5e6 Mon Sep 17 00:00:00 2001
From: Yann Rouillard <yann@pleiades.fr.eu.org>
Date: Sat, 29 Sep 2012 17:19:27 +0200
Subject: [PATCH] Remove GNU ld specific options

---
 configure | 1 -
 1 file changed, 1 deletion(-)

diff --git a/configure b/configure
index fa6e845..505508e 100755
--- a/configure
+++ b/configure
@@ -3372,7 +3372,6 @@ enabled extra_warnings && check_cflags -Winline
 
 # add some linker flags
 check_ldflags -Wl,--warn-common
-check_ldflags -Wl,-rpath-link=libpostproc:libswresample:libswscale:libavfilter:libavdevice:libavformat:libavcodec:libavutil:libavresample
 test_ldflags -Wl,-Bsymbolic && append SHFLAGS -Wl,-Bsymbolic
 
 enabled xmm_clobber_test &&                             \
-- 
1.7.12

