From 14fb2a6aa9fc69358b7d0148da0674279fdcdf1c Mon Sep 17 00:00:00 2001
From: Jan Holzhueter <jh@opencsw.org>
Date: Mon, 1 Oct 2012 14:47:53 +0200
Subject: [PATCH] remove-linkerflags

---
 configure | 1 -
 1 file changed, 1 deletion(-)

diff --git a/configure b/configure
index ac429ef..03992c4 100755
--- a/configure
+++ b/configure
@@ -3813,7 +3813,6 @@ enabled extra_warnings && check_cflags -Winline
 
 # add some linker flags
 check_ldflags -Wl,--warn-common
-check_ldflags -Wl,-rpath-link=libpostproc:libswresample:libswscale:libavfilter:libavdevice:libavformat:libavcodec:libavutil:libavresample
 test_ldflags -Wl,-Bsymbolic && append SHFLAGS -Wl,-Bsymbolic
 
 enabled xmm_clobber_test &&
-- 
1.7.12

