From 9b4689bad0183bd155cb57b512aad9de693e8884 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Tue, 20 Jan 2015 15:39:53 +0100
Subject: [PATCH] Use GTK_DOC_USE_LIBTOOL

---
 configure.ac | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/configure.ac b/configure.ac
index da9936c..c04ccb7 100644
--- a/configure.ac
+++ b/configure.ac
@@ -28,6 +28,9 @@ GTK_DOC_CHECK([1.15],[--flavour no-tmpl])
 ],[
 AM_CONDITIONAL([ENABLE_GTK_DOC], false)
 ])
+m4_ifdef([GTK_DOC_USE_LIBTOOL], [], [
+AM_CONDITIONAL([GTK_DOC_USE_LIBTOOL], false)
+])
 
 #
 # enable creation of man pages
-- 
2.2.1

