From db3d12ec17a80fad52a5a463ad42bfbc6d70ad91 Mon Sep 17 00:00:00 2001
From: Oliver Kiddle <opk@opencsw.org>
Date: Mon, 25 Oct 2010 12:52:51 +0200
Subject: [PATCH] skip chgrp of inc

---
 uip/Makefile.in |    6 +-----
 1 files changed, 1 insertions(+), 5 deletions(-)

diff --git a/uip/Makefile.in b/uip/Makefile.in
index b3c6717..2919bb9 100644
--- a/uip/Makefile.in
+++ b/uip/Makefile.in
@@ -282,11 +282,7 @@ install-misc:
 
 # install commands with special installation needs (thus no $(SCMDS) use here)
 install-scmds:
-	if test x$(SETGID_MAIL) != x; then \
-	  $(INSTALL_PROGRAM) -g $(MAIL_SPOOL_GRP) -m 2755 inc $(DESTDIR)$(bindir)/$$cmd; \
-	else \
-	  $(INSTALL_PROGRAM)                 inc $(DESTDIR)$(bindir)/$$cmd; \
-	fi
+	$(INSTALL_PROGRAM)                 inc $(DESTDIR)$(bindir)/$$cmd
 
 uninstall:
 	for cmd in $(CMDS); do \
-- 
1.7.3

