From 707ba96fd821b215a03478f3d9acde08cd53714a Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Tue, 3 Dec 2013 22:47:57 +0100
Subject: [PATCH] realpath does not like seocnds argument to be NULL on Solaris

---
 src/lexer/lchdir.c | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/src/lexer/lchdir.c b/src/lexer/lchdir.c
index afa3a0a..df23ff8 100644
--- a/src/lexer/lchdir.c
+++ b/src/lexer/lchdir.c
@@ -6,8 +6,10 @@ void l_noChdir(Lexer *lp, char const *pathname)
 void l_chdir(Lexer *lp, char const *pathname)
 {
 // fprintf(stderr, "CHANGING WD\n");
- 
-    char *resolved = realpath(media_filename(lp->d_media_ptr), NULL);
+    char *resolved_name;
+  
+    resolved_name = malloc(PATH_MAX);
+    char *resolved = realpath(media_filename(lp->d_media_ptr), resolved_name);
 
     if (resolved == NULL)
         out_of_memory();
-- 
1.8.3.4

