diff --speed-large-files --minimal -Nru firefox.orig/configure firefox/configure
--- firefox.orig/configure      2009-07-07 21:46:57.267011389 +0200
+++ firefox/configure   	2009-07-07 21:49:10.319971404 +0200
@@ -4426,8 +4426,8 @@
     /usr/local/x11r5/include  \
     /usr/lpp/Xamples/include  \
                               \
-    /usr/openwin/include      \
-    /usr/openwin/share/include \
+    /opt/csw/X11/include      \
+    /opt/csw/X11/share/include \
     ; \
   do
     if test -r "$ac_dir/$x_direct_test_include"; then
@@ -4502,8 +4502,8 @@
     /usr/lpp/Xamples/lib  \
     /lib/usr/lib/X11     \
                           \
-    /usr/openwin/lib      \
-    /usr/openwin/share/lib \
+    /opt/csw/X11/lib      \
+    /opt/csw/X11/share/lib \
     ; \
 do
   for ac_extension in a so sl; do
@@ -5185,7 +5185,7 @@

 MOZ_JS_LIBS='-L$(LIBXUL_DIST)/bin -lmozjs'
 DYNAMIC_XPCOM_LIBS='-L$(LIBXUL_DIST)/bin -lxpcom -lxpcom_core'
-MOZ_FIX_LINK_PATHS='-Wl,-rpath-link,$(LIBXUL_DIST)/bin -Wl,-rpath-link,$(prefix)/lib'
+MOZ_FIX_LINK_PATHS='-Wl,-R,$(LIBXUL_DIST)/bin -Wl,-R,$(prefix)/lib'
 XPCOM_FROZEN_LDOPTS='-L$(LIBXUL_DIST)/bin -lxpcom'
 LIBXUL_LIBS='$(XPCOM_FROZEN_LDOPTS) -lxul'
 XPCOM_GLUE_LDOPTS='$(LIBXUL_DIST)/lib/$(LIB_PREFIX)xpcomglue_s.$(LIB_SUFFIX) $(XPCOM_FROZEN_LDOPTS)'
