diff --speed-large-files --minimal -Nru firefox.orig/js/src/configure firefox/js/src/configure
--- firefox.orig/js/src/configure	2009-07-08 20:48:40.741770226 +0200
+++ firefox/js/src/configure		2009-07-08 20:49:34.735119508 +0200
@@ -4026,8 +4026,8 @@
     /usr/local/x11r5/include  \
     /usr/lpp/Xamples/include  \
                               \
-    /usr/openwin/include      \
-    /usr/openwin/share/include \
+    /opt/csw/X11/include      \
+    /opt/csw/X11/share/include \
     ; \
   do
     if test -r "$ac_dir/$x_direct_test_include"; then
@@ -4102,8 +4102,8 @@
     /usr/lpp/Xamples/lib  \
     /lib/usr/lib/X11	  \
                           \
-    /usr/openwin/lib      \
-    /usr/openwin/share/lib \
+    /opt/csw/X11/lib      \
+    /opt/csw/X11/share/lib \
     ; \
 do
   for ac_extension in a so sl; do
