diff --speed-large-files --minimal -Nru firefox.orig/js/src/nanojit/Nativei386.h firefox/js/src/nanojit/Nativei386.h
--- firefox.orig/js/src/nanojit/Nativei386.h	2008-12-19 16:11:09.743712966 +0800
+++ firefox/js/src/nanojit/Nativei386.h		2008-12-19 16:12:32.156744648 +0800
@@ -87,6 +87,15 @@
 #define count_imt()
 #endif

+#undef EAX
+#undef EBX
+#undef ECX
+#undef EDX
+#undef ESP
+#undef EBP
+#undef ESI
+#undef EDI
+
 namespace nanojit
 {
	const int NJ_LOG2_PAGE_SIZE     = 12;           // 4K
