diff --speed-large-files --minimal -Nru firefox.orig/modules/plugin/base/public/npruntime.h firefox/modules/plugin/base/public/npruntime.h
--- firefox.orig/modules/plugin/base/public/npruntime.h    2007-10-21 15:36:05.000000000 +0000
+++ firefox/modules/plugin/base/public/npruntime.h         2008-11-24 21:59:04.857036600 +0000
@@ -65,6 +65,10 @@
 #ifndef _NP_RUNTIME_H_
 #define _NP_RUNTIME_H_

+typedef int bool;
+#define true   1
+#define false  0
+
 #ifdef __cplusplus
 extern "C" {
 #endif
