diff --speed-large-files --minimal -Nru firefox.orig/security/nss/lib/softoken/config.mk firefox/security/nss/lib/softoken/config.mk
--- firefox.orig/security/nss/lib/softoken/config.mk    2009-07-10 22:58:22.364751249 +0200
+++ firefox/security/nss/lib/softoken/config.mk       2009-07-10 22:58:22.364751249 +0200
@@ -101,7 +101,7 @@
 ifeq ($(OS_TARGET),SunOS)
 # The -R '$ORIGIN' linker option instructs this library to search for its
 # dependencies in the same directory where it resides.
-MKSHLIB += -R '$$ORIGIN'
+MKSHLIB += -R '$$ORIGIN' -L../../../../dist/lib
 OS_LIBS += -lbsm 
 endif

