From 071f0c1b11cf21628aa3ec8c092cfd050a11e9d9 Mon Sep 17 00:00:00 2001
From: Maciej Blizinski <maciej@opencsw.org>
Date: Sat, 8 May 2010 08:47:26 +0200
Subject: [PATCH] libtool dir used for linking

---
 python/setup.py |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/python/setup.py b/python/setup.py
index 7c86194..71074c3 100644
--- a/python/setup.py
+++ b/python/setup.py
@@ -3,7 +3,7 @@ from distutils.core import setup, Extension
 
 magic_module = Extension('magic',
     libraries = ['magic'],
-    library_dirs = ['./','../','../src','/usr/lib/'],
+    library_dirs = ['./','../','../src','../src/.libs','/usr/lib/'],
     include_dirs = ['./','../','../src','/usr/include/'],
     sources = ['py_magic.c'])
 
-- 
1.7.0

