From 19ff890df2c01f4c0fa2f606017ff2ab421240c9 Mon Sep 17 00:00:00 2001
From: Maciej Blizinski <maciej@opencsw.org>
Date: Mon, 7 Jun 2010 21:25:39 +0200
Subject: [PATCH] CSW runpath

---
 python/setup.py |    5 +++--
 1 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/python/setup.py b/python/setup.py
index 71074c3..086211e 100644
--- a/python/setup.py
+++ b/python/setup.py
@@ -3,8 +3,9 @@ from distutils.core import setup, Extension
 
 magic_module = Extension('magic',
     libraries = ['magic'],
-    library_dirs = ['./','../','../src','../src/.libs','/usr/lib/'],
-    include_dirs = ['./','../','../src','/usr/include/'],
+    library_dirs = ['./','../','../src','../src/.libs','/usr/lib/','/opt/csw/lib'],
+    include_dirs = ['./','../','../src','/usr/include/','/opt/csw/include'],
+    runtime_library_dirs = ['/opt/csw/lib'],
     sources = ['py_magic.c'])
 
 setup (name = 'Magic file extensions',
-- 
1.7.1

