--- a/Makefile	2010-01-05 22:40:37.832643496 +0100
+++ b/Makefile	2010-01-05 23:01:25.428084486 +0100
@@ -1,19 +1,22 @@
 V:=$(shell /bin/sh keychain.sh --version 2>&1 | \
 	awk -F'[ ;]' '/^K/{print $$2; exit}')
 D:=$(shell date +'%d %b %Y')
+ifndef SED
+SED = sed
+endif
 TARBALL_CONTENTS=keychain README.rst ChangeLog COPYING keychain.pod keychain.1 \
 				 keychain.spec
 
 all: keychain.1.gz keychain keychain.spec
 
 keychain.spec: keychain.spec.in keychain.sh
-	sed 's/KEYCHAIN_VERSION/$V/' keychain.spec.in > keychain.spec
+	$(SED) 's/KEYCHAIN_VERSION/$V/' keychain.spec.in > keychain.spec
 
 keychain.1: keychain.pod keychain.sh
 	pod2man --name=keychain --release=$V \
 		--center='http://gentoo.org/proj/en/keychain.xml' \
 		keychain.pod keychain.1
-	sed -i.orig -e "s/^'br/.br/" keychain.1
+	$(SED) -i.orig -e "s/^'br/.br/" keychain.1
 
 keychain.1.gz: keychain.1
 	gzip -9 keychain.1
