--- xtrans-1.2.5/Makefile.in.orig	2009-11-26 15:01:50.752826994 +0100
+++ xtrans-1.2.5/Makefile.in	2009-11-26 15:05:44.512854519 +0100
@@ -222,7 +222,7 @@
 
 aclocaldir = $(datadir)/aclocal
 aclocal_DATA = xtrans.m4
-pkgconfigdir = $(datadir)/pkgconfig
+pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = xtrans.pc
 EXTRA_DIST = xtrans.pc.in ${aclocal_DATA} ChangeLog Xtrans.mm
 MAINTAINERCLEANFILES = ChangeLog
