diff --speed-large-files --minimal -Nru subversion-1.2.0-rc1.orig/subversion/libsvn_subr/utf.c subversion-1.2.0-rc1/subversion/libsvn_subr/utf.c
--- subversion-1.2.0-rc1.orig/subversion/libsvn_subr/utf.c	2005-04-14 15:41:36.420002000 -0700
+++ subversion-1.2.0-rc1/subversion/libsvn_subr/utf.c	2005-04-14 15:40:56.400014000 -0700
@@ -302,7 +302,7 @@
 static svn_error_t *
 get_ntou_xlate_handle_node (xlate_handle_node_t **ret, apr_pool_t *pool)
 {
-  return get_xlate_handle_node (ret, "UTF-8", APR_LOCALE_CHARSET,
+  return get_xlate_handle_node (ret, "UTF-8", "ASCII",
                                 SVN_UTF_NTOU_XLATE_HANDLE, pool);
 }
 
@@ -315,7 +315,7 @@
 static svn_error_t *
 get_uton_xlate_handle_node (xlate_handle_node_t **ret, apr_pool_t *pool)
 {
-  return get_xlate_handle_node (ret, APR_LOCALE_CHARSET, "UTF-8",
+  return get_xlate_handle_node (ret, "ASCII", "UTF-8",
                                 SVN_UTF_UTON_XLATE_HANDLE, pool);
 }
 
