diff --speed-large-files --minimal -Nru Python-2.5c1.orig/Lib/site.py Python-2.5c1/Lib/site.py
--- Python-2.5c1.orig/Lib/site.py	2006-08-16 01:01:42.000000000 -0400
+++ Python-2.5c1/Lib/site.py	2006-09-04 23:19:13.331586000 -0400
@@ -186,6 +186,8 @@
                                          "python" + sys.version[:3],
                                          "site-packages"),
                             os.path.join(prefix, "lib", "site-python")]
+
+                sitedirs.append("/opt/csw/lib/python2.3/site-packages")
             else:
                 sitedirs = [prefix, os.path.join(prefix, "lib", "site-packages")]
             if sys.platform == 'darwin':
