From 61edd606d46be2a533197b7ab3e30807dc285999 Mon Sep 17 00:00:00 2001
From: Rafael Ostertag <rafisol@opencsw.org>
Date: Sun, 18 Dec 2011 12:55:10 +0100
Subject: [PATCH] Fix shell on configure

---
 configure |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/configure b/configure
index c93937c..3788551 100755
--- a/configure
+++ b/configure
@@ -1,4 +1,4 @@
-#! /bin/sh
+#! /bin/bash
 
 if test `uname -s | grep -c MINGW 2>/dev/null` != "0"; then
   msyshost=1
-- 
1.7.6.1

