diff --speed-large-files --minimal -Nru firefox.orig/toolkit/crashreporter/nsExceptionHandler.cpp firefox/toolkit/crashreporter/nsExceptionHandler.cpp
--- firefox.orig/toolkit/crashreporter/nsExceptionHandler.cpp      2008-07-20 22:19:32.000000000 +0000
+++ firefox/toolkit/crashreporter/nsExceptionHandler.cpp      2008-11-24 21:46:58.112282800 +0000
@@ -301,7 +301,7 @@
   else if (pid == 0) {
     // need to clobber this, as libcurl might load NSS,
     // and we want it to load the system NSS.
-    unsetenv("LD_LIBRARY_PATH");
+    // unsetenv("LD_LIBRARY_PATH");
     (void) execl(crashReporterPath,
                  crashReporterPath, minidumpPath, (char*)0);
     _exit(1);
