From 020e8d0a3bad76bfd045e685787eafea4995000a Mon Sep 17 00:00:00 2001
From: Peter Felecan <pfelecan@opencsw.org>
Date: Tue, 19 Nov 2013 13:43:08 +0100
Subject: [PATCH] plugin needs limits

---
 dictdplugin_popen.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/dictdplugin_popen.cpp b/dictdplugin_popen.cpp
index 3c04da7..5b88bc9 100644
--- a/dictdplugin_popen.cpp
+++ b/dictdplugin_popen.cpp
@@ -42,6 +42,7 @@
 #include <stdlib.h>
 #include <stdio.h>
 #include <errno.h>
+#include <limits.h>
 #include <string>
 #include <unistd.h>
 #include <sstream>
-- 
1.8.4.1

