From 3e7862a7efa384b5957a864c1b809afe480d3e51 Mon Sep 17 00:00:00 2001
From: Freddy DISSAUX <fdissaux+opencsw@lncsa.fr>
Date: Sat, 20 Sep 2014 17:31:33 +0200
Subject: [PATCH] Include string.h to mitigate warnings

---
 src/makeheaders.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/makeheaders.c b/src/makeheaders.c
index 858426f..087af9b 100644
--- a/src/makeheaders.c
+++ b/src/makeheaders.c
@@ -44,6 +44,8 @@
 #    define WIN32
 #  endif
 # include <string.h>
+#elif defined(__sun) && defined(__SVR4)
+# include <string.h>
 #else
 # include <unistd.h>
 #endif
-- 
2.0.3

