From 4ab202a1367d352937b101c7994cd99100950822 Mon Sep 17 00:00:00 2001
From: Rafael Ostertag <raos@opencsw.org>
Date: Sun, 17 Jun 2012 10:46:20 +0200
Subject: [PATCH] Optimization/debug flags adjusted for use with SunStudio

---
 configure |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure b/configure
index d8df74d..c929f0a 100755
--- a/configure
+++ b/configure
@@ -13685,9 +13685,9 @@ fi
 
 
 if test x"$debug" = x"yes"; then
-   CFLAGS="$CFLAGS -g3 -O0"
+   CFLAGS="$CFLAGS -g -xO0"
 else
-   CFLAGS="$CFLAGS -O2 -DNDEBUG"
+   CFLAGS="$CFLAGS -xO3 -DNDEBUG"
 fi
 
 
-- 
1.7.10.3

