From 9063c218bfd3fcdeeeab7a2f40d46312bb3d5f23 Mon Sep 17 00:00:00 2001
From: Rafael Ostertag <raos@opencsw.org>
Date: Sat, 16 Jun 2012 21:15:44 +0200
Subject: [PATCH] Add include alloca.h in gcov-stub.c

---
 tests/unit/gcov-stub.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/tests/unit/gcov-stub.c b/tests/unit/gcov-stub.c
index 39cfd00..241b92a 100644
--- a/tests/unit/gcov-stub.c
+++ b/tests/unit/gcov-stub.c
@@ -1,5 +1,6 @@
 #include "cf3.defs.h"
 #include "cf3.extern.h"
+#include <alloca.h>
 
 /*
  * Stubs which allow CFEngine compiled with gcov support to link against unit
-- 
1.7.10.3

