--- netpbm-10.26.19/manweb	2005-11-20 12:40:53.984621000 -0500
+++ netpbm-10.26.19/manweb	2005-11-20 13:00:57.566416000 -0500
@@ -300,7 +300,7 @@
 
 sub readConfFile($) {
 #-----------------------------------------------------------------------------
-#  Read the configuration file (/etc/manweb.conf or value of
+#  Read the configuration file (/opt/csw/etc/manweb.conf or value of
 #  MANWEB_CONF_FILE or named by our argument).  Return values set in
 #  it, or defaults.
 #-----------------------------------------------------------------------------
@@ -315,7 +315,7 @@
         if (defined($envVblValue)) {
             $confFile = $envVblValue;
         } else {
-            $confFile = "/etc/manweb.conf";
+            $confFile = "/opt/csw/etc/manweb.conf";
         }
     }
 
