diff -ur cyrus-imapd-2.4.4/perl/imap/Makefile.PL cyrus-imapd-2.4.4.new/perl/imap/Makefile.PL
--- cyrus-imapd-2.4.4/perl/imap/Makefile.PL	2010-11-11 23:15:33.000000000 +0100
+++ cyrus-imapd-2.4.4.new/perl/imap/Makefile.PL	2010-11-21 22:36:28.368907061 +0100
@@ -65,7 +65,7 @@
     'clean'	=> {'FILES' => 'libcyrperl.a cyradm'},
     'OBJECT'    => 'IMAP.o',
     'MYEXTLIB'  => '../../lib/libcyrus.a ../../lib/libcyrus_min.a',
-    'LIBS'	=> ["$BDB_LIB $SASL_LIB $OPENSSL_LIB $LIB_RT $ZLIB -lssl -lcrypto"],
+    'LIBS'	=> ["$BDB_LIB $SASL_LIB $OPENSSL_LIB $LIB_RT $ZLIB -lssl -lcrypto -lperl"],
     'DEFINE'	=> '-DPERL_POLLUTE',    # e.g., '-DHAVE_SOMETHING' 
     'INC'	=> "-I../../lib -I../.. -I../../com_err/et $SASL_INC $OPENSSL_INC", 
     'EXE_FILES' => [cyradm],
diff -ur cyrus-imapd-2.4.4/perl/sieve/managesieve/Makefile.PL cyrus-imapd-2.4.4.new/perl/sieve/managesieve/Makefile.PL
--- cyrus-imapd-2.4.4/perl/sieve/managesieve/Makefile.PL	2010-11-11 23:15:33.000000000 +0100
+++ cyrus-imapd-2.4.4.new/perl/sieve/managesieve/Makefile.PL	2010-11-21 22:36:09.746290674 +0100
@@ -62,7 +62,7 @@
     'ABSTRACT'  => 'Cyurs Sieve management interface',
     'VERSION_FROM' => 'managesieve.pm', # finds $VERSION
     'MYEXTLIB'  => '../lib/isieve.o ../lib/lex.o ../lib/mystring.o ../lib/request.o ../../../lib/libcyrus.a ../../../lib/libcyrus_min.a',
-    'LIBS'	=> ["$BDB_LIB $SASL_LIB $OPENSSL_LIB $LIB_RT $ZLIB -lssl -lcrypto"], 
+    'LIBS'	=> ["$BDB_LIB $SASL_LIB $OPENSSL_LIB $LIB_RT $ZLIB -lssl -lcrypto -lperl"], 
     'DEFINE'	=> '-DPERL_POLLUTE',     # e.g., '-DHAVE_SOMETHING' 
     'INC'	=> "-I../lib/ -I../../../lib/ $SASL_INC $OPENSSL_INC",
 );
