diff --speed-large-files --minimal -Nru ./tools/config2header ./tools/config2header.new
--- ./tools/config2header	Sat Nov 20 00:11:21 2010
+++ ./tools/config2header.new	Sat Nov 20 00:11:11 2010
@@ -71,7 +71,7 @@
 # Look for CC=xxx "assigments" in the argument list.
 #
 while ($#ARGV >= 0) {
-	last unless ($ARGV[0] =~ m/^(\S+)=(\S+)$/);
+	last unless ($ARGV[0] =~ m/^(\S+)=(.+)$/);
 	eval "\$$1='$2';";
 	die "$@" if ($@);
 	shift @ARGV;
