From cb2ee195ba14e16060115e90d41d8744cff31b29 Mon Sep 17 00:00:00 2001
From: Juergen Arndt <ja@opencsw.org>
Date: Sat, 11 Feb 2012 19:11:07 +0100
Subject: [PATCH] changed install and command options in configure

---
 configure |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure b/configure
index bb7b3bf..bf774a8 100755
--- a/configure
+++ b/configure
@@ -5664,7 +5664,7 @@ cat >>confdefs.h <<_ACEOF
 #define DEFAULT_NAGIOS_GROUP "$nagios_grp"
 _ACEOF
 
-INSTALL_OPTS="-o $nagios_user -g $nagios_grp"
+INSTALL_OPTS="-o $install_user -g $install_grp"
 
 
 
@@ -5685,7 +5685,7 @@ else
 fi;
 
 
-COMMAND_OPTS="-o $command_user -g $command_grp"
+COMMAND_OPTS="-o $install_user -g $install_grp"
 
 
 MAIL_PROG=no
-- 
1.7.8.3

