diff --speed-large-files --minimal -Nru nagios-3.2.0.orig/configure nagios-3.2.0/configure
--- nagios-3.2.0.orig/configure	2009-08-12 20:28:10.000000000 +0200
+++ nagios-3.2.0/configure	2009-08-15 22:09:52.647057188 +0200
@@ -7754,7 +7754,7 @@
 s,@CPP@,$CPP,;t t
 s,@EGREP@,$EGREP,;t t
 s,@SNPRINTF_O@,$SNPRINTF_O,;t t
-s,@SOCKETLIBS@,$SOCKETLIBS,;t t
+s,@SOCKETLIBS@,-lnsl -lsocket,;t t
 s,@THREADLIBS@,$THREADLIBS,;t t
 s,@nagios_user@,$nagios_user,;t t
 s,@nagios_grp@,$nagios_grp,;t t
