diff --speed-large-files --minimal -Nru nagios-3.2.0.orig/cgi/cmd.c nagios-3.2.0/cgi/cmd.c
--- nagios-3.2.0.orig/cgi/cmd.c	2009-08-11 16:35:43.000000000 +0200
+++ nagios-3.2.0/cgi/cmd.c	2009-08-15 21:33:36.069423261 +0200
@@ -1886,7 +1886,7 @@
         }
 
 
-__attribute__((format(printf, 2, 3)))
+/* __attribute__((format(printf, 2, 3))) */
 static int cmd_submitf(int id, const char *fmt, ...){
 	char cmd[MAX_EXTERNAL_COMMAND_LENGTH];
 	const char *command;
diff --speed-large-files --minimal -Nru nagios-3.2.0.orig/include/nebmodules.h nagios-3.2.0/include/nebmodules.h
--- nagios-3.2.0.orig/include/nebmodules.h	2008-11-30 18:22:59.000000000 +0100
+++ nagios-3.2.0/include/nebmodules.h	2009-08-15 21:34:31.707033245 +0200
@@ -31,7 +31,7 @@
 
 /***** MODULE VERSION INFORMATION *****/
 
-#define NEB_API_VERSION(x) int __neb_api_version = x;
+#define NEB_API_VERSION(x) int __neb_api_version = x
 #define CURRENT_NEB_API_VERSION    3
 
 
diff --speed-large-files --minimal -Nru nagios-3.2.0.orig/module/Makefile.in nagios-3.2.0/module/Makefile.in
--- nagios-3.2.0.orig/module/Makefile.in	2007-12-15 00:02:24.000000000 +0100
+++ nagios-3.2.0/module/Makefile.in	2009-08-15 21:38:56.408558356 +0200
@@ -25,13 +25,13 @@
 
 CP=@CP@
 
-all:	helloworld.o
+all:	helloworld
 
-helloworld.o: helloworld.c
-	$(CC) $(MOD_CFLAGS) $(CFLAGS) -o helloworld.o helloworld.c $(MOD_LDFLAGS) $(LDFLAGS) $(LIBS)
+helloworld: helloworld.c
+	$(CC) $(MOD_CFLAGS) $(CFLAGS) -o helloworld helloworld.c $(MOD_LDFLAGS) $(LDFLAGS) $(LIBS)
 
 clean:
-	rm -f helloworld.o
+	rm -f helloworld
 	rm -f core *.o
 	rm -f *~ *.*~
 
