diff --speed-large-files --minimal -Nru nagios-3.0.6.orig/Makefile.in nagios-3.0.6/Makefile.in
--- nagios-3.0.6.orig/Makefile.in	2008-12-19 08:48:37.252900900 +0100
+++ nagios-3.0.6/Makefile.in	2008-12-19 08:48:20.644359694 +0100
@@ -243,7 +243,7 @@
 	@echo ""
 
 install-webconf:
-	$(INSTALL) -m 644 sample-config/httpd.conf $(DESTDIR)$(HTTPD_CONF)/nagios.conf
+	$(INSTALL) -m 644 sample-config/httpd.conf $(DESTDIR)$(HTTPD_CONF)/httpd-nagios.conf
 
 	@echo ""
 	@echo "*** Nagios/Apache conf file installed ***"
