diff --git a/generic/tclDTrace.d b/generic/tclDTrace.d
index bdbcb6e..4e4d3a4 100644
--- a/generic/tclDTrace.d
+++ b/generic/tclDTrace.d
@@ -24,7 +24,7 @@ provider tcl {
      *		arg1: number of arguments		(int)
      *		arg2: array of proc argument objects	(Tcl_Obj**)
      */
-    probe proc__entry(char* name, int objc, Tcl_Obj **objv);
+    probe proc__entry(char* name, int objc, struct Tcl_Obj **objv);
     /*
      *	tcl*:::proc-return probe
      *	    triggered immediately after proc bytecode execution
@@ -40,7 +40,7 @@ provider tcl {
      *		arg2: proc result			(string)
      *		arg3: proc result object		(Tcl_Obj*)
      */
-    probe proc__result(char* name, int code, char* result, Tcl_Obj *resultobj);
+    probe proc__result(char* name, int code, char* result, struct Tcl_Obj *resultobj);
     /*
      *	tcl*:::proc-args probe
      *	    triggered before proc-entry probe, gives access to string
@@ -73,7 +73,7 @@ provider tcl {
      *		arg1: number of arguments		(int)
      *		arg2: array of command argument objects	(Tcl_Obj**)
      */
-    probe cmd__entry(char* name, int objc, Tcl_Obj **objv);
+    probe cmd__entry(char* name, int objc, struct Tcl_Obj **objv);
     /*
      *	tcl*:::cmd-return probe
      *	    triggered immediately after commmand execution
@@ -89,7 +89,7 @@ provider tcl {
      *		arg2: command result			(string)
      *		arg3: command result object		(Tcl_Obj*)
      */
-    probe cmd__result(char* name, int code, char* result, Tcl_Obj *resultobj);
+    probe cmd__result(char* name, int code, char* result, struct Tcl_Obj *resultobj);
     /*
      *	tcl*:::cmd-args probe
      *	    triggered before cmd-entry probe, gives access to string
@@ -122,7 +122,7 @@ provider tcl {
      *		arg1: depth of stack			(int)
      *		arg2: top of stack			(Tcl_Obj**)
      */
-    probe inst__start(char* name, int depth, Tcl_Obj **stack);
+    probe inst__start(char* name, int depth, struct Tcl_Obj **stack);
     /*
      *	tcl*:::inst-done probe
      *	    triggered immediately after execution of a bytecode
@@ -130,7 +130,7 @@ provider tcl {
      *		arg1: depth of stack			(int)
      *		arg2: top of stack			(Tcl_Obj**)
      */
-    probe inst__done(char* name, int depth, Tcl_Obj **stack);
+    probe inst__done(char* name, int depth, struct Tcl_Obj **stack);
 
     /***************************** obj probes ******************************/
     /*
@@ -138,13 +138,13 @@ provider tcl {
      *	    triggered immediately after a new Tcl_Obj has been created
      *		arg0: object created			(Tcl_Obj*)
      */
-    probe obj__create(Tcl_Obj* obj);
+    probe obj__create(struct Tcl_Obj* obj);
     /*
      *	tcl*:::obj-free probe
      *	    triggered immediately before a Tcl_Obj is freed
      *		arg0: object to be freed		(Tcl_Obj*)
      */
-    probe obj__free(Tcl_Obj* obj);
+    probe obj__free(struct Tcl_Obj* obj);
 
     /***************************** tcl probes ******************************/
     /*
