From ded5e4cdfd4ee2c3091f159db81425e4ae442ab7 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Tue, 17 May 2011 14:02:20 +0200
Subject: [PATCH] Remove linker version

---
 configure |    8 --------
 1 files changed, 0 insertions(+), 8 deletions(-)

diff --git a/configure b/configure
index ae375d5..b28749d 100755
--- a/configure
+++ b/configure
@@ -12747,14 +12747,6 @@ if test "x${ac_enable_gcc_werror}" = xyes ; then
 fi
 
 
-case "$target_os" in
-  linux* | solaris*|k*bsd*-gnu*|gnu*)
-    SHLIB_VERSION_ARG="-Wl,--version-script=Version_script"
-    ;;
-  *)
-    ;;
-esac
-
 # Checks for library functions.
 
 
-- 
1.7.3.2

