diff --speed-large-files --minimal -Nru autoconf-2.61.orig/bin/autoreconf.in autoconf-2.61/bin/autoreconf.in
--- autoconf-2.61.orig/bin/autoreconf.in	2006-10-16 22:58:44.000000000 +0200
+++ autoconf-2.61/bin/autoreconf.in	2008-01-26 07:02:06.717466000 +0100
@@ -108,10 +108,10 @@
 # Lib files.
 my $autoconf   = $ENV{'AUTOCONF'}   || '@bindir@/@autoconf-name@';
 my $autoheader = $ENV{'AUTOHEADER'} || '@bindir@/@autoheader-name@';
-my $automake   = $ENV{'AUTOMAKE'}   || 'automake';
-my $aclocal    = $ENV{'ACLOCAL'}    || 'aclocal';
-my $libtoolize = $ENV{'LIBTOOLIZE'} || 'libtoolize';
-my $autopoint  = $ENV{'AUTOPOINT'}  || 'autopoint';
+my $automake   = $ENV{'AUTOMAKE'}   || '@bindir@/automake';
+my $aclocal    = $ENV{'ACLOCAL'}    || '@bindir@/aclocal';
+my $libtoolize = $ENV{'LIBTOOLIZE'} || '@bindir@/libtoolize';
+my $autopoint  = $ENV{'AUTOPOINT'}  || '@bindir@/gautopoint';
 
 # --install -- as --add-missing in other tools.
 my $install = 0;
