diff -Naur expat-1.95.8.orig/Makefile.in expat-1.95.8.patch/Makefile.in
--- expat-1.95.8.orig/Makefile.in	2004-05-07 22:00:48.000000000 +0200
+++ expat-1.95.8.patch/Makefile.in	2008-10-08 21:37:37.308957000 +0200
@@ -44,6 +44,9 @@
 APIHEADER = $(srcdir)/lib/expat.h $(srcdir)/lib/expat_external.h
 LIBRARY = libexpat.la
 
+ifndef INSTALL_ROOT
+INSTALL_ROOT=$(DESTDIR)
+endif
 
 default:  buildlib xmlwf/xmlwf
 
@@ -73,22 +76,22 @@
 	tests/runtests
 
 install: xmlwf/xmlwf installlib
-	$(mkinstalldirs) $(bindir) $(man1dir)
-	$(LIBTOOL) --mode=install $(INSTALL_PROGRAM) xmlwf/xmlwf $(bindir)/xmlwf
-	$(INSTALL_DATA) $(MANFILE) $(man1dir)
+	$(mkinstalldirs) $(INSTALL_ROOT)$(bindir) $(INSTALL_ROOT)$(man1dir)
+	$(LIBTOOL) --mode=install $(INSTALL_PROGRAM) xmlwf/xmlwf $(INSTALL_ROOT)$(bindir)/xmlwf
+	$(INSTALL_DATA) $(MANFILE) $(INSTALL_ROOT)$(man1dir)
 
 installlib: $(LIBRARY) $(APIHEADER)
-	$(mkinstalldirs) $(libdir) $(includedir)
-	$(LIBTOOL) --mode=install $(INSTALL) $(LIBRARY) $(libdir)/$(LIBRARY)
-	$(INSTALL_DATA) $(APIHEADER) $(includedir)
+	$(mkinstalldirs) $(INSTALL_ROOT)$(libdir) $(INSTALL_ROOT)$(includedir)
+	$(LIBTOOL) --mode=install $(INSTALL) $(LIBRARY) $(INSTALL_ROOT)$(libdir)/$(LIBRARY)
+	for FN in $(APIHEADER) ; do $(INSTALL_DATA) $$FN $(INSTALL_ROOT)$(includedir) ; done
 
 uninstall: uninstalllib
-	$(LIBTOOL) --mode=uninstall rm -f $(bindir)/xmlwf
-	rm -f $(man1dir)/xmlwf.1
+	$(LIBTOOL) --mode=uninstall rm -f $(INSTALL_ROOT)$(bindir)/xmlwf
+	rm -f $(INSTALL_ROOT)$(man1dir)/xmlwf.1
 
 uninstalllib:
-	$(LIBTOOL) --mode=uninstall rm -f $(libdir)/$(LIBRARY)
-	rm -f $(includedir)/$(APIHEADER)
+	$(LIBTOOL) --mode=uninstall rm -f $(INSTALL_ROOT)$(libdir)/$(LIBRARY)
+	rm -f $(INSTALL_ROOT)$(includedir)/$(APIHEADER)
 
 # for VPATH builds (invoked by configure)
 mkdir-init:
