From d3924ea3a11fe747edf84bd4e907ce17f61beaad Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Thu, 28 Apr 2011 19:34:10 +0200
Subject: [PATCH] Use stdint.h only when it is available, fall back to inttypes.h

---
 docs/examples/anyauthput.c |    7 ++++++-
 1 files changed, 6 insertions(+), 1 deletions(-)

diff --git a/docs/examples/anyauthput.c b/docs/examples/anyauthput.c
index 6b3d74a..679b0f3 100644
--- a/docs/examples/anyauthput.c
+++ b/docs/examples/anyauthput.c
@@ -27,7 +27,12 @@
 #  ifdef __VMS
      typedef int intptr_t;
 #  else
-#    include <stdint.h>
+#    ifdef CURL_PULL_STDINT_H
+#      include <stdint.h>
+#    endif
+#    ifdef CURL_PULL_INTTYPES_H
+#      include <inttypes.h>
+#    endif
 #  endif
 #  include <unistd.h>
 #endif
-- 
1.7.3.2

