From fe5caa8e5e281e1feea1770a07da292bc71e36b2 Mon Sep 17 00:00:00 2001
From: Rafael Ostertag <raos@opencsw.org>
Date: Tue, 12 Jun 2012 22:09:07 +0200
Subject: [PATCH] Adjust Makefile.in

---
 Makefile.in |    7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/Makefile.in b/Makefile.in
index 8846c38..5282ec2 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -112,11 +112,12 @@ install :
 	mkdir -p $(DESTDIR)$(INCLUDEDIR)
 	cp -Rf $(HEADERFILES) $(DESTDIR)$(INCLUDEDIR)
 	mkdir -p $(DESTDIR)$(LIBDIR)
-	cp -Rf $(LIBRARYFILES) $(DESTDIR)$(LIBDIR)
+	cp -RPf $(LIBRARYFILES) $(DESTDIR)$(LIBDIR)
 	mkdir -p $(DESTDIR)$(BINDIR)
 	cp -Rf $(COMMANDFILES) $(DESTDIR)$(BINDIR)
-	mkdir -p $(DESTDIR)$(LIBEXECDIR)
-	cp -Rf $(CGIFILES) $(DESTDIR)$(LIBEXECDIR)
+# We don't want this
+#	mkdir -p $(DESTDIR)$(LIBEXECDIR)
+#	cp -Rf $(CGIFILES) $(DESTDIR)$(LIBEXECDIR)
 	mkdir -p $(DESTDIR)$(DATADIR)
 	cp -Rf $(DOCUMENTFILES) $(DESTDIR)$(DATADIR)
 	mkdir -p $(DESTDIR)$(MAN1DIR)
-- 
1.7.10.3

