From eee3e782fa7694aa0f03a4995b41419ee51878e0 Mon Sep 17 00:00:00 2001
From: Rafael Ostertag <raos@opencsw.org>
Date: Tue, 12 Jun 2012 22:30:30 +0200
Subject: [PATCH] Use make variable

---
 Makefile.in |   40 ++++++++++++++++++++--------------------
 1 file changed, 20 insertions(+), 20 deletions(-)

diff --git a/Makefile.in b/Makefile.in
index 5282ec2..a6e4f01 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -134,7 +134,7 @@ install :
 
 
 install-strip :
-	make DESTDIR=$(DESTDIR) install
+	$(MAKE) DESTDIR=$(DESTDIR) install
 	cd $(DESTDIR)$(BINDIR) && strip $(COMMANDFILES)
 
 
@@ -151,26 +151,26 @@ uninstall :
 
 
 dist :
-	make version
-	make untabify
-	make distclean
+	$(MAKE) version
+	$(MAKE) untabify
+	$(MAKE) distclean
 	cd .. && tar cvf - $(PACKAGEDIR) | gzip -c > $(PACKAGETGZ)
 	sync ; sync
 
 
 distclean : clean
-	cd example && make clean
-	cd bros && make clean
+	cd example && $(MAKE) clean
+	cd bros && $(MAKE) clean
 	rm -rf Makefile tokyocabinet.pc config.cache config.log config.status autom4te.cache
 
 
 check :
-	make check-util
-	make check-hdb
-	make check-bdb
-	make check-fdb
-	make check-tdb
-	make check-adb
+	$(MAKE) check-util
+	$(MAKE) check-hdb
+	$(MAKE) check-bdb
+	$(MAKE) check-fdb
+	$(MAKE) check-tdb
+	$(MAKE) check-adb
 	rm -rf casket*
 	@printf '\n'
 	@printf '#================================================================\n'
@@ -586,7 +586,7 @@ check-adb :
 
 
 check-valgrind :
-	make RUNCMD="valgrind --tool=memcheck --log-file=%p.vlog" check
+	$(MAKE) RUNCMD="valgrind --tool=memcheck --log-file=%p.vlog" check
 	grep ERROR *.vlog | grep -v ' 0 errors' ; true
 	grep 'at exit' *.vlog | grep -v ' 0 bytes' ; true
 
@@ -630,13 +630,13 @@ check-race :
 check-forever :
 	while true ; \
 	  do \
-	    make check || break ; \
-	    make check || break ; \
-	    make check-thread || break ; \
-	    make check-race || break ; \
-	    make check-race || break ; \
-	    make check-compare || break ; \
-	    make check-compare || break ; \
+	    $(MAKE) check || break ; \
+	    $(MAKE) check || break ; \
+	    $(MAKE) check-thread || break ; \
+	    $(MAKE) check-race || break ; \
+	    $(MAKE) check-race || break ; \
+	    $(MAKE) check-compare || break ; \
+	    $(MAKE) check-compare || break ; \
 	  done
 
 
-- 
1.7.10.3

