From 9832238e2cbd4edfa29eb996333c28f4d10d9b28 Mon Sep 17 00:00:00 2001
From: Juraj Lutter <wilbury@opencsw.org>
Date: Tue, 27 Sep 2011 22:29:37 +0200
Subject: [PATCH] Use /opt/csw/bin/ggrep

---
 m4/pandora_vc_build.m4 |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/m4/pandora_vc_build.m4 b/m4/pandora_vc_build.m4
index a1fb60b..fc45ccc 100644
--- a/m4/pandora_vc_build.m4
+++ b/m4/pandora_vc_build.m4
@@ -71,7 +71,7 @@ AC_DEFUN([PANDORA_BUILDING_FROM_VC],[
       PANDORA_GIT_REVID=`git --no-pager log --max-count=1 | cut -f2 -d' ' | head -1`
       if test "x$PANDORA_GIT_REVID" != "x${PANDORA_VC_REVNO}" ; then
          PANDORA_VC_REVID="${PANDORA_GIT_REVID}"
-         PANDORA_VC_BRANCH=`git branch | grep -Ei "\* (.*)" | cut -f2 -d' '`
+         PANDORA_VC_BRANCH=`git branch | /opt/csw/bin/ggrep -Ei "\* (.*)" | cut -f2 -d' '`
       fi
     fi
 
-- 
1.7.6.1

