From ba65af2026be2372d6d3d1c7a85f518892479221 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <daÂm@opencsw.org>
Date: Tue, 17 Apr 2012 17:09:07 +0200
Subject: [PATCH] Use posix_memalign only when available

---
 rak/allocators.h |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/rak/allocators.h b/rak/allocators.h
index 0a1b711..cfbd090 100644
--- a/rak/allocators.h
+++ b/rak/allocators.h
@@ -77,7 +77,9 @@ public:
 
   static pointer alloc_size(size_type size) {
     pointer ptr = NULL;
+#ifdef HAVE_POSIX_MEMALIGN
     int __UNUSED result = posix_memalign((void**)&ptr, LT_SMP_CACHE_BYTES, size);
+#endif
 
     return ptr;
   }
-- 
1.7.10

