From 9f0bae87a21f0256f1848b8d8f74d90e533ae4d6 Mon Sep 17 00:00:00 2001
From: Jan Holzhueter <jh@current9s.(none)>
Date: Thu, 20 May 2010 12:43:55 +0200
Subject: [PATCH] more-UNUSED

---
 src/torrent/resume.cc |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/src/torrent/resume.cc b/src/torrent/resume.cc
index dd53f71..5480220 100644
--- a/src/torrent/resume.cc
+++ b/src/torrent/resume.cc
@@ -58,6 +58,8 @@
 
 #include "resume.h"
 
+#define __UNUSED
+
 namespace torrent {
 
 void
-- 
1.7.1

