From c6154d71819493358176edfb73049d85883e321e Mon Sep 17 00:00:00 2001
From: Juraj Lutter <wilbury@opencsw.org>
Date: Sat, 5 May 2012 14:35:37 +0200
Subject: [PATCH] Use "gar" instead of "ar" for ranlib.

---
 pdns/ext/polarssl-1.1.2/library/Makefile |    3 +
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/pdns/ext/polarssl-1.1.2/library/Makefile b/pdns/ext/polarssl-1.1.2/library/Makefile
index 26de4d7..efcca3e 100644
--- a/pdns/ext/polarssl-1.1.2/library/Makefile
+++ b/pdns/ext/polarssl-1.1.2/library/Makefile
@@ -22,6 +22,9 @@ DLEXT=so
 # Windows shared library extension:
 # DLEXT=dll
 
+# Solaris OpenCSW
+AR=/opt/csw/bin/gar
+
 OBJS=	aes.o		arc4.o		asn1parse.o		\
 		base64.o	bignum.o	camellia.o		\
 		certs.o		cipher.o	cipher_wrap.o	\
-- 
1.7.10

