From f0c281399bfdd40dd9d02d5a3fe07417dd20b070 Mon Sep 17 00:00:00 2001
From: Juraj Lutter <wilbury@opencsw.org>
Date: Wed, 4 Apr 2012 13:12:24 +0200
Subject: [PATCH] Use correct namespace for timegm()

---
 modules/ldapbackend/utils.hh |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/modules/ldapbackend/utils.hh b/modules/ldapbackend/utils.hh
index ba60f9f..25392c3 100644
--- a/modules/ldapbackend/utils.hh
+++ b/modules/ldapbackend/utils.hh
@@ -147,7 +147,7 @@ inline time_t str2tstamp( const string& str )
 
         if( tmp != NULL && *tmp == 0 )
         {
-        	return timegm( &tm );
+        	return Utility::timegm( &tm );
         }
 
         return 0;
-- 
1.7.9

