diff --speed-large-files --minimal -Nru hatari-1.2.0.orig/src/int.c hatari-1.2.0/src/int.c
--- hatari-1.2.0.orig/src/int.c	2009-01-24 15:12:44.000000000 +0100
+++ hatari-1.2.0/src/int.c	2009-03-14 02:54:10.150429990 +0100
@@ -66,7 +66,12 @@
 
 const char Int_fileid[] = "Hatari int.c : " __DATE__ " " __TIME__;
 
+#if defined (__SVR4)  && defined (__sun)
+#include <sys/types.h>
+#else
 #include <stdint.h>
+#endif
+
 #include "main.h"
 #include "blitter.h"
 #include "dmaSnd.h"
diff --speed-large-files --minimal -Nru hatari-1.2.0.orig/src/uae-cpu/sysdeps.h hatari-1.2.0/src/uae-cpu/sysdeps.h
--- hatari-1.2.0.orig/src/uae-cpu/sysdeps.h	2009-01-24 15:12:45.000000000 +0100
+++ hatari-1.2.0/src/uae-cpu/sysdeps.h	2009-03-14 02:54:19.410281923 +0100
@@ -30,7 +30,11 @@
 #endif
 
 #include <stdarg.h>
+#if defined (__SVR4)  && defined (__sun)
+#include <sys/types.h>
+#else
 #include <stdint.h>
+#endif
 
 
 #if EEXIST == ENOTEMPTY
