diff -ur multitail-5.2.2.orig/makefile.solaris_sunwspro multitail-5.2.2/makefile.solaris_sunwspro
--- multitail-5.2.2.orig/makefile.solaris_sunwspro	2009-03-07 13:52:50.277658365 +0100
+++ multitail-5.2.2/makefile.solaris_sunwspro	2009-03-07 14:07:17.458014674 +0100
@@ -1,11 +1,9 @@
 include version
 
-DESTDIR=/
-CONFIG_FILE=$(DESTDIR)/etc/multitail.conf
+CONFIG_FILE=$(sysconfdir)/multitail.conf
 
-DEBUG=#-g -D_DEBUG #-pg #-fprofile-arcs
-CFLAGS=${EXTRA_CFLAGS} -O2 -I/usr/local/include/ -D_STDC_C99 -DVERSION=\"$(VERSION)\" $(DEBUG) -DCONFIG_FILE=\"$(CONFIG_FILE)\"
-LDFLAGS=${EXTRA_LDFLAGS} -L/usr/local/lib/ -lsocket -lpanel -lncurses -lnsl -lm $(DEBUG)
+CFLAGS +=${EXTRA_CFLAGS} -D_STDC_C99 -DVERSION=\"$(VERSION)\" -DCONFIG_FILE=\"$(CONFIG_FILE)\"
+LDFLAGS+=${EXTRA_LDFLAGS} -lsocket -lpanel -lncurses -lnsl -lm
 
 OBJS=utils.o mt.o error.o my_pty.o term.o scrollback.o help.o mem.o cv.o color.o stripstring.o selbox.o misc.o ui.o exec.o diff.o config.o cmdline.o globals.o history.o
 
