From be1063f6b42879daeab42f956fc9e59f2bdda6c5 Mon Sep 17 00:00:00 2001
From: Juraj Lutter <wilbury@opencsw.org>
Date: Thu, 5 Apr 2012 23:15:17 +0200
Subject: [PATCH] Add -llber to make OpenLDAP happy.

---
 modules/ldapbackend/OBJECTLIBS |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/modules/ldapbackend/OBJECTLIBS b/modules/ldapbackend/OBJECTLIBS
index a787b8d..4adc2f0 100644
--- a/modules/ldapbackend/OBJECTLIBS
+++ b/modules/ldapbackend/OBJECTLIBS
@@ -1 +1 @@
--lldap_r
\ No newline at end of file
+-lldap_r -llber
-- 
1.7.9

