From 14de0bf2c22ad53e3e57776e71e696f8f6068b2c Mon Sep 17 00:00:00 2001
From: Ralph Boehme <sloowfranklin@gmail.com>
Date: Mon, 5 May 2014 15:05:03 +0200
Subject: [PATCH] Fix POSIX and XOPEN defines

---
 configure | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure b/configure
index 17b0a17..00bda6a 100755
--- a/configure
+++ b/configure
@@ -23796,10 +23796,10 @@ fi
 case $host in
   *-*-solaris* )
 
-$as_echo "#define _XOPEN_SOURCE_EXTENDED 1" >>confdefs.h
+# $as_echo "#define _XOPEN_SOURCE_EXTENDED 1" >>confdefs.h
 
 
-$as_echo "#define _XOPEN_SOURCE 2" >>confdefs.h
+$as_echo "#define _XOPEN_SOURCE 600" >>confdefs.h
 
 
 $as_echo "#define __EXTENSIONS__ 1" >>confdefs.h
-- 
1.8.4.1

