From 5e52aa40abe0e1089cef06517e49a5f84b1f49e0 Mon Sep 17 00:00:00 2001
From: Rafael Ostertag <rafisol@opencsw.org>
Date: Mon, 5 Sep 2011 20:47:36 +0200
Subject: [PATCH] Use proper locale for collate tests.

---
 glib/tests/collate.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/glib/tests/collate.c b/glib/tests/collate.c
index eaae049..09da374 100644
--- a/glib/tests/collate.c
+++ b/glib/tests/collate.c
@@ -184,9 +184,9 @@ main (int argc, char *argv[])
 
   g_test_init (&argc, &argv, NULL);
 
-  g_setenv ("LC_ALL", "en_US", TRUE);
+  g_setenv ("LC_ALL", "en_US.UTF-8", TRUE);
   locale = setlocale (LC_ALL, "");
-  if (locale == NULL || strcmp (locale, "en_US") != 0)
+  if (locale == NULL || strcmp (locale, "en_US.UTF-8") != 0)
     {
       g_test_message ("No suitable locale, skipping test");
       return 0;
-- 
1.7.6

