From 903ddf339228d00341b397ec4c172ca64d3b6887 Mon Sep 17 00:00:00 2001
From: Rafael Ostertag <raos@opencsw.org>
Date: Mon, 30 Jan 2012 08:00:40 +0100
Subject: [PATCH] Replace sh by svcs in utils/find-program

---
 glib/tests/utils.c |    9 +++++----
 1 files changed, 5 insertions(+), 4 deletions(-)

diff --git a/glib/tests/utils.c b/glib/tests/utils.c
index 81f2706..c8344c7 100644
--- a/glib/tests/utils.c
+++ b/glib/tests/utils.c
@@ -210,12 +210,13 @@ test_find_program (void)
 {
   gchar *res;
 
-  res = g_find_program_in_path ("sh");
-  g_assert_cmpstr (res, ==, "/bin/sh");
+  /* sh may be ambigous on Solaris */
+  res = g_find_program_in_path ("svcs");
+  g_assert_cmpstr (res, ==, "/usr/bin/svcs");
   g_free (res);
 
-  res = g_find_program_in_path ("/bin/sh");
-  g_assert_cmpstr (res, ==, "/bin/sh");
+  res = g_find_program_in_path ("/usr/bin/svcs");
+  g_assert_cmpstr (res, ==, "/usr/bin/svcs");
   g_free (res);
 
   res = g_find_program_in_path ("this_program_does_not_exit");
-- 
1.7.8.3

