From c26dd7b80189f5d0a19a0b85d00ecbbd93b45624 Mon Sep 17 00:00:00 2001
From: Rafael Ostertag <rafisol@opencsw.org>
Date: Thu, 23 Aug 2012 15:33:09 +0200
Subject: [PATCH] Adjust zoneinfo database path

---
 glib/gtimezone.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/glib/gtimezone.c b/glib/gtimezone.c
index 7bb48d2..8cbb487 100644
--- a/glib/gtimezone.c
+++ b/glib/gtimezone.c
@@ -416,7 +416,11 @@ zone_info_unix (const gchar *identifier)
 
       tzdir = getenv ("TZDIR");
       if (tzdir == NULL)
+#ifndef __sun
         tzdir = "/usr/share/zoneinfo";
+#else
+        tzdir = "/usr/share/lib/zoneinfo";
+#endif
 
       if (*identifier == ':')
         identifier ++;
-- 
1.8.1.4

