From ae24538028785188fb5640dd74984b7341ff93d9 Mon Sep 17 00:00:00 2001
From: Rafael Ostertag <rafisol@opencsw.org>
Date: Wed, 7 Sep 2011 11:40:26 +0200
Subject: [PATCH] Use smtp instead of http in network-address test

---
 gio/tests/network-address.c |    4 +++-
 1 files changed, 3 insertions(+), 1 deletions(-)

diff --git a/gio/tests/network-address.c b/gio/tests/network-address.c
index aabb8ef..0968395 100644
--- a/gio/tests/network-address.c
+++ b/gio/tests/network-address.c
@@ -34,7 +34,9 @@ static ParseTest tests[] =
 {
   { "www.gnome.org", "www.gnome.org", 1234, -1 },
   { "www.gnome.org:8080", "www.gnome.org", 8080, -1 },
-  { "www.gnome.org:http", "www.gnome.org", 80, -1 },
+  /* On my Solaris 10, http is not /etc/services, so switching to smtp
+     is not a bad idead */
+  { "www.gnome.org:smtp", "www.gnome.org", 25, -1 },
   { "[2001:db8::1]", "2001:db8::1", 1234, -1 },
   { "[2001:db8::1]:888", "2001:db8::1", 888, -1 },
   { "[hostname", NULL, 0, G_IO_ERROR_INVALID_ARGUMENT },
-- 
1.7.6

