From ba33d77921abb4a37738ece7702d0631af50d8e4 Mon Sep 17 00:00:00 2001
From: Rafael Ostertag <rafisol@opencsw.org>
Date: Wed, 7 Sep 2011 19:07:36 +0200
Subject: [PATCH] Fix hard coded paths

---
 gio/xdgmime/xdgmime.c |    2 +-
 glib/gutils.c         |    4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/gio/xdgmime/xdgmime.c b/gio/xdgmime/xdgmime.c
index 846be39..e7d004a 100644
--- a/gio/xdgmime/xdgmime.c
+++ b/gio/xdgmime/xdgmime.c
@@ -257,7 +257,7 @@ xdg_run_command_on_dirs (XdgDirectoryFunc  func,
 
   xdg_data_dirs = getenv ("XDG_DATA_DIRS");
   if (xdg_data_dirs == NULL)
-    xdg_data_dirs = "/usr/local/share/:/usr/share/";
+    xdg_data_dirs = "/opt/csw/share/:/usr/share/";
 
   ptr = xdg_data_dirs;
 
diff --git a/glib/gutils.c b/glib/gutils.c
index 92b7492..ba64597 100644
--- a/glib/gutils.c
+++ b/glib/gutils.c
@@ -2935,7 +2935,7 @@ g_get_system_data_dirs (void)
       gchar *data_dirs = (gchar *) g_getenv ("XDG_DATA_DIRS");
 
       if (!data_dirs || !data_dirs[0])
-          data_dirs = "/usr/local/share/:/usr/share/";
+          data_dirs = "/opt/csw/share/:/usr/share/";
 
       data_dir_vector = g_strsplit (data_dirs, G_SEARCHPATH_SEPARATOR_S, 0);
 #endif
@@ -2997,7 +2997,7 @@ g_get_system_config_dirs (void)
       conf_dirs = (gchar *) g_getenv ("XDG_CONFIG_DIRS");
 
       if (!conf_dirs || !conf_dirs[0])
-          conf_dirs = "/etc/xdg";
+          conf_dirs = "/etc/opt/csw/xdg:/opt/csw/etc/xdg:/etc/xdg";
 
       conf_dir_vector = g_strsplit (conf_dirs, G_SEARCHPATH_SEPARATOR_S, 0);
 #endif
-- 
1.7.6

