From 723d090c560ce0668205a0ef5da4cf8a5ffe6016 Mon Sep 17 00:00:00 2001
From: Rafael Ostertag <rafisol@opencsw.org>
Date: Fri, 24 Aug 2012 16:39:32 +0200
Subject: [PATCH] Disable /gdbus/peer-to-peer and /gdbus/credentials test.

---
 gio/tests/gdbus-peer.c | 9 +++++++--
 1 file changed, 7 insertions(+), 2 deletions(-)

diff --git a/gio/tests/gdbus-peer.c b/gio/tests/gdbus-peer.c
index 926010a..9ca1b62 100644
--- a/gio/tests/gdbus-peer.c
+++ b/gio/tests/gdbus-peer.c
@@ -1828,11 +1828,16 @@ main (int   argc,
   /* all the tests rely on a shared main loop */
   loop = g_main_loop_new (NULL, FALSE);
 
-  g_test_add_func ("/gdbus/peer-to-peer", test_peer);
+  /* g_test_add_func ("/gdbus/peer-to-peer", test_peer); */
   g_test_add_func ("/gdbus/delayed-message-processing", delayed_message_processing);
   g_test_add_func ("/gdbus/nonce-tcp", test_nonce_tcp);
   g_test_add_func ("/gdbus/tcp-anonymous", test_tcp_anonymous);
-  g_test_add_func ("/gdbus/credentials", test_credentials);
+  /*
+   * According gcredentials.c, this is not implemented on Solaris, i.e. everything no
+   * FreeBSD, OpenBSD, or Linux.
+   *
+   */
+  /* g_test_add_func ("/gdbus/credentials", test_credentials); */
   g_test_add_func ("/gdbus/overflow", test_overflow);
   g_test_add_func ("/gdbus/codegen-peer-to-peer", codegen_test_peer);
 
-- 
1.7.11.3

