From 32655b53b5b27c57c36901ed5e3a0e518178427e Mon Sep 17 00:00:00 2001
From: Rafael Ostertag <rafisol@opencsw.org>
Date: Sat, 10 Sep 2011 14:01:22 +0200
Subject: [PATCH] Fix configure _XOPEN_SOURCE

---
 configure |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/configure b/configure
index 86472d5..a6831c3 100755
--- a/configure
+++ b/configure
@@ -23497,7 +23497,7 @@ case $host in
 $as_echo "#define _XOPEN_SOURCE_EXTENDED 1" >>confdefs.h
 
 
-$as_echo "#define _XOPEN_SOURCE 2" >>confdefs.h
+$as_echo "#define _XOPEN_SOURCE 600" >>confdefs.h
 
 
 $as_echo "#define __EXTENSIONS__ 1" >>confdefs.h
-- 
1.7.6

