From d61f7f5a9c68a1d8d6c05b10d793b933252e8834 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Fri, 23 Sep 2011 23:23:52 +0200
Subject: [PATCH] Do not ellipses in switch cases

---
 glib/tests/checksum.c |   18 ++++++++++++++++--
 1 files changed, 16 insertions(+), 2 deletions(-)

diff --git a/glib/tests/checksum.c b/glib/tests/checksum.c
index 0e51736..26469a5 100644
--- a/glib/tests/checksum.c
+++ b/glib/tests/checksum.c
@@ -615,9 +615,23 @@ hexval (const gchar c)
 {
   switch (c)
     {
-     case '0' ... '9':
+     case '0':
+     case '1':
+     case '2':
+     case '3':
+     case '4':
+     case '5':
+     case '6':
+     case '7':
+     case '8':
+     case '9':
        return c - '0';
-     case 'a' ... 'f':
+     case 'a':
+     case 'b':
+     case 'c':
+     case 'd':
+     case 'e':
+     case 'f':
        return 10 + c - 'a';
      default:
        return 0;
-- 
1.7.5.4

