From 45edb4fed7b58e5ec7ed6b055f0dd73536821a7a Mon Sep 17 00:00:00 2001
From: Rafael Ostertag <raos@opencsw.org>
Date: Sat, 24 Sep 2011 17:06:12 +0200
Subject: [PATCH] Format string for Solaris lt. 10u9

---
 glib/tests/string.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/glib/tests/string.c b/glib/tests/string.c
index ff172bd..8f85c93 100644
--- a/glib/tests/string.c
+++ b/glib/tests/string.c
@@ -102,7 +102,7 @@ test_string_printf (void)
 #ifndef G_OS_WIN32
   /* MSVC and mingw32 use the same run-time C library, which doesn't like
      the %10000.10000f format... */
-  g_string_printf (string, "%s|%0100d|%s|%0*d|%*.*f|%10000.10000f",
+  g_string_printf (string, "%s|%0100d|%s|%0*d|%*.*f|%100.100f",
 		   "this pete guy sure is a wuss, like he's the number ",
 		   1,
 		   " wuss.  everyone agrees.\n",
-- 
1.7.6.1

