From c7e81851af3b8aae37e3e7d7b7799dd188cf47cc Mon Sep 17 00:00:00 2001
From: Rafael Ostertag <raos@opencsw.org>
Date: Sat, 1 Oct 2011 12:36:20 +0200
Subject: [PATCH] Fix gdbus-peer tests

---
 gio/tests/gdbus-peer.c |   37 +++++++++++++++++++++++++++++++++++--
 1 files changed, 35 insertions(+), 2 deletions(-)

diff --git a/gio/tests/gdbus-peer.c b/gio/tests/gdbus-peer.c
index e304618..51d155f 100644
--- a/gio/tests/gdbus-peer.c
+++ b/gio/tests/gdbus-peer.c
@@ -41,6 +41,7 @@
 #ifdef G_OS_UNIX
 #include <gio/gunixconnection.h>
 #include <errno.h>
+#include <stdlib.h>
 #endif
 
 #include "gdbus-tests.h"
@@ -51,6 +52,7 @@ static gboolean is_unix = TRUE;
 static gboolean is_unix = FALSE;
 #endif
 
+static gchar *tmp_address = NULL;
 static gchar *test_guid = NULL;
 static GMainLoop *service_loop = NULL;
 static GDBusServer *server = NULL;
@@ -328,7 +330,7 @@ service_thread_func (gpointer user_data)
 
   error = NULL;
   observer = g_dbus_auth_observer_new ();
-  server = g_dbus_server_new_sync (is_unix ? "unix:tmpdir=/tmp/gdbus-test-" : "nonce-tcp:",
+  server = g_dbus_server_new_sync (tmp_address,
                                    G_DBUS_SERVER_FLAGS_NONE,
                                    test_guid,
                                    observer,
@@ -1002,7 +1004,7 @@ dmp_thread_func (gpointer user_data)
 
   error = NULL;
   guid = g_dbus_generate_guid ();
-  data->server = g_dbus_server_new_sync ("unix:tmpdir=/tmp/gdbus-test-",
+  data->server = g_dbus_server_new_sync (tmp_address,
                                          G_DBUS_SERVER_FLAGS_NONE,
                                          guid,
                                          NULL, /* GDBusAuthObserver */
@@ -1444,6 +1446,8 @@ main (int   argc,
 {
   gint ret;
   GDBusNodeInfo *introspection_data = NULL;
+  gchar *tmpdir = NULL;
+  gchar *template = NULL;
 
   g_type_init ();
   g_thread_init (NULL);
@@ -1455,6 +1459,21 @@ main (int   argc,
 
   test_guid = g_dbus_generate_guid ();
 
+  if (is_unix)
+    {
+      if (g_unix_socket_address_abstract_names_supported ())
+	tmp_address = g_strdup ("unix:tmpdir=/tmp/gdbus-test-");
+      else
+	{
+	  template = g_strdup("/tmp/gdbus-test-XXXXXX");
+	  tmpdir = mktemp(template);
+	  mkdir(tmpdir, S_IRWXU | S_IRWXG | S_IROTH | S_IXOTH);
+	  tmp_address = g_strdup_printf ("unix:tmpdir=%s", tmpdir);
+	}
+    }
+  else
+    tmp_address = g_strdup ("nonce-tcp:");
+
   /* all the tests rely on a shared main loop */
   loop = g_main_loop_new (NULL, FALSE);
 
@@ -1463,14 +1482,28 @@ main (int   argc,
 #ifdef BUG_631379_FIXED
   g_test_add_func ("/gdbus/nonce-tcp", test_nonce_tcp);
 #endif
+#if defined(__linux__) || defined(__FreeBSD__)
   g_test_add_func ("/gdbus/credentials", test_credentials);
+#endif
+  /* Won't work on sparc */
+#ifndef __sparc
   g_test_add_func ("/gdbus/overflow", test_overflow);
+#endif
 
   ret = g_test_run();
 
   g_main_loop_unref (loop);
   g_free (test_guid);
   g_dbus_node_info_unref (introspection_data);
+  if (is_unix)
+    g_free (tmp_address);
+  if (template)
+    g_free (template);
+  if (tmpdir)
+    {
+      g_rmdir (tmpdir);
+      g_free (tmpdir);
+    }
 
   return ret;
 }
-- 
1.7.6.1

