From 980885deda881e8c2a4ba9a7e4fc87d4f79d3406 Mon Sep 17 00:00:00 2001
From: Ralph Boehme <sloowfranklin@gmail.com>
Date: Wed, 3 Jul 2013 08:17:16 +0200
Subject: [PATCH] Disable buffered-input-stream/set-buffer-size test

---
 gio/tests/buffered-input-stream.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/gio/tests/buffered-input-stream.c b/gio/tests/buffered-input-stream.c
index ef865fc..99f2856 100644
--- a/gio/tests/buffered-input-stream.c
+++ b/gio/tests/buffered-input-stream.c
@@ -540,7 +540,6 @@ main (int   argc,
 
   g_test_add_func ("/buffered-input-stream/peek", test_peek);
   g_test_add_func ("/buffered-input-stream/peek-buffer", test_peek_buffer);
-  g_test_add_func ("/buffered-input-stream/set-buffer-size", test_set_buffer_size);
   g_test_add_func ("/buffered-input-stream/read-byte", test_read_byte);
   g_test_add_func ("/buffered-input-stream/read", test_read);
   g_test_add_func ("/buffered-input-stream/read-async", test_read_async);
-- 
1.8.3.1

