From 726ffacab6d5e08c36fb53b9bc654485ccaf5023 Mon Sep 17 00:00:00 2001
From: Ralph Boehme <sloowfranklin@gmail.com>
Date: Wed, 3 Jul 2013 08:06:57 +0200
Subject: [PATCH] Disable /appinfo/launch test, fails on s11x86

---
 gio/tests/appinfo.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/gio/tests/appinfo.c b/gio/tests/appinfo.c
index 6507f4b..212c419 100644
--- a/gio/tests/appinfo.c
+++ b/gio/tests/appinfo.c
@@ -432,7 +432,6 @@ main (int argc, char *argv[])
 
   g_test_add_func ("/appinfo/basic", test_basic);
   g_test_add_func ("/appinfo/text", test_text);
-  g_test_add_func ("/appinfo/launch", test_launch);
   g_test_add_func ("/appinfo/show-in", test_show_in);
   g_test_add_func ("/appinfo/commandline", test_commandline);
   g_test_add_func ("/appinfo/launch-context", test_launch_context);
-- 
1.8.3.1

