--- glib-2.23.1/gio/gio.h.orig	2010-01-09 15:33:51.097448405 +0100
+++ glib-2.23.1/gio/gio.h	2010-01-09 15:34:37.201482946 +0100
@@ -97,6 +97,22 @@
 #include <gio/gzlibcompressor.h>
 #include <gio/gzlibdecompressor.h>
 
+#ifndef CMSG_ALIGN
+#   ifdef __sun__
+#       define CMSG_ALIGN _CMSG_DATA_ALIGN
+#   else
+#       define CMSG_ALIGN(len) (((len)+sizeof(long)-1) & ~(sizeof(long)-1))
+#   endif
+#endif
+
+#ifndef CMSG_SPACE
+#   define CMSG_SPACE(len) (CMSG_ALIGN(sizeof(struct cmsghdr))+CMSG_ALIGN(len))
+#endif
+
+#ifndef CMSG_LEN
+#   define CMSG_LEN(len) (CMSG_ALIGN(sizeof(struct cmsghdr))+(len))
+#endif
+
 #undef __GIO_GIO_H_INSIDE__
 
 #endif /* __G_IO_H__ */
