From 3aaf73298717337b2002ae64d843d1342778c370 Mon Sep 17 00:00:00 2001
From: Juergen Arndt <ja@opencsw.org>
Date: Tue, 28 Feb 2012 14:43:46 +0100
Subject: [PATCH] changing packetbuffer length to 8k

---
 include/common.h |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/common.h b/include/common.h
index fb3b429..6591d10 100755
--- a/include/common.h
+++ b/include/common.h
@@ -41,7 +41,7 @@
 #define DEFAULT_SOCKET_TIMEOUT	10	/* timeout after 10 seconds */
 #define DEFAULT_CONNECTION_TIMEOUT 300	/* timeout if daemon is waiting for connection more than this time */
 
-#define MAX_INPUT_BUFFER	2048	/* max size of most buffers we use */
+#define MAX_INPUT_BUFFER	8192	/* max size of most buffers we use */
 #define MAX_FILENAME_LENGTH     256
 
 #define MAX_HOST_ADDRESS_LENGTH	256	/* max size of a host address */
@@ -60,7 +60,7 @@
 #define NRPE_PACKET_VERSION_2   2               
 #define NRPE_PACKET_VERSION_1	1		/* older packet version identifiers (no longer supported) */
 
-#define MAX_PACKETBUFFER_LENGTH	1024		/* max amount of data we'll send in one query/response */
+#define MAX_PACKETBUFFER_LENGTH	8192		/* max amount of data we'll send in one query/response */
 
 typedef struct packet_struct{
 	int16_t   packet_version;
-- 
1.7.9

