diff --speed-large-files --minimal -Nru nrpe-2.12.orig/Makefile.in nrpe-2.12/Makefile.in
--- nrpe-2.12.orig/Makefile.in	2008-12-25 13:05:01.688902701 +0100
+++ nrpe-2.12/Makefile.in	2008-12-25 13:04:44.279858247 +0100
@@ -8,6 +8,7 @@
 # Source code directories
 SRC_BASE=./src/
 SRC_INCLUDE=./include/
+DOC_BASE=./docs/
 
 CC=@CC@
 CFLAGS=@CFLAGS@ @DEFS@
@@ -19,6 +20,8 @@
 BINDIR=@bindir@
 SBINDIR=@sbindir@
 LIBEXECDIR=@libexecdir@
+DATADIR=@datadir@
+DOCDIR=$(DATADIR)/doc/nrpe
 INSTALL=@INSTALL@
 NAGIOS_INSTALL_OPTS=
 NRPE_INSTALL_OPTS=
@@ -62,6 +65,15 @@
 	$(INSTALL) -m 775 $(NAGIOS_INSTALL_OPTS) -d $(DESTDIR)$(CFGDIR)
 	$(INSTALL) -m 644 $(NRPE_INSTALL_OPTS) sample-config/nrpe.cfg $(DESTDIR)$(CFGDIR)
 
+install-docs:
+	$(INSTALL) -m 775 $(NAGIOS_INSTALL_OPTS) -d $(DESTDIR)$(DOCDIR)
+	$(INSTALL) -m 644 $(NRPE_INSTALL_OPTS) $(DOC_BASE)/NRPE.pdf $(DESTDIR)$(DOCDIR)
+	$(INSTALL) -m 644 $(NRPE_INSTALL_OPTS) ./LEGAL $(DESTDIR)$(DOCDIR)
+	$(INSTALL) -m 644 $(NRPE_INSTALL_OPTS) ./README $(DESTDIR)$(DOCDIR)
+	$(INSTALL) -m 644 $(NRPE_INSTALL_OPTS) ./README.SSL $(DESTDIR)$(DOCDIR)
+	$(INSTALL) -m 644 $(NRPE_INSTALL_OPTS) ./SECURITY $(DESTDIR)$(DOCDIR)
+
+
 clean:
 	cd $(SRC_BASE); $(MAKE) $@ ; cd ..
 	rm -f core
