diff --speed-large-files --minimal -Nru seamonkey.orig/toolkit/mozapps/installer/packager.mk seamonkey/toolkit/mozapps/installer/packager.mk
--- seamonkey.orig/toolkit/mozapps/installer/packager.mk	2008-04-28 19:14:12.000000000 +0000
+++ seamonkey/toolkit/mozapps/installer/packager.mk	2008-11-15 20:28:41.679741300 +0000
@@ -332,7 +332,7 @@
 endif # MOZ_PKG_MANIFEST
 ifndef PKG_SKIP_STRIP
 	@echo "Stripping package directory..."
-	@cd $(DIST)/$(STAGEPATH)$(MOZ_PKG_APPNAME); find . ! -type d \
+	@cd $(DIST)/$(STAGEPATH)$(MOZ_PKG_APPNAME); gfind . ! -type d \
 			! -name "*.js" \
 			! -name "*.xpt" \
 			! -name "*.gif" \
