--- protobuf-2.3.0/src/google/protobuf/compiler/command_line_interface_unittest.cc.orig	Tue Feb 23 11:17:08 2010
+++ protobuf-2.3.0/src/google/protobuf/compiler/command_line_interface_unittest.cc	Tue Feb 23 11:18:09 2010
@@ -1009,7 +1009,8 @@
   }
 #endif
 
-  ExpectErrorSubstring("foo.proto/: Not a directory");
+  ExpectErrorSubstring("foo.proto/");
+  ExpectErrorSubstring("Not a directory");
 }
 
 TEST_F(CommandLineInterfaceTest, GeneratorError) {
