--- a/Makefile.in	Tue Dec 30 22:04:25 2008
+++ b/Makefile.in	Thu Jun 27 00:14:39 2013
@@ -11,7 +11,7 @@
 WFLAGS=-Wall -W -Wshadow -Wpointer-arith -Wwrite-strings -pedantic -ffor-scope -Wcast-align -Wsign-compare -Wpointer-arith -Wwrite-strings -Wformat-security -Wswitch-enum -Winit-self $(MORE_WARNINGS)
 CFLAGS=-O2 @debug@ -DNDEBUG $(WFLAGS) $(MORECFLAGS)
 CXX=@CXX@ $(CFLAGS)
-LINK=@CXX@
+LINK=@CXX@ $(LDFLAGS)
 THREAD_LFLAGS=@thread_ldflags@
 
 INSTALL=@INSTALL@
