diff --speed-large-files --minimal -Nru flex-2.5.4.orig/Makefile.in flex-2.5.4/Makefile.in
--- flex-2.5.4.orig/Makefile.in	2005-04-21 13:52:20.679323000 -0400
+++ flex-2.5.4/Makefile.in	2005-04-21 13:52:16.031842000 -0400
@@ -28,7 +28,7 @@
 # Raw, unformatted troff source will be installed if INSTALLMAN=man,
 # nroff preformatted versions will be installed if INSTALLMAN=cat.
 
-prefix = @prefix@
+prefix = $(DESTDIR)@prefix@
 exec_prefix = @exec_prefix@
 bindir = $(exec_prefix)/bin
 libdir = $(exec_prefix)/lib
