From 3ecc8abf7b82e1dc359d788d0410c4e9d00a6302 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Tue, 3 Jan 2012 14:28:05 +0100
Subject: [PATCH] Use putenv() instead of setenv()

---
 arguments.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arguments.c b/arguments.c
index a92fe10..3a03e14 100644
--- a/arguments.c
+++ b/arguments.c
@@ -686,7 +686,7 @@ parse_arguments(int argc, const char **argv, cmdline *result, GPtrArray *ctrls)
 	}		
 
 	if (result->profileonlyp)
-		if (setenv("LDAPNOINIT", "thanks", 1)) syserr();
+		if (putenv("LDAPNOINIT=thanks")) syserr();
 
 	/* don't free! */
 /* 	poptFreeContext(ctx); */
-- 
1.7.6.1

