From 6301179ba982f8a6f4d0f28e8dd70952d449ca54 Mon Sep 17 00:00:00 2001
From: Rafael Ostertag <raos@opencsw.org>
Date: Sun, 9 Dec 2012 12:23:07 +0100
Subject: [PATCH] Fix preprocessor macro FUNC_DEPRECATED to avoid syntax
 errors

---
 src/compiler.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/compiler.h b/src/compiler.h
index 16987c7..32db658 100644
--- a/src/compiler.h
+++ b/src/compiler.h
@@ -49,7 +49,7 @@
 #  define FUNC_DEPRECATED \
     __attribute__((deprecated))
 #else
-#  define FUNC_DEPRECATED(warning_text)
+#  define FUNC_DEPRECATED
 #endif
 
 #endif
-- 
1.8.0

