diff --speed-large-files --minimal -Nru rubygems-0.8.11.orig/post-install.rb rubygems-0.8.11/post-install.rb
--- rubygems-0.8.11.orig/post-install.rb	2005-11-14 13:19:58.117793000 -0500
+++ rubygems-0.8.11/post-install.rb	2005-11-14 13:19:50.794822000 -0500
@@ -66,7 +66,9 @@
     load "sources.gemspec"
     spec = Gem.sources_spec
     gem_file = Gem::Builder.new(spec).build
-    Gem::Installer.new(gem_file).install(true, Gem.dir, false)
+    gem_path = ENV['DESTDIR'] + Gem.dir
+    FileUtils.mkdir_p(gem_path)
+    Gem::Installer.new(gem_file).install(true, gem_path, false)
   end
 end
 
