--- awesome-3.4.4/build-utils/gperf.sh.orig	2010-03-12 14:08:00.910378535 +0000
+++ awesome-3.4.4/build-utils/gperf.sh	2010-03-12 14:08:26.207943113 +0000
@@ -50,7 +50,7 @@
 
 typedef enum awesome_token_t {
     A_TK_UNKNOWN,
-`tr '[:lower:]' '[:upper:]' | sed -e "s/^[^/].*/    A_TK_&,/"`
+`LC_ALL=C tr a-z A-Z | sed -e "s/^[^/].*/    A_TK_&,/"`
 } awesome_token_t;
 
 __attribute__((pure)) enum awesome_token_t a_tokenize(const char *s, int len);
@@ -63,7 +63,7 @@
         case "$tok" in
             "") continue;;
             *)
-                echo "$tok, A_TK_`echo $tok | tr '[:lower:]' '[:upper:]'`"
+                echo "$tok, A_TK_`echo $tok | LC_ALL=C tr a-z A-Z`"
                 ;;
         esac
     done
