diff --speed-large-files --minimal -Nru gtkpod-0.99.8.orig/src/mp4file.c gtkpod-0.99.8/src/mp4file.c
--- gtkpod-0.99.8.orig/src/mp4file.c	2006-09-18 03:10:44.000000000 -0400
+++ gtkpod-0.99.8/src/mp4file.c	2006-10-06 18:26:03.944839000 -0400
@@ -133,6 +133,12 @@
 #include <inttypes.h>
 #include <stdlib.h>
 #include <string.h>
+
+/* Gtkpod doesn't set up HAVE_GLIB_H when it should. */
+#ifndef HAVE_GLIB_H
+#define HAVE_GLIB_H
+#endif
+
 #include "mp4.h"
 #undef PACKAGE
 #undef VERSION
