--- lftp-4.0.6/m4/getopt.m4.orig	Sat Mar 27 14:36:10 2010
+++ lftp-4.0.6/m4/getopt.m4	Sat Mar 27 14:39:57 2010
@@ -57,7 +57,7 @@
   gl_CHECK_NEXT_HEADERS([getopt.h])
   AC_CHECK_HEADERS_ONCE([getopt.h])
   if test $ac_cv_header_getopt_h = yes; then
-    HAVE_GETOPT_H=1
+    HAVE_GETOPT_H=0
   else
     HAVE_GETOPT_H=0
   fi
@@ -67,12 +67,12 @@
 
   dnl Test whether <getopt.h> is available.
   if test -z "$gl_replace_getopt" && test $gl_getopt_required = GNU; then
-    AC_CHECK_HEADERS([getopt.h], [], [gl_replace_getopt=yes])
+    AC_CHECK_HEADERS([getopt.h], [gl_replace_getopt=yes], [gl_replace_getopt=yes])
   fi
 
   dnl Test whether the function getopt_long is available.
   if test -z "$gl_replace_getopt" && test $gl_getopt_required = GNU; then
-    AC_CHECK_FUNCS([getopt_long_only], [], [gl_replace_getopt=yes])
+    AC_CHECK_FUNCS([getopt_long_only], [gl_replace_getopt=yes], [gl_replace_getopt=yes])
   fi
 
   dnl BSD getopt_long uses an incompatible method to reset option processing.
