diff --speed-large-files --minimal -Nru httpd-2.0.54.orig/configure httpd-2.0.54/configure
--- httpd-2.0.54.orig/configure	2005-09-26 19:29:02.294952000 -0400
+++ httpd-2.0.54/configure	2005-09-26 19:26:47.624888000 -0400
@@ -17035,7 +17035,7 @@
 _ACEOF
 
 
-perlbin=`$ac_aux_dir/PrintPath perl`
+perlbin=/usr/bin/perl
 if test "x$perlbin" = "x"; then
     perlbin="/replace/with/path/to/perl/interpreter"
 fi
