--- nss-3.12.4-with-nspr-4.8/mozilla/security/coreconf/UNIX.mk.orig	2009-11-22 02:25:17.411354784 +0100
+++ nss-3.12.4-with-nspr-4.8/mozilla/security/coreconf/UNIX.mk	2009-11-22 02:25:24.833876220 +0100
@@ -46,7 +46,7 @@
 	DEFINES    += -UDEBUG -DNDEBUG
 else
 	OPTIMIZER  += -g
-	USERNAME   := $(shell whoami)
+	USERNAME   := $(shell gwhoami)
 	USERNAME   := $(subst -,_,$(USERNAME))
 	DEFINES    += -DDEBUG -UNDEBUG -DDEBUG_$(USERNAME)
 endif
