From 4ddab6dc87cad733a900256d82e44199fe3e3d46 Mon Sep 17 00:00:00 2001
From: Ben Walton <bwalton@opencsw.org>
Date: Thu, 28 Jun 2012 02:42:26 +0200
Subject: [PATCH] Bludgeon the build system to do what we need

Signed-off-by: Ben Walton <bwalton@opencsw.org>
---
 src/Makefile |   14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/src/Makefile b/src/Makefile
index 3938401..3ee04e0 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -1,15 +1,15 @@
 DESTDIR =
-PREFIX = /usr/local
+PREFIX = /opt/csw
 BINDIR = $(PREFIX)/bin
 SHAREDIR = $(PREFIX)/share/imapfilter
-MANDIR = $(PREFIX)/man
+MANDIR = $(PREFIX)/share/man
 
-MYCFLAGS =
-MYLDFLAGS =
-MYLIBS =
+MYCFLAGS := $(CFLAGS)
+MYLDFLAGS := $(LDFLAGS)
+MYLIBS := $(LIBS)
 
-INCDIRS =
-LIBDIRS =
+INCDIRS = -I$(PREFIX)/include
+LIBDIRS = -L$(PREFIX)/lib
 
 LIBLUA = -llua
 LIBPCRE = -lpcre
-- 
1.7.10.3

