From b60beed92d4d9eca62a718a4c1fe1b5d18abb03b Mon Sep 17 00:00:00 2001
From: Juraj Lutter <wilbury@opencsw.org>
Date: Mon, 15 Apr 2013 15:52:58 +0200
Subject: [PATCH] Include proper memcached.h

---
 php_libmemcached_compat.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/php_libmemcached_compat.h b/php_libmemcached_compat.h
index 6e5f58b..fc6ee7d 100644
--- a/php_libmemcached_compat.h
+++ b/php_libmemcached_compat.h
@@ -2,6 +2,6 @@
 #define PHP_LIBMEMCACHED_COMPAT
 
 /* this is the version(s) we support */
-#include <libmemcached-1.0/memcached.h>
+#include <libmemcached/memcached.h>
 
 #endif
-- 
1.8.1.4

