diff --speed-large-files --minimal -Nru bind2nsd-0.7.2.orig/setup.py bind2nsd-0.7.2/setup.py
--- bind2nsd-0.7.2.orig/setup.py	2008-06-27 02:22:28.000000000 +0200
+++ bind2nsd-0.7.2/setup.py	2009-04-18 11:22:14.530664816 +0200
@@ -37,8 +37,8 @@
       packages = [ 'bind2nsd' ],
       scripts = [ 'scripts/bind2nsd', 'scripts/s64-sync', 'scripts/nsd-sync',
                   'scripts/s64-mkpw', 'scripts/bind2nsd-slave', ],
-      data_files = [ ('/usr/share/doc/bind2nsd', ['./README', './TODO']),
-                     ('/etc/bind2nsd', ['etc/bind2nsd.conf']),
+      data_files = [ ('/opt/csw/share/doc/bind2nsd', ['./README', './TODO']),
+                     ('/etc/opt/csw/bind2nsd', ['etc/bind2nsd.conf']),
 		   ]
      )
 
