Index: Source/NSTimeZone.m
===================================================================
--- a/Source/NSTimeZone.m	(revision 38165)
+++ b/Source/NSTimeZone.m	(working copy)
@@ -112,7 +112,6 @@
 #ifdef HAVE_TZHEAD
 #include <tzfile.h>
 #else
-#define NOID
 #include "nstzfile.h"
 #endif
 
Index: Source/nstzfile.h
===================================================================
--- a/Source/nstzfile.h	(revision 38165)
+++ b/Source/nstzfile.h	(working copy)
@@ -16,23 +16,9 @@
 */
 
 /*
-** ID
-*/
-
-#ifndef lint
-#ifndef NOID
-static char	tzfilehid[] = "@(#)tzfile.h	7.14";
-#endif /* !defined NOID */
-#endif /* !defined lint */
-
-/*
 ** Information about time zone files.
 */
 
-#ifndef TZDIR
-#define TZDIR	"/usr/local/etc/zoneinfo" /* Time zone object file directory */
-#endif /* !defined TZDIR */
-
 #ifndef TZDEFAULT
 #define TZDEFAULT	"localtime"
 #endif /* !defined TZDEFAULT */
