From c0f3d3cfdff8a4838b903eb80bd63c2966f511a2 Mon Sep 17 00:00:00 2001
From: Maciej Blizinski <maciej@opencsw.org>
Date: Sat, 3 Jul 2010 22:37:03 +0200
Subject: [PATCH] Allowing to modify CFLAGS

---
 Makefile.am |    4 ++--
 Makefile.in |    4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index e6fcc76..b667595 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -12,10 +12,10 @@
 #
 
 # CFLAGS =
-CFLAGS = -O2
+CFLAGS ?= -O2
 # CFLAGS = -g -Wall
 
-LDFLAGS = -s
+LDFLAGS ?= -s
 # LDFLAGS =
 
 bin_PROGRAMS =
diff --git a/Makefile.in b/Makefile.in
index 9166e52..6f72858 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -128,7 +128,7 @@ CC = @CC@
 CCDEPMODE = @CCDEPMODE@
 
 # CFLAGS =
-CFLAGS = -O2
+CFLAGS ?= -O2
 CPP = @CPP@
 CPPFLAGS = @CPPFLAGS@
 CYGPATH_W = @CYGPATH_W@
@@ -150,7 +150,7 @@ INSTALL_PROGRAM = @INSTALL_PROGRAM@
 INSTALL_SCRIPT = @INSTALL_SCRIPT@
 INSTALL_STRIP_PROGRAM = @INSTALL_STRIP_PROGRAM@
 # CFLAGS = -g -Wall
-LDFLAGS = -s
+LDFLAGS ?= -s
 LIBOBJS = @LIBOBJS@
 LIBS = @LIBS@
 LN_S = @LN_S@
-- 
1.7.1

