--- ../build-global/Libnet-1.0.2a/configure.in	2001-01-17 21:59:30.000000000 +0100
+++ Libnet-1.0.2a/configure.in	2009-03-24 09:29:18.117017012 +0100
@@ -39,6 +39,7 @@
 dnl Check compiler type.
 dnl
 AC_PROG_CC
+AC_TRY_CPP([#include <wchar.h>])
 
 dnl
 dnl Check for typedefs, structures, and compiler characteristics.
@@ -203,7 +204,7 @@
 
 *solaris*)
     AC_DEFINE(HAVE_SOLARIS)
-    AC_CHECK_HEADERS(sys/bufmod.h sys/dlpi_ext.h)
+    AC_CHECK_HEADERS(sys/bufmod.h sys/dlpi_ext.h sys/sockio.h)
     AC_MSG_CHECKING(for /dev/dlpi device)
     if test -c /dev/dlpi; then
         AC_MSG_RESULT(yes)
@@ -237,10 +238,10 @@
 
     ADDITIONAL_LIBS='-lsocket -lnsl'
     LIBNET_CONFIG_LIBS="$LIBNET_CONFIG_LIBS -lsocket -lnsl"
-    MAN_PREFIX="$prefix/man/man3/"
-    LIB_PREFIX="$prefix/lib/"
-    INC_PREFIX="$prefix/include/"
-    BIN_PREFIX="$prefix/bin/"
+    MAN_PREFIX="$mandir/man3/"
+    LIB_PREFIX="$libdir/"
+    INC_PREFIX="$includedri/"
+    BIN_PREFIX="$bindir/"
     ;;
 
 *)
