From 647781bd34ae85a97a317f38c910bc355b8900c9 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Tue, 31 Dec 2013 17:35:27 +0100
Subject: [PATCH] Honour LDFLAGS

---
 Makefile.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.in b/Makefile.in
index 1bacf82..361bcf7 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -100,7 +100,7 @@ libedit.a: ${OOBJS}
 	ar -r $@ ${OOBJS}
 
 libedit.so: ${OOBJS}
-	${CC} --shared -o $@ ${OOBJS}
+	${CC} $(LDFLAGS) -G -h libedit.so.1 -o $@ ${OOBJS}
 
 # minimal dependency to make "make depend" optional
 editline.o editline.po editline.so editline.ln:	\
-- 
1.8.4.1

