diff -ur imap-2004f.orig/src/osdep/unix/Makefile imap-2004f/src/osdep/unix/Makefile
--- imap-2004f.orig/src/osdep/unix/Makefile	2005-08-31 14:10:13.962828000 -0400
+++ imap-2004f/src/osdep/unix/Makefile	2005-08-31 14:20:10.914947000 -0400
@@ -41,11 +41,12 @@
 
 # Extended flags needed for SSL.  You may need to modify.
 
-SSLDIR=/usr/local/ssl
+PREFIX=/opt/csw
+SSLDIR=$(PREFIX)/ssl
 SSLCERTS=$(SSLDIR)/certs
 SSLKEYS=$(SSLCERTS)
-SSLINCLUDE=$(SSLDIR)/include
-SSLLIB=$(SSLDIR)/lib
+SSLINCLUDE=$(PREFIX)/include
+SSLLIB=$(PREFIX)/lib
 
 SSLCRYPTO=-lcrypto
 
