From 0e3ebcc0dab328d289f88b2bb07b4a3ecc68c377 Mon Sep 17 00:00:00 2001
From: Juergen Arndt <ja@opencsw.org>
Date: Sun, 3 Jun 2012 18:24:50 +0200
Subject: [PATCH] paths in munin-htaccess.in

---
 master/www/munin-htaccess.in |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/master/www/munin-htaccess.in b/master/www/munin-htaccess.in
index f6c6321..1838408 100644
--- a/master/www/munin-htaccess.in
+++ b/master/www/munin-htaccess.in
@@ -7,9 +7,9 @@
 #
 # As a config file enclose it in <directory> like so:
 # 
-# <directory @@HTMLDIR@@>
+# <directory /var/opt/csw/munin/www>
 
-AuthUserFile @@CONFDIR@@/munin-htpasswd
+AuthUserFile /etc/opt/csw/munin/munin-htpasswd
 AuthName "Munin"
 AuthType Basic
 require valid-user
-- 
1.7.10.3

