From 02a69c5ed4cd96c71068e1cb1b2f0a2006321396 Mon Sep 17 00:00:00 2001
From: Juergen Arndt <ja@opencsw.org>
Date: Sun, 3 Jun 2012 18:11:50 +0200
Subject: [PATCH] Use CSWtop in memory.in

---
 plugins/node.d.sunos/memory.in |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/plugins/node.d.sunos/memory.in b/plugins/node.d.sunos/memory.in
index 98f4bce..fea0a31 100644
--- a/plugins/node.d.sunos/memory.in
+++ b/plugins/node.d.sunos/memory.in
@@ -17,7 +17,7 @@ top installed.
 The following shows the default settings for this plugin:
 
   [memory]
-     env.top /usr/local/bin/top
+     env.top /opt/csw/bin/top
 
 You can also set warning and critical limits with these env settings:
 
@@ -61,7 +61,7 @@ Fixme: Using kstat would be much better
 
 . $MUNIN_LIBDIR/plugins/plugin.sh
 
-TOP=${top:-/usr/local/bin/top}
+TOP=${top:-/opt/csw/bin/top}
 
 if [ "$1" = "autoconf" ]; then
     if [ -x $TOP ] ; then
-- 
1.7.10.3

