diff --speed-large-files --minimal -Nru munin-1.4.0.orig/node/munin-node.conf.in munin-1.4.0/node/munin-node.conf.in
--- munin-1.4.0.orig/node/munin-node.conf.in	2009-11-27 12:17:13.000000000 +0100
+++ munin-1.4.0/node/munin-node.conf.in	2009-11-28 11:59:12.681167885 +0100
@@ -26,7 +26,7 @@
 # Set this if the client doesn't report the correct hostname when
 # telnetting to localhost, port 4949
 #
-#host_name @@HOSTNAME@@
+#host_name yourhostname
 
 # A list of addresses that are allowed to connect.  This must be a
 # regular expression, since Net::Server does not understand CIDR-style
