From c7b05c6b5304a67c1b4226ba9473407a4953822d Mon Sep 17 00:00:00 2001
From: Juergen Arndt <ja@opencsw.org>
Date: Sun, 11 Jul 2010 15:21:09 +0200
Subject: [PATCH] set path names to /var/opt/csw/munin

---
 master/_bin/munin-graph.in |    8 ++++----
 1 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/master/_bin/munin-graph.in b/master/_bin/munin-graph.in
index 804d2bd..1ea9ddc 100755
--- a/master/_bin/munin-graph.in
+++ b/master/_bin/munin-graph.in
@@ -135,10 +135,10 @@ needed. To force redrawing of graphs (after setup-changes et alia), use
 
 =head1 FILES
 
-	@@CONFDIR@@/munin.conf
-	@@DBDIR@@/*
-	@@LOGDIR@@/munin-graph
-	@@STATEDIR@@/*
+	/etc/opt/csw/munin/munin.conf
+	/var/opt/csw/munin/*
+	/var/opt/csw/munin/munin-graph
+	/var/opt/csw/munin/*
 
 =head1 AUTHORS
 
-- 
1.7.1

