diff --speed-large-files --minimal -Nru munin-1.4.0.orig/master/_bin/munin-graph.in munin-1.4.0/master/_bin/munin-graph.in
--- munin-1.4.0.orig/master/_bin/munin-graph.in	2009-11-27 12:17:18.000000000 +0100
+++ munin-1.4.0/master/_bin/munin-graph.in	2009-11-28 13:41:02.135025435 +0100
@@ -135,10 +135,10 @@
 
 =head1 FILES
 
-	@@CONFDIR@@/munin.conf
-	@@DBDIR@@/*
-	@@LOGDIR@@/munin-graph
-	@@STATEDIR@@/*
+	/etc/opt/csw/munin/munin.conf
+	/var/opt/csw/munin/*
+	/var/opt/csw/munin/munin-graph
+	/var/opt/csw/munin/*
 
 =head1 AUTHORS
 
