From f94f6692eeb6875abf5e52ecf8a192ac3430c6e3 Mon Sep 17 00:00:00 2001
From: Juergen Arndt <ja@opencsw.org>
Date: Sun, 11 Jul 2010 15:25:17 +0200
Subject: [PATCH] changed default hostname to "yourhostname"

---
 master/munin.conf.in |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/master/munin.conf.in b/master/munin.conf.in
index 3688d88..79d9729 100644
--- a/master/munin.conf.in
+++ b/master/munin.conf.in
@@ -57,7 +57,7 @@ includedir @@CONFDIR@@/munin-conf.d
 #contact.nagios.command /usr/bin/send_nsca nagios.host.comm -c /etc/nsca.conf
 
 # a simple host tree
-[@@HOSTNAME@@]
+[yourhostname]
     address 127.0.0.1
     use_node_name yes
 
-- 
1.7.1

