From 272b43ac170c907476ad5698c960fd84b31996c3 Mon Sep 17 00:00:00 2001
From: Juergen Arndt <ja@opencsw.org>
Date: Sun, 11 Jul 2010 18:25:14 +0200
Subject: [PATCH] munin-node.conf.in: set some paths for OpenCSW, change hostname to "yourhostname"

---
 node/munin-node.conf.in |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/node/munin-node.conf.in b/node/munin-node.conf.in
index 40793b2..67a92a5 100644
--- a/node/munin-node.conf.in
+++ b/node/munin-node.conf.in
@@ -3,8 +3,8 @@
 #
 
 log_level 4
-log_file @@LOGDIR@@/munin-node.log
-pid_file @@STATEDIR@@/munin-node.pid
+log_file /var/opt/csw/munin/munin-node.log
+pid_file /var/opt/csw/munin/run/munin-node.pid
 
 background 1
 setsid 1
@@ -26,7 +26,7 @@ ignore_file \.pod$
 # Set this if the client doesn't report the correct hostname when
 # telnetting to localhost, port 4949
 #
-#host_name @@HOSTNAME@@
+#host_name yourhostname
 
 # A list of addresses that are allowed to connect.  This must be a
 # regular expression, since Net::Server does not understand CIDR-style
-- 
1.7.1

