From 52ea662193a0922aef3c4f82fe2b672e70e5ee13 Mon Sep 17 00:00:00 2001
From: Juergen Arndt <ja@opencsw.org>
Date: Mon, 12 Jul 2010 20:55:23 +0200
Subject: [PATCH] munin-htaccess.in: set paths for OpenCSW

---
 master/www/munin-htaccess.in |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/master/www/munin-htaccess.in b/master/www/munin-htaccess.in
index f6c6321..c5f355d 100644
--- a/master/www/munin-htaccess.in
+++ b/master/www/munin-htaccess.in
@@ -2,14 +2,14 @@
 # config file.
 #
 # For the .htaccess file option to work the munin www directory
-# (@@HTMLDIR@@) must have "AllowOverride all" or something close
+# (/var/opt/csw/munin/www) must have "AllowOverride all" or something close
 # to that set.
 #
 # As a config file enclose it in <directory> like so:
 # 
-# <directory @@HTMLDIR@@>
+# <directory /var/opt/csw/munin/www>
 
-AuthUserFile @@CONFDIR@@/munin-htpasswd
+AuthUserFile /etc/opt/csw/munin/munin-htpasswd
 AuthName "Munin"
 AuthType Basic
 require valid-user
-- 
1.7.1

