diff --speed-large-files --minimal -Nru munin-1.4.3.orig/master/_bin/munin-update.in munin-1.4.3/master/_bin/munin-update.in
--- munin-1.4.3.orig/master/_bin/munin-update.in	2009-12-30 13:55:56.000000000 +0100
+++ munin-1.4.3/master/_bin/munin-update.in	2010-01-27 15:45:19.655925252 +0100
@@ -1,4 +1,4 @@
-#! /usr/bin/perl
+#!/opt/csw/bin/perl
 
 use warnings;
 use strict;
diff --speed-large-files --minimal -Nru munin-1.4.3.orig/node/bin/munindoc munin-1.4.3/node/bin/munindoc
--- munin-1.4.3.orig/node/bin/munindoc	2009-12-30 13:55:56.000000000 +0100
+++ munin-1.4.3/node/bin/munindoc	2010-01-27 15:45:11.095300986 +0100
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#/opt/csw/bin/perl -w
 # -*- perl -*-
 #
 # This script provides the basis for a plugin documentation system for
diff --speed-large-files --minimal -Nru munin-1.4.3.orig/node/sbin/munin-node munin-1.4.3/node/sbin/munin-node
--- munin-1.4.3.orig/node/sbin/munin-node	2009-12-30 13:55:56.000000000 +0100
+++ munin-1.4.3/node/sbin/munin-node	2010-01-27 15:45:47.944337631 +0100
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -wT
+#!/opt/csw/bin/perl -wT
 # -*- cperl -*-
 #
 # Copyright (C) 2002-2009 Audun Ytterdal, Jimmy Olsen, Tore Anderson,
diff --speed-large-files --minimal -Nru munin-1.4.3.orig/node/sbin/munin-node-configure munin-1.4.3/node/sbin/munin-node-configure
--- munin-1.4.3.orig/node/sbin/munin-node-configure	2009-12-30 13:55:56.000000000 +0100
+++ munin-1.4.3/node/sbin/munin-node-configure	2010-01-27 15:46:18.906920790 +0100
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/opt/csw/bin/perl -w
 # -*- cperl -*-
 #
 # Copyright (C) 2003-2006 Jimmy Olsen, Nicolai Langfeldt.
diff --speed-large-files --minimal -Nru munin-1.4.3.orig/node/sbin/munin-run munin-1.4.3/node/sbin/munin-run
--- munin-1.4.3.orig/node/sbin/munin-run	2009-12-30 13:55:56.000000000 +0100
+++ munin-1.4.3/node/sbin/munin-run	2010-01-27 15:46:36.558632214 +0100
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -wT
+#!/opt/csw/bin/perl -wT
 # -*- perl -*-
 
 # Copyright (C) 2004-2009
