diff --speed-large-files --minimal -Nru munin-1.4.3.orig/plugins/node.d.sunos/memory.in munin-1.4.3/plugins/node.d.sunos/memory.in
--- munin-1.4.3.orig/plugins/node.d.sunos/memory.in	2009-12-30 13:55:56.000000000 +0100
+++ munin-1.4.3/plugins/node.d.sunos/memory.in	2010-01-27 15:08:58.425391715 +0100
@@ -17,7 +17,7 @@
 The following shows the default settings for this plugin:
 
   [memory]
-     env.top /usr/local/bin/top
+     env.top /opt/csw/bin/top
 
 You can also set warning and critical limits with these env settings:
 
@@ -61,7 +61,7 @@
 
 . $MUNIN_LIBDIR/plugins/plugin.sh
 
-TOP=${top:-/usr/local/bin/top}
+TOP=${top:-/opt/csw/bin/top}
 
 if [ "$1" = "autoconf" ]; then
     if [ -x $TOP ] ; then
