diff --speed-large-files --minimal -Nru munin-1.4.0.orig/plugins/node.d.sunos/memory.in munin-1.4.0/plugins/node.d.sunos/memory.in
--- munin-1.4.0.orig/plugins/node.d.sunos/memory.in	2009-11-27 12:17:12.000000000 +0100
+++ munin-1.4.0/plugins/node.d.sunos/memory.in	2009-11-28 11:53:12.048911726 +0100
@@ -15,7 +15,7 @@
 
 =item top
 
-The path to "top" (Default: "/usr/local/bin/top")
+The path to "top" (Default: "/opt/csw/bin/top")
 
 =back
 
@@ -50,7 +50,7 @@
 
 . $MUNIN_LIBDIR/plugins/plugin.sh
 
-TOP=${top:-/usr/local/bin/top}
+TOP=${top:-/opt/csw/bin/top}
 
 if [ "$1" = "autoconf" ]; then
     if [ -x $TOP ] ; then
