--- poppler-0.12.4/goo/gstrtod.cc.orig	2010-03-11 09:12:33.246797323 +0100
+++ poppler-0.12.4/goo/gstrtod.cc	2010-03-11 09:13:57.881353247 +0100
@@ -22,10 +22,10 @@
 
 #include "gstrtod.h"
 
-#include <clocale>
-#include <cerrno>
-#include <cstdlib>
-#include <cstring>
+#include <locale.h>
+#include <errno.h>
+#include <stdlib.h>
+#include <string.h>
 
 #define ascii_isspace(c) \
   (c == ' ' || c == '\f' || c == '\n' || c == '\r' || c == '\t' || c == '\v')
