From 553c8845bdbca567eedd35fa8aa37af653ced013 Mon Sep 17 00:00:00 2001
From: Ralph Boehme <sloowfranklin@gmail.com>
Date: Fri, 9 Aug 2013 16:27:48 +0200
Subject: [PATCH] libtracker-common: Use procfs psinfo instead of cmdline on
 Solaris

---
 src/libtracker-common/tracker-dbus.c | 13 +++++++++++++
 1 file changed, 13 insertions(+)

diff --git a/src/libtracker-common/tracker-dbus.c b/src/libtracker-common/tracker-dbus.c
index c086f83..beb1814 100644
--- a/src/libtracker-common/tracker-dbus.c
+++ b/src/libtracker-common/tracker-dbus.c
@@ -31,6 +31,10 @@
 #include <kvm.h>
 #endif
 
+#ifdef __sun
+#include <procfs.h>
+#endif
+
 #include "tracker-dbus.h"
 #include "tracker-log.h"
 
@@ -153,6 +157,9 @@ client_data_new (gchar *sender)
 		gchar *contents = NULL;
 		GError *error = NULL;
 		gchar **strv;
+#ifdef __sun /* Solaris */
+		psinfo_t psinfo = { 0 };
+#endif
 
 		pid_str = g_strdup_printf ("%ld", cd->pid);
 		filename = g_build_filename (G_DIR_SEPARATOR_S,
@@ -173,7 +180,13 @@ client_data_new (gchar *sender)
 
 		g_free (filename);
 
+#ifdef __sun /* Solaris */
+		memcpy (&psinfo, contents, sizeof (psinfo));
+		/* won't work with paths containing spaces :( */
+		strv = g_strsplit (psinfo.pr_psargs, " ", 2);
+#else
 		strv = g_strsplit (contents, "^@", 2);
+#endif
 		if (strv && strv[0]) {
 			cd->binary = g_path_get_basename (strv[0]);
 		}
-- 
1.8.3.4

