--- php-5.2.10/configure.o	2009-06-23 13:56:57.118204433 -0500
+++ php-5.2.10/configure	2009-06-23 14:01:59.104988013 -0500
@@ -4776,7 +4776,7 @@
   $APXS -q -S CFLAGS="$APXS_CFLAGS" CFLAGS >/dev/null 2>&1
 
   if test "$?" != "0"; then
-    APACHE_INSTALL="$APXS -i -a -n php5 $SAPI_SHARED" # Old apxs does not have -S option
+    APACHE_INSTALL="$APXS -i -n php5 $SAPI_SHARED" # Old apxs does not have -S option
   else 
     APXS_LIBEXECDIR='$(INSTALL_ROOT)'`$APXS -q LIBEXECDIR`
     if test -z `$APXS -q SYSCONFDIR`; then
@@ -4789,7 +4789,7 @@
                       \$(mkinstalldirs) '$APXS_SYSCONFDIR' && \
                        $APXS -S LIBEXECDIR='$APXS_LIBEXECDIR' \
                              -S SYSCONFDIR='$APXS_SYSCONFDIR' \
-                             -i -a -n php5 $SAPI_SHARED"
+                             -i -n php5 $SAPI_SHARED"
     fi
   fi
 
@@ -5937,7 +5937,7 @@
                 \$(mkinstalldirs) '$APXS_SYSCONFDIR' && \
                  $APXS -S LIBEXECDIR='$APXS_LIBEXECDIR' \
                        -S SYSCONFDIR='$APXS_SYSCONFDIR' \
-                       -i -a -n php5"
+                       -i -n php5"
   fi
 
   case $host_alias in
@@ -6785,7 +6785,7 @@
                 \$(mkinstalldirs) '$APXS_SYSCONFDIR' && \
                  $APXS -S LIBEXECDIR='$APXS_LIBEXECDIR' \
                        -S SYSCONFDIR='$APXS_SYSCONFDIR' \
-                       -i -a -n php5"
+                       -i -n php5"
   fi
 
   case $host_alias in
@@ -7802,7 +7802,7 @@
   $APXS -q -S CFLAGS="$APXS_CFLAGS" CFLAGS >/dev/null 2>&1
 
   if test "$?" != "0"; then
-    APACHE_HOOKS_INSTALL="$APXS -i -a -n php5 $SAPI_SHARED" # Old apxs does not have -S option
+    APACHE_HOOKS_INSTALL="$APXS -i -n php5 $SAPI_SHARED" # Old apxs does not have -S option
   else 
     APXS_LIBEXECDIR='$(INSTALL_ROOT)'`$APXS -q LIBEXECDIR`
     if test -z `$APXS -q SYSCONFDIR`; then
@@ -7815,7 +7815,7 @@
                       \$(mkinstalldirs) '$APXS_SYSCONFDIR' && \
                        $APXS -S LIBEXECDIR='$APXS_LIBEXECDIR' \
                              -S SYSCONFDIR='$APXS_SYSCONFDIR' \
-                             -i -a -n php5 $SAPI_SHARED"
+                             -i -n php5 $SAPI_SHARED"
     fi
   fi
 
@@ -46657,7 +46657,7 @@
 
   if test "$iconv_avail" != "no"; then
     if test -z "$ICONV_DIR"; then
-      for i in /usr/local /usr; do
+      for i in /opt/csw /usr/local /usr; do
         if test -f "$i/include/iconv.h" || test -f "$i/include/giconv.h"; then
           PHP_ICONV_PREFIX="$i"
           break
@@ -68725,7 +68725,7 @@
     echo $ac_n "checking for unixODBC support""... $ac_c" 1>&6
 echo "configure:68727: checking for unixODBC support" >&5
     if test "$PHP_UNIXODBC" = "yes"; then
-      PHP_UNIXODBC=/usr/local
+      PHP_UNIXODBC=/opt/csw
     fi
     ODBC_INCDIR=$PHP_UNIXODBC/include
     ODBC_LIBDIR=$PHP_UNIXODBC/$PHP_LIBDIR
@@ -77388,8 +77388,8 @@
   echo $ac_n "checking for pg_config""... $ac_c" 1>&6
 echo "configure:77390: checking for pg_config" >&5
   for i in $PHP_PDO_PGSQL $PHP_PDO_PGSQL/bin /usr/local/pgsql/bin /usr/local/bin /usr/bin ""; do
-    if test -x $i/pg_config; then
-      PG_CONFIG="$i/pg_config"
+    if test -x $i/pg_config32; then
+      PG_CONFIG="$i/pg_config32"
       break;
     fi
   done
@@ -80202,8 +80202,8 @@
   echo $ac_n "checking for pg_config""... $ac_c" 1>&6
 echo "configure:80204: checking for pg_config" >&5
   for i in $PHP_PGSQL $PHP_PGSQL/bin /usr/local/pgsql/bin /usr/local/bin /usr/bin ""; do
-	if test -x $i/pg_config; then
-      PG_CONFIG="$i/pg_config"
+	if test -x $i/pg_config32; then
+      PG_CONFIG="$i/pg_config32"
       break;
     fi
   done
@@ -83426,7 +83426,7 @@
   echo $ac_n "(cached) $ac_c" 1>&6
 else
   ac_save_LIBS="$LIBS"
-LIBS="-lreadline  $LIBS"
+LIBS="-lreadline  -lcurses $LIBS"
 cat > conftest.$ac_ext <<EOF
 #line 83432 "configure"
 #include "confdefs.h"
@@ -103475,7 +103475,7 @@
   else
     echo $ac_n "checking for XMLRPC-EPI in default path""... $ac_c" 1>&6
 echo "configure:103478: checking for XMLRPC-EPI in default path" >&5
-    for i in /usr/local /usr; do
+    for i in /opt/csw /usr/local /usr; do
       if test -r $i/include/xmlrpc.h; then
         XMLRPC_DIR=$i/include
         echo "$ac_t""found in $i" 1>&6
