--- php-5.2.6/configure.orig	2009-03-02 19:48:06.545907523 -0600
+++ php-5.2.6/configure	2009-03-02 19:44:49.225016959 -0600
@@ -4714,7 +4714,7 @@
   $APXS -q -S CFLAGS="$APXS_CFLAGS" CFLAGS >/dev/null 2>&1
 
   if test "$?" != "0"; then
-    APACHE_INSTALL="$APXS -i -a -n php5 $SAPI_SHARED" # Old apxs does not have -S option
+    APACHE_INSTALL="$APXS -i -n php5 $SAPI_SHARED" # Old apxs does not have -S option
   else 
     APXS_LIBEXECDIR='$(INSTALL_ROOT)'`$APXS -q LIBEXECDIR`
     if test -z `$APXS -q SYSCONFDIR`; then
@@ -4727,7 +4727,7 @@
                       \$(mkinstalldirs) '$APXS_SYSCONFDIR' && \
                        $APXS -S LIBEXECDIR='$APXS_LIBEXECDIR' \
                              -S SYSCONFDIR='$APXS_SYSCONFDIR' \
-                             -i -a -n php5 $SAPI_SHARED"
+                             -i -n php5 $SAPI_SHARED"
     fi
   fi
 
@@ -5875,7 +5875,7 @@
                 \$(mkinstalldirs) '$APXS_SYSCONFDIR' && \
                  $APXS -S LIBEXECDIR='$APXS_LIBEXECDIR' \
                        -S SYSCONFDIR='$APXS_SYSCONFDIR' \
-                       -i -a -n php5"
+                       -i -n php5"
   fi
 
   case $host_alias in
@@ -6723,7 +6723,7 @@
                 \$(mkinstalldirs) '$APXS_SYSCONFDIR' && \
                  $APXS -S LIBEXECDIR='$APXS_LIBEXECDIR' \
                        -S SYSCONFDIR='$APXS_SYSCONFDIR' \
-                       -i -a -n php5"
+                       -i -n php5"
   fi
 
   case $host_alias in
@@ -7740,7 +7740,7 @@
   $APXS -q -S CFLAGS="$APXS_CFLAGS" CFLAGS >/dev/null 2>&1
 
   if test "$?" != "0"; then
-    APACHE_HOOKS_INSTALL="$APXS -i -a -n php5 $SAPI_SHARED" # Old apxs does not have -S option
+    APACHE_HOOKS_INSTALL="$APXS -i -n php5 $SAPI_SHARED" # Old apxs does not have -S option
   else 
     APXS_LIBEXECDIR='$(INSTALL_ROOT)'`$APXS -q LIBEXECDIR`
     if test -z `$APXS -q SYSCONFDIR`; then
@@ -7753,7 +7753,7 @@
                       \$(mkinstalldirs) '$APXS_SYSCONFDIR' && \
                        $APXS -S LIBEXECDIR='$APXS_LIBEXECDIR' \
                              -S SYSCONFDIR='$APXS_SYSCONFDIR' \
-                             -i -a -n php5 $SAPI_SHARED"
+                             -i -n php5 $SAPI_SHARED"
     fi
   fi
 
@@ -19871,7 +19871,7 @@
   echo $ac_n "(cached) $ac_c" 1>&6
 else
   
-  for i in $PHP_LIBXML_DIR /usr/local /usr; do
+  for i in $PHP_LIBXML_DIR /opt/csw /usr/local /usr; do
     if test -x "$i/bin/xml2-config"; then
       ac_cv_php_xml2_config_path="$i/bin/xml2-config"
       break
@@ -45789,7 +45789,7 @@
 
         if test "$found_iconv" = "no"; then
 
-    for i in $PHP_ICONV /usr/local /usr; do
+    for i in $PHP_ICONV /opt/csw /usr/local /usr; do
       if test -r $i/include/giconv.h; then
         cat >> confdefs.h <<\EOF
 #define HAVE_GICONV_H 1
@@ -46270,7 +46270,7 @@
 
   if test "$iconv_avail" != "no"; then
     if test -z "$ICONV_DIR"; then
-      for i in /usr/local /usr; do
+      for i in /opt/csw /usr/local /usr; do
         if test -f "$i/include/iconv.h" || test -f "$i/include/giconv.h"; then
           PHP_ICONV_PREFIX="$i"
           break
@@ -68070,7 +68070,7 @@
     echo $ac_n "checking for unixODBC support""... $ac_c" 1>&6
 echo "configure:68072: checking for unixODBC support" >&5
     if test "$PHP_UNIXODBC" = "yes"; then
-      PHP_UNIXODBC=/usr/local
+      PHP_UNIXODBC=/opt/csw
     fi
     ODBC_INCDIR=$PHP_UNIXODBC/include
     ODBC_LIBDIR=$PHP_UNIXODBC/$PHP_LIBDIR
@@ -76646,8 +76646,8 @@
   echo $ac_n "checking for pg_config""... $ac_c" 1>&6
 echo "configure:76648: checking for pg_config" >&5
   for i in $PHP_PDO_PGSQL $PHP_PDO_PGSQL/bin /usr/local/pgsql/bin /usr/local/bin /usr/bin ""; do
-    if test -x $i/pg_config; then
-      PG_CONFIG="$i/pg_config"
+    if test -x $i/pg_config32; then
+      PG_CONFIG="$i/pg_config32"
       break;
     fi
   done
@@ -79427,8 +79427,8 @@
   echo $ac_n "checking for pg_config""... $ac_c" 1>&6
 echo "configure:79429: checking for pg_config" >&5
   for i in $PHP_PGSQL $PHP_PGSQL/bin /usr/local/pgsql/bin /usr/local/bin /usr/bin ""; do
-	if test -x $i/pg_config; then
-      PG_CONFIG="$i/pg_config"
+	if test -x $i/pg_config32; then
+      PG_CONFIG="$i/pg_config32"
       break;
     fi
   done
@@ -82574,7 +82574,7 @@
   echo $ac_n "(cached) $ac_c" 1>&6
 else
   ac_save_LIBS="$LIBS"
-LIBS="-lreadline  $LIBS"
+LIBS="-lreadline -lcurses $LIBS"
 cat > conftest.$ac_ext <<EOF
 #line 82580 "configure"
 #include "confdefs.h"
@@ -102614,7 +102614,7 @@
   else
     echo $ac_n "checking for XMLRPC-EPI in default path""... $ac_c" 1>&6
 echo "configure:102617: checking for XMLRPC-EPI in default path" >&5
-    for i in /usr/local /usr; do
+    for i in /opt/csw /usr/local /usr; do
       if test -r $i/include/xmlrpc.h; then
         XMLRPC_DIR=$i/include
         echo "$ac_t""found in $i" 1>&6
