diff -Naur gdome2-0.8.1.orig/gtk-doc/Makefile.am gdome2-0.8.1.patched/gtk-doc/Makefile.am
--- gdome2-0.8.1.orig/gtk-doc/Makefile.am	2002-04-04 08:58:04.000000000 +0200
+++ gdome2-0.8.1.patched/gtk-doc/Makefile.am	2009-09-11 15:29:22.320277946 +0200
@@ -9,7 +9,7 @@
 # The directory containing the source code (if it contains documentation).
 DOC_SOURCE_DIR=../libgdome
 
-HTML_DIR=@HTML_DIR@
+HTML_DIR=$(DESTDIR)@HTML_DIR@
 
 TARGET_DIR=$(HTML_DIR)/$(DOC_MODULE)
 
diff -Naur gdome2-0.8.1.orig/gtk-doc/Makefile.in gdome2-0.8.1.patched/gtk-doc/Makefile.in
--- gdome2-0.8.1.orig/gtk-doc/Makefile.in	2003-10-05 22:52:52.000000000 +0200
+++ gdome2-0.8.1.patched/gtk-doc/Makefile.in	2009-09-11 15:29:44.962917775 +0200
@@ -105,7 +105,7 @@
 # The directory containing the source code (if it contains documentation).
 DOC_SOURCE_DIR = ../libgdome
 
-HTML_DIR = @HTML_DIR@
+HTML_DIR = $(DESTDIR)@HTML_DIR@
 
 TARGET_DIR = $(HTML_DIR)/$(DOC_MODULE)
 
