--- src-tree/libtool-1.5.22/libtool	2006-06-27 09:01:20.017045000 -0400
+++ libtool/i386/libtool	2006-06-17 23:25:28.793268000 -0400
@@ -44,7 +44,7 @@
 
 # ### BEGIN LIBTOOL CONFIG
 
-# Libtool was configured on host thor:
+# Libtool was configured on host ra:
 
 # Shell to use when invoking shell scripts.
 SHELL="/bin/bash"
@@ -85,7 +85,7 @@
 LTCC="cc"
 
 # LTCC compiler flags.
-LTCFLAGS="-I/opt/csw/include -fsimple=2 -nofstore -xO5 -xalias_level=basic -xdepend -xarch=386 -xstrconst -xildoff"
+LTCFLAGS="-O"
 
 # A language-specific compiler.
 CC="cc"
@@ -382,6 +382,8 @@
 # positional parameter $0, within a function call, is the name of the
 # function.
 progpath="$0"
+confpath="$0"
+tagspath="$0"
 
 # The name of this program:
 progname=`echo "$progpath" | $SED $basename`
@@ -594,9 +596,9 @@
       # if we don't check for them as well.
       *)
 	for z in $available_tags; do
-	  if grep "^# ### BEGIN LIBTOOL TAG CONFIG: $z$" < "$progpath" > /dev/null; then
+	  if grep "^# ### BEGIN LIBTOOL TAG CONFIG: $z$" < "$tagspath" > /dev/null; then
 	    # Evaluate the configuration.
-	    eval "`${SED} -n -e '/^# ### BEGIN LIBTOOL TAG CONFIG: '$z'$/,/^# ### END LIBTOOL TAG CONFIG: '$z'$/p' < $progpath`"
+	    eval "`${SED} -n -e '/^# ### BEGIN LIBTOOL TAG CONFIG: '$z'$/,/^# ### END LIBTOOL TAG CONFIG: '$z'$/p' < $tagspath`"
 	    CC_quoted=
 	    for arg in $CC; do
 	    # Double-quote args containing other shell metacharacters.
@@ -777,10 +779,10 @@
 	# not specially marked.
 	;;
       *)
-	if grep "^# ### BEGIN LIBTOOL TAG CONFIG: $tagname$" < "$progpath" > /dev/null; then
+	if grep "^# ### BEGIN LIBTOOL TAG CONFIG: $tagname$" < "$tagspath" > /dev/null; then
 	  taglist="$taglist $tagname"
 	  # Evaluate the configuration.
-	  eval "`${SED} -n -e '/^# ### BEGIN LIBTOOL TAG CONFIG: '$tagname'$/,/^# ### END LIBTOOL TAG CONFIG: '$tagname'$/p' < $progpath`"
+	  eval "`${SED} -n -e '/^# ### BEGIN LIBTOOL TAG CONFIG: '$tagname'$/,/^# ### END LIBTOOL TAG CONFIG: '$tagname'$/p' < $tagspath`"
 	else
 	  $echo "$progname: ignoring unknown tag $tagname" 1>&2
 	fi
@@ -816,7 +818,7 @@
     ${SED} -e '1,/^# ### BEGIN LIBTOOL CONFIG/d' -e '/^# ### END LIBTOOL CONFIG/,$d' $progpath
     # Now print the configurations for the tags.
     for tagname in $taglist; do
-      ${SED} -n -e "/^# ### BEGIN LIBTOOL TAG CONFIG: $tagname$/,/^# ### END LIBTOOL TAG CONFIG: $tagname$/p" < "$progpath"
+      ${SED} -n -e "/^# ### BEGIN LIBTOOL TAG CONFIG: $tagname$/,/^# ### END LIBTOOL TAG CONFIG: $tagname$/p" < "$tagspath"
     done
     exit $?
     ;;
@@ -894,6 +896,17 @@
   exit $EXIT_FAILURE
 fi
 
+case `which "$nonopt"` in
+/opt/csw/gcc2/bin/*)
+  . /opt/csw/share/libtool/gcc2.conf
+  tagspath=/opt/csw/share/libtool/gcc2.tags
+  ;;
+/opt/csw/gcc3/bin/*)
+  . /opt/csw/share/libtool/gcc3.conf
+  tagspath=/opt/csw/share/libtool/gcc3.tags
+  ;;
+esac
+
 case $disable_libs in
 no) 
   ;;
@@ -7207,13 +7220,9 @@
 disable_libs=static
 # ### END LIBTOOL TAG CONFIG: disable-static
 
-# Local Variables:
-# mode:shell-script
-# sh-indentation:2
-# End:
 # ### BEGIN LIBTOOL TAG CONFIG: CXX
 
-# Libtool was configured on host thor:
+# Libtool was configured on host ra:
 
 # Shell to use when invoking shell scripts.
 SHELL="/bin/bash"
@@ -7254,7 +7263,7 @@
 LTCC="cc"
 
 # LTCC compiler flags.
-LTCFLAGS="-I/opt/csw/include -fsimple=2 -nofstore -xO5 -xalias_level=basic -xdepend -xarch=386 -xstrconst -xildoff"
+LTCFLAGS="-O"
 
 # A language-specific compiler.
 CC="CC"
@@ -7519,7 +7528,7 @@
 
 # ### BEGIN LIBTOOL TAG CONFIG: F77
 
-# Libtool was configured on host thor:
+# Libtool was configured on host ra:
 
 # Shell to use when invoking shell scripts.
 SHELL="/bin/bash"
@@ -7560,7 +7569,7 @@
 LTCC="cc"
 
 # LTCC compiler flags.
-LTCFLAGS="-I/opt/csw/include -fsimple=2 -nofstore -xO5 -xalias_level=basic -xdepend -xarch=386 -xstrconst -xildoff"
+LTCFLAGS="-O"
 
 # A language-specific compiler.
 CC="f77"
@@ -7822,4 +7831,3 @@
 include_expsyms=""
 
 # ### END LIBTOOL TAG CONFIG: F77
-
