diff --speed-large-files --minimal -Nru ndoutils-1.4b7.orig/config/misccommands.cfg ndoutils-1.4b7/config/misccommands.cfg
--- ndoutils-1.4b7.orig/config/misccommands.cfg	2007-01-08 01:35:58.000000000 +0100
+++ ndoutils-1.4b7/config/misccommands.cfg	2009-06-29 12:35:20.907885672 +0200
@@ -7,5 +7,5 @@
 
 define command{
 	command_name	rotate_ndo_log
-	command_line	/bin/mv /usr/local/nagios/var/ndo.dat /usr/local/nagios/var/ndo.`date +%s`
+	command_line	/bin/mv /var/opt/csw/ndoutils/ndo.dat /var/opt/csw/ndoutils/ndo.`date +%s`
         }
diff --speed-large-files --minimal -Nru ndoutils-1.4b7.orig/config/nagios.cfg ndoutils-1.4b7/config/nagios.cfg
--- ndoutils-1.4b7.orig/config/nagios.cfg	2007-01-08 01:35:58.000000000 +0100
+++ ndoutils-1.4b7/config/nagios.cfg	2009-06-29 12:36:01.020559826 +0200
@@ -5,7 +5,7 @@
 # configuration file (nagios.cfg).  Adjust the paths, etc. to suit your needs.
 
 # Uncomment the line below if you're running Nagios 2.x
-broker_module=/usr/local/nagios/bin/ndomod-2x.o config_file=/usr/local/nagios/etc/ndomod.cfg
+#broker_module=/opt/csw/nagios/ndoutils/bin/ndomod-2x.o config_file=/opt/csw/nagios/ndoutils/etc/ndomod.cfg
 
 # Uncomment the line below if you're running Nagios 3.x
-#broker_module=/usr/local/nagios/bin/ndomod-3x.o config_file=/usr/local/nagios/etc/ndomod.cfg
+#broker_module=/opt/csw/nagios/ndoutils/bin/ndomod-3x.o config_file=/opt/csw/nagios/ndoutils/etc/ndomod.cfg
diff --speed-large-files --minimal -Nru ndoutils-1.4b7.orig/config/ndo2db.cfg ndoutils-1.4b7/config/ndo2db.cfg
--- ndoutils-1.4b7.orig/config/ndo2db.cfg	2007-10-31 19:17:04.000000000 +0100
+++ ndoutils-1.4b7/config/ndo2db.cfg	2009-06-29 12:35:20.910744030 +0200
@@ -32,7 +32,7 @@
 # This option is only valid if the socket type specified above
 # is "unix".
 
-socket_name=/usr/local/nagios/var/ndo.sock
+socket_name=/var/opt/csw/ndoutils/ndo.sock
 
 
 
@@ -94,8 +94,8 @@
 # The user needs at least SELECT, INSERT, UPDATE, and DELETE privileges on
 # the database.
 
-db_user=ndouser
-db_pass=ndopassword
+db_user=nagios
+db_pass=nagios
 
 
 
@@ -152,7 +152,7 @@
 # DEBUG FILE
 # This option determines where the daemon should write debugging information.
 
-debug_file=@localstatedir@/ndo2db.debug
+debug_file=/var/opt/csw/ndoutils/ndo2db.debug
 
 
 
diff --speed-large-files --minimal -Nru ndoutils-1.4b7.orig/config/ndomod.cfg ndoutils-1.4b7/config/ndomod.cfg
--- ndoutils-1.4b7.orig/config/ndomod.cfg	2007-09-27 18:47:44.000000000 +0200
+++ ndoutils-1.4b7/config/ndomod.cfg	2009-06-29 12:35:20.912629356 +0200
@@ -35,9 +35,9 @@
 # of fully qualified domain name of the host that the module should
 # connect to for sending output.
 
-#output=/usr/local/nagios/var/ndo.dat
+#output=/var/opt/csw/ndoutils/ndo.dat
 #output=127.0.0.1
-output=/usr/local/nagios/var/ndo.sock
+output=/var/opt/csw/ndoutils/ndo.sock
 
 
 
@@ -68,7 +68,7 @@
 # Nagios (re)starts, the NDO NEB module will read the contents of this
 # file and send it to the NDO2DB daemon for processing.
 
-buffer_file=/usr/local/nagios/var/ndomod.tmp
+buffer_file=/var/opt/csw/ndoutils/ndomod.tmp
 
 
 
