From 4f44644052f34452a6a5be402b964ad0da55be54 Mon Sep 17 00:00:00 2001
From: Mark Phillips <none@none>
Date: Fri, 8 Jul 2011 13:13:11 +0200
Subject: [PATCH] add csw paths for sudoers

---
 lenses/sudoers.aug |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/lenses/sudoers.aug b/lenses/sudoers.aug
index a108ee2..ec3764a 100644
--- a/lenses/sudoers.aug
+++ b/lenses/sudoers.aug
@@ -493,6 +493,8 @@ let filter = (incl "/etc/sudoers")
     . (incl "/usr/local/etc/sudoers")
     . (incl "/etc/sudoers.d/*")
     . (incl "/usr/local/etc/sudoers.d/*")
+    . (incl "/opt/csw/etc/sudoers")
+    . (incl "/etc/opt/csw/sudoers")
     . Util.stdexcl
 
 let xfm = transform lns filter
-- 
1.7.6

