From 3c294e2e73c600d888329188b41adb7ed02577e7 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Mon, 1 Mar 2010 12:12:02 +0100
Subject: [PATCH] Hack in isblank as gnutools dont seem to be used

---
 src/augtool.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/src/augtool.c b/src/augtool.c
index ab412fd..0539b64 100644
--- a/src/augtool.c
+++ b/src/augtool.c
@@ -21,6 +21,7 @@
  */
 
 #include <config.h>
+# define isblank(ch) ((ch) == ' ' || (ch) == '\t')
 #include "augeas.h"
 #include "internal.h"
 #include "safe-alloc.h"
-- 
1.6.6

