diff -Nru expect-5.43.orig/exp_clib.c expect-5.43/exp_clib.c
--- expect-5.43.orig/exp_clib.c	2003-05-05 23:34:18.000000000 +0200
+++ expect-5.43/exp_clib.c	2010-03-20 06:10:27.258008060 +0100
@@ -38,6 +38,9 @@
 #endif
 
 #ifdef HAVE_SYS_FCNTL_H
+#ifdef __sun
+#  include <fcntl.h>
+#endif
 #  include <sys/fcntl.h>
 #else
 #  include <fcntl.h>
@@ -56,6 +59,9 @@
 
 #include <errno.h>
 
+/* For read, write,execvp... */
+#include <unistd.h>
+
 #ifdef NO_STDLIB_H
 
 /*
diff -Nru expect-5.43.orig/exp_pty.c expect-5.43/exp_pty.c
--- expect-5.43.orig/exp_pty.c	2002-04-08 01:09:00.000000000 +0200
+++ expect-5.43/exp_pty.c	2010-03-20 06:07:09.035909728 +0100
@@ -13,6 +13,10 @@
 #  include <unistd.h>
 #endif
 #ifdef HAVE_SYS_FCNTL_H
+/* open() is defined in fcntl.h not sys/fcntl.h */
+#ifdef __sun
+#  include <fcntl.h>
+#endif
 #  include <sys/fcntl.h>
 #else
 #  include <fcntl.h>
diff -Nru expect-5.43.orig/exp_win.c expect-5.43/exp_win.c
--- expect-5.43.orig/exp_win.c	2002-02-12 03:00:55.000000000 +0100
+++ expect-5.43/exp_win.c	2010-03-20 06:06:24.992305028 +0100
@@ -10,6 +10,10 @@
 #include "expect_cf.h"
 #include "tcl.h"
 
+/* Needed for ioctl */
+#include <unistd.h>
+#include <stropts.h>
+
 #ifdef NO_STDLIB_H
 #include "../compat/stdlib.h"
 #else
diff -Nru expect-5.43.orig/pty_termios.c expect-5.43/pty_termios.c
--- expect-5.43.orig/pty_termios.c	2010-03-20 03:14:25.000000000 +0100
+++ expect-5.43/pty_termios.c	2010-03-20 06:07:23.720256962 +0100
@@ -10,6 +10,10 @@
 #include <stdio.h>
 #include <string.h>
 #include <signal.h>
+/* for open() */
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <fcntl.h>
 
 #if defined(SIGCLD) && !defined(SIGCHLD)
 #define SIGCHLD SIGCLD
