--- balance-3.11.orig/Makefile	2003-10-04 02:41:44.000000000 -0700
+++ balance-3.11/Makefile	2004-05-14 13:02:03.201642000 -0700
@@ -1,12 +1,8 @@
 # $Id: build.diff,v 1.1.1.1 2004/08/17 17:07:34 comand Exp $
 
-#CFLAGS=-g -I.
-CFLAGS=-O2
-
 # uncomment for Solaris:
-# LIBRARIES=-lsocket -lnsl
+LIBRARIES=-lsocket -lnsl
 
-CC=gcc
 RELEASE=3.11
 
 balance:	balance.c butils.o balance.h
@@ -24,8 +20,9 @@
 		rm -f balance
 
 install:
-		install -o root -g root -m 755  balance /usr/sbin/balance 
-		install -o root -g root -m 755  balance.1 /usr/man/man1 
+		install -d $(DESTDIR)$(sbindir) $(DESTDIR)$(mandir)/man1
+		install -o root -g root -m 755  balance $(DESTDIR)$(sbindir)/balance 
+		install -o root -g root -m 755  balance.1 $(DESTDIR)$(mandir)/man1 
 		mkdir -p /var/run/balance
 		chmod 1777 /var/run/balance
 
