diff -Naur flac-1.1.2.orig/src/libFLAC/ia32/Makefile.in flac-1.1.2.patched/src/libFLAC/ia32/Makefile.in
--- flac-1.1.2.orig/src/libFLAC/ia32/Makefile.in	2005-02-05 03:23:38.000000000 +0100
+++ flac-1.1.2.patched/src/libFLAC/ia32/Makefile.in	2009-08-04 09:37:40.159783125 +0200
@@ -253,7 +253,7 @@
 sysconfdir = @sysconfdir@
 target_alias = @target_alias@
 SUFFIXES = .nasm .lo
-STRIP_NON_ASM = sh $(top_srcdir)/strip_non_asm_libtool_args.sh
+STRIP_NON_ASM = /opt/csw/bin/bash $(top_srcdir)/strip_non_asm_libtool_args.sh
 noinst_LTLIBRARIES = libFLAC-asm.la
 libFLAC_asm_la_SOURCES = \
 	cpu_asm.nasm \
diff -Naur flac-1.1.2.orig/strip_non_asm_libtool_args.sh flac-1.1.2.patched/strip_non_asm_libtool_args.sh
--- flac-1.1.2.orig/strip_non_asm_libtool_args.sh	2004-07-29 08:52:52.000000000 +0200
+++ flac-1.1.2.patched/strip_non_asm_libtool_args.sh	2009-08-04 09:38:06.129792997 +0200
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/opt/csw/bin/bash
 #
 # libtool assumes that the compiler can handle the -fPIC flag.
 # This isn't always true (for example, nasm can't handle it).
@@ -8,10 +8,12 @@
 while [ $1 ]; do
 	if [ "$1" != "-fPIC" ]; then
 		if [ "$1" != "-DPIC" ]; then
+		if [ "$1" != "-KPIC" ]; then
 			if [ "$1" != "-fno-common" ]; then
 				command="$command $1"
 			fi
 		fi
+		fi
 	fi
 	shift
 done
