From 12bba7cb6c0d79a212452ab0470c288a8b55fdec Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Thu, 8 Dec 2011 14:55:12 +0100
Subject: [PATCH] Enforce array size, even if zero length

---
 gegl/gegl-lookup.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/gegl/gegl-lookup.h b/gegl/gegl-lookup.h
index d1e239f..940690d 100644
--- a/gegl/gegl-lookup.h
+++ b/gegl/gegl-lookup.h
@@ -33,7 +33,7 @@ typedef struct GeglLookup
   gint               shift;
   guint32            positive_min, positive_max, negative_min, negative_max;
   guint32            bitmask[GEGL_LOOKUP_MAX_ENTRIES/32];
-  gfloat             table[];
+  gfloat             table[0];
 } GeglLookup;
 
 
-- 
1.7.5.4

