From a1350cd6d86051e18b4513f1e7babce8fd5d00e0 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Thu, 29 Oct 2009 14:00:09 +0100
Subject: [PATCH 3/3] Disable stop-on-error until cpp_test refcount error is fixed

---
 tests/test_wrapper.sh.in |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/tests/test_wrapper.sh.in b/tests/test_wrapper.sh.in
index 7fb2d8e..8a0557d 100755
--- a/tests/test_wrapper.sh.in
+++ b/tests/test_wrapper.sh.in
@@ -12,7 +12,7 @@ if [ ! -f sfversion ]; then
 sfversion=`./sfversion`
 
 # Force exit on errors.
-set -e
+# set -e
 
 # generic-tests
 uname -a
-- 
1.6.5

