--- Makefile.orig       2009-05-18 13:08:50.608183737 +0200
+++ Makefile
@@ -36,6 +36,11 @@ CONFIGURE_ENV+=              ac_cv_c_struct_hack=yes
 BUILDLINK_PASSTHRU_DIRS+=      /Developer
 .endif
 
+# Needed for sun studio having __func__ defined
+.if ${PKGSRC_COMPILER} == "sunpro"
+CFLAGS+=        -features=extensions
+.endif
+
 .include "../../audio/flac/buildlink3.mk"
 #BUILDLINK_API_DEPENDS.libvorbis+=     libvorbis>=1.2.1
 .include "../../audio/libvorbis/buildlink3.mk"
$NetBSD$

--- src/common.h.orig   2009-02-13 22:24:08.000000000 +0100
+++ src/common.h
@@ -39,7 +39,7 @@
 
 #if (SIZEOF_LONG == 8)
 #      define  SF_PLATFORM_S64(x)              x##l
-#elif COMPILER_IS_GCC
+#elif COMPILER_IS_GCC || __SUNPRO_C
 #      define  SF_PLATFORM_S64(x)              x##ll
 #elif OS_IS_WIN32
 #      define  SF_PLATFORM_S64(x)              x##I64
