From c65d23475d2bd592e9e6e6bbf7fa04f72bbec039 Mon Sep 17 00:00:00 2001
From: Sebastian Kayser <skayser@opencsw.org>
Date: Tue, 26 Oct 2010 01:36:57 +0200
Subject: [PATCH] Install man pages

---
 Makefile.solaris |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/Makefile.solaris b/Makefile.solaris
index d48453b..76b2ed5 100644
--- a/Makefile.solaris
+++ b/Makefile.solaris
@@ -51,3 +51,6 @@ cscope:
 install: $(PROGS) $(SCRIPTS)
 	$(INSTALL) -m755 -d $(DESTDIR)$(bindir)
 	$(INSTALL) $(PROGS) $(SCRIPTS) $(DESTDIR)$(bindir)
+	$(INSTALL) -m 755 -d $(DESTDIR)$(mandir)/man1
+	$(INSTALL) -m 644 fio.1 $(DESTDIR)$(mandir)/man1
+	$(INSTALL) -m 644 fio_generate_plots.1 $(DESTDIR)$(mandir)/man1
-- 
1.7.3

