diff --speed-large-files --minimal -Nru xfce-mcs-plugins-4.4.0.orig/configure.in xfce-mcs-plugins-4.4.0/configure.in
--- xfce-mcs-plugins-4.4.0.orig/configure.in	2007-01-20 16:45:48.000000000 +0100
+++ xfce-mcs-plugins-4.4.0/configure.in	2007-02-17 19:59:01.961709600 +0100
@@ -245,13 +245,49 @@
   [], [enable_debug=xfce_mcs_plugins_debug_default])
 AC_MSG_CHECKING([whether to enable debugging support])
 if test x"$enable_debug" = x"full" -o x"$enable_debug" = x"yes"; then
-  dnl Check whether the compiler accepts the -fno-strict-aliasing
+
+  dnl Check whether the compiler accepts the -errwarn=%all
+  save_CFLAGS="$CFLAGS"
+  CFLAGS="$CFLAGS -errwarn=%all"
+  AC_MSG_CHECKING([whether $CC accepts -errwarn=%all])
+  AC_COMPILE_IFELSE(AC_LANG_SOURCE([int x;]), [
+    AC_MSG_RESULT([yes])
+    PLATFORM_CFLAGS="$PLATFORM_CFLAGS -errwarn=%all"
+  ], [
+    AC_MSG_RESULT([no])
+  ])
+  CFLAGS="$save_CFLAGS"
+
+  dnl Check whether the compiler accepts the -Werror
+  save_CFLAGS="$CFLAGS"
+  CFLAGS="$CFLAGS -Werror"
+  AC_MSG_CHECKING([whether $CC accepts -Werror])
+  AC_COMPILE_IFELSE(AC_LANG_SOURCE([int x;]), [
+    AC_MSG_RESULT([yes])
+    PLATFORM_CFLAGS="$PLATFORM_CFLAGS -Werror"
+  ], [
+    AC_MSG_RESULT([no])
+  ])
+  CFLAGS="$save_CFLAGS"
+
+  dnl Check whether the compiler accepts the -Wall
+  save_CFLAGS="$CFLAGS"
+  CFLAGS="$CFLAGS -Wall"
+  AC_MSG_CHECKING([whether $CC accepts -Wall])
+  AC_COMPILE_IFELSE(AC_LANG_SOURCE([int x;]), [
+    AC_MSG_RESULT([yes])
+    PLATFORM_CFLAGS="$PLATFORM_CFLAGS -Wall"
+  ], [
+    AC_MSG_RESULT([no])
+  ])
+  CFLAGS="$save_CFLAGS"
+
   dnl switch, as certain packages - like Glib - still don't work
   dnl properly with newer compilers.
   save_CFLAGS="$CFLAGS"
   CFLAGS="$CFLAGS -fno-strict-aliasing"
   AC_MSG_CHECKING([whether $CC accepts -fno-strict-aliasing])
-  AC_COMPILE_IFELSE(AC_LANG_SOURCE([int x;]), [
+  AC_COMPILE_IFELSE(AC_LANG_SOURCE([int x; int main(void) { x=0; x=x+1; return x; }]), [
     AC_MSG_RESULT([yes])
     PLATFORM_CFLAGS="$PLATFORM_CFLAGS -fno-strict-aliasing"
   ], [
@@ -259,9 +295,6 @@
   ])
   CFLAGS="$save_CFLAGS"
 
-  dnl Make sure we detect possible errors
-  PLATFORM_CFLAGS="$PLATFORM_CFLAGS -Werror -Wall"
-
   dnl Paranoia for --enable-debug=full
   if test x"$enable_debug" = x"full"; then
     PLATFORM_CFLAGS="$PLATFORM_CFLAGS -O0 -g3"
