From 7b2c6d4e46b124924a82474d21ab51db5de64830 Mon Sep 17 00:00:00 2001
From: Juraj Lutter <wilbury@opencsw.org>
Date: Wed, 21 Sep 2011 16:06:58 +0200
Subject: [PATCH] Use default cache_dir.

---
 src/cf.data.pre |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/cf.data.pre b/src/cf.data.pre
index a5aaefc..8301b58 100644
--- a/src/cf.data.pre
+++ b/src/cf.data.pre
@@ -2412,7 +2412,7 @@ DOC_START
 NOCOMMENT_START
 
 # Uncomment and adjust the following to add a disk cache directory.
-#cache_dir ufs @DEFAULT_SWAP_DIR@ 100 16 256
+cache_dir ufs @DEFAULT_SWAP_DIR@ 100 16 256
 NOCOMMENT_END
 DOC_END
 
-- 
1.7.6.1

