diff -Naur tcp_wrappers_7.6-ipv6.4.orig/Makefile tcp_wrappers_7.6-ipv6.4.patched/Makefile
--- tcp_wrappers_7.6-ipv6.4.orig/Makefile	2003-04-16 16:25:03.000000000 +0200
+++ tcp_wrappers_7.6-ipv6.4.patched/Makefile	2009-10-20 17:46:10.069122055 +0200
@@ -197,6 +197,24 @@
 	BUGS="$(BUGS) -DSOLARIS_24_GETHOSTBYNAME_BUG" IPV6="$(IPV6)" \
 	EXTRA_CFLAGS=-DUSE_STRERROR all
 
+sunos5-shared:
+	@$(MAKE) REAL_DAEMON_DIR=$(REAL_DAEMON_DIR) STYLE=$(STYLE) \
+	LIBS="-lsocket -lnsl" RANLIB=/bin/true \
+	AR=ld ARFLAGS="-G -h libwrap.so.1 -o " \
+	LIB=libwrap.so.1 VSYSLOG= \
+	NETGROUP=-DNETGROUP AUX_OBJ="setenv.o sharedhack.o" TLI=-DTLI \
+	BUGS="$(BUGS) -DSOLARIS_24_GETHOSTBYNAME_BUG" IPV6="$(IPV6)" \
+	EXTRA_CFLAGS="-DUSE_STRERROR -Kpic" all
+
+sunos5-sharedext:
+	@$(MAKE) REAL_DAEMON_DIR=$(REAL_DAEMON_DIR) STYLE=$(STYLE) \
+	LIBS="-lsocket -lnsl" RANLIB=/bin/true \
+	AR=ld ARFLAGS="-G -h libwrap.so.1 -o " \
+	LIB=libwrap.so.1 VSYSLOG= STYLE=-DPROCESS_OPTIONS \
+	NETGROUP=-DNETGROUP AUX_OBJ="setenv.o sharedhack.o" TLI=-DTLI \
+	BUGS="$(BUGS) -DSOLARIS_24_GETHOSTBYNAME_BUG" IPV6="$(IPV6)" \
+	EXTRA_CFLAGS="-DUSE_STRERROR -Kpic" all
+
 # Generic SYSV40
 esix sysv4:
 	@make REAL_DAEMON_DIR=$(REAL_DAEMON_DIR) STYLE=$(STYLE) \
@@ -481,7 +499,7 @@
 # If your system has IPv6 and supports getipnode* and inet_pton/inet_ntop
 # uncomment the following (Solaris 8)
 
-# IPV6 = -DHAVE_IPV6
+IPV6 = -DHAVE_IPV6
 
 # If your system does not have getipnodebyname() but uses the obsolete
 # gethostbyname2() instead, use this (AIX)
@@ -527,7 +545,7 @@
 #
 # The LOG_XXX names below are taken from the /usr/include/syslog.h file.
 
-FACILITY= LOG_MAIL	# LOG_MAIL is what most sendmail daemons use
+FACILITY= LOG_LOCAL1	# LOG_MAIL is what most sendmail daemons use
 
 # The syslog priority at which successful connections are logged.
 
@@ -603,7 +621,7 @@
 # look for access control information. Watch out for the quotes and
 # backslashes when you make changes.
 
-TABLES	= -DHOSTS_DENY=\"/etc/hosts.deny\" -DHOSTS_ALLOW=\"/etc/hosts.allow\"
+TABLES	= -DHOSTS_DENY=\"/etc/opt/csw/hosts.deny\" -DHOSTS_ALLOW=\"/etc/opt/csw/hosts.allow\"
 
 ####################################################
 # Optional: dealing with host name/address conflicts
@@ -725,7 +743,7 @@
 	$(CC) $(CFLAGS) -o $@ miscd.o $(LIB) $(LIBS)
 
 safe_finger: safe_finger.o $(LIB)
-	$(CC) $(CFLAGS) -o $@ safe_finger.o $(LIB) $(LIBS)
+	$(CC) $(CFLAGS) -o $@ safe_finger.o $(LIBS)
 
 TCPDMATCH_OBJ = tcpdmatch.o fakelog.o inetcf.o scaffold.o
 
diff -Naur tcp_wrappers_7.6-ipv6.4.orig/sharedhack.c tcp_wrappers_7.6-ipv6.4.patched/sharedhack.c
--- tcp_wrappers_7.6-ipv6.4.orig/sharedhack.c	1970-01-01 01:00:00.000000000 +0100
+++ tcp_wrappers_7.6-ipv6.4.patched/sharedhack.c	2009-10-20 17:47:08.156958125 +0200
@@ -0,0 +1,7 @@
+/* This must be compiled with Sun CC for the pragma */
+#pragma weak deny_severity
+#pragma weak allow_severity
+#pragma weak hard_severity
+int deny_severity=0;
+int allow_severity=0;
+int hard_severity=0;
