From 40c7d0836f4433f9908ffa814fb67507cee07051 Mon Sep 17 00:00:00 2001
From: Peter Felecan <pfelecan@opencsw.org>
Date: Fri, 20 Jul 2012 15:47:41 +0200
Subject: [PATCH] OpenSolaris gdb solib svr4

---
 gdb/solib-svr4.c |    5 +++++
 1 file changed, 5 insertions(+)

diff --git a/gdb/solib-svr4.c b/gdb/solib-svr4.c
index 69d3cb5..640ef70 100644
--- a/gdb/solib-svr4.c
+++ b/gdb/solib-svr4.c
@@ -130,6 +130,11 @@ svr4_same_1 (const char *gdb_so_name, const char *inferior_so_name)
   if (strcmp (gdb_so_name, "/usr/lib/sparcv9/ld.so.1") == 0
       && strcmp (inferior_so_name, "/lib/sparcv9/ld.so.1") == 0)
     return 1;
+  /* Similarly, we observed the same issue with amd64, but with
+     different locations.  */
+  if (strcmp (gdb_so_name, "/usr/lib/amd64/ld.so.1") == 0
+      && strcmp (inferior_so_name, "/lib/amd64/ld.so.1") == 0)
+    return 1;
 
   return 0;
 }
-- 
1.7.10.3

