From b1487ee041bde06527b29fbabf636d7425d08d50 Mon Sep 17 00:00:00 2001
From: Yann Rouillard <yann@pleiades.fr.eu.org>
Date: Tue, 7 Aug 2012 20:58:23 +0200
Subject: [PATCH] Fixes SVN_LIB macro

---
 Source/setup_configure.py |    9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/Source/setup_configure.py b/Source/setup_configure.py
index fddc636..4c3d1ae 100644
--- a/Source/setup_configure.py
+++ b/Source/setup_configure.py
@@ -221,6 +221,7 @@ class Setup:
 
         print( 'Info: Using tool chain %s' % (self.c_utils.__class__.__name__,) )
 
+
         self.c_utils.setupUtilities()
         self.c_pysvn.setupPySvn()
 
@@ -1211,7 +1212,7 @@ class CygwinCompilerGCC(UnixCompilerGCC):
 
     def _getLdLibs( self ):
         py_ld_libs = [
-                '-L%(svn_lib_dir)s',
+                '-L%(SVN_LIB)s',
                 '-L/usr/lib/python%d.%d/config -lpython%d.%d.dll' %
                     (sys.version_info[0], sys.version_info[1], sys.version_info[0], sys.version_info[1]),
                 '-lsvn_client-1',
@@ -1251,7 +1252,7 @@ class AixCompilerGCC(UnixCompilerGCC):
                 python_exp = 'python.exp'
 
         py_ld_libs = [
-                '-L%(svn_lib_dir)s',
+                '-L%(SVN_LIB)s',
                 '-lsvn_client-1',
                 '-lsvn_repos-1',
                 '-lsvn_subr-1',
@@ -1284,8 +1285,8 @@ class SunOsCompilerGCC(UnixCompilerGCC):
 
     def _getLdLibs( self ):
         py_ld_libs = [
-                '-L%(svn_lib_dir)s',
-                '-Wl,--rpath -Wl,%(svn_lib_dir)s',
+                '-L%(SVN_LIB)s',
+                '-Wl,--rpath -Wl,%(SVN_LIB)s',
                 '-lsvn_client-1',
                 '-lsvn_diff-1',
                 '-lsvn_repos-1',
-- 
1.7.10.3

