diff --speed-large-files --minimal -Nru nagvis-1.3.2.orig/etc/nagvis.ini.php-sample nagvis-1.3.2/etc/nagvis.ini.php-sample
--- nagvis-1.3.2.orig/etc/nagvis.ini.php-sample	2008-06-04 19:57:06.000000000 +0200
+++ nagvis-1.3.2/etc/nagvis.ini.php-sample	2009-03-26 11:57:24.335438559 +0100
@@ -84,7 +84,7 @@
 ; path options
 [paths]
 ; absolute physical NagVis path
-;base="/usr/local/nagios/share/nagvis/"
+base="/opt/csw/nagios/share/share/nagvis/"
 ; absolute html NagVis path
 ;htmlbase="/nagios/nagvis"
 ; absolute html NagVis cgi path
@@ -102,9 +102,9 @@
 ; database-name for NDO-db
 ;dbname="nagios"
 ; username for NDO-db
-;dbuser="root"
+dbuser="nagios"
 ; password for NDO-db
-;dbpass=""
+dbpass="nagios"
 ; prefix for tables in NDO-db
 ;dbprefix="nagios_"
 ; instace-name for tables in NDO-db
