diff --speed-large-files --minimal -Nru libxfcegui4-4.4.0.orig/configure.in libxfcegui4-4.4.0/configure.in
--- libxfcegui4-4.4.0.orig/configure.in	2007-01-20 16:17:32.000000000 +0100
+++ libxfcegui4-4.4.0/configure.in	2007-02-17 20:21:31.853513200 +0100
@@ -120,6 +120,43 @@
   [], [enable_debug=libxfcegui4_debug_default])
 AC_MSG_CHECKING([whether to enable debugging support])
 if test x"$enable_debug" = x"full" -o x"$enable_debug" = x"yes"; then
+
+  dnl Check whether the compiler accepts the -errwarn=%all
+  save_CFLAGS="$CFLAGS"
+  CFLAGS="$CFLAGS -errwarn=%all"
+  AC_MSG_CHECKING([whether $CC accepts -errwarn=%all])
+  AC_COMPILE_IFELSE(AC_LANG_SOURCE([int x;]), [
+    AC_MSG_RESULT([yes])
+    PLATFORM_CFLAGS="$PLATFORM_CFLAGS -errwarn=%all"
+  ], [
+    AC_MSG_RESULT([no])
+  ])
+  CFLAGS="$save_CFLAGS"
+
+  dnl Check whether the compiler accepts the -Werror
+  save_CFLAGS="$CFLAGS"
+  CFLAGS="$CFLAGS -Werror"
+  AC_MSG_CHECKING([whether $CC accepts -Werror])
+  AC_COMPILE_IFELSE(AC_LANG_SOURCE([int x;]), [
+    AC_MSG_RESULT([yes])
+    PLATFORM_CFLAGS="$PLATFORM_CFLAGS -Werror"
+  ], [
+    AC_MSG_RESULT([no])
+  ])
+  CFLAGS="$save_CFLAGS"
+
+  dnl Check whether the compiler accepts the -Wall
+  save_CFLAGS="$CFLAGS"
+  CFLAGS="$CFLAGS -Wall"
+  AC_MSG_CHECKING([whether $CC accepts -Wall])
+  AC_COMPILE_IFELSE(AC_LANG_SOURCE([int x;]), [
+    AC_MSG_RESULT([yes])
+    PLATFORM_CFLAGS="$PLATFORM_CFLAGS -Wall"
+  ], [
+    AC_MSG_RESULT([no])
+  ])
+  CFLAGS="$save_CFLAGS"
+
   dnl Check whether the compiler accepts the -fno-strict-aliasing
   dnl switch, as certain packages - like Glib - still don't work
   dnl properly with newer compilers.
@@ -134,9 +171,6 @@
   ])
   CFLAGS="$save_CFLAGS"
 
-  dnl Make sure we detect possible errors
-  PLATFORM_CFLAGS="$PLATFORM_CFLAGS -Werror -Wall"
-
   dnl Paranoia for --enable-debug=full
   if test x"$enable_debug" = x"full"; then
     PLATFORM_CFLAGS="$PLATFORM_CFLAGS -O0 -g3"
