From 023a228660a6cdb955db4d4a5b3da3b5edfcde30 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Tue, 1 Jan 2013 19:18:22 +0100
Subject: [PATCH] Use __sparc and __i386 for Solaris

---
 include/libEMF/wine/winnt.h | 14 +++++++++++---
 1 file changed, 11 insertions(+), 3 deletions(-)

diff --git a/include/libEMF/wine/winnt.h b/include/libEMF/wine/winnt.h
index f76c08a..3a8d6f2 100644
--- a/include/libEMF/wine/winnt.h
+++ b/include/libEMF/wine/winnt.h
@@ -29,7 +29,11 @@
 
 /* Architecture dependent settings. */
 /* These are hardcoded to avoid dependencies on config.h in Winelib apps. */
-#if defined(__i386__)
+#if defined(__i386)
+# undef  WORDS_BIGENDIAN
+# undef  BITFIELDS_BIGENDIAN
+# define ALLOW_UNALIGNED_ACCESS
+#elif defined(__i386__)
 # undef  WORDS_BIGENDIAN
 # undef  BITFIELDS_BIGENDIAN
 # define ALLOW_UNALIGNED_ACCESS
@@ -45,6 +49,10 @@
 # undef  WORDS_BIGENDIAN
 # undef  BITFIELDS_BIGENDIAN
 # undef  ALLOW_UNALIGNED_ACCESS
+#elif defined(__sparc)
+# define WORDS_BIGENDIAN
+# define BITFIELDS_BIGENDIAN
+# undef  ALLOW_UNALIGNED_ACCESS
 #elif defined(__sparc__)
 # define WORDS_BIGENDIAN
 # define BITFIELDS_BIGENDIAN
@@ -676,7 +684,7 @@ typedef struct _CONTEXT86
 #define CONTEXT86_FULL (CONTEXT86_CONTROL | CONTEXT86_INTEGER | CONTEXT86_SEGMENTS)
 
 /* i386 context definitions */
-#ifdef __i386__
+#if defined(__i386__) || defined(__i386)
 
 #define CONTEXT_CONTROL         CONTEXT86_CONTROL
 #define CONTEXT_INTEGER         CONTEXT86_INTEGER
@@ -1225,7 +1233,7 @@ typedef struct _STACK_FRAME_HEADER
 
 #endif  /* __PPC__ */
 
-#ifdef __sparc__
+#if defined(__sparc__) || defined(__sparc)
 
 /* 
  * FIXME:  
-- 
1.8.0

