From 3d08d8589aa7974d0eea96f59e6b5c5faf2354ea Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Thu, 12 Dec 2013 15:48:49 +0100
Subject: [PATCH] Use __sparc and __i386 for Solaris

---
 include/libEMF/wine/winnt.h | 18 +++++++++++++++---
 1 file changed, 15 insertions(+), 3 deletions(-)

diff --git a/include/libEMF/wine/winnt.h b/include/libEMF/wine/winnt.h
index f76c08a..ad66499 100644
--- a/include/libEMF/wine/winnt.h
+++ b/include/libEMF/wine/winnt.h
@@ -29,7 +29,11 @@
 
 /* Architecture dependent settings. */
 /* These are hardcoded to avoid dependencies on config.h in Winelib apps. */
-#if defined(__i386__)
+#if defined(__i386)
+# undef  WORDS_BIGENDIAN
+# undef  BITFIELDS_BIGENDIAN
+# define ALLOW_UNALIGNED_ACCESS
+#elif defined(__i386__)
 # undef  WORDS_BIGENDIAN
 # undef  BITFIELDS_BIGENDIAN
 # define ALLOW_UNALIGNED_ACCESS
@@ -37,6 +41,10 @@
 # undef  WORDS_BIGENDIAN
 # undef  BITFIELDS_BIGENDIAN
 # define ALLOW_UNALIGNED_ACCESS
+#elif defined(__amd64)
+# undef  WORDS_BIGENDIAN
+# undef  BITFIELDS_BIGENDIAN
+# define ALLOW_UNALIGNED_ACCESS
 #elif defined(__alpha__)
 # undef  WORDS_BIGENDIAN
 # undef  BITFIELDS_BIGENDIAN
@@ -45,6 +53,10 @@
 # undef  WORDS_BIGENDIAN
 # undef  BITFIELDS_BIGENDIAN
 # undef  ALLOW_UNALIGNED_ACCESS
+#elif defined(__sparc)
+# define WORDS_BIGENDIAN
+# define BITFIELDS_BIGENDIAN
+# undef  ALLOW_UNALIGNED_ACCESS
 #elif defined(__sparc__)
 # define WORDS_BIGENDIAN
 # define BITFIELDS_BIGENDIAN
@@ -676,7 +688,7 @@ typedef struct _CONTEXT86
 #define CONTEXT86_FULL (CONTEXT86_CONTROL | CONTEXT86_INTEGER | CONTEXT86_SEGMENTS)
 
 /* i386 context definitions */
-#ifdef __i386__
+#if defined(__i386__) || defined(__i386)
 
 #define CONTEXT_CONTROL         CONTEXT86_CONTROL
 #define CONTEXT_INTEGER         CONTEXT86_INTEGER
@@ -1225,7 +1237,7 @@ typedef struct _STACK_FRAME_HEADER
 
 #endif  /* __PPC__ */
 
-#ifdef __sparc__
+#if defined(__sparc__) || defined(__sparc)
 
 /* 
  * FIXME:  
-- 
1.8.4.1

