From 1df10ad110f4661217c2f68bbfbdcaa7b1cde2cf Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Tue, 23 Nov 2010 14:37:55 +0100
Subject: [PATCH] Add __sparc and __i386 in addition to __*__

---
 include/libEMF/wine/winnt.h |   21 +++++++++++----------
 1 files changed, 11 insertions(+), 10 deletions(-)

diff --git a/include/libEMF/wine/winnt.h b/include/libEMF/wine/winnt.h
index a455290..b052679 100644
--- a/include/libEMF/wine/winnt.h
+++ b/include/libEMF/wine/winnt.h
@@ -29,11 +29,11 @@
 
 /* Architecture dependent settings. */
 /* These are hardcoded to avoid dependencies on config.h in Winelib apps. */
-#if defined(__i386__)
+#if defined(__i386__) || defined(__i386)
 # undef  WORDS_BIGENDIAN
 # undef  BITFIELDS_BIGENDIAN
 # define ALLOW_UNALIGNED_ACCESS
-#elif defined(__sparc__)
+#elif defined(__sparc__) || defined(__sparc)
 # define WORDS_BIGENDIAN
 # define BITFIELDS_BIGENDIAN
 # undef  ALLOW_UNALIGNED_ACCESS
@@ -48,7 +48,7 @@
 
 /* Calling conventions definitions */
 
-#ifdef __i386__
+#if defined(__i386__) || defined(__i386)
 # ifndef _X86_
 #  define _X86_
 # endif
@@ -56,7 +56,8 @@
 #  define __stdcall __attribute__((__stdcall__))
 #  define __cdecl   __attribute__((__cdecl__))
 # else
-#  error You need gcc >= 2.7 to build Wine on a 386
+#  define __stdcall
+#  define __cdecl
 # endif  /* __GNUC__ */
 #else  /* __i386__ */
 # define __stdcall
@@ -639,7 +640,7 @@ typedef struct _CONTEXT86
 #define CONTEXT86_FULL (CONTEXT86_CONTROL | CONTEXT86_INTEGER | CONTEXT86_SEGMENTS)
 
 /* i386 context definitions */
-#ifdef __i386__
+#if defined(__i386__) || defined(__i386)
 
 #define CONTEXT_CONTROL         CONTEXT86_CONTROL
 #define CONTEXT_INTEGER         CONTEXT86_INTEGER
@@ -969,7 +970,7 @@ typedef struct _STACK_FRAME_HEADER
 
 #endif  /* __PPC__ */
 
-#ifdef __sparc__
+#if defined(__sparc__) || defined(__sparc)
 
 /* 
  * FIXME:  
@@ -1138,7 +1139,7 @@ typedef CONTEXT *PCONTEXT;
 
 #endif  /* __i386__ */
 
-#ifdef __sparc__
+#if defined(__sparc__) || defined(__sparc)
 /* FIXME: use getcontext() to retrieve full context */
 #define _GET_CONTEXT \
     CONTEXT context;   \
@@ -1204,7 +1205,7 @@ typedef CONTEXT *PCONTEXT;
     static void func(void) __attribute__((constructor)); \
     static void func(void)
 #else  /* __GNUC__ */
-# ifdef __i386__
+# if defined(__i386__) || defined(__i386)
 #  define DECL_GLOBAL_CONSTRUCTOR(func) \
     static void __dummy_init_##func(void) { \
         asm(".section .init,\"ax\"\n\t" \
@@ -1218,7 +1219,7 @@ typedef CONTEXT *PCONTEXT;
 
 /* Segment register access */
 
-#ifdef __i386__
+#if defined(__i386__) || defined(__i386)
 # ifdef __GNUC__
 #  define __DEFINE_GET_SEG(seg) \
     extern inline unsigned short __get_##seg(void) \
@@ -2347,7 +2348,7 @@ typedef struct _NT_TIB
 
 struct _TEB;
 
-#if defined(__i386__) && defined(__GNUC__) && !defined(__CHECKER__)
+#if (defined(__i386__) || defined(__i386)) && defined(__GNUC__) && !defined(__CHECKER__)
 extern inline struct _TEB WINAPI *NtCurrentTeb(void);
 extern inline struct _TEB WINAPI *NtCurrentTeb(void)
 {
-- 
1.7.3

