From 36faee190243cdb3e44b3f9a6f2898fb54eba387 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Mon, 22 Nov 2010 15:56:47 +0100
Subject: [PATCH 3/3] Always use csw libtool

---
 configure |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/configure b/configure
index d419f8f..6a8cc02 100755
--- a/configure
+++ b/configure
@@ -8419,7 +8419,7 @@ fi
 LIBTOOL_DEPS="$ltmain"
 
 # Always use our own libtool.
-LIBTOOL='$(SHELL) /usr/bin/libtool'
+LIBTOOL='$(SHELL) /opt/csw/bin/libtool'
 
 
 
-- 
1.7.3

