From eba1a1f1276e40b68e7b7084863469a0c97ef41a Mon Sep 17 00:00:00 2001
From: Martyn Russell <martyn@lanedo.com>
Date: Fri, 7 Mar 2014 09:23:23 +0000
Subject: [PATCH] extract: Don't break on dummy implementation due to incorrect
 args in init()

---
 libmediaart/extractdummy.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/libmediaart/extractdummy.c b/libmediaart/extractdummy.c
index 8d25b74..195de31 100644
--- a/libmediaart/extractdummy.c
+++ b/libmediaart/extractdummy.c
@@ -23,13 +23,15 @@
 #include "extractgeneric.h"
 
 void
-media_art_plugin_init (void)
+media_art_plugin_init (gint max_width)
 {
+	/* Initialize something */
 }
 
 void
 media_art_plugin_shutdown (void)
 {
+	/* Shutdown something */
 }
 
 gboolean
-- 
1.9.1

