From 8ea189fd88c2e2fb0c0d5df0e9addcd07b7a637d Mon Sep 17 00:00:00 2001
From: Rafael Ostertag <raos@opencsw.org>
Date: Sun, 18 Dec 2011 22:09:16 +0100
Subject: [PATCH] Run tests in /var/tmp/pgsql/tmp_check

---
 src/test/regress/GNUmakefile |    7 ++++---
 1 files changed, 4 insertions(+), 3 deletions(-)

diff --git a/src/test/regress/GNUmakefile b/src/test/regress/GNUmakefile
index 83b02cb..85884ee 100644
--- a/src/test/regress/GNUmakefile
+++ b/src/test/regress/GNUmakefile
@@ -141,7 +141,8 @@ tablespace-setup:
 pg_regress_call = ./pg_regress --inputdir=$(srcdir) --dlpath=. --multibyte=$(MULTIBYTE) --load-language=plpgsql $(NOLOCALE)
 
 check: all
-	$(pg_regress_call) --temp-install=./tmp_check --top-builddir=$(top_builddir) --schedule=$(srcdir)/parallel_schedule $(MAXCONNOPT) $(TEMP_CONF)
+	install -d /var/tmp/pgsql/tmp_check
+	$(pg_regress_call) --temp-install=/var/tmp/pgsql/tmp_check --top-builddir=$(top_builddir) --schedule=$(srcdir)/parallel_schedule $(MAXCONNOPT) $(TEMP_CONF)
 
 installcheck: all
 	$(pg_regress_call) --psqldir="$(PSQLDIR)" --schedule=$(srcdir)/serial_schedule
@@ -160,7 +161,7 @@ bigtest: all
 	$(pg_regress_call) --psqldir="$(PSQLDIR)" --schedule=$(srcdir)/serial_schedule numeric_big 
 
 bigcheck: all
-	$(pg_regress_call) --temp-install=./tmp_check --top-builddir=$(top_builddir) --schedule=$(srcdir)/parallel_schedule $(MAXCONNOPT) numeric_big
+	$(pg_regress_call) --temp-install=/var/tmp/pgsql/tmp_check --top-builddir=$(top_builddir) --schedule=$(srcdir)/parallel_schedule $(MAXCONNOPT) numeric_big
 
 
 ##
@@ -173,5 +174,5 @@ clean distclean maintainer-clean: clean-lib
 # things created by various check targets
 	rm -f $(output_files) $(input_files)
 	rm -rf testtablespace
-	rm -rf results tmp_check log
+	rm -rf results /var/tmp/pgsql/tmp_check log
 	rm -f regression.diffs regression.out regress.out run_check.out
-- 
1.7.6.1

