From 0242e1aa1d6f6a95a1b1addb750c35c526ee20fe Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Wed, 1 Dec 2010 12:27:55 +0100
Subject: [PATCH] Dont return value in dequote_bytea

---
 quote.c |    5 +++--
 1 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/quote.c b/quote.c
index 96ca12f..f8a7249 100644
--- a/quote.c
+++ b/quote.c
@@ -529,9 +529,10 @@ void dequote_bytea(char *string, STRLEN *retlen, int estring)
 		return;
 
 	if ('\\' == *string && 'x' == *(string+1))
-		return _dequote_bytea_hex(string, retlen, estring);
+		_dequote_bytea_hex(string, retlen, estring);
 	else
-		return _dequote_bytea_escape(string, retlen, estring);
+		_dequote_bytea_escape(string, retlen, estring);
+	return;
 }
 
 /*
-- 
1.7.3

