From 37c9568e6a474cf7ff1e125613b1a16c139e2074 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <daÂm@opencsw.org>
Date: Fri, 9 Mar 2012 22:38:48 +0100
Subject: [PATCH] Patch from Phil to add ACCESSPERMS

---
 tests/document-saver.c |    5 +++++
 1 files changed, 5 insertions(+), 0 deletions(-)

diff --git a/tests/document-saver.c b/tests/document-saver.c
index 9f44d5a..1fcb49c 100644
--- a/tests/document-saver.c
+++ b/tests/document-saver.c
@@ -27,6 +27,11 @@
 #include <string.h>
 #include <sys/stat.h>
 
+/* linux/bsd has it. others such as Solaris, do not */
+#ifndef ACCESSPERMS
+#define ACCESSPERMS (S_IRWXU|S_IRWXG|S_IRWXO) 
+#endif
+
 #define DEFAULT_LOCAL_URI "/tmp/gedit-document-saver-test.txt"
 #define DEFAULT_REMOTE_URI "sftp://localhost/tmp/gedit-document-saver-test.txt"
 #define DEFAULT_CONTENT "hello world!"
-- 
1.7.9

