diff --speed-large-files --minimal -Nru libntlm-0.4.1.orig/ntlm.h.in libntlm-0.4.1/ntlm.h.in
--- libntlm-0.4.1.orig/ntlm.h.in	2007-09-23 10:46:48.000000000 +0200
+++ libntlm-0.4.1/ntlm.h.in	2008-02-29 16:39:46.423843000 +0100
@@ -37,6 +37,18 @@
 #define NTLM_VERSION "@PACKAGE_VERSION@"
 
 /*
+ * all bytes in our structures are aligned so just swap bytes so 
+ * we have just to swap order 
+ */
+#ifdef WORDS_BIGENDIAN
+# define UI16LE(n) byteswap16(n)
+# define UI32LE(n) byteswap32(n)
+#else
+# define UI16LE(n) (n)
+# define UI32LE(n) (n)
+#endif
+
+/*
  * These structures are byte-order dependant, and should not
  * be manipulated except by the use of the routines provided
  */
@@ -90,7 +102,7 @@
 
 /* public: */
 
-#define SmbLength(ptr) (((ptr)->buffer - (uint8*)(ptr)) + (ptr)->bufIndex)
+#define SmbLength(ptr) (((ptr)->buffer - (uint8*)(ptr)) + UI32LE((ptr)->bufIndex))
 
   extern void
     dumpSmbNtlmAuthRequest (FILE * fp, tSmbNtlmAuthRequest * request);
diff --speed-large-files --minimal -Nru libntlm-0.4.1.orig/smbutil.c libntlm-0.4.1/smbutil.c
--- libntlm-0.4.1.orig/smbutil.c	2007-09-23 10:46:48.000000000 +0200
+++ libntlm-0.4.1/smbutil.c	2008-02-29 16:39:46.418635000 +0100
@@ -257,6 +257,7 @@
   /* FIXME this should be workstation, not username */
   AddBytes (request, user, user, user_len);
   AddString (request, domain, domain);
+  request->bufIndex = UI32LE( request->bufIndex );
 }
 
 void
@@ -309,6 +310,8 @@
   AddString (response, sessionKey, NULL);
 
   response->flags = challenge->flags;
+
+  response->bufIndex = UI32LE( response->bufIndex );
 }
 
 void
