--- Python-2.6.1/configure.orig	2009-03-01 22:10:31.322676775 -0600
+++ Python-2.6.1/configure	2009-03-01 22:11:19.595609202 -0600
@@ -13131,8 +13131,7 @@
 	SunOS*) if test "$GCC" = yes;
 		then CCSHARED="-fPIC";
 		elif test `uname -p` = sparc;
-		then CCSHARED="-xcode=pic32";
-		else CCSHARED="-Kpic";
+            then CCSHARED="-xO3";
 		fi;;
 	hp*|HP*) if test "$GCC" = yes;
 		 then CCSHARED="-fPIC";
