--- Python-2.6.2/setup.py.o	2009-07-29 15:44:48.389269300 -0500
+++ Python-2.6.2/setup.py	2009-07-29 15:43:32.509619500 -0500
@@ -309,9 +309,9 @@
         return sys.platform
 
     def detect_modules(self):
-        # Ensure that /usr/local is always used
-        add_dir_to_list(self.compiler.library_dirs, '/usr/local/lib')
-        add_dir_to_list(self.compiler.include_dirs, '/usr/local/include')
+        # Ensure that /opt/csw is always used
+        add_dir_to_list(self.compiler.library_dirs, '/opt/csw/lib')
+        add_dir_to_list(self.compiler.include_dirs, '/opt/csw/include')
 
         # Add paths specified in the environment variables LDFLAGS and
         # CPPFLAGS for header and library files.
@@ -989,7 +989,7 @@
         # we do not build this one.  Otherwise this build will pick up
         # the more recent berkeleydb's db.h file first in the include path
         # when attempting to compile and it will fail.
-        f = "/usr/include/db.h"
+        f = "/opt/csw/include/db_185.h"
         if os.path.exists(f) and not db_incs:
             data = open(f).read()
             m = re.search(r"#s*define\s+HASHVERSION\s+2\s*", data)
@@ -1300,6 +1300,14 @@
                 )
             libraries = []
 
+        elif platform == 'sunos5':
+            macros = dict(                  # Solaris
+                HAVE_SEM_OPEN=0,        # Not Implemented
+                HAVE_SEM_TIMEDWAIT=0,   # Not Implemented
+                HAVE_FD_TRANSFER=1,
+                )
+            libraries = ['rt']
+
         else:                                   # Linux and other unices
             macros = dict(
                 HAVE_SEM_OPEN=1,
