diff --speed-large-files --minimal -Nru Python-2.3.orig/Makefile.pre.in Python-2.3/Makefile.pre.in
--- Python-2.3.orig/Makefile.pre.in	2003-07-13 03:10:42.000000000 -0700
+++ Python-2.3/Makefile.pre.in	2003-07-30 18:22:32.177323000 -0700
@@ -642,7 +642,7 @@
 	plat-mac/lib-scriptpackages/SystemEvents \
 	plat-mac/lib-scriptpackages/Terminal
 PLATMACPATH=:plat-mac:plat-mac/lib-scriptpackages
-LIBSUBDIRS=	lib-old lib-tk site-packages test test/output test/data \
+LIBSUBDIRS=	lib-old lib-tk site-packages \
 		encodings email email/test email/test/data compiler hotshot \
 		logging bsddb bsddb/test csv idlelib idlelib/Icons \
 		distutils distutils/command $(XMLLIBSUBDIRS) curses $(MACHDEPS)
diff --speed-large-files --minimal -Nru Python-2.3.orig/Modules/Setup.dist Python-2.3/Modules/Setup.dist
--- Python-2.3.orig/Modules/Setup.dist	2003-06-14 14:03:05.000000000 -0700
+++ Python-2.3/Modules/Setup.dist	2003-07-30 18:31:32.471869000 -0700
@@ -157,7 +157,7 @@
 # it, depending on your system -- see the GNU readline instructions.
 # It's okay for this to be a shared library, too.
 
-#readline readline.c -lreadline -ltermcap
+readline readline.c -I$(prefix)/include -L$(prefix)/lib -R$(prefix)/lib -lreadline -ltermcap
 
 
 # Modules that should always be present (non UNIX dependent):
@@ -200,10 +200,10 @@
 
 # Socket module helper for SSL support; you must comment out the other
 # socket line above, and possibly edit the SSL variable:
-#SSL=/usr/local/ssl
-#_ssl _ssl.c \
-#	-DUSE_SSL -I$(SSL)/include -I$(SSL)/include/openssl \
-#	-L$(SSL)/lib -lssl -lcrypto
+SSL=$(prefix)
+_ssl _ssl.c \
+	-DUSE_SSL -I$(SSL)/include -I$(SSL)/include/openssl \
+	-L$(SSL)/lib -R$(SSL)/lib -lssl -lcrypto
 
 # The crypt module is now disabled by default because it breaks builds
 # on many systems (where -lcrypt is needed), e.g. Linux (I believe).
@@ -322,15 +322,15 @@
 # every system.
 
 # *** Always uncomment this (leave the leading underscore in!):
-# _tkinter _tkinter.c tkappinit.c -DWITH_APPINIT \
+ _tkinter _tkinter.c tkappinit.c -DWITH_APPINIT \
 # *** Uncomment and edit to reflect where your Tcl/Tk libraries are:
-#	-L/usr/local/lib \
+	-L/usr/local/Tcl/lib -R/usr/local/Tcl/lib \
 # *** Uncomment and edit to reflect where your Tcl/Tk headers are:
-#	-I/usr/local/include \
+	-I/usr/local/Tcl/include \
 # *** Uncomment and edit to reflect where your X11 header files are:
 #	-I/usr/X11R6/include \
 # *** Or uncomment this for Solaris:
-#	-I/usr/openwin/include \
+	-I/usr/openwin/include \
 # *** Uncomment and edit for Tix extension only:
 #	-DWITH_TIX -ltix8.1.8.2 \
 # *** Uncomment and edit for BLT extension only:
@@ -341,17 +341,17 @@
 # *** Uncomment and edit for TOGL extension only:
 #	-DWITH_TOGL togl.c \
 # *** Uncomment and edit to reflect your Tcl/Tk versions:
-#	-ltk8.2 -ltcl8.2 \
+	-ltk8.4 -ltcl8.4 \
 # *** Uncomment and edit to reflect where your X11 libraries are:
 #	-L/usr/X11R6/lib \
 # *** Or uncomment this for Solaris:
-#	-L/usr/openwin/lib \
+	-L/usr/openwin/lib \
 # *** Uncomment these for TOGL extension only:
 #	-lGL -lGLU -lXext -lXmu \
 # *** Uncomment for AIX:
 #	-lld \
 # *** Always uncomment this; X11 libraries to link with:
-#	-lX11
+	-lX11
 
 # Lance Ellinghaus's modules:
 
@@ -399,7 +399,7 @@
 #
 # First, look at Setup.config; configure may have set this for you.
 
-#gdbm gdbmmodule.c -I/usr/local/include -L/usr/local/lib -lgdbm
+gdbm gdbmmodule.c -I$(prefix)/include -L$(prefix)/lib -R$(prefix)/lib -lgdbm
 
 
 # Sleepycat Berkeley DB interface.
@@ -414,11 +414,11 @@
 #
 # Edit the variables DB and DBLIBVERto point to the db top directory
 # and the subdirectory of PORT where you built it.
-#DB=/usr/local/BerkeleyDB.4.0
-#DBLIBVER=4.0
-#DBINC=$(DB)/include
-#DBLIB=$(DB)/lib
-#_bsddb _bsddb.c -I$(DBINC) -L$(DBLIB) -ldb-$(DBLIBVER)
+DB=$(prefix)
+DBLIBVER=4.1
+DBINC=$(DB)/include
+DBLIB=$(DB)/lib
+_bsddb _bsddb.c -I$(DBINC) -L$(DBLIB) -R$(DBLIB) -ldb-$(DBLIBVER)
 
 # Historical Berkeley DB 1.85
 #
@@ -463,7 +463,7 @@
 # Andrew Kuchling's zlib module.
 # This require zlib 1.1.3 (or later).
 # See http://www.cdrom.com/pub/infozip/zlib/
-#zlib zlibmodule.c -I$(prefix)/include -L$(exec_prefix)/lib -lz
+zlib zlibmodule.c -I$(prefix)/include -L$(prefix)/lib -R$(prefix)/lib -lz
 
 # Interface to the Expat XML parser
 #
@@ -476,8 +476,8 @@
 #
 # More information on Expat can be found at www.libexpat.org.
 #
-#EXPAT_DIR=/usr/local/src/expat-1.95.2
-#pyexpat pyexpat.c -DHAVE_EXPAT_H -I$(EXPAT_DIR)/lib -L$(EXPAT_DIR) -lexpat
+EXPAT_DIR=$(prefix)
+pyexpat pyexpat.c -DHAVE_EXPAT_H -I$(EXPAT_DIR)/include -L$(EXPAT_DIR)/lib -R$(EXPAT_DIR)/lib -lexpat
 
 # Example -- included for reference only:
 # xx xxmodule.c
