From ceab3487bb09c08ca527ca41d94bea68f1370ac7 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Wed, 5 Sep 2012 15:55:32 +0200
Subject: [PATCH] Enable SONAME in library

---
 unix/Makefile.in |    4 +++-
 unix/tcl.m4      |    1 +
 2 files changed, 4 insertions(+), 1 deletion(-)

diff --git a/unix/Makefile.in b/unix/Makefile.in
index a527bf0..3659916 100644
--- a/unix/Makefile.in
+++ b/unix/Makefile.in
@@ -722,7 +722,9 @@ install-binaries: binaries
 	    fi
 	@echo "Installing $(LIB_FILE) to $(DLL_INSTALL_DIR)/"
 	@@INSTALL_LIB@
-	@chmod 555 "$(DLL_INSTALL_DIR)"/$(LIB_FILE)
+	@mv "$(DLL_INSTALL_DIR)"/$(LIB_FILE) "$(DLL_INSTALL_DIR)"/$(LIB_FILE).0
+	@ln -sf $(LIB_FILE).0 "$(DLL_INSTALL_DIR)"/$(LIB_FILE)
+	@chmod 555 "$(DLL_INSTALL_DIR)"/$(LIB_FILE).0
 	@echo "Installing ${TCL_EXE} as $(BIN_INSTALL_DIR)/tclsh$(VERSION)@EXEEXT@"
 	@$(INSTALL_PROGRAM) ${TCL_EXE} "$(BIN_INSTALL_DIR)"/tclsh$(VERSION)@EXEEXT@
 	@echo "Installing tclConfig.sh to $(CONFIG_INSTALL_DIR)/"
diff --git a/unix/tcl.m4 b/unix/tcl.m4
index 3974753..cb31f50 100644
--- a/unix/tcl.m4
+++ b/unix/tcl.m4
@@ -1989,6 +1989,7 @@ AC_DEFUN([SC_CONFIG_CFLAGS], [
 			use_sunmath=no
 		])
 	    ])
+	    TCL_SHLIB_LD_EXTRAS="-h \${TCL_LIB_FILE}.0"
 	    SHLIB_SUFFIX=".so"
 	    DL_OBJS="tclLoadDl.o"
 	    DL_LIBS="-ldl"
-- 
1.7.10.3

