From 3686fdb384225d622c118c8302fc20b6f8d0824f Mon Sep 17 00:00:00 2001
From: Carsten Grzemba <cgrzemba@opencsw.org>
Date: Mon, 27 Feb 2012 13:01:13 +0100
Subject: [PATCH] fixup plugin_syntax.c

---
 ldap/servers/slapd/plugin_syntax.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/ldap/servers/slapd/plugin_syntax.c b/ldap/servers/slapd/plugin_syntax.c
index 0438ceb..840fa7a 100644
--- a/ldap/servers/slapd/plugin_syntax.c
+++ b/ldap/servers/slapd/plugin_syntax.c
@@ -991,5 +991,5 @@ slapi_attr_value_normalize(
 	char **retval
 )
 {
-	return slapi_attr_value_normalize_ext(pb, sattr, type, val, trim_spaces, retval, 0);
+	slapi_attr_value_normalize_ext(pb, sattr, type, val, trim_spaces, retval, 0);
 }
-- 
1.7.9

