diff -Naur ctags-5.8.orig/Makefile.in ctags-5.8.patched/Makefile.in
--- ctags-5.8.orig/Makefile.in	2009-07-04 07:29:28.000000000 +0200
+++ ctags-5.8.patched/Makefile.in	2009-09-07 14:02:27.768451180 +0200
@@ -16,14 +16,14 @@
 # GNU Autoconf variables. These are set by the "configure" script when it
 # runs.
 #
-exec_prefix = @exec_prefix@
-datarootdir = @datarootdir@
-prefix	= @prefix@
-bindir	= @bindir@
+exec_prefix = $(DESTDIR)@exec_prefix@
+datarootdir = $(DESTDIR)@datarootdir@
+prefix	= $(DESTDIR)@prefix@
+bindir	= $(DESTDIR)@bindir@
 srcdir	= @srcdir@
-libdir	= @libdir@
-incdir	= @includedir@
-mandir	= @mandir@
+libdir	= $(DESTDIR)@libdir@
+incdir	= $(DESTDIR)@includedir@
+mandir	= $(DESTDIR)@mandir@
 SLINK	= @LN_S@
 STRIP	= @STRIP@
 CC	= @CC@
