From b437f073e7437c657eb5525c6df7dc08e624e0c6 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Thu, 17 Jun 2010 11:53:59 +0200
Subject: [PATCH] Add workaround for echo -n

---
 config/icu-config-top |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/config/icu-config-top b/config/icu-config-top
index 874a3ab..f3dce7b 100644
--- a/config/icu-config-top
+++ b/config/icu-config-top
@@ -9,6 +9,9 @@
 # This script is designed to aid configuration of ICU.
 # rpath links a library search path right into the binaries.
 
+# Make sure we understand "echo -n" until this is fixed:
+#   http://bugs.icu-project.org/trac/ticket/7768
+PATH=/usr/ucb:$PATH
 
 ### END of icu-config-top
 
-- 
1.7.1

