diff --speed-large-files --minimal -Nru Net_SSLeay.pm-1.30.orig/Makefile.PL Net_SSLeay.pm-1.30/Makefile.PL
--- Net_SSLeay.pm-1.30.orig/Makefile.PL	2006-01-30 03:33:46.663552000 -0500
+++ Net_SSLeay.pm-1.30/Makefile.PL	2006-01-30 03:33:40.552977000 -0500
@@ -51,7 +51,7 @@
 shift if $ARGV[0] eq '--';   # Rest of args are for MakeMaker
 
 $openssl_vers = '0.9.6j';
-$openssl_vers2 = '0.9.7b';
+$openssl_vers2 = '0.9.8a';
 
 print "Checking for OpenSSL-$openssl_vers or $openssl_vers2 or newer...\n";
 
@@ -92,11 +92,13 @@
     . "see README for other options (like older versions of this module).\n"
     if $maj eq '0.9' && $min < 3;
 
+=pod
 warn "That's is newer than what this module was tested with ($openssl_vers\n"
     . "or $openssl_vers2). You should\n"
     . "consider checking if there is a newer release of this module\n"
     . "available. Everything will probably work OK, though.\n"
     if $maj > 0.9 || $min > 7;
+=cut
 
 warn "openssl-0.9.6i/0.9.7a and earlier versions have security flaws or bugs,\n"
     . "see advisories at www.openssl.org, upgrading to openssl-$openssl_vers\n"
