From 6677cc7bc95aeeee76c7741ee9edd03469b7936e Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Tue, 5 Jan 2010 15:35:21 +0100
Subject: [PATCH 2/2] Initial commit

--- a/configure
+++ b/configure
@@ -14683,7 +14683,7 @@ fi
 
 esac
 
-CFLAGS="-g -Wall -pedantic $CFLAGS"
+# CFLAGS="-g -Wall -pedantic $CFLAGS"
 ac_config_files="$ac_config_files libproxy-1.0.pc Makefile libproxy.spec src/Makefile src/lib/Makefile src/modules/Makefile src/bin/Makefile src/bindings/Makefile src/bindings/python/Makefile src/bindings/java/Makefile src/platform/Makefile src/platform/win32/Makefile src/bindings/dotnet/Makefile src/bindings/dotnet/libproxy-sharp-1.0.pc src/bindings/dotnet/AssemblyInfo.cs"
 
 cat >confcache <<\_ACEOF

-- 
1.6.6

