diff -Nru libproxy-0.3.0.orig/src/lib/stdbool.h libproxy-0.3.0/src/lib/stdbool.h
--- libproxy-0.3.0.orig/src/lib/stdbool.h	1970-01-01 01:00:00.000000000 +0100
+++ libproxy-0.3.0/src/lib/stdbool.h	2010-01-15 00:49:51.930719445 +0100
@@ -0,0 +1,17 @@
+#ifndef _STDBOOL_H
+#ifndef _MY_STDBOOL_H
+#define _MY_STDBOOL_H
+
+#if !defined(__GNUG__) && (defined(__SUNPRO_C) && ( defined(__SunOS_5_8) || defined(__SunOS_5_9)))
+
+#ifndef bool
+#define bool int
+#define false 0
+#define true  1
+#endif  /* bool */
+
+#endif /* Compiler has bool */
+
+
+#endif /* _MY_BOOL_H_*/
+#endif /* _STDBOOL_H_*/
diff -Nru libproxy-0.3.0.orig/src/lib/stdint.h libproxy-0.3.0/src/lib/stdint.h
--- libproxy-0.3.0.orig/src/lib/stdint.h	1970-01-01 01:00:00.000000000 +0100
+++ libproxy-0.3.0/src/lib/stdint.h	2010-01-15 01:00:57.971084112 +0100
@@ -0,0 +1,9 @@
+#ifndef _STDINT_H
+#ifndef _MY_STDINT_H
+#define _MY_STDINT_H
+
+#if !defined(__GNUG__) && (defined(__SUNPRO_C) && ( defined(__SunOS_5_8) || defined(__SunOS_5_9)))
+#include <sys/inttypes.h>
+#endif /* Compiler has stdint.h */
+#endif /* _MY_STDINT_H */
+#endif /* _STDINT_H */
