From 43f2a579cd9808557fa2d8ce4d5f5b6f94d627b0 Mon Sep 17 00:00:00 2001
From: Maciej Blizinski <maciej@opencsw.org>
Date: Sat, 25 May 2013 09:07:33 +0200
Subject: [PATCH] Forcefully disable the symbol versioning 2

---
 configure | 10 +---------
 1 file changed, 1 insertion(+), 9 deletions(-)

diff --git a/configure b/configure
index 80dd0fc..f314402 100755
--- a/configure
+++ b/configure
@@ -11876,18 +11876,10 @@ if test "$lt_cv_prog_gnu_ld" = yes; then
   VERSION_SCRIPT_FLAGS=-Wl,--version-script=
 else
   case $host in
-  *-*-sunos*) VERSION_SCRIPT_FLAGS="-Wl,-M -Wl,";;
+  *-*-sunos*) VERSION_SCRIPT_FLAGS="-Wl,-M";;
   esac
 fi
 
- if test -n "$VERSION_SCRIPT_FLAGS"; then
-  USE_VERSION_SCRIPT_TRUE=
-  USE_VERSION_SCRIPT_FALSE='#'
-else
-  USE_VERSION_SCRIPT_TRUE='#'
-  USE_VERSION_SCRIPT_FALSE=
-fi
-
 
 
 _cppflags="${CPPFLAGS}"
-- 
1.8.1.4

