From 8a1c3cdecc3befbe22961b78e4e2c107fce8a198 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Fri, 24 May 2013 15:18:01 +0200
Subject: [PATCH] Forcefully disable the symbol versioning

---
 configure | 11 +++--------
 1 file changed, 3 insertions(+), 8 deletions(-)

diff --git a/configure b/configure
index 80dd0fc..6203fd0 100755
--- a/configure
+++ b/configure
@@ -11876,17 +11876,12 @@ if test "$lt_cv_prog_gnu_ld" = yes; then
   VERSION_SCRIPT_FLAGS=-Wl,--version-script=
 else
   case $host in
-  *-*-sunos*) VERSION_SCRIPT_FLAGS="-Wl,-M -Wl,";;
+  *-*-sunos*) VERSION_SCRIPT_FLAGS="-Wl,-M";;
   esac
 fi
 
- if test -n "$VERSION_SCRIPT_FLAGS"; then
-  USE_VERSION_SCRIPT_TRUE=
-  USE_VERSION_SCRIPT_FALSE='#'
-else
-  USE_VERSION_SCRIPT_TRUE='#'
-  USE_VERSION_SCRIPT_FALSE=
-fi
+USE_VERSION_SCRIPT_TRUE='#'
+USE_VERSION_SCRIPT_FALSE=
 
 
 
-- 
1.8.1.4

