--- libxml2-2.7.7/configure.orig	2010-03-16 03:22:07.931504296 +0100
+++ libxml2-2.7.7/configure	2010-03-16 03:22:59.183232519 +0100
@@ -12157,17 +12157,12 @@
   VERSION_SCRIPT_FLAGS=-Wl,--version-script=
 else
   case $host in
-  *-*-sunos*) VERSION_SCRIPT_FLAGS="-Wl,-M -Wl,";;
+  *-*-sunos*) VERSION_SCRIPT_FLAGS="-Wl,-M";;
   esac
 fi
 
- if test -n "$VERSION_SCRIPT_FLAGS"; then
-  USE_VERSION_SCRIPT_TRUE=
-  USE_VERSION_SCRIPT_FALSE='#'
-else
-  USE_VERSION_SCRIPT_TRUE='#'
-  USE_VERSION_SCRIPT_FALSE=
-fi
+USE_VERSION_SCRIPT_TRUE='#'
+USE_VERSION_SCRIPT_FALSE=
 
 
 
