diff --speed-large-files --minimal -Nru libevent-1.1a.orig/devpoll.c libevent-1.1a/devpoll.c
--- libevent-1.1a.orig/devpoll.c	2005-12-13 14:33:50.263925000 -0500
+++ libevent-1.1a/devpoll.c	2005-12-13 14:33:26.417073000 -0500
@@ -140,7 +140,7 @@
 
 	if (getrlimit(RLIMIT_NOFILE, &rl) == 0 &&
 	    rl.rlim_cur != RLIM_INFINITY)
-		nfiles = rl.rlim_cur;
+		nfiles = rl.rlim_cur - 1;
 
 	/* Initialize the kernel queue */
 	if ((dpfd = open("/dev/poll", O_RDWR)) == -1) {
