From 5cb574becae82337673dbd093176ed87b0de4be1 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Thu, 19 Jan 2012 16:18:24 +0100
Subject: [PATCH] Do not return value in void function

---
 test/regress_dns.c |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/test/regress_dns.c b/test/regress_dns.c
index b2ca47c..24eb6b7 100644
--- a/test/regress_dns.c
+++ b/test/regress_dns.c
@@ -1665,7 +1665,8 @@ static void
 cnt_free(void *ptr)
 {
 	allocated_chunks -= 1;
-	return free(ptr);
+	free(ptr);
+        return;
 }
 
 struct testleak_env_t {
-- 
1.7.8.3

