From 4df7207a66d0160dd99ea8cd9940194bc84ed6e0 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Tue, 6 Jul 2010 11:09:13 +0200
Subject: [PATCH] Replace ... not understood by Sun Studio

---
 glib/tests/checksum.c |   18 ++++++++++++++++--
 1 files changed, 16 insertions(+), 2 deletions(-)

diff --git a/glib/tests/checksum.c b/glib/tests/checksum.c
index 0e51736..26469a5 100644
--- a/glib/tests/checksum.c
+++ b/glib/tests/checksum.c
@@ -615,9 +615,23 @@ hexval (const gchar c)
 {
   switch (c)
     {
-     case '0' ... '9':
+     case '0':
+     case '1':
+     case '2':
+     case '3':
+     case '4':
+     case '5':
+     case '6':
+     case '7':
+     case '8':
+     case '9':
        return c - '0';
-     case 'a' ... 'f':
+     case 'a':
+     case 'b':
+     case 'c':
+     case 'd':
+     case 'e':
+     case 'f':
        return 10 + c - 'a';
      default:
        return 0;
-- 
1.7.1

