From 0ec0789eab4126e3deee2b807da92d6a28dbdad6 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Tue, 9 Mar 2010 15:02:57 +0100
Subject: [PATCH] Use G_STRFUNC instead of explicit gcc-only __PRETTY_FUNC__

---
 glib/gvariant.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/glib/gvariant.c b/glib/gvariant.c
index adc9030..d484ee2 100644
--- a/glib/gvariant.c
+++ b/glib/gvariant.c
@@ -282,7 +282,7 @@
  */
 #define TYPE_CHECK(value, TYPE, val) \
   if G_UNLIKELY (!g_variant_is_of_type (value, TYPE)) {           \
-    g_return_if_fail_warning (G_LOG_DOMAIN, __PRETTY_FUNCTION__,  \
+    g_return_if_fail_warning (G_LOG_DOMAIN, G_STRFUNC,            \
                               "g_variant_is_of_type (" #value     \
                               ", " #TYPE ")");                    \
     return val;                                                   \
-- 
1.7.0

