From 475c9fd0ca749f581c59260b8efe87633b083d0a Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Tue, 25 May 2010 16:12:45 +0200
Subject: [PATCH] Make sure we have the include for major()/minor()

---
 gio/gdbusmessage.c |    5 +++++
 1 files changed, 5 insertions(+), 0 deletions(-)

diff --git a/gio/gdbusmessage.c b/gio/gdbusmessage.c
index eabe171..bb8cad1 100644
--- a/gio/gdbusmessage.c
+++ b/gio/gdbusmessage.c
@@ -30,6 +30,11 @@
 #include <unistd.h>
 #endif
 
+#ifdef __sun
+/* For defines of major() and minor() */
+#include <sys/mkdev.h>
+#endif
+
 #include "gdbusutils.h"
 #include "gdbusmessage.h"
 #include "gdbuserror.h"
-- 
1.7.1

