From 525582d45e936cdbbcf6dc4b6e0413661771d264 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Tue, 20 Jan 2015 11:51:20 +0100
Subject: [PATCH] Preserve -norunpath

---
 build-aux/ltmain.in | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/build-aux/ltmain.in b/build-aux/ltmain.in
index 46a0bb5..f02e9fe 100644
--- a/build-aux/ltmain.in
+++ b/build-aux/ltmain.in
@@ -5343,6 +5343,7 @@ func_mode_link ()
       # -64, -mips[0-9]      enable 64-bit mode for the SGI compiler
       # -r[0-9][0-9]*        specify processor for the SGI compiler
       # -xarch=*, -xtarget=* enable 64-bit mode for the Sun compiler
+      # -norunpath, -xnorunpath do not include compiler path in RPATH
       # +DA*, +DD*           enable 64-bit mode for the HP compiler
       # -q*                  compiler args for the IBM compiler
       # -m*, -t[45]*, -txscale* architecture-specific flags for GCC
@@ -5356,7 +5357,7 @@ func_mode_link ()
       # -stdlib=*            select c++ std lib with clang
       -64|-mips[0-9]|-r[0-9][0-9]*|-xarch=*|-xtarget=*|+DA*|+DD*|-q*|-m*| \
       -t[45]*|-txscale*|-p|-pg|--coverage|-fprofile-*|-F*|@*|-tp=*|--sysroot=*| \
-      -O*|-g*|-flto*|-fwhopr*|-fuse-linker-plugin|-fstack-protector*|-stdlib=*)
+      -O*|-g*|-flto*|-fwhopr*|-fuse-linker-plugin|-fstack-protector*|-stdlib=*|-norunpath|-xnorunpath)
         func_quote_for_eval "$arg"
 	arg=$func_quote_for_eval_result
         func_append compile_command " $arg"
-- 
2.2.1

