From 184e7e0619ef31887fc1986e368d48f0a3677b6b Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Wed, 24 Nov 2010 11:15:29 +0100
Subject: [PATCH] Forward-port GCC tags patch

---
 libtool |   24 ++++++++++++++++++------
 1 files changed, 18 insertions(+), 6 deletions(-)

diff --git a/libtool b/libtool
index 4539216..8ac45c1 100755
--- a/libtool
+++ b/libtool
@@ -576,6 +576,8 @@ $lt_unset CDPATH
 # positional parameter $0, within a function call, is the name of the
 # function.
 progpath="$0"
+confpath="$0"
+tagspath="$0"
 
 
 
@@ -1347,11 +1349,11 @@ func_config ()
     re_endcf='^# ### END LIBTOOL'
 
     # Default configuration.
-    $SED "1,/$re_begincf CONFIG/d;/$re_endcf CONFIG/,\$d" < "$progpath"
+    $SED "1,/$re_begincf CONFIG/d;/$re_endcf CONFIG/,\$d" < "$confpath"
 
     # Now print the configurations for the tags.
     for tagname in $taglist; do
-      $SED -n "/$re_begincf TAG CONFIG: $tagname\$/,/$re_endcf TAG CONFIG: $tagname\$/p" < "$progpath"
+      $SED -n "/$re_begincf TAG CONFIG: $tagname\$/,/$re_endcf TAG CONFIG: $tagname\$/p" < "$tagspath"
     done
 
     exit $?
@@ -1401,14 +1403,14 @@ func_enable_tag ()
   case $tagname in
     CC) ;;
     *)
-      if $GREP "$re_begincf" "$progpath" >/dev/null 2>&1; then
+      if $GREP "$re_begincf" "$tagspath" >/dev/null 2>&1; then
 	taglist="$taglist $tagname"
 
 	# Evaluate the configuration.  Be careful to quote the path
 	# and the sed script, to avoid splitting on whitespace, but
 	# also don't use non-portable quotes within backquotes within
 	# quotes we have to do it in 2 steps:
-	extractedcf=`$SED -n -e "$sed_extractcf" < "$progpath"`
+	extractedcf=`$SED -n -e "$sed_extractcf" < "$tagspath"`
 	eval "$extractedcf"
       else
 	func_error "ignoring unknown tag $tagname"
@@ -1609,6 +1611,16 @@ func_enable_tag "$optarg"
 
   # save first non-option argument
   if test "$#" -gt 0; then
+    case `which "$opt"` in
+      /opt/csw/gcc3/bin/*)
+        . /opt/csw/share/libtool/gcc3.conf
+        tagspath=/opt/csw/share/libtool/gcc3.tags
+        ;;
+      /opt/csw/gcc4/bin/*|/opt/csw/bin/gcc|/opt/csw/bin/gcc-4.6)
+        . /opt/csw/share/libtool/gcc4.conf
+        tagspath=/opt/csw/share/libtool/gcc4.tags
+        ;;
+    esac
     nonopt="$opt"
     shift
   fi
@@ -1828,9 +1840,9 @@ func_infer_tag ()
       # if we don't check for them as well.
       *)
 	for z in $available_tags; do
-	  if $GREP "^# ### BEGIN LIBTOOL TAG CONFIG: $z$" < "$progpath" > /dev/null; then
+	  if $GREP "^# ### BEGIN LIBTOOL TAG CONFIG: $z$" < "$tagspath" > /dev/null; then
 	    # Evaluate the configuration.
-	    eval "`${SED} -n -e '/^# ### BEGIN LIBTOOL TAG CONFIG: '$z'$/,/^# ### END LIBTOOL TAG CONFIG: '$z'$/p' < $progpath`"
+	    eval "`${SED} -n -e '/^# ### BEGIN LIBTOOL TAG CONFIG: '$z'$/,/^# ### END LIBTOOL TAG CONFIG: '$z'$/p' < $tagspath`"
 	    CC_quoted=
 	    for arg in $CC; do
 	      # Double-quote args containing other shell metacharacters.
-- 
1.7.3

