diff --speed-large-files --minimal -Nru pnp-0.4.13.orig/src/npcd.c pnp-0.4.13/src/npcd.c
--- pnp-0.4.13.orig/src/npcd.c	2009-02-19 18:28:41.000000000 +0100
+++ pnp-0.4.13/src/npcd.c	2009-03-04 20:39:15.803376720 +0100
@@ -56,6 +56,11 @@
 const char *directory = NULL;
 const char progname[5] = "npcd";
 
+int alphasort(const struct dirent **a, const struct dirent **b)
+{
+  return(strcmp((*a)->d_name, (*b)->d_name));
+}
+
 static void start_daemon(const char *log_name, int facility) {
 	int i;
 	pid_t pid;
