From 62585d67b81bd3b47872a061f8c369504bce986f Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Thu, 19 Jan 2012 15:51:37 +0100
Subject: [PATCH] Do not reference tm.tm_gmtoff on Solaris

---
 src/rrd_fetch_libdbi.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/src/rrd_fetch_libdbi.c b/src/rrd_fetch_libdbi.c
index 21fbc08..9d437aa 100644
--- a/src/rrd_fetch_libdbi.c
+++ b/src/rrd_fetch_libdbi.c
@@ -464,7 +464,9 @@ rrd_fetch_fn_libdbi(
     struct tm tm;
     time_t t=time(NULL);
     localtime_r(&t,&tm);
+#if !defined(__sun) && !defined(__sun__)
     gmt_offset=tm.tm_gmtoff;
+#endif
     isunixtime=0; table_help.timestamp++;
   }
   /* hex-unescape the value */
-- 
1.7.8.3

