From c3cecb517aad9f2d69df2b4c9898ec0306b4ff76 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Sat, 22 May 2010 22:16:52 +0200
Subject: [PATCH] Hardcode library position

---
 mtcp    |    3 +--
 mtgtar  |    3 +--
 mtstar  |    3 +--
 mtunzip |    3 +--
 4 files changed, 4 insertions(+), 8 deletions(-)

diff --git a/mtcp b/mtcp
index 456ef54..431fe6d 100755
--- a/mtcp
+++ b/mtcp
@@ -1,6 +1,5 @@
 #!/bin/sh
-INSTALLDIR=`dirname $0`/../lib
-LD_PRELOAD=$INSTALLDIR/`uname -s`-`uname -m`/mtwrite.so
+LD_PRELOAD=/opt/csw/lib/mtwrite.so
 MTWRITE_VERBOSE=${MTWRITE_VERBOSE:=1}			# default
 MTWRITE_MAXTHREADS=${MTWRITE_MAXTHREADS:=32}		# default
 MTWRITE_MEMPERTHREAD=${MTWRITE_MEMPERTHREAD:=16M}	# default
diff --git a/mtgtar b/mtgtar
index 0e6e129..5a0fe40 100755
--- a/mtgtar
+++ b/mtgtar
@@ -1,6 +1,5 @@
 #!/bin/sh
-INSTALLDIR=`dirname $0`/../lib
-LD_PRELOAD=$INSTALLDIR/`uname -s`-`uname -m`/mtwrite.so
+LD_PRELOAD=/opt/csw/lib/mtwrite.so
 MTWRITE_VERBOSE=${MTWRITE_VERBOSE:=1}			# default
 MTWRITE_MAXTHREADS=${MTWRITE_MAXTHREADS:=32}		# default
 MTWRITE_MEMPERTHREAD=${MTWRITE_MEMPERTHREAD:=16M}	# default
diff --git a/mtstar b/mtstar
index 63d5002..64de85a 100755
--- a/mtstar
+++ b/mtstar
@@ -1,6 +1,5 @@
 #!/bin/sh
-INSTALLDIR=`dirname $0`/../lib
-LD_PRELOAD=$INSTALLDIR/`uname -s`-`uname -m`/mtwrite.so
+LD_PRELOAD=/opt/csw/lib/mtwrite.so
 MTWRITE_VERBOSE=${MTWRITE_VERBOSE:=1}			# default
 MTWRITE_MAXTHREADS=${MTWRITE_MAXTHREADS:=32}		# default
 MTWRITE_MEMPERTHREAD=${MTWRITE_MEMPERTHREAD:=16M}	# default
diff --git a/mtunzip b/mtunzip
index db4e93f..8e4ead4 100755
--- a/mtunzip
+++ b/mtunzip
@@ -1,6 +1,5 @@
 #!/bin/sh
-INSTALLDIR=`dirname $0`/../lib
-LD_PRELOAD=$INSTALLDIR/`uname -s`-`uname -m`/mtwrite.so
+LD_PRELOAD=/opt/csw/lib/mtwrite.so
 MTWRITE_VERBOSE=${MTWRITE_VERBOSE:=1}			# default
 MTWRITE_MAXTHREADS=${MTWRITE_MAXTHREADS:=32}		# default
 MTWRITE_MEMPERTHREAD=${MTWRITE_MEMPERTHREAD:=16M}	# default
-- 
1.7.1

