From bbacd109a1314356064f26397ff68dd1a7fa72f4 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Wed, 5 May 2010 11:55:37 +0200
Subject: [PATCH 2/2] Calculate threadcount instead of hardcoding

---
 mtcp    |    4 ++--
 mtgtar  |    4 ++--
 mtstar  |    4 ++--
 mttar   |    2 +-
 mtunzip |    4 ++--
 5 files changed, 9 insertions(+), 9 deletions(-)

diff --git a/mtcp b/mtcp
index 5e9a373..62a38f2 100755
--- a/mtcp
+++ b/mtcp
@@ -1,8 +1,8 @@
 #!/bin/ksh
 INSTALLDIR=`dirname $0`/../lib
-LD_PRELOAD=$INSTALLDIR/`uname -s`-`uname -m`/mtwrite.so
+LD_PRELOAD=${LD_PRELOAD:-}:$INSTALLDIR/`uname -s`-`uname -m`/mtwrite.so
 MTWRITE_VERBOSE=${MTWRITE_VERBOSE:=1}			# default
-MTWRITE_MAXTHREADS=${MTWRITE_MAXTHREADS:=32}		# default
+MTWRITE_MAXTHREADS=${MTWRITE_MAXTHREADS:=`/usr/sbin/psrinfo | wc -l`}		# default
 MTWRITE_MEMPERTHREAD=${MTWRITE_MEMPERTHREAD:=16M}	# default
 export LD_PRELOAD MTWRITE_VERBOSE MTWRITE_MAXTHREADS MTWRITE_MEMPERTHREAD
 exec cp $@
diff --git a/mtgtar b/mtgtar
index b56399c..0a8e9f3 100755
--- a/mtgtar
+++ b/mtgtar
@@ -1,8 +1,8 @@
 #!/bin/ksh
 INSTALLDIR=`dirname $0`/../lib
-LD_PRELOAD=$INSTALLDIR/`uname -s`-`uname -m`/mtwrite.so
+LD_PRELOAD=${LD_PRELOAD:-}:$INSTALLDIR/`uname -s`-`uname -m`/mtwrite.so
 MTWRITE_VERBOSE=${MTWRITE_VERBOSE:=1}			# default
-MTWRITE_MAXTHREADS=${MTWRITE_MAXTHREADS:=32}		# default
+MTWRITE_MAXTHREADS=${MTWRITE_MAXTHREADS:=`/usr/sbin/psrinfo | wc -l`}		# default
 MTWRITE_MEMPERTHREAD=${MTWRITE_MEMPERTHREAD:=16M}	# default
 export LD_PRELOAD MTWRITE_VERBOSE MTWRITE_MAXTHREADS MTWRITE_MEMPERTHREAD
 exec gtar $@
diff --git a/mtstar b/mtstar
index ed37a89..3cb2b1f 100755
--- a/mtstar
+++ b/mtstar
@@ -1,8 +1,8 @@
 #!/bin/ksh
 INSTALLDIR=`dirname $0`/../lib
-LD_PRELOAD=$INSTALLDIR/`uname -s`-`uname -m`/mtwrite.so
+LD_PRELOAD=${LD_PRELOAD:-}:$INSTALLDIR/`uname -s`-`uname -m`/mtwrite.so
 MTWRITE_VERBOSE=${MTWRITE_VERBOSE:=1}			# default
-MTWRITE_MAXTHREADS=${MTWRITE_MAXTHREADS:=32}		# default
+MTWRITE_MAXTHREADS=${MTWRITE_MAXTHREADS:=`/usr/sbin/psrinfo | wc -l`}		# default
 MTWRITE_MEMPERTHREAD=${MTWRITE_MEMPERTHREAD:=16M}	# default
 export LD_PRELOAD MTWRITE_VERBOSE MTWRITE_MAXTHREADS MTWRITE_MEMPERTHREAD
 exec star $@
diff --git a/mttar b/mttar
index ec45cec..49e985f 100755
--- a/mttar
+++ b/mttar
@@ -2,7 +2,7 @@
 INSTALLDIR=`dirname $0`/../lib
 LD_PRELOAD=${LD_PRELOAD:-}:$INSTALLDIR/`uname -s`-`uname -m`/mtwrite.so
 MTWRITE_VERBOSE=${MTWRITE_VERBOSE:=1}			# default
-MTWRITE_MAXTHREADS=${MTWRITE_MAXTHREADS:=32}		# default
+MTWRITE_MAXTHREADS=${MTWRITE_MAXTHREADS:=`/usr/sbin/psrinfo | wc -l`}		# default
 MTWRITE_MEMPERTHREAD=${MTWRITE_MEMPERTHREAD:=16M}	# default
 export LD_PRELOAD MTWRITE_VERBOSE MTWRITE_MAXTHREADS MTWRITE_MEMPERTHREAD
 exec tar $@
diff --git a/mtunzip b/mtunzip
index 1763686..200d0d6 100755
--- a/mtunzip
+++ b/mtunzip
@@ -1,8 +1,8 @@
 #!/bin/ksh
 INSTALLDIR=`dirname $0`/../lib
-LD_PRELOAD=$INSTALLDIR/`uname -s`-`uname -m`/mtwrite.so
+LD_PRELOAD=${LD_PRELOAD:-}:$INSTALLDIR/`uname -s`-`uname -m`/mtwrite.so
 MTWRITE_VERBOSE=${MTWRITE_VERBOSE:=1}			# default
-MTWRITE_MAXTHREADS=${MTWRITE_MAXTHREADS:=32}		# default
+MTWRITE_MAXTHREADS=${MTWRITE_MAXTHREADS:=`/usr/sbin/psrinfo | wc -l`}		# default
 MTWRITE_MEMPERTHREAD=${MTWRITE_MEMPERTHREAD:=16M}	# default
 export LD_PRELOAD MTWRITE_VERBOSE MTWRITE_MAXTHREADS MTWRITE_MEMPERTHREAD
 exec unzip $@
-- 
1.7.0

