From 5ddb8ae66e212ce2dfcb46a39dea91c0c12ce81d Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Fri, 18 Oct 2013 15:54:24 +0200
Subject: [PATCH] Add definition of INT32_MIN

---
 lib/timegm.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/lib/timegm.c b/lib/timegm.c
index dced800..fb4cb88 100644
--- a/lib/timegm.c
+++ b/lib/timegm.c
@@ -29,6 +29,8 @@
 
 #include <stdint.h>
 
+#define INT32_MIN  (~ INT32_MAX)
+
 /* Counter the number of leap year in the range [0, y). The |y| is the
    year, including century (e.g., 2012) */
 static int count_leap_year(int y)
-- 
1.8.3.4

