From a97f1c4bf7bc6b1df79d21bd5dc7a1dd765665c5 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Fri, 18 Oct 2013 15:59:00 +0200
Subject: [PATCH] Use pkgconfig for expat libs

---
 configure | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure b/configure
index 7f2dc88..2e8afb8 100755
--- a/configure
+++ b/configure
@@ -12079,7 +12079,7 @@ if test "x$have_libexpat" = "xyes"; then
 
 $as_echo "#define HAVE_LIBEXPAT 1" >>confdefs.h
 
-    EXPAT_LIBS=-lexpat
+    EXPAT_LIBS=`/opt/csw/bin/pkg-config --libs expat`
     EXPAT_CFLAGS=
 
 
-- 
1.8.3.4

