From 08e4cb277db7cfb9a6456167d1c3d0200ad16f8f Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Fri, 18 Oct 2013 16:27:37 +0200
Subject: [PATCH 154/154] bzr commit #154

---
 configure.ac               | 4 +++-
 doc/examples/metalinkcat.c | 7 +++++--
 2 files changed, 8 insertions(+), 3 deletions(-)

diff --git a/doc/examples/metalinkcat.c b/doc/examples/metalinkcat.c
index 8bb7c21..081a41e 100644
--- a/doc/examples/metalinkcat.c
+++ b/doc/examples/metalinkcat.c
@@ -51,7 +51,9 @@ int main(int argc, char** argv)
     file = metalink->files[0];
     printf("name: %s\n", file->name);
     printf("size: %" PRId64 "\n", (int64_t)file->size);
-    printf("os  : %s\n", file->os);
+    if(file->os) {
+      printf("os  : %s\n", file->os);
+    }
 
     if(file->checksums) {
       checksums = file->checksums;
@@ -82,7 +84,8 @@ int main(int argc, char** argv)
       resources = file->resources;
       while(*resources && count < 5) {
         printf("type=%s, location=%s, preference=%d, url=%s\n",
-               (*resources)->type, (*resources)->location,
+               (*resources)->type ? (*resources)->type : "<null>",
+               (*resources)->location ? (*resources)->location : "<null>",
                (*resources)->preference, (*resources)->url);
         ++resources;
         ++count;
-- 
1.8.3.4

