diff --speed-large-files --minimal -Nru vsftpd-2.1.2.orig/sysutil.c vsftpd-2.1.2/sysutil.c
--- vsftpd-2.1.2.orig/sysutil.c	2009-05-27 08:14:59.000000000 +0200
+++ vsftpd-2.1.2/sysutil.c	2009-06-09 22:56:52.905511283 +0200
@@ -2779,6 +2779,7 @@
 void
 vsf_sysutil_set_no_procs()
 {
+#ifdef RLIMIT_NPROC
   int ret;
   struct rlimit rlim;
   rlim.rlim_cur = 0;
@@ -2788,6 +2789,7 @@
   {
     die("setrlimit NPROC");
   }
+#endif
 }
 
 void
