diff --speed-large-files --minimal -Nru vsftpd-2.1.0.orig/tunables.c vsftpd-2.1.0/tunables.c
--- vsftpd-2.1.0.orig/tunables.c	2009-02-21 21:30:09.674664421 +0100
+++ vsftpd-2.1.0/tunables.c	2009-02-21 21:29:49.474053248 +0100
@@ -248,7 +248,7 @@
   install_str_setting("/var/log/xferlog", &tunable_xferlog_file);
   install_str_setting("/var/log/vsftpd.log", &tunable_vsftpd_log_file);
   install_str_setting(".message", &tunable_message_file);
-  install_str_setting("nobody", &tunable_nopriv_user);
+  install_str_setting("vsftpd", &tunable_nopriv_user);
   install_str_setting(0, &tunable_ftpd_banner);
   install_str_setting("/opt/csw/etc/vsftpd/vsftpd.banned_emails", &tunable_banned_email_file);
   install_str_setting("/opt/csw/etc/vsftpd/vsftpd.chroot_list", &tunable_chroot_list_file);
