From 2cedc89069d4f66c9ce9e24addaa4c13ebd1bca4 Mon Sep 17 00:00:00 2001
From: Carsten Grzemba <cgrzemba@opencsw.org>
Date: Fri, 13 Jan 2012 13:04:22 +0100
Subject: [PATCH] change includedir preference in Makefile.am

---
 Makefile.am |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 792f782..25ae853 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -99,7 +99,7 @@ libadmin_a_SOURCES = lib/libadmin/cluster.c \
 	lib/libadmin/error.c lib/libadmin/form_get.c lib/libadmin/httpcon.c lib/libadmin/install.c \
 	lib/libadmin/referer.c lib/libadmin/template.c lib/libadmin/util.c
 
-AM_CPPFLAGS = $(DEBUG_DEFINES) $(PATH_DEFINES) $(OTHER_DEFINES) @adminutil_inc@ @icu_inc@ @openldap_inc@ @ldapsdk_inc@ @nss_inc@ @nspr_inc@ -DUSE_ADMSERV=1 \
+AM_CPPFLAGS = $(DEBUG_DEFINES) $(PATH_DEFINES) $(OTHER_DEFINES) @adminutil_inc@ @openldap_inc@ @ldapsdk_inc@ @nss_inc@ @nspr_inc@ @icu_inc@ -DUSE_ADMSERV=1 \
 	-I$(srcdir)/include -I$(srcdir)/include/base
 if WINNT
 AM_CPPFLAGS += -DXP_WINNT
-- 
1.7.8.2

