--- a/wrappers/initscript.in
+++ b/wrappers/initscript.in
@@ -7,6 +7,9 @@
 # processname: @sbindir@/httpd
 # piddir:      @localstatedir@/run/@PACKAGE_BASE_NAME@
 #
+#AUTOENABLE no # no SMF enable at install!
+#
+
 
 # Source function library.
 if [ -f /etc/rc.d/init.d/functions ] ; then
@@ -94,7 +97,9 @@ prog="@package_name@"
 piddir="@localstatedir@/run/@PACKAGE_BASE_NAME@"
 # PID file
 pidfile=$piddir/admin-serv.pid
-lockfile=/var/lock/subsys/@package_name@
+lockfile=@localstatedir@/lock/subsys/@package_name@
+lockdir=`dirname $lockfile`
+
 
 [ -f $exec ] || exit 0
 
@@ -163,7 +168,8 @@ start() {
             RETVAL=1
         fi
     fi
-    [ $RETVAL -eq 0 -a -d /var/lock/subsys ] && touch $lockfile
+    [ ! -d $lockdir ] && mkdir -p $lockdir
+    [ $RETVAL -eq 0 -a -d $lockdir ] && touch $lockfile
     if [ -f $lockfile ]; then
         restorecon $lockfile
     fi
