From fc16535d4003f8fc7df6f451b6c785e975ce9bc0 Mon Sep 17 00:00:00 2001
From: Carsten Grzemba <cgrzemba@opencsw.org>
Date: Mon, 23 Jan 2012 14:27:18 +0100
Subject: [PATCH 4/5] disable SMF service at install

---
 wrappers/initscript.in |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/wrappers/initscript.in b/wrappers/initscript.in
index 82dd20e..2971c35 100644
--- a/wrappers/initscript.in
+++ b/wrappers/initscript.in
@@ -7,6 +7,8 @@
 # processname: @sbindir@/httpd
 # piddir:      @localstatedir@/run/@PACKAGE_BASE_NAME@
 #
+#AUTOENABLE no # no CSW SMF enable at install!
+#
 
 # Source function library.
 if [ -f /etc/rc.d/init.d/functions ] ; then
-- 
1.7.8.3

