From 0fde3ae66c2bbca4d211ad8a232b488ab3b389cc Mon Sep 17 00:00:00 2001
From: Carsten Grzemba <cgrzemba@opencsw.org>
Date: Mon, 23 Jan 2012 14:29:10 +0100
Subject: [PATCH 5/5] comment out httpd thread options in httpd.conf

---
 admserv/cfgstuff/httpd.conf.in |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/admserv/cfgstuff/httpd.conf.in b/admserv/cfgstuff/httpd.conf.in
index a66eb21..ea12a56 100644
--- a/admserv/cfgstuff/httpd.conf.in
+++ b/admserv/cfgstuff/httpd.conf.in
@@ -102,9 +102,9 @@ StartServers 1
 ServerLimit 1
 
 MaxClients 64
-MinSpareThreads 32
-MaxSpareThreads 64
-ThreadsPerChild 64
+# MinSpareThreads 32
+# MaxSpareThreads 64
+# ThreadsPerChild 64
 
 #
 # Dynamic Shared Object (DSO) Support
-- 
1.7.8.3

