--- a/makefile
+++ b/makefile
@@ -14,10 +14,16 @@ $(OBJ)/%.o:$(SRC)/%.c
 	${CC} $(CFLAGS) -c $< -o $@
 
 jhead: $(objs) jhead.h
-	${CC} -o jhead $(objs) -lm
+	${CC} $(LDFLAGS) -o jhead $(objs) -lm
 
 clean:
 	rm -f $(objs) jhead
 
 install:
-	cp jhead ${DESTDIR}/usr/local/bin/
+	echo DESTDIR=$(DESTDIR)
+	sed '1,/^Liscence:/d' < readme.txt >COPYING
+	test -x /usr/ccs/bin/strip && /usr/ccs/bin/strip jhead
+	test -x /usr/ccs/bin/mcs && /usr/ccs/bin/mcs -d jhead
+	ginstall -D jhead $(DESTDIR)$(prefix)/bin/jhead
+	ginstall -D jhead.1 $(DESTDIR)$(mandir)/man1/jhead.1
+	ginstall -D changes.txt $(DESTDIR)$(docdir)/jhead/changes.tx
