From 442db668872dfbe469e8b272a86109511f45523f Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Fri, 6 Aug 2010 15:38:01 +0200
Subject: [PATCH 6/6] Remove definition on Solaris confusing includes

---
 protocols/twitter/twitter_lib.c |    5 ++++-
 1 files changed, 4 insertions(+), 1 deletions(-)

diff --git a/protocols/twitter/twitter_lib.c b/protocols/twitter/twitter_lib.c
index d1b65c2..8e9c5af 100644
--- a/protocols/twitter/twitter_lib.c
+++ b/protocols/twitter/twitter_lib.c
@@ -22,7 +22,10 @@
 ****************************************************************************/
 
 /* For strptime(): */
+#if(__sun)
+#else
 #define _XOPEN_SOURCE
+#endif
 
 #include "twitter_http.h"
 #include "twitter.h"
@@ -735,4 +738,4 @@ void twitter_friendships_create_destroy(struct im_connection *ic, char *who, int
 	args[0] = "screen_name";
 	args[1] = who;
 	twitter_http(ic, create ? TWITTER_FRIENDSHIPS_CREATE_URL : TWITTER_FRIENDSHIPS_DESTROY_URL, twitter_http_post, ic, 1, args, 2);
-}
\ No newline at end of file
+}
-- 
1.7.1

