From 8795f71f0d7a9b6ab49f7b921c869920cf5277c3 Mon Sep 17 00:00:00 2001
From: Peter Felecan <pfelecan@opencsw.org>
Date: Thu, 21 Feb 2013 15:13:08 +0100
Subject: [PATCH] Fix installation paths

---
 Makefile | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index e52e0f6..aa93904 100644
--- a/Makefile
+++ b/Makefile
@@ -2,13 +2,14 @@
 # Compile using ocamlopt, to use ocamlc set TARGET=byte
 TARGET=opt
 # Install prefix
-PREFIX=/usr/local
+PREFIX=$(prefix)
 # Library directory of hevea
 LIBDIR=$(PREFIX)/lib/hevea
 # Where to install programms
 BINDIR=$(PREFIX)/bin
 #Where to install hevea.sty
-LATEXLIBDIR=$(PREFIX)/lib/hevea
+LATEXLIBDIR=$(PREFIX)/share/texmf-local/tex/latex/hevea
+
 ##### Advanced configuration parameters
 SUF=
 DIR=
-- 
1.8.1

