From f9882d5b5ac794019eba7d1f95098f19e7497c1e Mon Sep 17 00:00:00 2001
From: Peter Felecan <pfelecan@opencsw.org>
Date: Wed, 13 Mar 2013 18:55:56 +0100
Subject: [PATCH] Fix installation paths

---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index dd90bcb..5816684 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 # Compile using ocamlopt, to use ocamlc set TARGET=byte
 TARGET=opt
 # Install prefix
-PREFIX=/usr/local
+PREFIX=$(prefix)
 # Library directory of hevea
 LIBDIR=$(PREFIX)/lib/hevea
 # Where to install programms
@@ -10,7 +10,7 @@ BINDIR=$(PREFIX)/bin
 # Install prefix prefix
 DESTDIR=
 #Where to install hevea.sty
-LATEXLIBDIR=$(PREFIX)/lib/hevea
+LATEXLIBDIR=$(PREFIX)/share/texmf-local/tex/latex/hevea
 ##### Advanced configuration parameters
 SUF=
 DIR=
-- 
1.8.1.4

