--- a/configure.d/config_project_with_enable
+++ b/configure.d/config_project_with_enable
@@ -1037,7 +1037,7 @@ NETSNMP_ARG_ENABLE(embedded-perl,
 NETSNMP_ARG_ENABLE(perl-cc-checks,
 [  --disable-perl-cc-checks        Disable configure checks for whether Perl's
                                   C Compiler is compatible with ours when
-                                  embedded Perl is enabled.])
+                                  embedded Perl is enabled.], enable_perl_cc_checks="no")
 
 
 ##
--- a/configure.d/config_project_perl_python
+++ b/configure.d/config_project_perl_python
@@ -84,7 +84,7 @@ if test "x$install_perl" != "xno" ; then
 
     #       What compiler was used to build the perl binary?
     #
-    if test "xenable_perl_cc_checks" != "xno" ; then
+    if test "x$enable_perl_cc_checks" != "xno" ; then
         AC_MSG_CHECKING([for Perl cc])
         changequote(, )
         PERLCC=`$myperl -V:cc | $myperl -n -e 'print if (s/^\s*cc=.([-=\w\s\/]+).;\s*/$1/);'`
@@ -98,42 +98,42 @@ if test "x$install_perl" != "xno" ; then
                 AC_MSG_ERROR([Could not determine the compiler that was used to build $myperl. Either set the environment variable PERLPROG to a different perl binary or use --without-perl-modules to build without Perl.])
             fi
         fi
-    fi
 
-    #       Was GCC used to build the perl binary?
-    #
-    if test "x$install_perl" != "xno" ; then
-        AC_MSG_CHECKING([whether $PERLCC is a GNU C compiler])
-        OLDCC=$CC
-        CC="$PERLCC"
-        AC_COMPILE_IFELSE([
-            AC_LANG_PROGRAM([], [[
+        #       Was GCC used to build the perl binary?
+        #
+        if test "x$install_perl" != "xno" ; then
+            AC_MSG_CHECKING([whether $PERLCC is a GNU C compiler])
+            OLDCC=$CC
+            CC="$PERLCC"
+            AC_COMPILE_IFELSE([
+                AC_LANG_PROGRAM([], [[
 #ifndef __GNUC__
   choke me
 #endif
-            ]])], [perlcc_is_gnu=yes], [perlcc_is_gnu=no])
-        AC_MSG_RESULT([$perlcc_is_gnu])
-        CC=$OLDCC
-
-        #       Check compatability:  Gnu Net-SNMP vs Non-Gnu perl
-        #
-        if test "x$GCC" = "xyes" -a "x$perlcc_is_gnu" = "xno" ; then
-            if test "x$install_perl" = "xtry" ; then
-      	        install_perl="no"
-      	    else
-      	        AC_MSG_ERROR([This build is using a GNU C compiler ($CC) while Perl has been compiled with a non-GNU (or non-working) compiler ($PERLCC). This likely won't work for building with Perl support. Either specify a different compiler (--with-cc=PATH), disable this check (--disable-perl-cc-checks) or build without Perl (--without-perl-modules).])
-      	    fi
+                ]])], [perlcc_is_gnu=yes], [perlcc_is_gnu=no])
+            AC_MSG_RESULT([$perlcc_is_gnu])
+            CC=$OLDCC
+
+            #       Check compatability:  Gnu Net-SNMP vs Non-Gnu perl
+            #
+            if test "x$GCC" = "xyes" -a "x$perlcc_is_gnu" = "xno" ; then
+                if test "x$install_perl" = "xtry" ; then
+      	            install_perl="no"
+          	    else
+          	        AC_MSG_ERROR([This build is using a GNU C compiler ($CC) while Perl has been compiled with a non-GNU (or non-working) compiler ($PERLCC). This likely won't work for building with Perl support. Either specify a different compiler (--with-cc=PATH), disable this check (--disable-perl-cc-checks) or build without Perl (--without-perl-modules).])
+          	    fi
+            fi
         fi
-    fi
 
-    #       Check compatability:  Non-Gnu Net-SNMP vs Gnu perl
-    #
-    if test "x$install_perl" != "xno" ; then
-        if test "x$GCC" != "xyes" -a "x$perlcc_is_gnu" = "xyes" ; then
-      	    if test "x$install_perl" = "xtry" ; then
-      	        install_perl="no"
-      	    else
-                AC_MSG_ERROR([This build is using a non-GNU C compiler ($CC) while Perl has been compiled with a GNU compiler ($PERLCC). This likely won't work for building with Perl support. Either specify a different compiler (--with-cc=PATH), disable this check (--disable-perl-cc-checks) or build without Perl (--without-perl-modules).])
+        #       Check compatability:  Non-Gnu Net-SNMP vs Gnu perl
+        #
+        if test "x$install_perl" != "xno" ; then
+            if test "x$GCC" != "xyes" -a "x$perlcc_is_gnu" = "xyes" ; then
+          	    if test "x$install_perl" = "xtry" ; then
+      	            install_perl="no"
+      	        else
+                    AC_MSG_ERROR([This build is using a non-GNU C compiler ($CC) while Perl has been compiled with a GNU compiler ($PERLCC). This likely won't work for building with Perl support. Either specify a different compiler (--with-cc=PATH), disable this check (--disable-perl-cc-checks) or build without Perl (--without-perl-modules).])
+                fi
       	    fi
         fi
     fi
