--- a/Makefile.in
+++ b/Makefile.in
@@ -198,13 +198,13 @@ perlrealclean:
 #
 PYMAKE=$(PYTHON) setup.py $(PYTHONARGS)
 pythonmodules: subdirs
-	@(dir=`pwd`; cd python; $(PYMAKE) build --basedir=$$dir) ; \
+	@(dir=`pwd`; cd python; LDFLAGS= $(PYMAKE) build --basedir=$$dir) ; \
         if test $$? != 0 ; then \
            exit 1 ; \
         fi
 
 pythoninstall:
-	@(dir=`pwd`; cd python; $(PYMAKE) install --basedir=$$dir) ; \
+	@(dir=`pwd`; cd python; $(PYMAKE) install --root=$(DESTDIR) --basedir=$$dir) ; \
         if test $$? != 0 ; then \
            exit 1 ; \
         fi
