--- a/perl/ASN/Makefile.PL
+++ b/perl/ASN/Makefile.PL
@@ -74,7 +74,7 @@ sub InitMakeParams {
     else
     {
       $opts = NetSNMPGetOpts();
-      $Params{'LDDLFLAGS'} = "$Config{lddlflags} " . `$opts->{'nsconfig'} --ldflags`;
+      $Params{'LDDLFLAGS'} = "-L../../snmplib/.libs -L../../snmplib -L/opt/csw/lib $Config{lddlflags} " . `$opts->{'nsconfig'} --ldflags`;
       $Params{'LIBS'}    = `$opts->{'nsconfig'} --libs`;
       chomp($Params{'LIBS'});
       $Params{'CCFLAGS'} = `$opts->{'nsconfig'} --cflags`;
--- a/perl/OID/Makefile.PL
+++ b/perl/OID/Makefile.PL
@@ -73,7 +73,7 @@ sub InitMakeParams {
     }
     else {
 	$opts = NetSNMPGetOpts();
-	$Params{'LDDLFLAGS'} = "$Config{lddlflags} " . `$opts->{'nsconfig'} --ldflags`;
+	$Params{'LDDLFLAGS'} = "-L../../snmplib/.libs -L../../snmplib -L/opt/csw/lib $Config{lddlflags} " . `$opts->{'nsconfig'} --ldflags`;
  	$Params{'LIBS'}    = `$opts->{'nsconfig'} --libs`;
 	chomp($Params{'LIBS'});
 	$Params{'CCFLAGS'} = `$opts->{'nsconfig'} --cflags`;
--- a/perl/SNMP/Makefile.PL
+++ b/perl/SNMP/Makefile.PL
@@ -77,7 +77,7 @@ sub InitMakeParams {
     }
     else {
 	$opts = NetSNMPGetOpts();
-	$Params{'LDDLFLAGS'} = "$Config{lddlflags} " . `$opts->{'nsconfig'} --ldflags`;
+	$Params{'LDDLFLAGS'} = "-L../../snmplib/.libs -L../../snmplib -L/opt/csw/lib $Config{lddlflags} " . `$opts->{'nsconfig'} --ldflags`;
 	if (!$ENV{'NETSNMP_LIBS'}) {
 	    $Params{'LIBS'}    = `$opts->{'nsconfig'} --libs`;
 	    chomp($Params{'LIBS'});
--- a/perl/TrapReceiver/Makefile.PL
+++ b/perl/TrapReceiver/Makefile.PL
@@ -106,7 +106,7 @@ sub InitMakeParams {
     }
     else {
 	$opts = NetSNMPGetOpts();
-	$Params{'LDDLFLAGS'} = "$Config{lddlflags} " . `$opts->{'nsconfig'} --ldflags`;
+	$Params{'LDDLFLAGS'} = "-L../../snmplib/.libs -L../../snmplib -L../../apps/.libs -L../../agent/.libs -L/opt/csw/lib $Config{lddlflags} " . `$opts->{'nsconfig'} --ldflags`;
 	$Params{'LIBS'} = '-lnetsnmptrapd ' . `$opts->{'nsconfig'} --base-agent-libs`;
 	chomp($Params{'LIBS'});
 	if (!$ENV{'NETSNMP_CCFLAGS'}) {
--- a/perl/agent/Makefile.PL
+++ b/perl/agent/Makefile.PL
@@ -78,7 +78,7 @@ sub InitMakeParams {
     }
     else {
 	$opts = NetSNMPGetOpts();
-	$Params{'LDDLFLAGS'} = "$Config{lddlflags} " . `$opts->{'nsconfig'} --ldflags`;
+	$Params{'LDDLFLAGS'} = "-L../../snmplib/.libs -L../../snmplib -L../../apps/.libs -L../../agent/.libs -L/opt/csw/lib $Config{lddlflags} " . `$opts->{'nsconfig'} --ldflags`;
 	$Params{'LIBS'}   = `$opts->{'nsconfig'} --base-agent-libs`;
 	chomp($Params{'LIBS'});
 	$Params{'CCFLAGS'} = `$opts->{'nsconfig'} --cflags`;
--- a/perl/default_store/Makefile.PL
+++ b/perl/default_store/Makefile.PL
@@ -68,7 +68,7 @@ sub InitMakeParams {
     }
     else {
 	$opts = NetSNMPGetOpts();
-	$Params{'LDDLFLAGS'} = "$Config{lddlflags} " . `$opts->{'nsconfig'} --ldflags`;
+	$Params{'LDDLFLAGS'} = "-L../../snmplib/.libs -L../../snmplib -L../../apps/.libs -L../../agent/.libs -L/opt/csw/lib $Config{lddlflags} " . `$opts->{'nsconfig'} --ldflags`;
 	$Params{'LIBS'}    = `$opts->{'nsconfig'} --libs`;
 	chomp($Params{'LIBS'});
 	$Params{'CCFLAGS'} = `$opts->{'nsconfig'} --cflags`;
--- a/perl/agent/default_store/Makefile.PL
+++ b/perl/agent/default_store/Makefile.PL
@@ -69,6 +69,7 @@ sub InitMakeParams {
     }
     else {
        $opts = NetSNMPGetOpts("../../");
+       $Params{'LDDLFLAGS'} = "$Config{lddlflags} -L../../../snmplib/.libs";
        $Params{'LIBS'}    = `$opts->{'nsconfig'} --libs`;
        chomp($Params{'LIBS'});
        $Params{'CCFLAGS'} = `$opts->{'nsconfig'} --cflags`;

