From 6c411f42f71cab30ee81d30b142596ad331cdeb8 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Mon, 17 May 2010 22:15:13 +0200
Subject: [PATCH] Do not cast lvalue as left side is already of type char*

---
 outgoing.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/outgoing.c b/outgoing.c
index 11d58aa..b4b3a50 100644
--- a/outgoing.c
+++ b/outgoing.c
@@ -470,7 +470,7 @@ static apr_status_t socket_writev(serf_connection_t *conn)
                             sizeof(struct iovec) * (conn->vec_len - i));
                     conn->vec_len -= i;
                 }
-                ((char *) conn->vec[0].iov_base) += conn->vec[0].iov_len - (len - written);
+                conn->vec[0].iov_base += conn->vec[0].iov_len - (len - written);
                 conn->vec[0].iov_len = len - written;
                 break;
             }
-- 
1.7.0

