From cb2f6976156d04bc2329bac9dca53b03a94c25f1 Mon Sep 17 00:00:00 2001
From: Peter Felecan <pfelecan@opencsw.org>
Date: Wed, 20 Jun 2012 10:12:40 +0200
Subject: [PATCH] Fix shebang perl

---
 bin/ps2eps |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/bin/ps2eps b/bin/ps2eps
index f02e5a2..c8898e6 100755
--- a/bin/ps2eps
+++ b/bin/ps2eps
@@ -1,5 +1,4 @@
-eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}' && eval 'exec perl -S $0 $argv:q' # -*-perl-*-
-        if 0;
+#! /usr/bin/env perl
 # The expression in the previous line replaces the unix specific line 
 # {#!/usr/bin/perl}.   
 # ps2eps - convert PostScript to EPS (Encapsulated PostScript) files
-- 
1.7.10.3

