From c082b42e7d6c6acf4ae8969e21fc55cc4059bca5 Mon Sep 17 00:00:00 2001
From: Rafael Ostertag <raos@opencsw.org>
Date: Thu, 14 Nov 2013 17:28:15 +0100
Subject: [PATCH] Use default man path instead of manpath command.

---
 share/functions/man.fish | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/share/functions/man.fish b/share/functions/man.fish
index 2b9bab5..abbe136 100644
--- a/share/functions/man.fish
+++ b/share/functions/man.fish
@@ -8,7 +8,9 @@ function man --description "Format and display the on-line manual pages"
 		set -l fish_manpath (dirname $__fish_datadir)/fish/man
 		if test -d "$fish_manpath"
 			# Notice local but exported variable
-			set -lx MANPATH "$fish_manpath":(command manpath)
+			#
+			# Solaris has no `manpath' command, so I stick with the default
+			set -lx MANPATH "$fish_manpath":/usr/share/man
 			
 			# Invoke man with this manpath, and we're done
 			command man $argv
-- 
1.8.4.1

