From 1d9434d46cafd3e4e2ce434bb5e4791e20ca1b8d Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Tue, 11 Sep 2012 11:40:11 +0200
Subject: [PATCH] Keep -norunpath during linkage

---
 build-aux/ltmain.sh |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build-aux/ltmain.sh b/build-aux/ltmain.sh
index e589475..c03c27f 100644
--- a/build-aux/ltmain.sh
+++ b/build-aux/ltmain.sh
@@ -1690,7 +1690,7 @@ EOF
       # -F/path gives path to uninstalled frameworks, gcc on darwin
       # @file GCC response files
       -64|-mips[0-9]|-r[0-9][0-9]*|-xarch=*|-xtarget=*|+DA*|+DD*|-q*|-m*| \
-      -t[45]*|-txscale*|-p|-pg|--coverage|-fprofile-*|-F*|@*)
+      -t[45]*|-txscale*|-p|-pg|--coverage|-fprofile-*|-F*|@*|-norunpath)
 
 	# Unknown arguments in both finalize_command and compile_command need
 	# to be aesthetically quoted because they are evaled later.
-- 
1.7.10.3

