From 1b32d9bec0d1851f5d5e9c57123f309ba43ed6ed Mon Sep 17 00:00:00 2001
From: Jan Holzhueter <jh@opencsw.org>
Date: Thu, 12 Feb 2015 16:31:56 +0100
Subject: [PATCH] fix-makefile

---
 Makefile | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/Makefile b/Makefile
index 7b2c322..dc599a1 100644
--- a/Makefile
+++ b/Makefile
@@ -50,10 +50,10 @@ stud: $(OBJS)
 	$(CC) -o $@ $^ $(LDFLAGS)
 
 install: $(ALL)
-	install -d $(DESTDIR)$(BINDIR)
-	install stud $(DESTDIR)$(BINDIR)
-	install -d $(DESTDIR)$(MANDIR)/man8
-	install -m 644 stud.8 $(DESTDIR)$(MANDIR)/man8
+	ginstall -d $(DESTDIR)$(BINDIR)
+	ginstall stud $(DESTDIR)$(BINDIR)
+	ginstall -d $(DESTDIR)$(MANDIR)/man8
+	ginstall -m 644 stud.8 $(DESTDIR)$(MANDIR)/man8
 
 clean:
 	rm -f stud $(OBJS)
-- 
2.2.1

