From 87ea1e52b98d43252aecc3911ff96d3a6444dbe3 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Fri, 7 Jan 2011 13:43:07 +0100
Subject: [PATCH 1/5] Fix ambiguity between double and long double

---
 src/bzflag/ServerMenu.cxx |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/bzflag/ServerMenu.cxx b/src/bzflag/ServerMenu.cxx
index ebd00f2..f5db381 100644
--- a/src/bzflag/ServerMenu.cxx
+++ b/src/bzflag/ServerMenu.cxx
@@ -354,7 +354,7 @@ void ServerMenu::setSelected(int index, bool forcerefresh)
 	}
 	else {
 	  // colorize servers: many shots->red, jumping->green, CTF->blue
-	  const float rf = std::min(1.0f, logf(server.ping.maxShots) / logf(20.0f));
+	  const float rf = std::min(1.0f, logf( (double) server.ping.maxShots) / logf( (double) 20.0f));
 	  const float gf = gameStyle & JumpingGameStyle ? 1.0f : 0.0f;
 	  const float bf = gameStyle & TeamFlagGameStyle ? 1.0f : 0.0f;
 	  label->setColor(0.5f + rf * 0.5f, 0.5f + gf * 0.5f, 0.5f + bf * 0.5f);
-- 
1.7.3.2

