From d116c7dabfe30093c09b151f9344fbc296716a05 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Fri, 7 Jan 2011 13:47:21 +0100
Subject: [PATCH 2/5] Make function location explicit

---
 src/common/WordFilter.cxx |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/common/WordFilter.cxx b/src/common/WordFilter.cxx
index 2d65dc2..fe62f4d 100644
--- a/src/common/WordFilter.cxx
+++ b/src/common/WordFilter.cxx
@@ -49,7 +49,7 @@ bool WordFilter::simpleFilter(char *input) const
 
     word = line.substr(startPosition, endPosition-startPosition);
     // transform to lowercase
-    transform (word.begin(), word.end(), word.begin(), tolower);
+    std::transform (word.begin(), word.end(), word.begin(), tolower);
 
     findWord.word = word;
     firstchar = (unsigned char)word[0];
@@ -778,7 +778,7 @@ unsigned int WordFilter::loadFromFile(const std::string &fileName, bool verbose)
     }
 
     // convert the word to lowercase
-    transform (filterWord.begin(),filterWord.end(), filterWord.begin(), tolower);
+    std::transform (filterWord.begin(),filterWord.end(), filterWord.begin(), tolower);
 
     bool added = addToFilter(filterWord, std::string(""));
     if ((!added) && (verbose)) {
@@ -800,7 +800,7 @@ unsigned int WordFilter::loadFromFile(const std::string &fileName, bool verbose)
  * pattern match (default) catching hundreds of variations per filter
  * word or using a simple exact word match technique (original).
  */
-bool WordFilter::filter(char *input, bool simple) const
+bool WordFilter::filter(char *input, const bool simple) const
 {
 #ifdef DEBUG
   TimeKeeper before = TimeKeeper::getCurrent();
-- 
1.7.3.2

