From 0d728c7a76f0a0e37e88262059fce9d850963242 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Fri, 7 Jan 2011 13:47:41 +0100
Subject: [PATCH 3/5] Force inclusion of glew.h before gl.h

---
 src/platform/XWindow.cxx |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/src/platform/XWindow.cxx b/src/platform/XWindow.cxx
index 3c56476..c802bf8 100644
--- a/src/platform/XWindow.cxx
+++ b/src/platform/XWindow.cxx
@@ -10,6 +10,7 @@
  * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE.
  */
 
+#include "GL/glew.h"
 #include "XWindow.h"
 #include "XVisual.h"
 #include "OpenGLGState.h"
-- 
1.7.3.2

