From 8516009f9e7d257c396a7e9729be468089bd3da0 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Fri, 7 Jan 2011 13:48:16 +0100
Subject: [PATCH 4/5] Do not use const as non-const is required at some place

---
 src/bzflag/HUDuiList.cxx |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/bzflag/HUDuiList.cxx b/src/bzflag/HUDuiList.cxx
index 201a711..77afc01 100644
--- a/src/bzflag/HUDuiList.cxx
+++ b/src/bzflag/HUDuiList.cxx
@@ -57,7 +57,7 @@ void			HUDuiList::update()
   setIndex(index);
 }
 
-void			HUDuiList::createSlider(const int numValues)
+void			HUDuiList::createSlider(int numValues)
 {
   // create a slider with numValues options
   /* createSlider(4) does the equivalent of
-- 
1.7.3.2

