From 6b65d1feeec8c055edafb80172beff75e617abac Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Fri, 7 Jan 2011 13:48:46 +0100
Subject: [PATCH 5/5] Make arguments constant as required

---
 src/common/VotingBooth.cxx |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/common/VotingBooth.cxx b/src/common/VotingBooth.cxx
index e36f446..29f9238 100644
--- a/src/common/VotingBooth.cxx
+++ b/src/common/VotingBooth.cxx
@@ -85,7 +85,7 @@ bool VotingBooth::hasVoted(const std::string voterName) const
 }
 
 
-bool VotingBooth::vote(std::string voterName, std::string response)
+bool VotingBooth::vote(const std::string voterName, const std::string response)
 {
   if (this->hasVoted(voterName)) {
     /* voters are not allowed to vote multiple times */
-- 
1.7.3.2

