From 6b17727747523731d64399166f1182bf24988b41 Mon Sep 17 00:00:00 2001
From: Maciej Blizinski <maciej@opencsw.org>
Date: Sat, 12 Jun 2010 11:04:27 +0200
Subject: [PATCH 09/10] lockf instead of flock

---
 misc.c |    5 +++--
 1 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/misc.c b/misc.c
index 933552f..13def32 100644
--- a/misc.c
+++ b/misc.c
@@ -42,7 +42,7 @@ static char rcsid[] = "$Id: misc.c,v 2.9 1994/01/15 20:43:43 vixie Exp $";
 #if defined(SYSLOG)
 # include <syslog.h>
 #endif
-
+#include <sys/file.h>
 
 #if defined(LOG_DAEMON) && !defined(LOG_CRON)
 #define LOG_CRON LOG_DAEMON
@@ -308,7 +308,8 @@ acquire_daemonlock(closeflag)
 			exit(ERROR_EXIT);
 		}
 
-		if (flock(fd, LOCK_EX|LOCK_NB) < OK) {
+    /* This bit originally used flock(), switched to lockf() */
+    if (lockf(fd, F_TLOCK, 0) < OK) {
 			int save_errno = errno;
 
 			fscanf(fp, "%d", &otherpid);
-- 
1.7.1

