From a04d34c996c8f87f92f96243cd80dd54ff4a9a1a Mon Sep 17 00:00:00 2001
From: Sebastian Kayser <skayser@opencsw.org>
Date: Wed, 24 Feb 2010 16:42:46 +0100
Subject: [PATCH 2/2] tmpwatch.c: include stpcpy.c

---
 tmpwatch.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/tmpwatch.c b/tmpwatch.c
index 4091013..a6b47a1 100644
--- a/tmpwatch.c
+++ b/tmpwatch.c
@@ -35,6 +35,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include "stpcpy.c"
 #include <time.h>
 #include <sys/stat.h>
 #include <sys/time.h>
-- 
1.6.6

