From 5ca8b992d77a3479f1081432d21f8a96382b41c6 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Sat, 1 Feb 2014 14:56:06 +0100
Subject: [PATCH] Do not take ref on a simple scalar string or it wont match

---
 tools/scanner.pl | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/scanner.pl b/tools/scanner.pl
index ab9bcb8..4216721 100755
--- a/tools/scanner.pl
+++ b/tools/scanner.pl
@@ -774,7 +774,7 @@ sub ftp_readdir
   return unless defined $ftp;
   my $text = ftp_cont($ftp, "$url/$name");
 
-  if(!ref($text) && \$text =~ m/^(\d\d\d)\s/) {	# some FTP status code? Not good.
+  if(!ref($text) && $text =~ m/^(\d\d\d)\s/) {	# some FTP status code? Not good.
 
     # Bug: Net::FTP wrongly reports timeouts (421) as code 550:
     # sunsite.informatik.rwth-aachen.de: ftp dir: ftp://sunsite.informatik.rwth-aachen.de/pub/linux/opensuse/distribution/11.0/repo/debug/suse/i686
-- 
1.8.4.1

