From b9bad73a01288801847ba7f01c1980453885cab7 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Sun, 26 Jan 2014 21:28:31 +0100
Subject: [PATCH] Exit if geoip file cannot be opened

---
 tools/geoiplookup_city.c      | 4 ++++
 tools/geoiplookup_continent.c | 4 ++++
 2 files changed, 8 insertions(+)

diff --git a/tools/geoiplookup_city.c b/tools/geoiplookup_city.c
index 2b33c1d..6faed7e 100644
--- a/tools/geoiplookup_city.c
+++ b/tools/geoiplookup_city.c
@@ -58,6 +58,10 @@ int main(int argc, char **argv) {
         }
 
 	gip = GeoIP_open(geoipfilename, GEOIP_STANDARD);
+	if( gip == NULL ) {
+		printf("An error occured during GeoIP_open\n");
+		return 1;
+	}
 	edition = GeoIP_database_edition(gip);
 
 	if (edition == GEOIP_COUNTRY_EDITION) {
diff --git a/tools/geoiplookup_continent.c b/tools/geoiplookup_continent.c
index 983afe2..871b247 100644
--- a/tools/geoiplookup_continent.c
+++ b/tools/geoiplookup_continent.c
@@ -56,6 +56,10 @@ int main(int argc, char **argv) {
         }
 
 	gip = GeoIP_open(geoipfilename, GEOIP_STANDARD);
+	if( gip == NULL ) {
+		printf("An error occured during GeoIP_open\n");
+		return 1;
+	}
 	edition = GeoIP_database_edition(gip);
 
 	if (edition == GEOIP_COUNTRY_EDITION) {
-- 
1.8.4.1

