From 92b69bbbee79a5f80085290dacf8af607692740e Mon Sep 17 00:00:00 2001
From: Kester Habermann <kester@opencsw.org>
Date: Tue, 26 Oct 2010 10:57:19 +0200
Subject: [PATCH] openldap-path

---
 setup.cfg |    5 ++---
 1 files changed, 2 insertions(+), 3 deletions(-)

diff --git a/setup.cfg b/setup.cfg
index cf765ab..f039820 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -2,8 +2,8 @@
 extra_objects = 
 extra_compile_args = 
 libs = ldap_r lber sasl2 ssl crypto
-library_dirs = /usr/local/openldap-2.3/lib
-include_dirs = /usr/local/openldap-2.3/include /usr/include/sasl
+library_dirs = /opt/csw/lib
+include_dirs = /opt/csw/include /opt/csw/include/sasl
 
 [egg_info]
 tag_build = 
@@ -20,4 +20,3 @@ provides = python-ldap
 release = 0
 packager = Michael Stroeder <michael@stroeder.com>
 requires = python libldap.so.2
-
-- 
1.7.3

