diff --speed-large-files --minimal -Nru cyrus-imapd-2.3.7.orig/imap/append.c cyrus-imapd-2.3.7/imap/append.c
--- cyrus-imapd-2.3.7.orig/imap/append.c	2006-05-25 08:57:31.000000000 -0400
+++ cyrus-imapd-2.3.7/imap/append.c	2006-07-28 16:50:51.024126000 -0400
@@ -735,7 +735,7 @@
     }
 
     /* Copy and parse message */
-    r = message_copy_strict(messagefile, destfile, size, 0);
+    r = message_copy_strict(messagefile, destfile, size, 1);
     if (!r) {
 	if (!*body || (as->nummsg - 1))
 	    r = message_parse_file(destfile, NULL, NULL, body);
