diff --speed-large-files --minimal -Nru clearsilver-0.10.1.orig/cs/Makefile clearsilver-0.10.1/cs/Makefile
--- clearsilver-0.10.1.orig/cs/Makefile	2006-01-09 18:55:23.556262000 -0500
+++ clearsilver-0.10.1/cs/Makefile	2006-01-09 18:55:07.902459000 -0500
@@ -23,7 +23,7 @@
 CSDUMP_OBJ = $(CSDUMP_SRC:%.c=%.o)
 
 CFLAGS += -I$(NEOTONIC_ROOT)
-LIBS += -L$(LIB_DIR) -lneo_cs -lneo_utl # -lefence
+LIBS := -L$(LIB_DIR) -lneo_cs -lneo_utl $(LIBS) # -lefence
 
 TARGETS = $(CS_LIB) $(CSTEST_EXE) $(CSR_EXE) test
 
