From 713c89789b8e5f7e4a0cbe69d11d07829141201f Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Wed, 3 Apr 2013 10:38:51 +0200
Subject: [PATCH] If alloca.h is available it is required (e.g. on Solaris 10)

---
 configure.ac             | 2 ++
 dbus/dbus-sysdeps-unix.c | 3 +++
 2 files changed, 5 insertions(+)

diff --git a/configure.ac b/configure.ac
index ee89dcb..3bd2405 100644
--- a/configure.ac
+++ b/configure.ac
@@ -688,6 +688,8 @@ AC_CHECK_HEADERS(ws2tcpip.h)
 
 AC_CHECK_HEADERS(wspiapi.h)
 
+AC_CHECK_HEADERS(alloca.h)
+
 # Add -D_POSIX_PTHREAD_SEMANTICS if on Solaris
 #
 case $host_os in
diff --git a/dbus/dbus-sysdeps-unix.c b/dbus/dbus-sysdeps-unix.c
index 30606ff..88db415 100644
--- a/dbus/dbus-sysdeps-unix.c
+++ b/dbus/dbus-sysdeps-unix.c
@@ -72,6 +72,9 @@
 #ifdef HAVE_GETPEERUCRED
 #include <ucred.h>
 #endif
+#ifdef HAVE_ALLOCA_H
+#include <alloca.h>
+#endif
 
 #ifdef HAVE_ADT
 #include <bsm/adt.h>
-- 
1.8.1.4

