From e54601c6645a7e6041c808866029866990d27f69 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Tue, 23 Aug 2011 16:52:46 +0200
Subject: [PATCH] Add missing inclusion of inttypes.h

---
 dbus/dbus-sysdeps.h |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/dbus/dbus-sysdeps.h b/dbus/dbus-sysdeps.h
index f66efaa..f8438f4 100644
--- a/dbus/dbus-sysdeps.h
+++ b/dbus/dbus-sysdeps.h
@@ -31,6 +31,10 @@
 #include <stdint.h>
 #endif
 
+#ifdef HAVE_INTTYPES_H
+#include <inttypes.h>
+#endif
+
 #include <dbus/dbus-errors.h>
 #include <dbus/dbus-file.h>
 #include <dbus/dbus-string.h>
-- 
1.7.6

