--- jhead-2.5.orig/makefile	2006-01-08 09:13:16.000000000 -0800
+++ jhead-2.5/makefile	2006-04-26 14:23:15.782840000 -0700
@@ -9,7 +9,19 @@
 objs = $(OBJ)/jhead.o $(OBJ)/jpgfile.o $(OBJ)/exif.o $(OBJ)/gpsinfo.o $(OBJ)/makernote.o 
 
 $(OBJ)/%.o:$(SRC)/%.c
-	${CC} -O3 -Wall -c $< -o $@
+	${CC} $(CFLAGS) -c $< -o $@
 
 jhead: $(objs) jhead.h
-	${CC} -o jhead $(objs) -lm
+	${CC} $(LDFLAGS) -o jhead $(objs) -lm
+
+install: all
+	sed '1,/^Liscence:/d' < readme.txt >COPYING
+	test -x /usr/ccs/bin/strip && /usr/ccs/bin/strip jhead
+	test -x /usr/ccs/bin/mcs && /usr/ccs/bin/mcs -d jhead
+	ginstall -D jhead $(DESTDIR)/bin/jhead
+	ginstall -D jhead.1.gz $(DESTDIR)/share/man/man1/jhead.1.gz
+	ginstall -D changes.txt $(DESTDIR)/share/doc/jhead/changes.txt
+	gzip -d $(DESTDIR)/share/man/man1/jhead.1.gz
+
+clean:
+	rm -f *.o jhead
