diff -Nru gftp-2.0.16.orig/src/gtk/gftp-gtk.h gftp-2.0.16/src/gtk/gftp-gtk.h
--- gftp-2.0.16.orig/src/gtk/gftp-gtk.h	2003-11-02 18:14:04.000000000 -0800
+++ gftp-2.0.16/src/gtk/gftp-gtk.h	2004-03-19 16:13:36.806207000 -0800
@@ -291,7 +291,7 @@
 void ftp_log					( gftp_logging_level level,
 						  gftp_request * request,
 						  const char *string,
-						  ... ) __attribute__((format(printf, 3, 4)));
+						  ... );
 
 void refresh 					( gftp_window_data * wdata );
 
