From 0ea01de68d4fb8b80a141d4607f463e37902e8cd Mon Sep 17 00:00:00 2001
From: Ben Walton <bwalton@opencsw.org>
Date: Sat, 16 Jun 2012 02:06:45 +0200
Subject: [PATCH] apply patch from upstream to handle building with a newer
 libgif

Signed-off-by: Ben Walton <bwalton@opencsw.org>
---
 frmts/gif/gifdataset.cpp |   31 +++++++++++++++++++++++++------
 1 file changed, 25 insertions(+), 6 deletions(-)

diff --git a/frmts/gif/gifdataset.cpp b/frmts/gif/gifdataset.cpp
index c89f34c..67fdb1f 100644
--- a/frmts/gif/gifdataset.cpp
+++ b/frmts/gif/gifdataset.cpp
@@ -469,6 +469,29 @@ GDALDataset *GIFDataset::Open( GDALOpenInfo * poOpenInfo )
     return poDS;
 }
 
+/************************************************************************/ 
+/*                        GDALPrintGifError()                           */ 
+/************************************************************************/ 
+ 
+static void GDALPrintGifError(const char* pszMsg) 
+{ 
+/* GIFLIB_MAJOR is only defined in libgif >= 4.2.0 */ 
+/* libgif 4.2.0 has retired PrintGifError() and added GifErrorString() */ 
+#if defined(GIFLIB_MAJOR) && defined(GIFLIB_MINOR) && \ 
+	((GIFLIB_MAJOR == 4 && GIFLIB_MINOR >= 2) || GIFLIB_MAJOR > 4) 
+    /* Static string actually, hence the const char* cast */ 
+    const char* pszGIFLIBError = (const char*) GifErrorString();
+    if (pszGIFLIBError == NULL) 
+      pszGIFLIBError = "Unknown error"; 
+		
+    CPLError( CE_Failure, CPLE_AppDefined, 
+	      "%s. GIFLib Error : %s", pszMsg, pszGIFLIBError ); 
+#else
+    PrintGifError(); 
+    CPLError( CE_Failure, CPLE_AppDefined, "%s", pszMsg ); 
+#endif 
+} 
+
 /************************************************************************/
 /*                             CreateCopy()                             */
 /************************************************************************/
@@ -596,9 +619,7 @@ GIFDataset::CreateCopy( const char * pszFilename, GDALDataset *poSrcDS,
                           psGifCT->ColorCount, 255, psGifCT) == GIF_ERROR)
     {
         FreeMapObject(psGifCT);
-        PrintGifError();
-        CPLError( CE_Failure, CPLE_AppDefined, 
-                  "Error writing gif file." );
+        GDALPrintGifError("Error writing gif file." );
         EGifCloseFile(hGifFile);
         VSIFCloseL( fp );
         return NULL;
@@ -622,9 +643,7 @@ GIFDataset::CreateCopy( const char * pszFilename, GDALDataset *poSrcDS,
 
     if (EGifPutImageDesc(hGifFile, 0, 0, nXSize, nYSize, bInterlace, NULL) == GIF_ERROR )
     {
-        PrintGifError();
-        CPLError( CE_Failure, CPLE_AppDefined, 
-                  "Error writing gif file." );
+        GDALPrintGifError("Error writing gif file." );
         EGifCloseFile(hGifFile);
         VSIFCloseL( fp );
         return NULL;
-- 
1.7.10.3

