From ebbffb3199cc4f9cd1fd0a429f3eac19a4570f06 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Fri, 26 Feb 2010 22:35:01 +0100
Subject: [PATCH 4/4] Use ucb echo for -n

---
 testsuite/check.sh |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/testsuite/check.sh b/testsuite/check.sh
index a57224d..77eb0ec 100755
--- a/testsuite/check.sh
+++ b/testsuite/check.sh
@@ -5,6 +5,10 @@
 # This script is free software: you have unlimited permission
 # to copy, distribute and modify it.
 
+# Only ucb echo has -n
+PATH=/usr/ucb:$PATH
+export PATH
+
 LC_ALL=C
 export LC_ALL
 objdir=`pwd`
-- 
1.6.6

