diff --speed-large-files --minimal -Nru up-imapproxy-1.2.5rc1.orig/include/imapproxy.h up-imapproxy-1.2.5rc1/include/imapproxy.h
--- up-imapproxy-1.2.5rc1.orig/include/imapproxy.h	2006-02-16 20:49:35.000000000 -0500
+++ up-imapproxy-1.2.5rc1/include/imapproxy.h	2006-07-23 10:25:05.497403000 -0400
@@ -169,7 +169,7 @@
 #define SELECT_STATUS_BUF_SIZE  256               /* size of select status   */
 
 #ifndef DEFAULT_CONFIG_FILE
-#define DEFAULT_CONFIG_FILE     "/etc/imapproxy.conf"
+#define DEFAULT_CONFIG_FILE     "/opt/csw/etc/imapproxy.conf"
 #endif
 
 #define LITERAL_PASSWORD        1
