From 9577cd3a6ec5cf5075419ebc99425236d2a61d3e Mon Sep 17 00:00:00 2001
From: builder user <builder@opencsw.org>
Date: Mon, 19 Sep 2011 15:53:13 +0200
Subject: [PATCH 0/1] Use /opt/csw/bin/bash for bootstrap.sh

---
 bootstrap.sh |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/bootstrap.sh b/bootstrap.sh
index b41b147..e079783 100755
--- a/bootstrap.sh
+++ b/bootstrap.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/opt/csw/bin/bash
 # Used to setup the configure.ac, autoheader and Makefile.in's if configure
 # has not been generated. This script is only needed for developers when
 # configure has not been run, or if a Makefile.am in a non-configured directory
-- 
1.7.6

