From 3ac648c430aa319daee17c52dee48e6b89b4cf63 Mon Sep 17 00:00:00 2001
From: Peter Felecan <pfelecan@opencsw.org>
Date: Mon, 8 Jul 2013 17:30:08 +0200
Subject: [PATCH] Do not compress info file

---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 1e266bd..e0c621d 100644
--- a/Makefile
+++ b/Makefile
@@ -197,7 +197,7 @@ install-info:	$(DESTDIR)$(infodir)slib.info
 info:	install-info
 $(DESTDIR)$(infodir)slib.info.gz: $(DESTDIR)$(infodir)slib.info
 	gzip -f $<
-install-infoz:	$(DESTDIR)$(infodir)slib.info.gz
+install-infoz:	$(DESTDIR)$(infodir)slib.info
 infoz:	install-infoz
 
 slib.doc: slib.1
@@ -205,7 +205,7 @@ slib.doc: slib.1
 install-man: slib.1 installdirs
 	-$(INSTALL_DATA) $< $(DESTDIR)$(mandir)man1/
 
-docs:	$(DESTDIR)$(infodir)slib.info.gz \
+docs:	$(DESTDIR)$(infodir)slib.info \
 	$(DESTDIR)$(htmldir)slib_toc.html slib.dvi \
 	$(DESTDIR)$(pdfdir)slib.pdf \
 	slib.doc
-- 
1.8.3.1

