From af449151a800845da508097fb85bae299b2e15ab Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Tue, 16 Aug 2011 17:12:43 +0200
Subject: [PATCH] Allow -norunpath for libtool

---
 ltmain.sh |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/ltmain.sh b/ltmain.sh
index 3061e3c..5817d54 100755
--- a/ltmain.sh
+++ b/ltmain.sh
@@ -5840,7 +5840,7 @@ func_mode_link ()
       # -O*, -flto*, -fwhopr*, -fuse-linker-plugin GCC link-time optimization
       -64|-mips[0-9]|-r[0-9][0-9]*|-xarch=*|-xtarget=*|+DA*|+DD*|-q*|-m*| \
       -t[45]*|-txscale*|-p|-pg|--coverage|-fprofile-*|-F*|@*|-tp=*|--sysroot=*| \
-      -O*|-flto*|-fwhopr*|-fuse-linker-plugin)
+      -O*|-flto*|-fwhopr*|-fuse-linker-plugin|-norunpath)
         func_quote_for_eval "$arg"
 	arg="$func_quote_for_eval_result"
         func_append compile_command " $arg"
-- 
1.7.6

