From 8d080cb9ee95738c6b446f1a1fcb8cddbbae7559 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Wed, 20 Jan 2010 15:55:55 +0100
Subject: [PATCH] Add missing inttypes.h

---
 pcre_compile.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/pcre_compile.c b/pcre_compile.c
index beae0e4..8a54b54 100644
--- a/pcre_compile.c
+++ b/pcre_compile.c
@@ -1,3 +1,4 @@
+#include <inttypes.h>
 /*************************************************
 *      Perl-Compatible Regular Expressions       *
 *************************************************/
-- 
1.6.6

