From 3fea8bd4c59e4efa53cc9cf842c8836dacdcba89 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Sat, 4 Feb 2012 23:19:53 +0100
Subject: [PATCH] Remove bashism

---
 RunTest |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/RunTest b/RunTest
index aae292e..0d542f9 100755
--- a/RunTest
+++ b/RunTest
@@ -135,7 +135,7 @@ $sim ./pcretest -C pcre8 >/dev/null
 support8=$?
 $sim ./pcretest -C pcre16 >/dev/null
 support16=$?
-if [ $(( $support8 + $support16 )) -eq 2 ] ; then
+if [ `expr $support8 + $support16` -eq 2 ] ; then
   test8=
   test16=-16
   if [ "$arg8" = yes -a "$arg16" != yes ] ; then
-- 
1.7.8.3

