From 914449712a08ff8797249162e2e32e8ffaa71d93 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Sat, 6 Apr 2013 19:32:26 +0200
Subject: [PATCH] Add missing sighandler_t on Solaris

---
 common/toolcommon/tool-daemon-common.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/common/toolcommon/tool-daemon-common.h b/common/toolcommon/tool-daemon-common.h
index a436bcb..eaaf4fa 100644
--- a/common/toolcommon/tool-daemon-common.h
+++ b/common/toolcommon/tool-daemon-common.h
@@ -24,6 +24,8 @@
 
 int daemonize_common (const char *pidfile);
 
+typedef void (*sighandler_t)(int);
+
 /* can pass NULL for no callback */
 int daemon_signal_handler_setup (sighandler_t cb);
 
-- 
1.8.1.4

