--- sunc.OLD	Tue Oct 12 18:34:30 2010
+++ sunc	Tue Oct 12 18:35:43 2010
@@ -37,6 +37,7 @@
     ngx_amd64="-m64"
 
 else
+    ngx_sparc="-xarch=v8"
     ngx_sparc32="-xarch=v8plus"
     ngx_sparc64="-xarch=v9"
     ngx_amd64="-xarch=amd64"
@@ -94,6 +95,14 @@
         CPU_OPT="$CPU_OPT -xcache=64/64/2:1024/64/16"
     ;;
 
+    sparc)
+        # build 32-bit Sparc v8 binary
+        CPU_OPT="$ngx_sparc"
+        CORE_LINK="$CORE_LINK $ngx_sparc"
+        CC_AUX_FLAGS="$CC_AUX_FLAGS $ngx_sparc"
+        NGX_CPU_CACHE_LINE=64
+    ;;
+
     sparc32)
         # build 32-bit UltraSparc binary
         CPU_OPT="$ngx_sparc32"
