From 921cb8295bd8d0e17a067f0782e20437e4b11d53 Mon Sep 17 00:00:00 2001
From: Peter Bonivart <bonivart@opencsw.org>
Date: Wed, 1 Feb 2012 17:40:08 +0100
Subject: [PATCH] fix-Makefile-in-cf-dir

---
 cf/cf/Makefile |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/cf/cf/Makefile b/cf/cf/Makefile
index 2b0f4b3..6e25162 100644
--- a/cf/cf/Makefile
+++ b/cf/cf/Makefile
@@ -16,7 +16,7 @@ CF=	sendmail
 # name of source for submit.cf (without extension)
 SUBMIT=	submit
 # directory for .cf files
-MAILDIR=/etc/mail
+MAILDIR=/etc/opt/csw/mail
 M4=	m4
 CFDIR=	..
 SED=  sed
@@ -25,7 +25,7 @@ CHMOD=	chmod
 ROMODE=	444
 RM=	rm -f
 # use our own install program; should be really confINSTALL
-INSTALL=../../devtools/bin/install.sh
+INSTALL=/usr/sbin/install
 # CF file ownership/permissions
 CFOWN=root
 CFGRP=bin
-- 
1.7.8.3

