From b53c60334676b2dcbf5f4d3de3ead8a391b8a43c Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Thu, 21 Oct 2010 14:12:56 +0200
Subject: [PATCH] Remove superflous extension breaking install with the shell

---
 doc/Makefile.in |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/doc/Makefile.in b/doc/Makefile.in
index 63b4893..4393def 100644
--- a/doc/Makefile.in
+++ b/doc/Makefile.in
@@ -344,7 +344,7 @@ distclean-local:
 
 install-data-local:
 	$(MKDIR_P) "$(DESTDIR)$(docdir)/html"  ## Didn't work with installdirs-local
-	$(INSTALL_DATA) html/*.{css,gif,html,png} "$(DESTDIR)$(docdir)/html/"
+	$(INSTALL_DATA) html/*.{css,html,png} "$(DESTDIR)$(docdir)/html/"
 
 uninstall-local:
 	-rm -Rf "$(DESTDIR)$(docdir)/html"
-- 
1.7.2.2

