Index: orca/data_gatherers/orcallator/orcallator.cfg.in
===================================================================
--- orca/data_gatherers/orcallator/orcallator.cfg.in	(revision 530)
+++ orca-new/data_gatherers/orcallator/orcallator.cfg.in	(working copy)
@@ -356,7 +356,7 @@
 plot {
 title			%g Interface Bits Per Second: $1
 source			orcallator
-data			1024 * 8 * ((?:(?:bge)|(?:ce)|(?:fjg[ei])|(?:v?ge)|(?:skge))\d+)InKB/s
+data			1024 * 8 * ((?:(?:bge)|(?:ce)|(?:fjg[ei])|(?:v?ge)|(?:skge)|(?:e1000g)|(?:ipge)|(?:bnx))\d+)InKB/s
 data			1024 * 8 * $1OuKB/s
 line_type		area
 line_type		line1
