diff -Nru --speed-large-files --minimal openssl-0.9.7d.orig/config openssl-0.9.7d/config
--- openssl-0.9.7d.orig/config	2004-01-21 01:58:17.000000000 -0800
+++ openssl-0.9.7d/config	2004-03-18 21:41:11.316059000 -0800
@@ -608,10 +608,10 @@
 	    if [ "$CC" = "cc" -a $CCVER -ge 50 ]; then
 		echo "WARNING! If you wish to build 64-bit library, then you have to"
 		echo "         invoke './Configure solaris64-sparcv9-cc' *manually*."
-		if [ "$TEST" = "false" ]; then
-		  echo "         You have about 5 seconds to press Ctrl-C to abort."
-		  (stty -icanon min 0 time 50; read waste) < /dev/tty
-		fi
+		#if [ "$TEST" = "false" ]; then
+		#  echo "         You have about 5 seconds to press Ctrl-C to abort."
+		#  (stty -icanon min 0 time 50; read waste) < /dev/tty
+		#fi
 	    elif [ "$CC" = "gcc" -a "$GCC_ARCH" = "-m64" ]; then
 		# $GCC_ARCH denotes default ABI chosen by compiler driver
 		# (first one found on the $PATH). I assume that user
@@ -620,18 +620,18 @@
 		OUT="solaris64-sparcv9-gcc"
 		echo "WARNING! If you wish to build 32-bit library, then you have to"
 		echo "         invoke './Configure solaris-sparcv9-gcc' *manually*."
-		if [ "$TEST" = "false" ]; then
-		  echo "         You have about 5 seconds to press Ctrl-C to abort."
-		  (stty -icanon min 0 time 50; read waste) < /dev/tty
-		fi
+		#if [ "$TEST" = "false" ]; then
+		#  echo "         You have about 5 seconds to press Ctrl-C to abort."
+		#  (stty -icanon min 0 time 50; read waste) < /dev/tty
+		#fi
 	    elif [ "$GCC_ARCH" = "-m32" ]; then
 		echo "NOTICE! If you *know* that your GNU C supports 64-bit/V9 ABI"
 		echo "        and wish to build 64-bit library, then you have to"
 		echo "        invoke './Configure solaris64-sparcv9-gcc' *manually*."
-		if [ "$TEST" = "false" ]; then
-		  echo "         You have about 5 seconds to press Ctrl-C to abort."
-		  (stty -icanon min 0 time 50; read waste) < /dev/tty
-		fi
+		#if [ "$TEST" = "false" ]; then
+		#  echo "         You have about 5 seconds to press Ctrl-C to abort."
+		#  (stty -icanon min 0 time 50; read waste) < /dev/tty
+		#fi
 	    fi
 	fi
 	;;
