From f0b7563d0a327d23fe8e6915e3d21059a0fb0d60 Mon Sep 17 00:00:00 2001
From: Ben Walton <bwalton@opencsw.org>
Date: Mon, 22 Dec 2014 17:13:19 +0000
Subject: [PATCH] use wc -w instead of awk in test suite

Signed-off-by: Ben Walton <bwalton@opencsw.org>
---
 t/t0090-cache-tree.sh | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/t/t0090-cache-tree.sh b/t/t0090-cache-tree.sh
index 158cf4f..508cce6 100755
--- a/t/t0090-cache-tree.sh
+++ b/t/t0090-cache-tree.sh
@@ -1,5 +1,7 @@
 #!/bin/sh
 
+set -x
+
 test_description="Test whether cache-tree is properly updated
 
 Tests whether various commands properly update and/or rewrite the
@@ -22,7 +24,7 @@ generate_expected_cache_tree_rec () {
 	# ls-files might have foo/bar, foo/bar/baz, and foo/bar/quux
 	# We want to count only foo because it's the only direct child
 	subtrees=$(git ls-files|grep /|cut -d / -f 1|uniq) &&
-	subtree_count=$(echo "$subtrees"|awk -v c=0 '$1 {++c} END {print c}') &&
+	subtree_count=$(echo "$subtrees"| wc -w) &&
 	entries=$(git ls-files|wc -l) &&
 	printf "SHA $dir (%d entries, %d subtrees)\n" "$entries" "$subtree_count" &&
 	for subtree in $subtrees
-- 
2.1.3

