From 351cb3ace190b99541c0a1b278ea60b7d953c5f8 Mon Sep 17 00:00:00 2001
From: Maciej Blizinski <maciej@opencsw.org>
Date: Mon, 7 Mar 2011 22:08:38 +0100
Subject: [PATCH 3/3] postgresql: Removed /usr/local from ecpg.c

---
 src/interfaces/ecpg/preproc/ecpg.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/interfaces/ecpg/preproc/ecpg.c b/src/interfaces/ecpg/preproc/ecpg.c
index db52bf5..05cad72 100644
--- a/src/interfaces/ecpg/preproc/ecpg.c
+++ b/src/interfaces/ecpg/preproc/ecpg.c
@@ -253,7 +253,7 @@ main(int argc, char *const argv[])
 	}
 
 	add_include_path(".");
-	add_include_path("/usr/local/include");
+	add_include_path("/opt/csw/include");
 	get_include_path(my_exec_path, include_path);
 	add_include_path(include_path);
 	add_include_path("/usr/include");
-- 
1.7.3.2

