--- a/mozilla/security/coreconf/SunOS5.mk
+++ b/mozilla/security/coreconf/SunOS5.mk
@@ -129,9 +129,9 @@ ifeq ($(BUILD_SUN_PKG), 1)
 # The -R '$ORIGIN' linker option instructs this library to search for its
 # dependencies in the same directory where it resides.
 ifeq ($(USE_64), 1)
-RPATH = -R '$$ORIGIN:/usr/lib/mps/secv1/64:/usr/lib/mps/64'
+RPATH = -R'/opt/csw/lib/64/$$ISALIST' -M ${HOME}/opencsw/.buildsys/v2/gar/lib/map.solaris10 -B direct -z ignore
 else
-RPATH = -R '$$ORIGIN:/usr/lib/mps/secv1:/usr/lib/mps'
+RPATH = -R'/opt/csw/lib/$$ISALIST' -M ${HOME}/opencsw/.buildsys/v2/gar/lib/map.solaris10 -B direct -z ignore
 endif
 else
 RPATH = -R '$$ORIGIN'
