--- a/mozilla/security/nss/cmd/platlibs.mk
+++ b/mozilla/security/nss/cmd/platlibs.mk
@@ -10,9 +10,9 @@ ifeq ($(BUILD_SUN_PKG), 1)
 
 ifeq ($(OS_ARCH), SunOS) 
 ifeq ($(USE_64), 1)
-EXTRA_SHARED_LIBS += -R '$$ORIGIN/../lib:/usr/lib/mps/secv1/64:/usr/lib/mps/64'
+EXTRA_SHARED_LIBS += -R'/opt/csw/lib/64/$$ISALIST' -M ${HOME}/opencsw/.buildsys/v2/gar/lib/map.solaris10 -B direct -z ignore
 else
-EXTRA_SHARED_LIBS += -R '$$ORIGIN/../lib:/usr/lib/mps/secv1:/usr/lib/mps'
+EXTRA_SHARED_LIBS += -R'/opt/csw/lib/$$ISALIST' -M ${HOME}/opencsw/.buildsys/v2/gar/lib/map.solaris10 -B direct -z ignore
 endif
 endif
 
