--- nss-3.12.4-with-nspr-4.8/mozilla/security/nss/config/Makefile.orig	2009-11-23 02:38:13.942222659 +0100
+++ nss-3.12.4-with-nspr-4.8/mozilla/security/nss/config/Makefile	2009-11-23 02:38:42.759423711 +0100
@@ -13,7 +13,7 @@
 export:
 	# Create the nss.pc file
 	mkdir -p $(DIST)/lib/pkgconfig
-	sed -e "s,@prefix@,$(PREFIX)," \
+	gsed -e "s,@prefix@,$(PREFIX)," \
 	    -e "s,@exec_prefix@,\$${prefix}," \
 	    -e "s,@libdir@,\$${prefix}/gentoo/nss," \
 	    -e "s,@includedir@,\$${prefix}/include/nss," \
@@ -21,18 +21,18 @@
 	    -e "s,@NSS_MINOR_VERSION@,$(NSS_MINOR_VERSION)," \
 	    -e "s,@NSS_PATCH_VERSION@,$(NSS_PATCH_VERSION)," \
 	    nss.pc.in > nss.pc
-	chmod 0644 nss.pc
-	ln -sf ../../../../../security/nss/config/nss.pc $(DIST)/lib/pkgconfig
+	gchmod 0644 nss.pc
+	gln -sf ../../../../../security/nss/config/nss.pc $(DIST)/lib/pkgconfig
 
 	# Create the nss-config script
 	mkdir -p $(DIST)/bin
-	sed -e "s,@prefix@,$(PREFIX)," \
+	gsed -e "s,@prefix@,$(PREFIX)," \
 	    -e "s,@NSS_MAJOR_VERSION@,$(NSS_MAJOR_VERSION)," \
 	    -e "s,@NSS_MINOR_VERSION@,$(NSS_MINOR_VERSION)," \
 	    -e "s,@NSS_PATCH_VERSION@,$(NSS_PATCH_VERSION)," \
 	    nss-config.in > nss-config
-	chmod 0755 nss-config
-	ln -sf ../../../../security/nss/config/nss-config $(DIST)/bin
+	gchmod 0755 nss-config
+	gln -sf ../../../../security/nss/config/nss-config $(DIST)/bin
 
 libs:
 
