--- nss-3.12.4-with-nspr-4.8/mozilla/security/nss/cmd/platlibs.mk.orig	2009-11-22 14:31:53.252227901 +0100
+++ nss-3.12.4-with-nspr-4.8/mozilla/security/nss/cmd/platlibs.mk	2009-11-22 14:32:46.536330277 +0100
@@ -48,6 +48,8 @@
 else
 EXTRA_SHARED_LIBS += -R '$$ORIGIN/../lib'
 endif
+EXTRA_SHARED_LIBS += -R/opt/csw/lib/$$ISALIST
+EXTRA_SHARED_LIBS += -R/opt/csw/lib/nspr/$$ISALIST
 endif
 
 ifeq ($(OS_ARCH), Linux)
