--- nss-3.12.4-with-nspr-4.8/mozilla/security/nss/lib/softoken/config.mk.orig	2009-11-21 16:42:33.499624641 +0100
+++ nss-3.12.4-with-nspr-4.8/mozilla/security/nss/lib/softoken/config.mk	2009-11-21 16:43:06.838729533 +0100
@@ -54,6 +54,7 @@
 
 ifdef NS_USE_GCC
 EXTRA_SHARED_LIBS += \
+	-L/opt/csw/lib \
 	-L$(DIST)/lib \
 	-L$(NSSUTIL_LIB_DIR) \
 	-lnssutil3 \
@@ -80,6 +81,7 @@
 # $(PROGRAM) has NO explicit dependencies on $(EXTRA_SHARED_LIBS)
 # $(EXTRA_SHARED_LIBS) come before $(OS_LIBS), except on AIX.
 EXTRA_SHARED_LIBS += \
+	-L/opt/csw/lib \
 	-L$(DIST)/lib \
 	-L$(NSSUTIL_LIB_DIR) \
 	-lnssutil3 \
