--- a/mozilla/security/nss/lib/softoken/config.mk.orig	2009-11-27 00:13:49.173159939 +0100
+++ b/mozilla/security/nss/lib/softoken/config.mk	2009-11-27 00:15:48.846636621 +0100
@@ -57,6 +57,7 @@
 	-L$(DIST)/lib \
 	-L$(NSSUTIL_LIB_DIR) \
 	-lnssutil3 \
+	-L/opt/csw/lib/64 \
 	-lsqlite3 \
 	-L$(NSPR_LIB_DIR) \
 	-lplc4 \
@@ -83,6 +84,7 @@
 	-L$(DIST)/lib \
 	-L$(NSSUTIL_LIB_DIR) \
 	-lnssutil3 \
+	-L/opt/csw/lib/64 \
 	-lsqlite3 \
 	-L$(NSPR_LIB_DIR) \
 	-lplc4 \
