diff --speed-large-files --minimal -Nru gd-2.0.22.orig/gdft.c gd-2.0.22/gdft.c
--- gd-2.0.22.orig/gdft.c	2004-01-15 09:15:47.000000000 -0800
+++ gd-2.0.22/gdft.c	2004-02-11 20:00:04.844456000 -0800
@@ -480,7 +480,7 @@
       encoding = charmap->encoding_id;
 
 /* EAM DEBUG - Newer versions of libfree2 make it easier by defining encodings */
-#if (defined(FREETYPE_MAJOR) && (FREETYPE_MAJOR >=2 ) && (FREETYPE_MINOR >= 1))
+#if (defined(FREETYPE_MAJOR) && (FREETYPE_MAJOR >=2 ) && (FREETYPE_MINOR >= 1) && (FREETYPE_PATCH > 0))
       if (charmap->encoding == FT_ENCODING_MS_SYMBOL
       ||  charmap->encoding == FT_ENCODING_ADOBE_CUSTOM
       ||  charmap->encoding == FT_ENCODING_ADOBE_STANDARD) {
@@ -1048,7 +1048,7 @@
 	}
 
 /* EAM DEBUG */
-#if (defined(FREETYPE_MAJOR) && (FREETYPE_MAJOR >=2 ) && (FREETYPE_MINOR >= 1))
+#if (defined(FREETYPE_MAJOR) && (FREETYPE_MAJOR >=2 ) && (FREETYPE_MINOR >= 1) && (FREETYPE_PATCH > 0))
       if (font->face->charmap->encoding == FT_ENCODING_MS_SYMBOL)
 	{
          /* I do not know the significance of the constant 0xf000. */
