From ee9340922a5a998e05b2ec123233f2bbf57a84a3 Mon Sep 17 00:00:00 2001
From: Super-User <root@cabbage.dub.corp.google.com>
Date: Wed, 9 Sep 2009 19:33:00 +0100
Subject: [PATCH 2/3] Adding the refcount member to mime_type_t

---
 scheduler/mime.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/scheduler/mime.h b/scheduler/mime.h
index 0194964..4bd64c2 100644
--- a/scheduler/mime.h
+++ b/scheduler/mime.h
@@ -85,6 +85,7 @@ typedef struct _mime_magic_s		/**** MIME Magic Data ****/
 
 typedef struct _mime_type_s		/**** MIME Type Data ****/
 {
+  int           refcount;               /* To allow sharing of this struct */
   mime_magic_t	*rules;			/* Rules used to detect this type */
   int		priority;		/* Priority of this type */
   char		super[MIME_MAX_SUPER],	/* Super-type name ("image", "application", etc.) */
-- 
1.6.3.2

