From 87110488b3509610756f0c21447a42124d3a092d Mon Sep 17 00:00:00 2001
From: =?utf-8?q?Maciej=20Blizi=C5=84ski?= <blizinski@google.com>
Date: Fri, 23 Oct 2009 11:08:23 +0100
Subject: [PATCH] Solaris zone support in init script

---
 init/cups.sh.in |   12 +++++++++++-
 1 files changed, 11 insertions(+), 1 deletions(-)

diff --git a/init/cups.sh.in b/init/cups.sh.in
index 2ff418c..d714b19 100755
--- a/init/cups.sh.in
+++ b/init/cups.sh.in
@@ -148,7 +148,7 @@ case "`uname`" in
 	HP-UX* | AIX* | SINIX*)
 		pid=`ps -e | awk '{if (match($4, ".*/cupsd$") || $4 == "cupsd") print $1}'`
 		;;
-	IRIX* | SunOS*)
+	IRIX*)
 		pid=`ps -e | nawk '{if (match($4, ".*/cupsd$") || $4 == "cupsd") print $1}'`
 		;;
 	UnixWare*)
@@ -161,6 +161,16 @@ case "`uname`" in
 	Linux* | *BSD* | Darwin*)
 		pid=`ps ax | awk '{if (match($5, ".*/cupsd$") || $5 == "cupsd") print $1}'`
 		;;
+	SunOS*)
+		if [ -x /usr/bin/zonename ]; then
+			zonename=`/usr/bin/zonename`
+			zoneopts="-z $zonename"
+		else
+			zoneopts=
+		fi
+		# The backslash before the ^ is important.
+		pid=`/usr/bin/pgrep ${zoneopts} \^cupsd$`
+		;;
 	*)
 		pid=""
 		;;
-- 
1.6.3.2

