diff --speed-large-files --minimal -Nru openssh-4.7p1.orig/channels.c openssh-4.7p1/channels.c
--- channels.c	2007-06-25 05:04:47.000000000 -0400
+++ channels.c	2008-04-05 15:08:54.648392000 -0400
@@ -2905,7 +2905,7 @@
 				debug2("bind port %d: %.100s", port, strerror(errno));
 				close(sock);
 
-				if (ai->ai_next)
+				if (errno == EADDRNOTAVAIL && ai->ai_next)
 					continue;
 
 				for (n = 0; n < num_socks; n++) {
