diff --speed-large-files --minimal -Nru pnp-0.4.14.orig/sample-config/pnp/nagios.cfg-sample pnp-0.4.14/sample-config/pnp/nagios.cfg-sample
--- pnp-0.4.14.orig/sample-config/pnp/nagios.cfg-sample	1970-01-01 01:00:00.000000000 +0100
+++ pnp-0.4.14/sample-config/pnp/nagios.cfg-sample	2009-05-19 20:46:35.753095929 +0200
@@ -0,0 +1,38 @@
+#
+# You can copy and uncomment the lines which fits your environment
+# to your nagios.cfg
+#
+# Proper process commands you can find in commands.cfg-sample
+#
+
+#
+# Default Mode
+#
+
+#process_performance_data=1
+#enable_environment_macros = 1
+#service_perfdata_command=process-service-perfdata
+#host_perfdata_command=process-host-perfdata
+
+#
+# Bulk Mode
+#
+
+#process_performance_data=1
+#service_perfdata_file=/var/opt/csw/pnp/service-perfdata
+#service_perfdata_file_template=DATATYPE::SERVICEPERFDATA\tTIMET::$TIMET$\tHOSTNAME::$HOSTNAME$\tSERVICEDESC::$SERVICEDESC$\tSERVICEPERFDATA::$SERVICEPERFDATA$\tSERVICECHECKCOMMAND::$SERVICECHECKCOMMAND$\tHOSTSTATE::$HOSTSTATE$\tHOSTSTATETYPE::$HOSTSTATETYPE$\tSERVICESTATE::$SERVICESTATE$\tSERVICESTATETYPE::$SERVICESTATETYPE$
+#service_perfdata_file_mode=a
+#service_perfdata_file_processing_interval=15
+#service_perfdata_file_processing_command=process-service-perfdata-file
+
+#host_perfdata_file=/var/opt/csw/pnp/host-perfdata
+#host_perfdata_file_template=DATATYPE::HOSTPERFDATA\tTIMET::$TIMET$\tHOSTNAME::$HOSTNAME$\tHOSTPERFDATA::$HOSTPERFDATA$\tHOSTCHECKCOMMAND::$HOSTCHECKCOMMAND$\tHOSTSTATE::$HOSTSTATE$\tHOSTSTATETYPE::$HOSTSTATETYPE$
+#host_perfdata_file_mode=a
+#host_perfdata_file_processing_interval=15
+#host_perfdata_file_processing_command=process-host-perfdata-file
+
+#
+# NPCD with Bulk Mode
+#
+
+# You need to change your nagios.cfg identically as with the Bulk Mode
