--- a/Makefile.common	jeu. juin 21 22:40:36 2012
+++ b/Makefile.common	mer. juil 24 21:42:39 2013
@@ -77,7 +77,7 @@
 # because $(INCLUDES) also contains -I's for some local directories, which
 # must go before -I importinc.  It is too complex for the Super Stable series.
 #
-INCLUDES2 := $(INCLUDES) -I$(SRCDIR)/$(SUBDIR) -I. -I importinc
+INCLUDES2 := -I$(SRCDIR)/$(SUBDIR) -I. $(INCLUDES) -I importinc
 
 ifeq ($(NETPBMLIBTYPE),unixstatic)
   NETPBMLIBFNAME = libnetpbm.$(STATICLIBSUFFIX)
