From 904e9283e718e1b14276ee8f7047f9106c48b6bc Mon Sep 17 00:00:00 2001
From: Peter Felecan <pfelecan@opencsw.org>
Date: Fri, 5 Oct 2012 11:41:59 +0200
Subject: [PATCH] activate Xft for configure

---
 configure | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/configure b/configure
index 3e3cec2..fb229b7 100755
--- a/configure
+++ b/configure
@@ -597,6 +597,9 @@ ac_includes_default="\
 #endif
 #ifdef HAVE_UNISTD_H
 # include <unistd.h>
+#endif
+#ifdef HAVE_X11R6
+# include <X11/Xlib.h>
 #endif"
 
 ac_header_list=
-- 
1.7.12

