From 2bf4fad0be05e3b11a3f233c16f0f681a11641bd Mon Sep 17 00:00:00 2001
From: Peter Felecan <pfelecan@opencsw.org>
Date: Tue, 22 May 2012 14:04:49 +0200
Subject: [PATCH] activate Xft for configure

---
 configure |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/configure b/configure
index d894fe6..affbb9f 100755
--- a/configure
+++ b/configure
@@ -589,6 +589,9 @@ ac_includes_default="\
 #endif
 #ifdef HAVE_UNISTD_H
 # include <unistd.h>
+#endif
+#ifdef HAVE_X11R6
+# include <X11/Xlib.h>
 #endif"
 
 ac_config_libobj_dir=src
-- 
1.7.10

