--- ./config.m4.orig	2009-02-25 10:49:36.267948251 -0600
+++ ./config.m4	2009-02-25 10:50:04.992804101 -0600
@@ -5,6 +5,8 @@
 PHP_ARG_WITH(libedit,for libedit readline replacement, 
 [  --with-libedit[=DIR]    Include libedit readline replacement (CLI/CGI only)])
 
+dnl Force the use of readline
+PHP_LIBEDIT="no"
 if test "$PHP_LIBEDIT" = "no"; then
   PHP_ARG_WITH(readline,for readline support,
   [  --with-readline[=DIR]   Include readline support (CLI/CGI only)])
