From 12209beaed0135b2bf11669cfc7ea27c2b36094f Mon Sep 17 00:00:00 2001
From: Jake Goerzen <jgoerzen@opencsw.org>
Date: Thu, 24 Apr 2014 18:57:38 +0200
Subject: [PATCH] add test to detech we are on Solaris

---
 src/test/test_bt_cl.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/src/test/test_bt_cl.c b/src/test/test_bt_cl.c
index 45ae82f..7352e9e 100644
--- a/src/test/test_bt_cl.c
+++ b/src/test/test_bt_cl.c
@@ -20,6 +20,11 @@ static int crashtype = 0;
 #define NORETURN __attribute__((noreturn))
 #endif
 
+#if defined(__sun) && defined(__SVR4)
+#define NOINLINE __attribute__((noinline))
+#define NORETURN __attribute__((noreturn))
+#endif
+
 int crash(int x) NOINLINE;
 int oh_what(int x) NOINLINE;
 int a_tangled_web(int x) NOINLINE;
-- 
1.8.4.1

