From b8063983295ef5629a106b61c735a90edd842412 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Fri, 25 Jun 2010 16:00:27 +0200
Subject: [PATCH] Fix isinf

---
 src/rrd_config_bottom.h |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/src/rrd_config_bottom.h b/src/rrd_config_bottom.h
index a569197..0a05a40 100644
--- a/src/rrd_config_bottom.h
+++ b/src/rrd_config_bottom.h
@@ -175,7 +175,8 @@ char *strchr (), *strrchr ();
 #  ifdef isinf
 #  undef isinf
 #  endif
-#  define isinf(a) (!!(fpclass(a) & (FP_SNAN|FP_QNAN)))
+/*  define isinf(a) (!!(fpclass(a) & (FP_SNAN|FP_QNAN))) */
+#define isinf(a) (fpclass(a) == FP_NINF || fpclass(a) == FP_PINF)
 
 #endif
 
-- 
1.7.1

