From dec1e5ffd2ec2e58d185f6e4753e38ff5a0af1cd Mon Sep 17 00:00:00 2001
From: Rafael Ostertag <rafisol@opencsw.org>
Date: Fri, 30 Dec 2011 16:27:27 +0100
Subject: [PATCH] ecpg /usr/local/include replacement

---
 src/interfaces/ecpg/preproc/ecpg.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/interfaces/ecpg/preproc/ecpg.c b/src/interfaces/ecpg/preproc/ecpg.c
index db52bf5..05cad72 100644
--- a/src/interfaces/ecpg/preproc/ecpg.c
+++ b/src/interfaces/ecpg/preproc/ecpg.c
@@ -253,7 +253,7 @@ main(int argc, char *const argv[])
 	}
 
 	add_include_path(".");
-	add_include_path("/usr/local/include");
+	add_include_path("/opt/csw/include");
 	get_include_path(my_exec_path, include_path);
 	add_include_path(include_path);
 	add_include_path("/usr/include");
-- 
1.7.6.1

