--- iperf-2.0.2/compat/Thread.c.orig	2007-01-03 22:57:59.258235849 +0100
+++ iperf-2.0.2/compat/Thread.c	2007-01-03 22:58:13.065711544 +0100
@@ -202,7 +202,7 @@
 #if   defined( HAVE_POSIX_THREAD )
             // Cray J90 doesn't have pthread_cancel; Iperf works okay without
 #ifdef HAVE_PTHREAD_CANCEL
-            pthread_cancel( oldTID );
+            pthread_cancel( thread->mTID );
 #endif
 #else // Win32
             // this is a somewhat dangerous function; it's not
