From d8dfa3860ec279f387c66e9c994bac7518e13ab0 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Thu, 6 Jan 2011 13:18:50 +0100
Subject: [PATCH] Use inttypes.h if stdint.h is not available

---
 compat/asprintf.c |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/compat/asprintf.c b/compat/asprintf.c
index cb5282b..ab45110 100644
--- a/compat/asprintf.c
+++ b/compat/asprintf.c
@@ -18,7 +18,11 @@
 
 #include <stdarg.h>
 #include <stdio.h>
+#ifdef HAVE_STDINT_H
 #include <stdint.h>
+#else
+#include <inttypes.h>
+#endif
 #include <string.h>
 
 #include "tmux.h"
-- 
1.7.3.2

