From 5b87d0623c11043f670ec4e2d11197a9030edda1 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Wed, 12 May 2010 14:38:28 +0200
Subject: [PATCH 2/3] Fix one more inttypes.h

---
 compat/asprintf.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/compat/asprintf.c b/compat/asprintf.c
index cb5282b..66b73d5 100644
--- a/compat/asprintf.c
+++ b/compat/asprintf.c
@@ -18,7 +18,7 @@
 
 #include <stdarg.h>
 #include <stdio.h>
-#include <stdint.h>
+#include <inttypes.h>
 #include <string.h>
 
 #include "tmux.h"
-- 
1.7.1

