From 38aff72c66f79339d3878cb6f15d88847133a39a Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Tue, 16 Apr 2013 14:19:24 +0200
Subject: [PATCH 2/3] Use lockf instead of flock as it is POSIX-compliant

---
 client.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/client.c b/client.c
index 91f4765..691ace3 100644
--- a/client.c
+++ b/client.c
@@ -78,8 +78,8 @@ client_get_lock(char *lockfile)
 	if ((lockfd = open(lockfile, O_WRONLY|O_CREAT, 0600)) == -1)
 		fatal("open failed");
 
-	if (flock(lockfd, LOCK_EX|LOCK_NB) == -1 && errno == EWOULDBLOCK) {
-		while (flock(lockfd, LOCK_EX) == -1 && errno == EINTR)
+	if (lockf(lockfd, F_TLOCK, 0) == -1 && errno == EAGAIN) {
+		while (lockf(lockfd, F_LOCK, 0) == -1 && errno == EINTR)
 			/* nothing */;
 		close(lockfd);
 		return (-1);
-- 
1.8.1.4

