From 374b542205a8ebee89c123f203600c627a1bdd45 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Fri, 23 Jul 2010 11:25:22 +0200
Subject: [PATCH 3/3] Add fix for missing MAXHOSTNAMELEN

---
 screen.c |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/screen.c b/screen.c
index e6d56c9..ee36024 100644
--- a/screen.c
+++ b/screen.c
@@ -22,6 +22,9 @@
 #include <string.h>
 #include <unistd.h>
 
+/* For MAXHOSTNAMELEN */
+#include <netdb.h>
+
 #include "tmux.h"
 
 void	screen_resize_x(struct screen *, u_int);
-- 
1.7.1

