From 8c2999a502b88621e39ec11e9cc60106236b5bd6 Mon Sep 17 00:00:00 2001
From: Gordon Marler <gmarler@opencsw.org>
Date: Fri, 20 Aug 2010 06:55:05 +0200
Subject: [PATCH] generate bzip2recover

---
 Makefile-libbz2_so |   11 +++++++++--
 1 files changed, 9 insertions(+), 2 deletions(-)

diff --git a/Makefile-libbz2_so b/Makefile-libbz2_so
index b980404..2eeea45 100644
--- a/Makefile-libbz2_so
+++ b/Makefile-libbz2_so
@@ -33,9 +33,16 @@ OBJS= blocksort.o  \
       decompress.o \
       bzlib.o
 
-all: $(OBJS)
-	$(CC) -G $(CFLAGS) $(LDFLAGS) -o libbz2.so.1.0.5 $(OBJS)
+all: libbz2.so.1.0.5 bzip2 bzip2recover
+
+bzip2: libbz2.so.1.0.5
 	$(CC) $(CFLAGS) $(LDFLAGS) -o bzip2 bzip2.c libbz2.so.1.0.5
+
+bzip2recover: bzip2recover.o
+	$(CC) $(CFLAGS) -o bzip2recover bzip2recover.o
+
+libbz2.so.1.0.5:  $(OBJS)
+	$(CC) -G $(CFLAGS) $(LDFLAGS) -o libbz2.so.1.0.5 $(OBJS)
 	rm -f libbz2.so.1.0
 	ln -s libbz2.so.1.0.5 libbz2.so.1.0
 
-- 
1.7.1

