--- Inline-0.46.orig/C/Makefile.PL	2010-02-17 22:12:18.040826000 +0100
+++ Inline-0.46/C/Makefile.PL	2010-02-17 22:15:07.194173580 +0100
@@ -73,10 +73,10 @@
 
 my $answer = '';
 my $default = $found ? "y" : "n";
-while (1) {
-    $answer = prompt ('Do you want to install Inline::C?', $default);
-    last if $answer =~ /^(y|yes|n|no)$/i;
-}
+#while (1) {
+    #$answer = prompt ('Do you want to install Inline::C?', $default);
+    #last if $answer =~ /^(y|yes|n|no)$/i;
+#}
 
 if ($answer =~ /^(y|yes)$/i) {
     WriteMakefile(
