From 7857370907e05070fb06074974a61fe3e936f64e Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Thu, 16 Oct 2014 10:24:18 +0200
Subject: [PATCH] These flags are for gnu gcc and fail on suns cc compiler

---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index c757a27..43b2179 100755
--- a/setup.py
+++ b/setup.py
@@ -199,7 +199,7 @@ def init_settings(zmq=None):
         for warning in ('unused-function', 'strict-aliasing'):
             extra_flags.append('-Wno-'+warning)
     
-    settings['extra_compile_args'] = extra_flags
+    settings['extra_compile_args'] = ""
     
     # include internal directories
     settings['include_dirs'] += [pjoin('zmq', sub) for sub in ('utils','core','devices')]
-- 
2.0.3

