From 00ff60e45eb1c49011e880b6566528ddf14488a8 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Thu, 16 Oct 2014 10:26:03 +0200
Subject: [PATCH] Include inttypes so builds on solaris 9

---
 zmq/utils/zmq_compat.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/zmq/utils/zmq_compat.h b/zmq/utils/zmq_compat.h
index 50a8ea8..39ac5b3 100644
--- a/zmq/utils/zmq_compat.h
+++ b/zmq/utils/zmq_compat.h
@@ -7,6 +7,10 @@
 
 #if defined(_MSC_VER)
 #define pyzmq_int64_t __int64
+#endif
+#if defined(__sun) || defined(__sun__)
+#include <sys/inttypes.h>
+#define pyzmq_int64_t int64_t
 #else
 #include <stdint.h>
 #define pyzmq_int64_t int64_t
-- 
2.0.3

