From 65752fa7bb4c2577352ed758f695f53a23edd734 Mon Sep 17 00:00:00 2001
From: Juraj Lutter <wilbury@opencsw.org>
Date: Sun, 12 May 2013 19:25:05 +0200
Subject: [PATCH] Do not activate php5 module during package build.

---
 sapi/apache2handler/config.m4 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sapi/apache2handler/config.m4 b/sapi/apache2handler/config.m4
index 702f91f..9e3d546 100644
--- a/sapi/apache2handler/config.m4
+++ b/sapi/apache2handler/config.m4
@@ -77,7 +77,7 @@ if test "$PHP_APXS2" != "no"; then
                 \$(mkinstalldirs) '$APXS_SYSCONFDIR' && \
                  $APXS -S LIBEXECDIR='$APXS_LIBEXECDIR' \
                        -S SYSCONFDIR='$APXS_SYSCONFDIR' \
-                       -i -a -n php5"
+                       -i -n php5"
   fi
 
   case $host_alias in
-- 
1.8.1.4

