diff -ru XML-Twig-3.32.orig/Makefile.PL XML-Twig-3.32/Makefile.PL
--- XML-Twig-3.32.orig/Makefile.PL	2009-03-17 20:16:43.000000000 +0100
+++ XML-Twig-3.32/Makefile.PL	2009-03-17 20:26:13.438626991 +0100
@@ -24,7 +24,7 @@
           "    'perl Makefile.PL -n' to skip installation\n";
     foreach my $prompt (@prompts)
       { my ($program, $default, $description) = @$prompt;
-        if( prompt("Do you want to install '$program' ($description)?", $default) =~ /^y/)
+#        if( prompt("Do you want to install '$program' ($description)?", $default) =~ /^y/)
           { push(@programs, $program); }
       }
   }
