diff -Nru libffi-3.0.9.orig/Makefile.in libffi-3.0.9/Makefile.in
--- libffi-3.0.9.orig/Makefile.in	2009-12-31 13:41:43.000000000 +0100
+++ libffi-3.0.9/Makefile.in	2010-01-30 08:19:09.733833212 +0100
@@ -478,7 +478,7 @@
 	$(am__append_24) $(am__append_25)
 libffi_convenience_la_SOURCES = $(libffi_la_SOURCES)
 nodist_libffi_convenience_la_SOURCES = $(nodist_libffi_la_SOURCES)
-AM_CFLAGS = -Wall -g -fexceptions
+AM_CFLAGS = 
 libffi_la_LDFLAGS = -version-info `grep -v '^\#' $(srcdir)/libtool-version` $(AM_LTLDFLAGS)
 AM_CPPFLAGS = -I. -I$(top_srcdir)/include -Iinclude -I$(top_srcdir)/src
 AM_CCASFLAGS = $(AM_CPPFLAGS)
diff -Nru libffi-3.0.9.orig/include/ffi_common.h libffi-3.0.9/include/ffi_common.h
--- libffi-3.0.9.orig/include/ffi_common.h	2009-12-29 16:22:26.000000000 +0100
+++ libffi-3.0.9/include/ffi_common.h	2010-01-30 08:20:33.829433399 +0100
@@ -84,7 +84,7 @@
 } extended_cif;
 
 /* Terse sized type definitions.  */
-#if defined(_MSC_VER) || defined(__sgi)
+#if defined(_MSC_VER) || defined(__sgi) || defined(__sun)
 typedef unsigned char UINT8;
 typedef signed char   SINT8;
 typedef unsigned short UINT16;
diff -Nru libffi-3.0.9.orig/src/x86/ffitarget.h libffi-3.0.9/src/x86/ffitarget.h
--- libffi-3.0.9.orig/src/x86/ffitarget.h	2009-12-29 16:22:26.000000000 +0100
+++ libffi-3.0.9/src/x86/ffitarget.h	2010-01-30 08:23:16.480398204 +0100
@@ -31,7 +31,7 @@
 
 /* ---- System specific configurations ----------------------------------- */
 
-#if defined (X86_64) && defined (__i386__)
+#if defined (X86_64) && (defined (__i386__) || defined (__i386))
 #undef X86_64
 #define X86
 #endif
@@ -74,10 +74,10 @@
 #else
 
   /* ---- Intel x86 and AMD x86-64 - */
-#if !defined(X86_WIN32) && (defined(__i386__) || defined(__x86_64__))
+#if !defined(X86_WIN32) && (defined(__i386__) || defined(__i386) || defined(__x86_64__) || defined(__x86_64))
   FFI_SYSV,
   FFI_UNIX64,   /* Unix variants all use the same ABI for x86-64  */
-#ifdef __i386__
+#if defined(__i386__) || defined(__i386)
   FFI_DEFAULT_ABI = FFI_SYSV,
 #else
   FFI_DEFAULT_ABI = FFI_UNIX64,
