diff -ru libgphoto2-2.3.1.orig/camlibs/digita/digita.c libgphoto2-2.3.1/camlibs/digita/digita.c
--- libgphoto2-2.3.1.orig/camlibs/digita/digita.c	2006-11-25 06:47:24.000000000 -0800
+++ libgphoto2-2.3.1/camlibs/digita/digita.c	2007-01-27 22:21:12.059500000 -0800
@@ -26,9 +26,7 @@
 #include <errno.h>
 #include <fcntl.h>
 #include <string.h>
-#ifdef HAVE_LIMITS_H
 #include <limits.h>
-#endif
 #ifdef OS2
 #include <db.h>
 #endif
diff -ru libgphoto2-2.3.1.orig/camlibs/mars/mars.c libgphoto2-2.3.1/camlibs/mars/mars.c
--- libgphoto2-2.3.1.orig/camlibs/mars/mars.c	2006-11-25 06:47:25.000000000 -0800
+++ libgphoto2-2.3.1/camlibs/mars/mars.c	2007-01-27 22:42:40.632238000 -0800
@@ -37,6 +37,7 @@
 #define INIT 		0xb5
 #define GET_DATA 	0x0f
 
+#undef M_READ
 static int 
 M_READ (GPPort *port, char *data, int size) 
 {
diff -ru libgphoto2-2.3.1.orig/camlibs/sierra/epson-desc.c libgphoto2-2.3.1/camlibs/sierra/epson-desc.c
--- libgphoto2-2.3.1.orig/camlibs/sierra/epson-desc.c	2006-11-25 06:47:22.000000000 -0800
+++ libgphoto2-2.3.1/camlibs/sierra/epson-desc.c	2007-01-27 22:21:12.089153000 -0800
@@ -42,7 +42,7 @@
  * the name/value union, and so it is the default value initialized.
  *
  * To init to a range, use:
- *	{ range: { 100.1, 2000.0, 10 } }, NULL 
+ *	{ .range = { 100.1, 2000.0, 10 } }, NULL 
  *
  * Casts are used to avoid warnings about using a const.
  *
@@ -185,7 +185,7 @@
  * I have previously set 0,255,1 is old camera_get_config_epson()
  */
 static const ValueNameType ep3000z_reg_23_val_names[] = {
-	{ { range: { 30, 600, 30  } }, NULL },
+	{ { .range = { 30, 600, 30  } }, NULL },
 };
 static const RegisterDescriptorType ep3000z_reg_23[] = { 
 	{
@@ -201,7 +201,7 @@
  * I have previously set 0,255,1 is old camera_get_config_epson()
  */
 static const ValueNameType ep3000z_reg_24_val_names[] = {
-	{ { range: { 30, 600, 30  } }, NULL },
+	{ { .range = { 30, 600, 30  } }, NULL },
 };
 static const RegisterDescriptorType ep3000z_reg_24[] = { 
 	{
diff -ru libgphoto2-2.3.1.orig/camlibs/sierra/nikon-desc.c libgphoto2-2.3.1/camlibs/sierra/nikon-desc.c
--- libgphoto2-2.3.1.orig/camlibs/sierra/nikon-desc.c	2006-11-25 06:47:22.000000000 -0800
+++ libgphoto2-2.3.1/camlibs/sierra/nikon-desc.c	2007-01-27 22:21:12.126487000 -0800
@@ -38,7 +38,7 @@
  * the name/value union, and so it is the default value initialized.
  *
  * To init to a range, use:
- *	{ range: { 100.1, 2000.0, 10 } }, NULL 
+ *	{ .range = { 100.1, 2000.0, 10 } }, NULL 
  *
  * Casts are used to avoid warnings about using a const.
  *
@@ -153,7 +153,7 @@
 #ifdef RANGE_FOR_SHUTTER
 static const ValueNameType cp880_reg_03_val_names[] = { 
 	{
-		{ range: { 0, 8000000 } }, NULL 
+		{ .range = { 0, 8000000 } }, NULL 
 	}
 };
 static const RegisterDescriptorType cp880_reg_03[] = { 
@@ -384,7 +384,7 @@
  * Register 35: lcd brightness.
  */
 static const ValueNameType cp880_reg_35_val_names[] = {
-	{ { range: { 0, 7 } }, NULL },
+	{ { .range = { 0, 7 } }, NULL },
 };
 static const RegisterDescriptorType cp880_reg_35[] = { 
 	{
@@ -399,7 +399,7 @@
  * what the maximum range should be.
  */
 static const ValueNameType cp880_reg_38_val_names[] = {
-	{ { range: { 0, 255 /* XXX? */ } }, NULL },
+	{ { .range = { 0, 255 /* XXX? */ } }, NULL },
 };
 static const RegisterDescriptorType cp880_reg_38[] = { 
 	{
@@ -450,7 +450,7 @@
 #ifdef RANGE_FOR_EXPOSURE
 static const ValueNameType cp880_reg_69_val_names[] = {
 	{
-		{ range: { -2.0, 2.0, .1 } }, NULL,
+		{ .range = { -2.0, 2.0, .1 } }, NULL,
 	}
 };
 static const RegisterDescriptorType cp880_reg_69[] = { 
@@ -508,7 +508,7 @@
  */
 static const ValueNameType cp880_reg_71_val_names[] = {
 	{
-		{ range: { 8.0, 20.0, .1 } }, NULL
+		{ .range = { 8.0, 20.0, .1 } }, NULL
 	}
 };
 static const RegisterDescriptorType cp880_reg_71[] = { 
diff -ru libgphoto2-2.3.1.orig/camlibs/sierra/olympus-desc.c libgphoto2-2.3.1/camlibs/sierra/olympus-desc.c
--- libgphoto2-2.3.1.orig/camlibs/sierra/olympus-desc.c	2006-11-25 06:47:22.000000000 -0800
+++ libgphoto2-2.3.1/camlibs/sierra/olympus-desc.c	2007-01-27 22:21:12.163140000 -0800
@@ -43,7 +43,7 @@
  * the name/value union, and so it is the default value initialized.
  *
  * To init to a range, use:
- *	{ range: { 100.1, 2000.0, 10 } }, NULL 
+ *	{ .range = { 100.1, 2000.0, 10 } }, NULL 
  *
  * Casts are used to avoid warnings about using a const.
  *
@@ -271,7 +271,7 @@
  */
 static const ValueNameType olyrange_reg_03_val_names[] = { 
 	{
-		{ range: { 0, 16000000, 100 } }, NULL 
+		{ .range = { 0, 16000000, 100 } }, NULL 
 	}
 };
 static const RegisterDescriptorType olyrange_reg_03[] = { 
@@ -523,7 +523,7 @@
  * Register 35: lcd brightness.
  */
 static const ValueNameType oly3040_reg_35_val_names[] = {
-	{ { range: { 0, 7 } }, NULL },
+	{ { .range = { 0, 7 } }, NULL },
 };
 static const RegisterDescriptorType oly3040_reg_35[] = { 
 	{
@@ -537,7 +537,7 @@
  * Register 24: camera power save (idle time before entering power save mode)
  */
 static const ValueNameType oly3040_reg_24_val_names[] = {
-	{ { range: { 30, 600, 30  } }, NULL },
+	{ { .range = { 30, 600, 30  } }, NULL },
 };
 static const RegisterDescriptorType oly3040_reg_24[] = { 
 	{
@@ -552,7 +552,7 @@
  * (idle time before entering power save mode)
  */
 static const ValueNameType oly3040_reg_23_val_names[] = {
-	{ { range: { 30, 600, 30  } }, NULL },
+	{ { .range = { 30, 600, 30  } }, NULL },
 };
 static const RegisterDescriptorType oly3040_reg_23[] = { 
 	{
@@ -565,7 +565,7 @@
  * Register 38: lcd auto shut off time
  */
 static const ValueNameType oly3040_reg_38_val_names[] = {
-	{ { range: { 30, 600, 30  } }, NULL },
+	{ { .range = { 30, 600, 30  } }, NULL },
 };
 static const RegisterDescriptorType oly3040_reg_38[] = { 
 	{
@@ -639,7 +639,7 @@
  */
 static const ValueNameType oly750uz_reg_71_val_names[] = {
 	{
-		{ range: { 6.3, 63.0, .1 } }, NULL
+		{ .range = { 6.3, 63.0, .1 } }, NULL
 	}
 };
 static const RegisterDescriptorType oly750uz_reg_71[] = { 
@@ -655,7 +655,7 @@
  */
 static const ValueNameType oly3040_reg_71_val_names[] = {
 	{
-		{ range: { 7.3, 21.0, .1 } }, NULL
+		{ .range = { 7.3, 21.0, .1 } }, NULL
 	}
 };
 static const RegisterDescriptorType oly3040_reg_71[] = { 
@@ -724,7 +724,7 @@
  *   121 to 240: normal positions
  */
 static const ValueNameType oly3040_reg_103_val_names[] = {
-	{ { range: { 1, 240, 1 } }, NULL },
+	{ { .range = { 1, 240, 1 } }, NULL },
 };
 static const RegisterDescriptorType oly3040_reg_103[] = { 
 	{
@@ -740,7 +740,7 @@
  *      0 to 120: non-macro mode
  */
 static const ValueNameType oly750uz_reg_103_val_names[] = {
-	{ { range: { 1, 120, 1 } }, NULL },
+	{ { .range = { 1, 120, 1 } }, NULL },
 };
 static const RegisterDescriptorType oly750uz_reg_103[] = { 
 	{
diff -ru libgphoto2-2.3.1.orig/libgphoto2_port/ptpip/ptpip.c libgphoto2-2.3.1/libgphoto2_port/ptpip/ptpip.c
--- libgphoto2-2.3.1.orig/libgphoto2_port/ptpip/ptpip.c	2006-11-25 06:47:35.000000000 -0800
+++ libgphoto2-2.3.1/libgphoto2_port/ptpip/ptpip.c	2007-01-27 22:21:12.196618000 -0800
@@ -63,6 +63,7 @@
 #define CHECK(result) {int r=(result); if (r<0) return (r);}
 
 struct _GPPortPrivateLibrary {
+	char dummy;
 };
 
 GPPortType
diff -ru libgphoto2-2.3.1.orig/packaging/generic/print-camera-list.c libgphoto2-2.3.1/packaging/generic/print-camera-list.c
--- libgphoto2-2.3.1.orig/packaging/generic/print-camera-list.c	2006-12-24 07:16:19.000000000 -0800
+++ libgphoto2-2.3.1/packaging/generic/print-camera-list.c	2007-01-27 22:48:23.740508000 -0800
@@ -1004,77 +1004,77 @@
 
 /** list of supported output formats */
 static const output_format_t formats[] = {
-	{name: "human-readable",
-	 descr: "human readable list of cameras",
-	 help: NULL,
-	 paramdescr: NULL,
-	 begin_func: human_begin_func,
-	 camera_func: human_camera_func,
-	 end_func: human_end_func
-	},
-	{name: "usb-usermap",
-	 descr: "usb.usermap include file for linux-hotplug",
-	 help: "If no <scriptname> is given, uses the script name "
+	{.name="human-readable",
+	 .descr="human readable list of cameras",
+	 .help=NULL,
+	 .paramdescr=NULL,
+	 .begin_func=human_begin_func,
+	 .camera_func=human_camera_func,
+	 .end_func=human_end_func
+	},
+	{.name="usb-usermap",
+	 .descr="usb.usermap include file for linux-hotplug",
+	 .help="If no <scriptname> is given, uses the script name "
 	 "\"" GP_USB_HOTPLUG_SCRIPT "\".\n        Put this into /etc/hotplug/usb/<scriptname>.usermap",
-	 paramdescr: "<NAME_OF_HOTPLUG_SCRIPT>",
-	 begin_func: empty_begin_func,
-	 camera_func: hotplug_camera_func,
-	 end_func: empty_end_func
-	},
-	{name: "hal-fdi",
-	 descr: "fdi file for HAL",
-	 help: "Put it into /usr/share/hal/fdi/information/10freedesktop/10-camera-libgphoto2.fdi",
-	 paramdescr: NULL,
-	 begin_func: fdi_begin_func,
-	 camera_func: fdi_camera_func,
-	 end_func: fdi_end_func
-	},
-	{name: "hal-fdi-device",
-	 descr: "fdi device file for HAL",
-	 help: "Put it into /usr/share/hal/fdi/information/10freedesktop/10-camera-libgphoto2-device.fdi",
-	 paramdescr: NULL,
-	 begin_func: fdi_device_begin_func,
-	 camera_func: fdi_device_camera_func,
-	 end_func: fdi_device_end_func
-	},
-	{name: "udev-rules",
-	 descr: "udev < 0.98 rules file",
-	 help: "Put it into /etc/udev/rules.d/90-libgphoto2.rules, set file mode, owner, group\n"
+	 .paramdescr="<NAME_OF_HOTPLUG_SCRIPT>",
+	 .begin_func=empty_begin_func,
+	 .camera_func=hotplug_camera_func,
+	 .end_func=empty_end_func
+	},
+	{.name="hal-fdi",
+	 .descr="fdi file for HAL",
+	 .help="Put it into /usr/share/hal/fdi/information/10freedesktop/10-camera-libgphoto2.fdi",
+	 .paramdescr=NULL,
+	 .begin_func=fdi_begin_func,
+	 .camera_func=fdi_camera_func,
+	 .end_func=fdi_end_func
+	},
+	{.name="hal-fdi-device",
+	 .descr="fdi device file for HAL",
+	 .help="Put it into /usr/share/hal/fdi/information/10freedesktop/10-camera-libgphoto2-device.fdi",
+	 .paramdescr=NULL,
+	 .begin_func=fdi_device_begin_func,
+	 .camera_func=fdi_device_camera_func,
+	 .end_func=fdi_device_end_func
+	},
+	{.name="udev-rules",
+	 .descr="udev < 0.98 rules file",
+	 .help="Put it into /etc/udev/rules.d/90-libgphoto2.rules, set file mode, owner, group\n"
 	 "        or add script to run. This rule files also uses the\n"
 	 "        check-ptp-camera script included in libgphoto2 source. Either put it to\n"
 	 "        /lib/udev/check-ptp-camera or adjust the path in the generated rules file.",
-	 paramdescr: "( <PATH_TO_SCRIPT> | [mode <mode>|owner <owner>|group <group>]* ) ",
-	 begin_func: udev_begin_func, 
-	 camera_func: udev_camera_func,
-	 end_func: udev_end_func
-	},
-	{name: "udev-rules-0.98",
-	 descr: "udev >= 0.98 rules file",
-	 help:
+	 .paramdescr="( <PATH_TO_SCRIPT> | [mode <mode>|owner <owner>|group <group>]* ) ",
+	 .begin_func=udev_begin_func, 
+	 .camera_func=udev_camera_func,
+	 .end_func=udev_end_func
+	},
+	{.name="udev-rules-0.98",
+	 .descr="udev >= 0.98 rules file",
+	 .help=
 	 "Put it into /etc/udev/rules.d/90-libgphoto2.rules, set file mode, owner, group\n"
 	 "        or add script to run, for udev >= 0.98. This rule files also uses the\n"
 	 "        check-ptp-camera script included in libgphoto2 source. Either put it to\n"
 	 "        /lib/udev/check-ptp-camera or adjust the path in the generated rules file.",
-	 paramdescr: "( <PATH_TO_SCRIPT> | [mode <mode>|owner <owner>|group <group>]* ) ",
-	 begin_func: udev_098_begin_func, 
-	 camera_func: udev_098_camera_func,
-	 end_func: udev_end_func
-	},
-	{name: "idlist",
-	 descr: "list of IDs and names",
-	 help: "grep for an ID to find the device name",
-	 paramdescr: NULL,
-	 begin_func: empty_begin_func, 
-	 camera_func: idlist_camera_func,
-	 end_func: empty_end_func
-	},
-	{name: "gp2ddb",
-	 descr: "gphoto2 device database (PRE-ALPHA)",
-	 help: "PRE-ALPHA test stage, do not use for production! Machine parseable.",
-	 paramdescr: NULL,
-	 begin_func: ddb_begin_func,
-	 camera_func: ddb_camera_func,
-	 end_func: ddb_end_func
+	 .paramdescr="( <PATH_TO_SCRIPT> | [mode <mode>|owner <owner>|group <group>]* ) ",
+	 .begin_func=udev_098_begin_func, 
+	 .camera_func=udev_098_camera_func,
+	 .end_func=udev_end_func
+	},
+	{.name="idlist",
+	 .descr="list of IDs and names",
+	 .help="grep for an ID to find the device name",
+	 .paramdescr=NULL,
+	 .begin_func=empty_begin_func, 
+	 .camera_func=idlist_camera_func,
+	 .end_func=empty_end_func
+	},
+	{.name="gp2ddb",
+	 .descr="gphoto2 device database (PRE-ALPHA)",
+	 .help="PRE-ALPHA test stage, do not use for production! Machine parseable.",
+	 .paramdescr=NULL,
+	 .begin_func=ddb_begin_func,
+	 .camera_func=ddb_camera_func,
+	 .end_func=ddb_end_func
 	},
 	{NULL, NULL, NULL, NULL, 
 	 NULL, NULL, NULL}
