From 9da02be4332b0effc5d3f8470867a01c33d39c28 Mon Sep 17 00:00:00 2001
From: Ralph Boehme <sloowfranklin@gmail.com>
Date: Wed, 30 Oct 2013 11:37:56 +0100
Subject: [PATCH 1/4] Disable noinst binary, fails to link on SPARC

---
 etc/spotlight/Makefile.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/etc/spotlight/Makefile.in b/etc/spotlight/Makefile.in
index c126a9e..c03983a 100644
--- a/etc/spotlight/Makefile.in
+++ b/etc/spotlight/Makefile.in
@@ -172,7 +172,7 @@ slmod_sparql_la_LINK = $(LIBTOOL) $(AM_V_lt) --tag=CC \
 	$(LDFLAGS) -o $@
 @HAVE_TRACKER_SPARQL_TRUE@am_slmod_sparql_la_rpath = -rpath \
 @HAVE_TRACKER_SPARQL_TRUE@	$(moduledir)
-@HAVE_TRACKER_SPARQL_TRUE@am__EXEEXT_1 = srp$(EXEEXT)
+@HAVE_TRACKER_SPARQL_TRUE@am__EXEEXT_1 =
 PROGRAMS = $(noinst_PROGRAMS)
 am__srp_SOURCES_DIST = slmod_sparql_map.c slmod_sparql_parser.y \
 	spotlight_rawquery_lexer.l
-- 
2.0.0

