From 1aa56c80e9e1393f15d080228e66b0c5f90fa9c1 Mon Sep 17 00:00:00 2001
From: Ralph Boehme <sloowfranklin@gmail.com>
Date: Thu, 3 Jul 2014 22:21:29 +0200
Subject: [PATCH] Disable dbus start by default

---
 etc/netatalk/netatalk.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/etc/netatalk/netatalk.c b/etc/netatalk/netatalk.c
index 3a470d0..d1cba32 100644
--- a/etc/netatalk/netatalk.c
+++ b/etc/netatalk/netatalk.c
@@ -418,7 +418,7 @@ int main(int argc, char **argv)
         setenv("XDG_CACHE_HOME", _PATH_STATEDIR, 0);
         setenv("TRACKER_USE_LOG_FILES", "1", 0);
 
-        if (atalk_iniparser_getboolean(obj.iniconfig, INISEC_GLOBAL, "start dbus", 1)) {
+        if (atalk_iniparser_getboolean(obj.iniconfig, INISEC_GLOBAL, "start dbus", 0)) {
             dbus_path = atalk_iniparser_getstring(obj.iniconfig, INISEC_GLOBAL, "dbus daemon", DBUS_DAEMON_PATH);
             LOG(log_debug, logtype_default, "DBUS: '%s'", dbus_path);
             if ((dbus_pid = run_process(dbus_path, "--config-file=" _PATH_CONFDIR "dbus-session.conf", NULL)) == NETATALK_SRV_ERROR) {
-- 
2.0.0

