From 0e56bd27a2801a7d7df210262adb2ec61e6dc03f Mon Sep 17 00:00:00 2001
From: Ralph Boehme <sloowfranklin@gmail.com>
Date: Sun, 18 May 2014 07:02:00 +0200
Subject: [PATCH] Disable dbus

---
 etc/netatalk/netatalk.c | 13 +------------
 1 file changed, 1 insertion(+), 12 deletions(-)

diff --git a/etc/netatalk/netatalk.c b/etc/netatalk/netatalk.c
index 640d386..a0e6f80 100644
--- a/etc/netatalk/netatalk.c
+++ b/etc/netatalk/netatalk.c
@@ -411,22 +411,12 @@ int main(int argc, char **argv)
     sigdelset(&blocksigs, SIGHUP);
     sigprocmask(SIG_SETMASK, &blocksigs, NULL);
 
-#ifdef HAVE_TRACKER
     if (obj.options.flags & OPTION_SPOTLIGHT) {
+#ifdef HAVE_TRACKER
         setenv("DBUS_SESSION_BUS_ADDRESS", "unix:path=" _PATH_STATEDIR "spotlight.ipc", 1);
         setenv("XDG_DATA_HOME", _PATH_STATEDIR, 0);
         setenv("XDG_CACHE_HOME", _PATH_STATEDIR, 0);
         setenv("TRACKER_USE_LOG_FILES", "1", 0);
-
-        dbus_path = atalk_iniparser_getstring(obj.iniconfig, INISEC_GLOBAL, "dbus daemon", DBUS_DAEMON_PATH);
-        LOG(log_debug, logtype_default, "DBUS: '%s'", dbus_path);
-        if ((dbus_pid = run_process(dbus_path, "--config-file=" _PATH_CONFDIR "dbus-session.conf", NULL)) == NETATALK_SRV_ERROR) {
-            LOG(log_error, logtype_default, "Error starting '%s'", dbus_path);
-            netatalk_exit(EXITERR_CONF);
-        }
-
-        /* Allow dbus some time to start up */
-        sleep(1);
 #endif
 
 #ifdef HAVE_TRACKER_SPARQL
@@ -437,7 +427,6 @@ int main(int argc, char **argv)
         setenv("TRACKER_LANGUAGE_STOPWORDS_DIR", TRACKER_PREFIX "/share/tracker/languages", 0);
 #endif
         set_sl_volumes();
-        system(TRACKER_PREFIX "/bin/tracker-control -s");
 #endif
     }
 
-- 
1.8.4.1

