From d2d5c9ae7dd66ac2e26fcb5d9ce8b430f645ef25 Mon Sep 17 00:00:00 2001
From: Ralph Boehme <sloowfranklin@gmail.com>
Date: Fri, 14 Mar 2014 18:16:21 +0100
Subject: [PATCH] Ensure gsettings is called from /opt/csw/bin

---
 etc/netatalk/netatalk.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/etc/netatalk/netatalk.c b/etc/netatalk/netatalk.c
index 4a20d7b..4cdc466 100644
--- a/etc/netatalk/netatalk.c
+++ b/etc/netatalk/netatalk.c
@@ -85,13 +85,13 @@ static int set_sl_volumes(void)
     }
 
     volnamelist = bjoin(vollist, sep);
-    cmd = bformat("gsettings set org.freedesktop.Tracker.Miner.Files index-recursive-directories \"[%s]\"",
+    cmd = bformat(TRACKER_PREFIX "/bin/gsettings set org.freedesktop.Tracker.Miner.Files index-recursive-directories \"[%s]\"",
                   bdata(volnamelist) ? bdata(volnamelist) : "");
     LOG(log_debug, logtype_sl, "set_sl_volumes: %s", bdata(cmd));
     system(bdata(cmd));
 
     /* Disable default root user home indexing */
-    system("gsettings set org.freedesktop.Tracker.Miner.Files index-single-directories \"[]\"");
+    system(TRACKER_PREFIX "/bin/gsettings set org.freedesktop.Tracker.Miner.Files index-single-directories \"[]\"");
 
 EC_CLEANUP:
     if (cmd)
-- 
1.8.4.1

