From f1eb5eeb4ad88dbc14b72c1e0d8483abec42013b Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Tue, 14 Dec 2010 11:52:45 +0100
Subject: [PATCH 3/3] Use inttypes instead of stdint

---
 src/bitfiddle.inl |    2 +-
 src/chmfile.h     |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/bitfiddle.inl b/src/bitfiddle.inl
index 3edd779..6bbb2ae 100644
--- a/src/bitfiddle.inl
+++ b/src/bitfiddle.inl
@@ -23,7 +23,7 @@
 
 
 #include <wx/string.h>
-#include <stdint.h>
+#include <inttypes.h>
 
 
 #define FIXENDIAN16(x) (x = wxUINT16_SWAP_ON_BE(x))
diff --git a/src/chmfile.h b/src/chmfile.h
index 4f57290..838f8ec 100644
--- a/src/chmfile.h
+++ b/src/chmfile.h
@@ -28,7 +28,7 @@
 #	include <config.h>
 #endif
 
-#include <stdint.h>
+#include <inttypes.h>
 #include <chm_lib.h>
 #include <wx/filefn.h>
 #include <wx/string.h>
-- 
1.7.3.2

