diff --speed-large-files --minimal -Nru sysstat-20080501.orig/Makefile sysstat-20080501/Makefile
--- sysstat-20080501.orig/Makefile	2008-05-02 19:40:43.000000000 +0200
+++ sysstat-20080501/Makefile	2008-05-13 06:12:48.272325000 +0200
@@ -68,10 +68,10 @@
 	-$(MKDIR) $(PREFIX)/bin
 	$(CP) $(CDIR)/sysstat $(PREFIX)/bin
 	-$(CHOWN) root:bin $(PREFIX)/bin/sysstat
-	-$(MKDIR) $(PREFIX)/man/man1m
-	$(CP) sysstat.1m $(PREFIX)/man/man1m
-	-$(CHOWN) root:bin $(PREFIX)/man/man1m/sysstat.1m
-	-$(CHMOD) 0644 $(PREFIX)/man/man1m/sysstat.1m
+	-$(MKDIR) $(PREFIX)/share/man/man1m
+	$(CP) sysstat.1m $(PREFIX)/share/man/man1m
+	-$(CHOWN) root:bin $(PREFIX)/share/man/man1m/sysstat.1m
+	-$(CHMOD) 0644 $(PREFIX)/share/man/man1m/sysstat.1m
 
 install_sysstatd: $(DDIR)/sysstatd
 	-$(MKDIR) $(PREFIX)/sbin
