From f072fece494256e6f7cc6ff06f79c059f18c9be9 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Wed, 28 Oct 2009 15:35:37 +0100
Subject: [PATCH 2/2] Add missing prototypes for db_sql

---
 include/db_sql.h |    4 +++-
 1 files changed, 3 insertions(+), 1 deletions(-)

diff --git a/include/db_sql.h b/include/db_sql.h
index 0907393..ecb1c41 100644
--- a/include/db_sql.h
+++ b/include/db_sql.h
@@ -22,7 +22,7 @@
 #define _DB_SQL_H_INCLUDED
 
 
-static __attribute__ ((used)) const char* db_sql_types[] = {
+static const char* db_sql_types[] = {
    "text unique primary key",/* "name",   */
    "text", 	         /* "lname",   */
    "int", 	         /* "perm",    */
@@ -51,5 +51,7 @@ static __attribute__ ((used)) const char* db_sql_types[] = {
 
 int db_close_sql(void*);
 db_line* db_readline_sql(int db, db_config* conf);
+int db_writeline_sql(db_line* line,db_config* conf);
+int db_writespec_sql(db_config* conf);
 
 #endif
-- 
1.6.5

