--- a/src/bindings/fluid_rtkit.c
+++ b/src/bindings/fluid_rtkit.c
@@ -290,28 +290,6 @@
         dbus_error_free(&error);
 
         return ret;
-}
-
-#else
-
-int rtkit_make_realtime(DBusConnection *connection, pid_t thread, int priority) {
-        return -ENOTSUP;
-}
-
-int rtkit_make_high_priority(DBusConnection *connection, pid_t thread, int nice_level) {
-        return -ENOTSUP;
-}
-
-int rtkit_get_max_realtime_priority(DBusConnection *connection) {
-        return -ENOTSUP;
-}
-
-int rtkit_get_min_nice_level(DBusConnection *connection, int* min_nice_level) {
-        return -ENOTSUP;
-}
-
-long long rtkit_get_rttime_nsec_max(DBusConnection *connection) {
-        return -ENOTSUP;
 }
 
 #endif
@@ -375,4 +353,31 @@
 	
 }
 
+
+#else
+
+int rtkit_make_realtime(DBusConnection *connection, pid_t thread, int priority) {
+        return -ENOTSUP;
+}
+
+int rtkit_make_high_priority(DBusConnection *connection, pid_t thread, int nice_level) {
+        return -ENOTSUP;
+}
+
+int rtkit_get_max_realtime_priority(DBusConnection *connection) {
+        return -ENOTSUP;
+}
+
+int rtkit_get_min_nice_level(DBusConnection *connection, int* min_nice_level) {
+        return -ENOTSUP;
+}
+
+long long rtkit_get_rttime_nsec_max(DBusConnection *connection) {
+        return -ENOTSUP;
+}
+
+int fluid_rtkit_make_realtime(pid_t thread, int priority) {
+        return -ENOTSUP;
+}
+
 #endif
