From c98a64ed5f18a50d9d249bd4bd0916374272a597 Mon Sep 17 00:00:00 2001
From: Maciej Blizinski <maciej@build8s.(none)>
Date: Sat, 29 Aug 2009 11:20:28 +0200
Subject: [PATCH 1/2] Using gwhoami instead of whoami

---
 wgetpaste |   10 +++++-----
 1 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/wgetpaste b/wgetpaste
index d4addbd..9b40432 100755
--- a/wgetpaste
+++ b/wgetpaste
@@ -99,7 +99,7 @@ notreadable() {
 noxclip() {
 	cat <<EOF >&2
 Could not find xclip on your system. In order to use --x$1 you must either
-emerge x11-misc/xclip or define x_$1() globally in /etc/wgetpaste.conf or
+emerge x11-misc/xclip or define x_$1() globally in /etc/opt/csw/wgetpaste.conf/g or
 per user in ~/.wgetpaste.conf to use another program (such as e.g. xcut or
 klipper) to $2 your clipboard.
 
@@ -288,11 +288,11 @@ Options:
         --debug                   be *very* verbose (implies -v)
 
     -h, --help                    show this help
-    -g, --ignore-configs          ignore /etc/wgetpaste.conf, ~/.wgetpaste.conf etc.
+    -g, --ignore-configs          ignore /etc/opt/csw/wgetpaste.conf/g, ~/.wgetpaste.conf etc.
         --version                 show version information
 
 Defaults (DEFAULT_{NICK,SERVICE,LANGUAGE,EXPIRATION}[_\${SERVICE}]) can be overridden
-globally in /etc/wgetpaste.conf or /etc/wgetpaste.d/*.conf or per user in any of
+globally in /etc/opt/csw/wgetpaste.conf/g or /etc/wgetpaste.d/*.conf or per user in any of
 ~/.wgetpaste.conf or ~/.wgetpaste.d/*.conf.
 EOF
 }
@@ -568,7 +568,7 @@ load_configs() {
 	if [[ ! $IGNORECONFIGS ]]; then
 		# compatibility code
 		local f deprecated=
-		for f in {/etc/,~/.}wgetpaste{.d/*.bash,}; do
+		for f in {/etc/,/etc/opt/csw/,~/.}wgetpaste{.d/*.bash,}; do
 			if [[ -f $f ]]; then
 				if [[ -z $deprecated ]]; then
 					echo "The config files for wgetpaste have changed to *.conf.$N" >&2
@@ -589,7 +589,7 @@ load_configs() {
 }
 load_configs
 [[ $SERVICESET ]] && verifyservice "$SERVICESET" && SERVICE=$(escape "$SERVICESET")
-DEFAULT_NICK=${DEFAULT_NICK:-$(whoami)} || die "whoami failed"
+DEFAULT_NICK=${DEFAULT_NICK:-$(gwhoami)} || die "whoami failed"
 DEFAULT_SERVICE=${DEFAULT_SERVICE:-dpaste}
 DEFAULT_LANGUAGE=${DEFAULT_LANGUAGE:-Plain Text}
 DEFAULT_EXPIRATION=${DEFAULT_EXPIRATION:-1 month}
-- 
1.6.3.2

