--- wgetpaste-2.16/wgetpaste.orig	2010-01-05 14:42:30.127801164 +0100
+++ wgetpaste-2.16/wgetpaste	2010-01-05 14:43:42.007282206 +0100
@@ -607,7 +607,7 @@
 }
 load_configs
 [[ $SERVICESET ]] && verifyservice "$SERVICESET" && SERVICE=$(escape "$SERVICESET")
-DEFAULT_NICK=${DEFAULT_NICK:-$(whoami)} || die "whoami failed"
+DEFAULT_NICK=${DEFAULT_NICK:-$(gwhoami)} || die "gwhoami failed"
 DEFAULT_SERVICE=${DEFAULT_SERVICE:-pocoo}
 DEFAULT_LANGUAGE=${DEFAULT_LANGUAGE:-Plain Text}
 DEFAULT_EXPIRATION=${DEFAULT_EXPIRATION:-1 month}
