From 30f91fb677f8da10b97c60e624c5c3a9173eebff Mon Sep 17 00:00:00 2001
From: Yann Rouillard <yann@pleiades.fr.eu.org>
Date: Thu, 23 Aug 2012 21:50:01 +0200
Subject: [PATCH] Opencsw path addition

---
 bash_completion | 11 +++++++----
 1 file changed, 7 insertions(+), 4 deletions(-)

diff --git a/bash_completion b/bash_completion
index d6ae8ae..4e9a3b5 100644
--- a/bash_completion
+++ b/bash_completion
@@ -40,8 +40,8 @@ fi
 
 # Set the following to the location of the backwards compat completion dir.
 #
-: ${BASH_COMPLETION_COMPAT_DIR:=/etc/bash_completion.d}
-readonly BASH_COMPLETION_COMPAT_DIR
+: ${BASH_COMPLETION_COMPAT_DIR_LIST:=/opt/csw/etc/bash_completion.d /etc/opt/csw/bash_completion.d}
+readonly BASH_COMPLETION_COMPAT_DIR_LIST
 
 # Blacklisted completions, causing problems with our code.
 #
@@ -1429,7 +1429,7 @@ _known_hosts_real()
     if [[ -n $configfile ]]; then
         [[ -r $configfile ]] && config+=( "$configfile" )
     else
-        for i in /etc/ssh/ssh_config ~/.ssh/config ~/.ssh2/config; do
+        for i in /etc/opt/csw/ssh/ssh_config /etc/ssh/ssh_config ~/.ssh/config ~/.ssh2/config; do
             [[ -r $i ]] && config+=( "$i" )
         done
     fi
@@ -1462,7 +1462,8 @@ _known_hosts_real()
 
     if [[ -z $configfile ]]; then
         # Global and user known_hosts files
-        for i in /etc/ssh/ssh_known_hosts /etc/ssh/ssh_known_hosts2 \
+        for i in /etc/opt/csw/ssh/ssh_known_hosts /etc/opt/csw/ssh/ssh_known_hosts2 \
+            /etc/ssh/ssh_known_hosts /etc/ssh/ssh_known_hosts2 \
             /etc/known_hosts /etc/known_hosts2 ~/.ssh/known_hosts \
             ~/.ssh/known_hosts2; do
             [[ -r $i ]] && kh+=( "$i" )
@@ -1947,6 +1948,7 @@ _xfunc()
 }
 
 # source compat completion directory definitions
+for BASH_COMPLETION_COMPAT_DIR in $BASH_COMPLETION_COMPAT_DIR_LIST; do
 if [[ -d $BASH_COMPLETION_COMPAT_DIR && -r $BASH_COMPLETION_COMPAT_DIR && \
     -x $BASH_COMPLETION_COMPAT_DIR ]]; then
     for i in $(LC_ALL=C command ls "$BASH_COMPLETION_COMPAT_DIR"); do
@@ -1955,6 +1957,7 @@ if [[ -d $BASH_COMPLETION_COMPAT_DIR && -r $BASH_COMPLETION_COMPAT_DIR && \
             && -f $i && -r $i ]] && . "$i"
     done
 fi
+done
 unset i _blacklist_glob
 
 # source user completion file
-- 
1.7.11.3

