diff --speed-large-files --minimal -Nru bash-completion-1.1.orig/bash_completion bash-completion-1.1/bash_completion
--- bash-completion-1.1.orig/bash_completion	2009-10-10 23:29:39.177572555 +0200
+++ bash-completion-1.1/bash_completion	2009-10-10 23:29:13.520556369 +0200
@@ -40,8 +40,8 @@
 
 # Alter the following to reflect the location of this file.
 #
-[ -n "$BASH_COMPLETION" ] || BASH_COMPLETION=/etc/bash_completion
-[ -n "$BASH_COMPLETION_DIR" ] || BASH_COMPLETION_DIR=/etc/bash_completion.d
+[ -n "$BASH_COMPLETION" ] || BASH_COMPLETION=/opt/csw/etc/bash_completion
+[ -n "$BASH_COMPLETION_DIR" ] || BASH_COMPLETION_DIR=/opt/csw/etc/bash_completion.d
 [ -n "$BASH_COMPLETION_COMPAT_DIR" ] || BASH_COMPLETION_COMPAT_DIR=/etc/bash_completion.d
 readonly BASH_COMPLETION BASH_COMPLETION_DIR BASH_COMPLETION_COMPAT_DIR
 
@@ -1196,6 +1196,8 @@
         [ -r "$configfile" ] &&
         config=( "${config[@]}" "$configfile" )
     else
+        [ -r /opt/csw/etc/ssh/ssh_config ] &&
+        config=( "${config[@]}" "/opt/csw/etc/ssh/ssh_config" )
         [ -r /etc/ssh/ssh_config ] &&
         config=( "${config[@]}" "/etc/ssh/ssh_config" )
         [ -r "${HOME}/.ssh/config" ] &&
@@ -1223,6 +1225,10 @@
     [ -r "$global_kh" ] &&
     kh=( "${kh[@]}" "${global_kh[@]}" )
     if [ -z "$configfile" ]; then
+        [ -r /opt/csw/etc/ssh/ssh_known_hosts ] &&
+        kh=( "${kh[@]}" /opt/csw/etc/ssh/ssh_known_hosts )
+        [ -r /opt/csw/etc/ssh/ssh_known_hosts2 ] &&
+        kh=( "${kh[@]}" /opt/csw/etc/ssh/ssh_known_hosts2 )
         [ -r /etc/ssh/ssh_known_hosts ] &&
         kh=( "${kh[@]}" /etc/ssh/ssh_known_hosts )
         [ -r /etc/ssh/ssh_known_hosts2 ] &&
