From bbdd76dd83f1e18436b41127f709aa0958753a7d Mon Sep 17 00:00:00 2001
From: builder user <builder@opencsw.org>
Date: Mon, 19 Sep 2011 16:25:15 +0200
Subject: [PATCH] Link against OpenSSL from OpenCSW.

---
 src/Makefile.am |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/src/Makefile.am b/src/Makefile.am
index 136a5cd..00ffb43 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -116,6 +116,8 @@ SUBDIRS += ssl
 SSL_LIBS = \
 	ssl/libsslutil.la \
 	ssl/libsslsquid.la
+XTRA_LIBS += -lssl -lcrypto
+LDFLAGS += -L/opt/csw/lib -R/opt/csw/lib
 else
 SSL_SOURCE = 
 SSL_LIBS = 
-- 
1.7.6

