--- graphviz-2.28.0/configure.old	2011-05-31 13:58:42.522392950 +0200
+++ graphviz-2.28.0/configure	2011-05-31 14:02:05.736756763 +0200
@@ -25270,7 +25270,8 @@
   $as_echo_n "(cached) " >&6
 else
   ac_check_lib_save_LIBS=$LIBS
-LIBS="-lgd $GD_LIBS $LIBS"
+GD_LIBS="$GD_LDFLAGS -lgd $GD_LIBS"
+LIBS="$GD_LIBS $LIBS"
 cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
 
@@ -25296,8 +25297,6 @@
 $as_echo "$ac_cv_lib_gd_main" >&6; }
 if test "x$ac_cv_lib_gd_main" = x""yes; then :
 
-	  GD_LIBS="$GD_LDFLAGS -lgd $GD_LIBS"
-
 cat >>confdefs.h <<_ACEOF
 #define HAVE_LIBGD 1
 _ACEOF
