--- graphviz2/cmd/lefty/ws/x11/libfilereq/Draw.c.orig	2010-01-26 13:27:07.000000000 -0500
+++ graphviz2/cmd/lefty/ws/x11/libfilereq/Draw.c	2010-01-26 13:27:39.000000000 -0500
@@ -1,4 +1,4 @@
-/* $Id: Draw.c,v 1.8 2009/10/13 14:20:29 ellson Exp $ $Revision: 1.8 $ */
+/* $Id: Draw.c,v 1.9 2010/01/26 18:24:35 ellson Exp $ $Revision: 1.9 $ */
 /* vim:set shiftwidth=4 ts=8: */
 
 /**********************************************************
@@ -43,8 +43,16 @@
  *         erik@sra.co.jp
  */
 
+#ifdef HAVE_CONFIG_H
+#include "config.h"
+#endif
 #include <stdio.h>
+#ifdef HAVE_STDINT_H
 #include <stdint.h>
+#endif
+#ifdef HAVE_INTTYPES_H
+#include <inttypes.h>
+#endif
 #include "SFinternal.h"
 #include "xstat.h"
 #include <X11/StringDefs.h>
