diff --speed-large-files --minimal -Nru lftp-3.5.4.orig/lib/argmatch.c lftp-3.5.4/lib/argmatch.c
--- lftp-3.5.4.orig/lib/argmatch.c	2006-07-24 03:54:53.000000000 -0400
+++ lftp-3.5.4/lib/argmatch.c	2006-08-19 14:08:45.545751000 -0400
@@ -27,7 +27,11 @@
 /* Specification.  */
 #include "argmatch.h"
 
-#include <stdbool.h>
+#ifdef HAVE_STDBOOL_H
+# include <stdbool.h>
+#else
+# include "../lib/stdbool.h"
+#endif
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
diff --speed-large-files --minimal -Nru lftp-3.5.4.orig/lib/human.h lftp-3.5.4/lib/human.h
--- lftp-3.5.4.orig/lib/human.h	2006-07-24 03:54:53.000000000 -0400
+++ lftp-3.5.4/lib/human.h	2006-08-19 14:09:22.415324000 -0400
@@ -23,7 +23,11 @@
 # define HUMAN_H_ 1
 
 # include <limits.h>
-# include <stdbool.h>
+# ifdef HAVE_STDBOOL_H
+#  include <stdbool.h>
+# else
+#  include "../lib/stdbool.h"
+# endif
 
 # if HAVE_STDINT_H
 #  include <stdint.h>
diff --speed-large-files --minimal -Nru lftp-3.5.4.orig/lib/quotearg.c lftp-3.5.4/lib/quotearg.c
--- lftp-3.5.4.orig/lib/quotearg.c	2006-07-24 03:54:53.000000000 -0400
+++ lftp-3.5.4/lib/quotearg.c	2006-08-19 14:08:45.545777000 -0400
@@ -30,7 +30,11 @@
 #include <ctype.h>
 #include <errno.h>
 #include <limits.h>
-#include <stdbool.h>
+#ifdef HAVE_STDBOOL_H
+# include <stdbool.h>
+#else
+# include "../lib/stdbool.h"
+#endif
 #include <stdlib.h>
 #include <string.h>
 
diff --speed-large-files --minimal -Nru lftp-3.5.4.orig/lib/stdbool_.h lftp-3.5.4/lib/stdbool_.h
--- lftp-3.5.4.orig/lib/stdbool_.h	2006-07-24 03:54:53.000000000 -0400
+++ lftp-3.5.4/lib/stdbool_.h	2006-08-19 14:08:45.545792000 -0400
@@ -74,7 +74,7 @@
   /* A compiler known to have 'bool'.  */
   /* If the compiler already has both 'bool' and '_Bool', we can assume they
      are the same types.  */
-# if !@HAVE__BOOL@
+# if @HAVE__BOOL@
 typedef bool _Bool;
 # endif
 #else
@@ -100,7 +100,7 @@
 enum { false = 0, true = 1 };
 # else
    /* With this compiler, trust the _Bool type if the compiler has it.  */
-#  if !@HAVE__BOOL@
+#  if @HAVE__BOOL@
 typedef enum { _Bool_must_promote_to_int = -1, false = 0, true = 1 } _Bool;
 #  endif
 # endif
