--- subversion-1.5.5/Makefile.in.orig	2008-08-26 12:27:56.000000000 -0500
+++ subversion-1.5.5/Makefile.in	2009-02-21 17:26:25.849467074 -0600
@@ -68,15 +69,15 @@
 swig_rb_libdir = @libdir@
 
 ### these possibly need further discussion
-swig_pydir = @libdir@/svn-python/libsvn
-swig_pydir_extra = @libdir@/svn-python/svn
-swig_pldir = @libdir@/svn-perl
+swig_pydir = @prefix@/lib/python/site-packages/libsvn
+swig_pydir_extra = @prefix@/lib/python/site-packages/svn
+#swig_pldir = @libdir@/svn-perl
 swig_rbdir = $(SWIG_RB_SITE_ARCH_DIR)/svn/ext
 contribdir = @bindir@/svn-contrib
 toolsdir = @bindir@/svn-tools
 
-javahl_javadir = @libdir@/svn-javahl
-javahl_javahdir = @libdir@/svn-javahl/include
+javahl_javadir = @libdir@
+javahl_javahdir = @libdir@/include
 javahl_libdir = @libdir@
 javahl_test_rootdir=$(abs_builddir)/subversion/bindings/javahl/test-work
 
--- subversion-1.5.5/subversion/bindings/swig/python/svn/fs.py.orig	2009-02-21 17:00:06.832094681 -0600
+++ subversion-1.5.5/subversion/bindings/swig/python/svn/fs.py	2009-02-21 16:59:46.198583440 -0600
@@ -97,7 +97,7 @@
 
     # use an array for the command to avoid the shell and potential
     # security exposures
-    cmd = ["diff"] \
+    cmd = ["gdiff"] \
           + self.diffoptions \
           + [self.tempfile1, self.tempfile2]
