From a14f0984beedd1b816a1e1c7365180b89716a672 Mon Sep 17 00:00:00 2001
From: Rafael Ostertag <rafi@guengel.ch>
Date: Sun, 29 Sep 2013 15:41:42 +0200
Subject: [PATCH] ecpg /usr/local/include replacement

---
 src/interfaces/ecpg/preproc/ecpg.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/interfaces/ecpg/preproc/ecpg.c b/src/interfaces/ecpg/preproc/ecpg.c
index 7b2634d..2befa06 100644
--- a/src/interfaces/ecpg/preproc/ecpg.c
+++ b/src/interfaces/ecpg/preproc/ecpg.c
@@ -253,7 +253,7 @@ main(int argc, char *const argv[])
 	}
 
 	add_include_path(".");
-	add_include_path("/usr/local/include");
+	add_include_path("/opt/csw/include");
 	get_include_path(my_exec_path, include_path);
 	add_include_path(include_path);
 	add_include_path("/usr/include");
-- 
1.8.3.4

