From 158e154cada106c38691140b578f7286be0aa387 Mon Sep 17 00:00:00 2001
From: Rafael Ostertag <rafi@guengel.ch>
Date: Sat, 5 Oct 2013 09:42:10 +0200
Subject: [PATCH] Use plain rpath

---
 src/makefiles/Makefile.solaris | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/makefiles/Makefile.solaris b/src/makefiles/Makefile.solaris
index e459de3..a234c30 100644
--- a/src/makefiles/Makefile.solaris
+++ b/src/makefiles/Makefile.solaris
@@ -6,7 +6,7 @@ ifeq ($(with_gnu_ld), yes)
 export_dynamic = -Wl,-E
 rpath = -Wl,-rpath,'$(rpathdir)'
 else
-rpath = -Wl,-R'$(rpathdir)'
+rpath = -R'$(rpathdir)'
 endif
 
 DLSUFFIX = .so
-- 
1.8.3.4

