From 3d38c6e422081129b880bf5bdd7ff8a7c370025a Mon Sep 17 00:00:00 2001
From: Rafael Ostertag <rafisol@opencsw.org>
Date: Sun, 16 Sep 2012 11:01:47 +0200
Subject: [PATCH 1/2] ecpg /usr/local/include replacement

---
 src/interfaces/ecpg/preproc/ecpg.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/interfaces/ecpg/preproc/ecpg.c b/src/interfaces/ecpg/preproc/ecpg.c
index 7e7bae3..2e9b28c 100644
--- a/src/interfaces/ecpg/preproc/ecpg.c
+++ b/src/interfaces/ecpg/preproc/ecpg.c
@@ -253,7 +253,7 @@ main(int argc, char *const argv[])
 	}
 
 	add_include_path(".");
-	add_include_path("/usr/local/include");
+	add_include_path("/opt/csw/include");
 	get_include_path(my_exec_path, include_path);
 	add_include_path(include_path);
 	add_include_path("/usr/include");
-- 
1.7.12

