From 5424924f8c92878e914f5ec5c9bfd00915cf439a Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Wed, 24 Feb 2010 14:58:17 +0100
Subject: [PATCH] Do not force inclusion of stdint.h

---
 src/common/sysdefs.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/common/sysdefs.h b/src/common/sysdefs.h
index fbc9176..f413863 100644
--- a/src/common/sysdefs.h
+++ b/src/common/sysdefs.h
@@ -34,7 +34,7 @@
 // C99 says that inttypes.h always includes stdint.h, but some systems
 // don't do that, and require including stdint.h separately.
 #ifdef HAVE_STDINT_H
-#	include <stdint.h>
+/* #	include <stdint.h> */
 #endif
 
 // Some pre-C99 systems have SIZE_MAX in limits.h instead of stdint.h. The
-- 
1.6.6

