diff --speed-large-files --minimal -Nru mod_transform-0.6.0.orig/src/Makefile.am mod_transform-0.6.0/src/Makefile.am
--- mod_transform-0.6.0.orig/src/Makefile.am	2005-06-02 13:00:32.598277000 -0700
+++ mod_transform-0.6.0/src/Makefile.am	2005-06-02 13:00:22.425212000 -0700
@@ -1,7 +1,7 @@
 CLEANFILES = .libs/libmod_transform *~
 
 libmod_transform_la_SOURCES = mod_transform.c transform_io.c transform_cache.c
-libmod_transform_la_CFLAGS = -Wall ${XSLT_CFLAGS} ${MODULE_CFLAGS}
+libmod_transform_la_CFLAGS = ${XSLT_CFLAGS} ${MODULE_CFLAGS}
 libmod_transform_la_LDFLAGS =  -rpath ${AP_LIBEXECDIR} -module -avoid-version ${XSLT_LIBS} -lexslt
 
 lib_LTLIBRARIES = libmod_transform.la 
diff --speed-large-files --minimal -Nru mod_transform-0.6.0.orig/src/Makefile.in mod_transform-0.6.0/src/Makefile.in
--- mod_transform-0.6.0.orig/src/Makefile.in	2005-06-02 13:00:32.600351000 -0700
+++ mod_transform-0.6.0/src/Makefile.in	2005-06-02 13:00:14.682611000 -0700
@@ -202,7 +202,7 @@
 target_vendor = @target_vendor@
 CLEANFILES = .libs/libmod_transform *~
 libmod_transform_la_SOURCES = mod_transform.c transform_io.c transform_cache.c
-libmod_transform_la_CFLAGS = -Wall ${XSLT_CFLAGS} ${MODULE_CFLAGS}
+libmod_transform_la_CFLAGS = ${XSLT_CFLAGS} ${MODULE_CFLAGS}
 libmod_transform_la_LDFLAGS = -rpath ${AP_LIBEXECDIR} -module -avoid-version ${XSLT_LIBS} -lexslt
 lib_LTLIBRARIES = libmod_transform.la 
 all: all-am
