diff --speed-large-files --minimal -Nru ruby-1.8.2.orig/ext/digest/md5/md5ossl.h ruby-1.8.2/ext/digest/md5/md5ossl.h
--- ruby-1.8.2.orig/ext/digest/md5/md5ossl.h	2005-09-06 12:39:32.761721000 -0400
+++ ruby-1.8.2/ext/digest/md5/md5ossl.h	2005-09-06 12:38:33.103186000 -0400
@@ -3,6 +3,7 @@
 #ifndef MD5OSSL_H_INCLUDED
 #define MD5OSSL_H_INCLUDED
 
+#include <sys/types.h>
 #include <openssl/md5.h>
 
 void MD5_End(MD5_CTX *pctx, unsigned char *hexdigest);
diff --speed-large-files --minimal -Nru ruby-1.8.2.orig/ext/digest/rmd160/rmd160ossl.h ruby-1.8.2/ext/digest/rmd160/rmd160ossl.h
--- ruby-1.8.2.orig/ext/digest/rmd160/rmd160ossl.h	2005-09-06 12:39:33.256387000 -0400
+++ ruby-1.8.2/ext/digest/rmd160/rmd160ossl.h	2005-09-06 12:38:54.414945000 -0400
@@ -3,6 +3,7 @@
 #ifndef RMD160OSSL_H_INCLUDED
 #define RMD160OSSL_H_INCLUDED
 
+#include <sys/types.h>
 #include <openssl/ripemd.h>
 
 #define RMD160_CTX	RIPEMD160_CTX
diff --speed-large-files --minimal -Nru ruby-1.8.2.orig/ext/digest/sha1/sha1ossl.h ruby-1.8.2/ext/digest/sha1/sha1ossl.h
--- ruby-1.8.2.orig/ext/digest/sha1/sha1ossl.h	2005-09-06 12:39:33.485422000 -0400
+++ ruby-1.8.2/ext/digest/sha1/sha1ossl.h	2005-09-06 12:39:11.155582000 -0400
@@ -3,6 +3,7 @@
 #ifndef SHA1OSSL_H_INCLUDED
 #define SHA1OSSL_H_INCLUDED
 
+#include <sys/types.h>
 #include <openssl/sha.h>
 
 #define SHA1_CTX	SHA_CTX
diff --speed-large-files --minimal -Nru ruby-1.8.2.orig/ext/openssl/ossl_x509store.c ruby-1.8.2/ext/openssl/ossl_x509store.c
--- ruby-1.8.2.orig/ext/openssl/ossl_x509store.c	2005-09-06 12:39:38.454612000 -0400
+++ ruby-1.8.2/ext/openssl/ossl_x509store.c	2005-09-06 12:38:02.395604000 -0400
@@ -538,7 +538,11 @@
 
     if(NIL_P(time)) {
 	GetX509StCtx(self, store);
+#if OPENSSL_VERSION_NUMBER >= 0x0090800fL
+	store->param->flags &= ~X509_V_FLAG_USE_CHECK_TIME;
+#else
 	store->flags &= ~X509_V_FLAG_USE_CHECK_TIME;
+#endif
     }
     else {
 	long t = NUM2LONG(rb_Integer(time));
