From fdd0d9cafc923f2a4eab29f8a1a1f3b76867dbae Mon Sep 17 00:00:00 2001
From: Maciej Blizinski <maciej@opencsw.org>
Date: Thu, 6 Oct 2011 00:27:44 +0200
Subject: [PATCH 2/2] versioned datadir in mysql_install_db 5.0

---
 scripts/mysql_install_db.sh |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/scripts/mysql_install_db.sh b/scripts/mysql_install_db.sh
index cca6244..1cdaa0d 100644
--- a/scripts/mysql_install_db.sh
+++ b/scripts/mysql_install_db.sh
@@ -217,7 +217,7 @@ else
   # We set up bootstrap-specific paths later, so skip this for now
   if test "$cross_bootstrap" -eq 0
   then
-    pkgdatadir=`find_in_basedir --dir fill_help_tables.sql share share/mysql`
+    pkgdatadir=`find_in_basedir --dir fill_help_tables.sql share share/mysql share/mysql/5.0/mysql`
     if test -z "$pkgdatadir"
     then
       missing_in_basedir fill_help_tables.sql
-- 
1.7.6.1

