--- a/configure.ac
+++ b/configure.ac
@@ -361,19 +361,19 @@ CFLAGS="$CFLAGS $extra_warnings"
 GIMP_DETECT_CFLAGS(extra_warnings, '-Wformat-non-literal')
 CFLAGS="$CFLAGS $extra_warnings"
 
-GIMP_DETECT_CFLAGS(extra_warnings, '-fno-common')
-CFLAGS="$CFLAGS $extra_warnings"
+# GIMP_DETECT_CFLAGS(extra_warnings, '-fno-common')
+# CFLAGS="$CFLAGS $extra_warnings"
 
-GIMP_DETECT_CFLAGS(extra_warnings, '-fdiagnostics-show-option')
-CFLAGS="$CFLAGS $extra_warnings"
+# GIMP_DETECT_CFLAGS(extra_warnings, '-fdiagnostics-show-option')
+# CFLAGS="$CFLAGS $extra_warnings"
 
 
 # Extra optimization flags for super performance
-GIMP_DETECT_CFLAGS(extra_warnings, '--omg-optimized')
-CFLAGS="$CFLAGS $extra_warnings"
-
-GIMP_DETECT_CFLAGS(extra_warnings, '--disable-instructions')
-CFLAGS="$CFLAGS $extra_warnings"
+# GIMP_DETECT_CFLAGS(extra_warnings, '--omg-optimized')
+# CFLAGS="$CFLAGS $extra_warnings"
+# 
+# GIMP_DETECT_CFLAGS(extra_warnings, '--disable-instructions')
+# CFLAGS="$CFLAGS $extra_warnings"
 
 
 # Ensure MSVC-compatible struct packing convention is used when
@@ -1801,8 +1801,6 @@ if test "x$enable_python" != xno; then
   AC_SUBST(PYGTK_DEFSDIR)
   AC_MSG_RESULT($PYGTK_DEFSDIR)
 
-  GIMP_DETECT_CFLAGS(PYGIMP_EXTRA_CFLAGS, '-fno-strict-aliasing')
-  AC_SUBST(PYGIMP_EXTRA_CFLAGS)
 fi
 AM_CONDITIONAL(BUILD_PYTHON, test "x$enable_python" != xno)
 
