--- c3270-3.3/configure.in-orig	Sat Jan  1 09:02:25 2005
+++ c3270-3.3/configure.in	Wed Oct 11 21:13:00 2006
@@ -85,7 +85,7 @@
 AC_CHECK_HEADERS(libutil.h)
 AC_CHECK_HEADERS(util.h)
 AC_CHECK_HEADERS(getopt.h)
-AC_CHECK_HEADERS(ncurses.h, , [AC_CHECK_HEADERS(curses.h, , [AC_MSG_ERROR(Can't find ncurses.h or curses.h)])])
+AC_CHECK_HEADERS(ncurses.h ncurses/ncurses.h, , [AC_CHECK_HEADERS(curses.h, , [AC_MSG_ERROR(Can't find ncurses.h or curses.h)])])
 
 dnl Check for libncurses/ncurses.h inconsistency
 if test $ac_cv_lib_ncurses_newterm != $ac_cv_header_ncurses_h
--- c3270-3.3/conf.h.in-orig	Tue Dec 24 16:14:31 2002
+++ c3270-3.3/conf.h.in	Tue Oct 10 23:35:16 2006
@@ -28,6 +28,7 @@
 #undef HAVE_LIBSSL
 
 /* Header files. */
+#undef HAVE_NCURSES_NCURSES_H
 #undef HAVE_NCURSES_H
 #undef HAVE_CURSES_H
 #undef HAVE_SYS_SELECT_H
--- c3270-3.3/configure-orig	Sat Jan  1 09:02:45 2005
+++ c3270-3.3/configure	Tue Oct 10 23:24:18 2006
@@ -3653,7 +3653,7 @@
 done
 
 
-for ac_header in ncurses.h
+for ac_header in ncurses/ncurses.h
 do
 as_ac_Header=`echo "ac_cv_header_$ac_header" | $as_tr_sh`
 if eval "test \"\${$as_ac_Header+set}\" = set"; then
--- c3270-3.3/keymap.c-orig	Thu Apr 10 10:24:01 2003
+++ c3270-3.3/keymap.c	Tue Oct 10 23:29:24 2006
@@ -40,7 +40,9 @@
 #undef COLOR_YELLOW
 #undef COLOR_BLUE   
 #undef COLOR_WHITE
-#if defined(HAVE_NCURSES_H) /*[*/
+#if defined(HAVE_NCURSES_NCURSES_H)
+#include <ncurses/ncurses.h>
+#elif defined(HAVE_NCURSES_H) /*[*/
 #include <ncurses.h>
 #else /*][*/
 #include <curses.h>
--- c3270-3.3/screen.c-orig	Mon Apr  4 14:34:58 2005
+++ c3270-3.3/screen.c	Tue Oct 10 23:29:30 2006
@@ -46,7 +46,9 @@
 #undef COLOR_YELLOW
 #undef COLOR_BLUE
 #undef COLOR_WHITE
-#if defined(HAVE_NCURSES_H) /*[*/
+#if defined(HAVE_NCURSES_NCURSES_H)
+#include <ncurses/ncurses.h>
+#elif defined(HAVE_NCURSES_H) /*[*/
 #include <ncurses.h>
 #else /*][*/ 
 #include <curses.h>
