From 024f67d496160ea6fe8b60b67f4832fed04fd199 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Thu, 7 Jul 2011 17:14:06 +0200
Subject: [PATCH] Be not that strict on standard

---
 awk.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/awk.h b/awk.h
index 25abf41..5034a46 100644
--- a/awk.h
+++ b/awk.h
@@ -60,7 +60,7 @@
 #endif /* LOCALEDIR */
 #endif
 
-#if !defined(__STDC__) || __STDC__ < 1
+#if (!defined(__STDC__) || __STDC__ < 1) && !defined(__EXTENSIONS__)
 #error "gawk no longer supports non-C89 environments (no __STDC__ or __STDC__ < 1)"
 #endif
 
-- 
1.7.6

