From 83506df0b040989eda490611936319f6c9e80cf3 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Fri, 20 Sep 2013 21:10:21 +0200
Subject: [PATCH] Check for explicit value or HAVE_UUID_UNPARSE_LOWER will
 always be set

---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index e12b6a0..6542546 100644
--- a/configure.ac
+++ b/configure.ac
@@ -721,7 +721,7 @@ case $enable_backend_chert$enable_backend_brass in
   dnl Older versions of libuuid (such as that on CentOS 4.7) don't have
   dnl uuid_unparse_lower(), only uuid_unparse().
   AC_LINK_IFELSE([AC_LANG_CALL([], [uuid_unparse_lower])], [found=true], [found=false])
-  if $found ; then
+  if test $found = "true"; then
     AC_DEFINE([HAVE_UUID_UNPARSE_LOWER], [1],
 	      [Define to 1 if you have the 'uuid_unparse_lower' function.])
   fi
-- 
1.8.3.4

