From 3e21ed4cad3df8d433c568cee1f040a7d076b6c3 Mon Sep 17 00:00:00 2001
From: Yann Rouillard <yann@pleiades.fr.eu.org>
Date: Fri, 17 May 2013 23:13:48 +0200
Subject: [PATCH] correctly handle the null key case

---
 src/certtool.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/certtool.c b/src/certtool.c
index 832ca53..a4b97da 100644
--- a/src/certtool.c
+++ b/src/certtool.c
@@ -2626,7 +2626,8 @@ print_bag_data (gnutls_pkcs12_bag_t bag)
       result = gnutls_pkcs12_bag_get_key_id (bag, i, &id);
       if (result < 0)
         error (EXIT_FAILURE, 0, "get_key_id: %s", gnutls_strerror (type));
-      fprintf (outfile, "\tKey ID: %s\n", raw_to_string (id.data, id.size));
+      if (id.size > 0)
+        fprintf (outfile, "\tKey ID: %s\n", raw_to_string (id.data, id.size));
 
       result = gnutls_pkcs12_bag_get_data (bag, i, &cdata);
       if (result < 0)
-- 
1.8.1.4

