From f3bb6835113ec86c74ef645cc45b6794637b9886 Mon Sep 17 00:00:00 2001
From: Yann Rouillard <yann@pleiades.fr.eu.org>
Date: Mon, 20 May 2013 21:21:09 +0200
Subject: [PATCH] workaround certificate expiration issue in the test suite

---
 tests/chainverify.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tests/chainverify.c b/tests/chainverify.c
index 151602e..6d90aea 100644
--- a/tests/chainverify.c
+++ b/tests/chainverify.c
@@ -738,12 +738,12 @@ static struct
   { "cacertrsamd5 fail", cacertrsamd5, &cacertrsamd5[2],
     0, GNUTLS_CERT_INSECURE_ALGORITHM | GNUTLS_CERT_INVALID },
   { "cacertrsamd5 ok", cacertrsamd5, &cacertrsamd5[2],
-    GNUTLS_VERIFY_ALLOW_SIGN_RSA_MD5, 0 },
+    GNUTLS_VERIFY_ALLOW_SIGN_RSA_MD5, GNUTLS_CERT_EXPIRED | GNUTLS_CERT_INVALID },
   { "cacertrsamd5 short-cut not ok", cacertrsamd5, &cacertrsamd5[0],
     GNUTLS_VERIFY_DO_NOT_ALLOW_SAME,
     GNUTLS_CERT_SIGNER_NOT_FOUND | GNUTLS_CERT_INVALID },
   { "cacertrsamd5 short-cut ok", cacertrsamd5, &cacertrsamd5[1],
-    0, 0 },
+    0, GNUTLS_CERT_EXPIRED | GNUTLS_CERT_INVALID },
   { NULL, NULL, NULL, 0, 0}
 };
 /* *INDENT-ON* */
-- 
1.8.1.4

