From 9f709dbe1015a11c96b75e328176dc0af1eb1946 Mon Sep 17 00:00:00 2001
From: Peter Bonivart <bonivart@opencsw.org>
Date: Wed, 13 Jul 2011 19:31:16 +0200
Subject: [PATCH] solaris-9-uses-inttypes.h-instead-of-stdint.h

---
 unix/unix.h |   11 ++++++++++-
 1 files changed, 10 insertions(+), 1 deletions(-)

diff --git a/unix/unix.h b/unix/unix.h
index 74f4174..abea183 100644
--- a/unix/unix.h
+++ b/unix/unix.h
@@ -6,7 +6,16 @@
 #endif
 
 #include <stdio.h>		       /* for FILENAME_MAX */
-#include <stdint.h>		       /* C99 int types */
+
+/* C99 int types, Solaris uses inttypes.h */
+#ifdef HAVE_STDINT_H
+#  include <stdint.h>
+#  else
+#  ifdef HAVE_INTTYPES_H
+#    include <inttypes.h>
+#  endif
+#endif
+
 #ifndef NO_LIBDL
 #include <dlfcn.h>		       /* Dynamic library loading */
 #endif /*  NO_LIBDL */
-- 
1.7.6

