From caa3df9aa2cf9a4e6f7defa95960a95fc76ce826 Mon Sep 17 00:00:00 2001
From: Maciej Blizinski <maciej@opencsw.org>
Date: Thu, 20 May 2010 12:24:12 +0200
Subject: [PATCH] Forcing -lintl on cxpm

---
 cxpm/Makefile.in |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/cxpm/Makefile.in b/cxpm/Makefile.in
index 1ca210f..f830ed7 100644
--- a/cxpm/Makefile.in
+++ b/cxpm/Makefile.in
@@ -52,7 +52,7 @@ am__installdirs = "$(DESTDIR)$(bindir)" "$(DESTDIR)$(appmandir)"
 PROGRAMS = $(bin_PROGRAMS)
 am_cxpm_OBJECTS = cxpm.$(OBJEXT)
 cxpm_OBJECTS = $(am_cxpm_OBJECTS)
-cxpm_LDADD = $(LDADD)
+cxpm_LDADD = $(LDADD) -lintl
 AM_V_lt = $(am__v_lt_$(V))
 am__v_lt_ = $(am__v_lt_$(AM_DEFAULT_VERBOSITY))
 am__v_lt_0 = --silent
-- 
1.7.1

