--- clearsilver-0.10.5/python/Makefile.orig	2006-07-13 01:51:49.000000000 -0500
+++ clearsilver-0.10.5/python/Makefile	2009-02-13 10:07:56.503478088 -0600
@@ -11,6 +11,7 @@
 NEO_UTIL_SRC = neo_cgi.c neo_util.c neo_cs.c
 NEO_UTIL_OBJ = $(NEO_UTIL_SRC:%.c=%.o)
 
+CC = cc
 CFLAGS += $(PYTHON_INC)
 DLIBS += -lneo_cgi -lneo_cs -lneo_utl 
 LIBS += $(DLIBS) $(DB2_LIB)
@@ -25,7 +26,7 @@
 
 $(NEO_UTIL_SO): setup.py $(NEO_UTIL_SRC) $(DEP_LIBS)
 	rm -f $(NEO_UTIL_SO)
-	CC="$(CC)" LDSHARED="$(LDSHARED)" $(PYTHON) setup.py build_ext --inplace
+	CC="$(CC)" CCSHARED="$(CC) -G" LDSHARED="$(CC) -G" $(PYTHON) setup.py build_ext --inplace
 
 OLD_NEO_UTIL_SO:
 	$(LDSHARED) -o $@ $(LDFLAGS) $(NEO_UTIL_OBJ) $(LIBS)
--- clearsilver-0.10.5/python/setup.py.orig	2009-02-13 10:04:10.707737561 -0600
+++ clearsilver-0.10.5/python/setup.py	2009-02-13 10:04:19.707153679 -0600
@@ -17,8 +17,8 @@
 INC_DIRS = ["../"]
 LIBRARIES = ["neo_cgi", "neo_cs", "neo_utl"]
 LIB_DIRS = ["../libs"]
-CC = "gcc"
-LDSHARED = "gcc -shared"
+CC = "cc"
+LDSHARED = "cc -shared"
 
 ## ARGGH!!  It looks like you can only specify a single item on the
 ## command-line or in the setup.cfg file for options which take multiple
