From f7b3397155b616637aa0b8ca51c78fa62b9a302b Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Thu, 16 Sep 2010 15:56:10 +0200
Subject: [PATCH] Add INADDR_NONE for Solaris

---
 src/assuan-socket-connect.c |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/src/assuan-socket-connect.c b/src/assuan-socket-connect.c
index 13b8aa5..8d5a09e 100644
--- a/src/assuan-socket-connect.c
+++ b/src/assuan-socket-connect.c
@@ -51,6 +51,9 @@
 #ifndef AF_LOCAL
 # define AF_LOCAL AF_UNIX
 #endif
+#ifndef INADDR_NONE
+# define INADDR_NONE ((unsigned long) -1)
+#endif
 
 #ifndef SUN_LEN
 # define SUN_LEN(ptr) ((size_t) (((struct sockaddr_un *) 0)->sun_path) \
-- 
1.7.1

