From 620ec77ec8e909bd492005b29762038668e3d77f Mon Sep 17 00:00:00 2001
From: Yann Rouillard <yann@pleiades.fr.eu.org>
Date: Tue, 23 Nov 2010 21:21:40 +0100
Subject: [PATCH] add /opt/csw/bin in the hardcoded paths used for auto-selecting the EDITOR variable.

---
 support/bashbug.sh |   28 ++++++++--------------------
 1 files changed, 8 insertions(+), 20 deletions(-)

diff --git a/support/bashbug.sh b/support/bashbug.sh
index 7b36d40..6dfd9bb 100644
--- a/support/bashbug.sh
+++ b/support/bashbug.sh
@@ -35,7 +35,7 @@ PATCHLEVEL="!PATCHLEVEL!"
 RELSTATUS="!RELSTATUS!"
 MACHTYPE="!MACHTYPE!"
 
-PATH=/bin:/usr/bin:/usr/local/bin:$PATH
+PATH=/opt/csw/bin:/bin:/usr/bin:/usr/local/bin:$PATH
 export PATH
 
 # Check if TMPDIR is set, default to /tmp
@@ -120,25 +120,13 @@ esac
 BUGADDR="${1-$BUGBASH}"
 
 if [ -z "$DEFEDITOR" ] && [ -z "$EDITOR" ]; then
-	if [ -x /usr/bin/editor ]; then
-		DEFEDITOR=editor
-	elif [ -x /usr/local/bin/ce ]; then
-		DEFEDITOR=ce
-	elif [ -x /usr/local/bin/emacs ]; then
-		DEFEDITOR=emacs
-	elif [ -x /usr/contrib/bin/emacs ]; then
-		DEFEDITOR=emacs
-	elif [ -x /usr/bin/emacs ]; then
-		DEFEDITOR=emacs
-	elif [ -x /usr/bin/xemacs ]; then
-		DEFEDITOR=xemacs
-	elif [ -x /usr/contrib/bin/jove ]; then
-		DEFEDITOR=jove
-	elif [ -x /usr/local/bin/jove ]; then
-		DEFEDITOR=jove
-	elif [ -x /usr/bin/vi ]; then
-		DEFEDITOR=vi
-	else
+	for E in editor ce emacs xemacs jove vi; do
+		if [ -x /opt/csw/bin/$E ] || [ -x /usr/bin/$E ] || [ -x /usr/local/bin/$E ] || [ -x /usr/contrib/bin/$E ]; then
+			DEFEDITOR=$E
+			break
+		fi
+	done	
+	if [ -z "$DEFEDITOR" ]; then
 		echo "$0: No default editor found: attempting to use vi" >&2
 		DEFEDITOR=vi
 	fi
-- 
1.7.3

