*** ../bash-4.1/builtins/printf.def	2009-11-20 15:31:23.000000000 -0500
--- builtins/printf.def	2010-01-07 08:50:06.000000000 -0500
***************
*** 173,177 ****
  
  #if !HAVE_VSNPRINTF
! extern int vsnprintf __P((char *, size_t, const char *, ...)) __attribute__((__format__ (printf, 3, 4)));
  #endif
  
--- 173,177 ----
  
  #if !HAVE_VSNPRINTF
! extern int vsnprintf __P((char *, size_t, const char *, va_list)) __attribute__((__format__ (printf, 3, 0)));
  #endif
  
