From e8821311d76859cc404718bfe4fd08947964ca63 Mon Sep 17 00:00:00 2001
From: Jake Goerzen <jgoerzen@opencsw.org>
Date: Tue, 28 May 2013 00:35:19 +0200
Subject: [PATCH] add stdio.h to quota-status.c

---
 src/plugins/quota/quota-status.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/plugins/quota/quota-status.c b/src/plugins/quota/quota-status.c
index 80268eb..4ef3c9a 100644
--- a/src/plugins/quota/quota-status.c
+++ b/src/plugins/quota/quota-status.c
@@ -11,6 +11,7 @@
 #include "mail-storage-service.h"
 #include "quota-private.h"
 #include "quota-plugin.h"
+#include <stdio.h>
 
 enum quota_protocol {
 	QUOTA_PROTOCOL_UNKNOWN = 0,
-- 
1.8.1.4

