--- ghostscript-8.70.orig/jpeg/jcinit.c	2010-01-09 03:15:02.698411311 +0100
+++ ghostscript-8.70/jpeg/jcinit.c	2010-01-09 03:46:28.021317069 +0100
@@ -42,7 +42,11 @@
   jinit_forward_dct(cinfo);
   /* Entropy encoding: either Huffman or arithmetic coding. */
   if (cinfo->arith_code) {
+#ifdef JERR_ARITH_NOTIMPL
     ERREXIT(cinfo, JERR_ARITH_NOTIMPL);
+#else
+		jinit_arith_encoder(cinfo);
+#endif
   } else {
     if (cinfo->progressive_mode) {
 #ifdef C_PROGRESSIVE_SUPPORTED
