diff -Nru ghostscript-8.70.orig/base/gs.mak ghostscript-8.70/base/gs.mak
--- ghostscript-8.70.orig/base/gs.mak	2009-01-27 21:43:37.000000000 +0100
+++ ghostscript-8.70/base/gs.mak	2010-01-10 23:24:54.511788159 +0100
@@ -363,7 +363,7 @@
 FEATURE_DEVS_EXTRA=
 DEVICE_DEVS_EXTRA=
 
-DEVS_ALL=$(GLGENDIR)$(D)$(PLATFORM).dev\
+DEVS_ALL=$(GLGENDIR)$(D)$(BUILD_PLATFORM).dev\
  $(FEATURE_DEVS) $(FEATURE_DEVS_EXTRA) \
  $(DEVICE_DEVS) $(DEVICE_DEVS1) \
  $(DEVICE_DEVS2) $(DEVICE_DEVS3) $(DEVICE_DEVS4) $(DEVICE_DEVS5) \
@@ -375,7 +375,7 @@
 
 devs_tr=$(GLGENDIR)$(D)devs.tr
 $(devs_tr) : $(GS_MAK) $(TOP_MAKEFILES) $(ECHOGS_XE)
-	$(EXP)$(ECHOGS_XE) -w $(devs_tr) - -include $(GLGENDIR)$(D)$(PLATFORM)
+	$(EXP)$(ECHOGS_XE) -w $(devs_tr) - -include $(GLGENDIR)$(D)$(BUILD_PLATFORM)
 	$(EXP)$(ECHOGS_XE) -a $(devs_tr) -+ $(FEATURE_DEVS)
 	$(EXP)$(ECHOGS_XE) -a $(devs_tr) -+ $(FEATURE_DEVS_EXTRA)
 	$(EXP)$(ECHOGS_XE) -a $(devs_tr) -+ $(DEVICE_DEVS)
diff -Nru ghostscript-8.70.orig/base/macos-mcp.mak ghostscript-8.70/base/macos-mcp.mak
--- ghostscript-8.70.orig/base/macos-mcp.mak	2009-01-08 10:17:18.000000000 +0100
+++ ghostscript-8.70/base/macos-mcp.mak	2010-01-10 23:25:55.154224777 +0100
@@ -75,7 +75,7 @@
 
 # Define the platform name.
 
-PLATFORM=macos_
+BUILD_PLATFORM=macos_
 
 
 # don't use any shared libraries, they are compiled into the Ghostscript Lib
@@ -267,7 +267,7 @@
 include $(GLSRCDIR)/unixhead.mak
 
 # Redefine the platform name before going on
-PLATFORM=macos_
+BUILD_PLATFORM=macos_
 
 include $(GLSRCDIR)/gs.mak
 # psromfs.mak must precede lib.mak
diff -Nru ghostscript-8.70.orig/base/msvclib.mak ghostscript-8.70/base/msvclib.mak
--- ghostscript-8.70.orig/base/msvclib.mak	2008-08-29 01:16:39.000000000 +0200
+++ ghostscript-8.70/base/msvclib.mak	2010-01-10 23:26:01.887097616 +0100
@@ -445,7 +445,7 @@
 
 LIB_ONLY=$(GLOBJDIR)\gslib.obj $(GLOBJDIR)\gsnogc.obj $(GLOBJDIR)\gconfig.obj $(GLOBJDIR)\gscdefs.obj $(GLOBJDIR)\gsromfs$(COMPILE_INITS).obj
 MAKEDLL=0
-PLATFORM=mslib32_
+BUILD_PLATFORM=mslib32_
 
 !include $(GLSRCDIR)\version.mak
 !include $(GLSRCDIR)\msvccmd.mak
diff -Nru ghostscript-8.70.orig/base/openvms.mak ghostscript-8.70/base/openvms.mak
--- ghostscript-8.70.orig/base/openvms.mak	2009-01-08 10:17:18.000000000 +0100
+++ ghostscript-8.70/base/openvms.mak	2010-01-10 23:25:36.911773955 +0100
@@ -270,7 +270,7 @@
 
 # Define the platform name.
 
-PLATFORM=openvms_
+BUILD_PLATFORM=openvms_
 
 # Define the name of the makefile -- used in dependencies.
 
diff -Nru ghostscript-8.70.orig/base/openvms.mmk ghostscript-8.70/base/openvms.mmk
--- ghostscript-8.70.orig/base/openvms.mmk	2009-01-08 10:17:18.000000000 +0100
+++ ghostscript-8.70/base/openvms.mmk	2010-01-10 23:25:27.461102799 +0100
@@ -298,7 +298,7 @@
 
 # Define the platform name.
 
-PLATFORM=openvms_
+BUILD_PLATFORM=openvms_
 
 # Define the name of the makefile -- used in dependencies.
 
diff -Nru ghostscript-8.70.orig/base/unixhead.mak ghostscript-8.70/base/unixhead.mak
--- ghostscript-8.70.orig/base/unixhead.mak	2007-12-31 19:07:59.000000000 +0100
+++ ghostscript-8.70/base/unixhead.mak	2010-01-10 23:22:33.391601312 +0100
@@ -21,7 +21,7 @@
 # Define the platform name.  For a "stock" System V platform,
 # use sysv_ instead of unix_.
 
-PLATFORM=unix_
+BUILD_PLATFORM=unix_
 
 # Define the syntax for command, object, and executable files.
 
diff -Nru ghostscript-8.70.orig/base/watclib.mak ghostscript-8.70/base/watclib.mak
--- ghostscript-8.70.orig/base/watclib.mak	2009-01-08 10:17:18.000000000 +0100
+++ ghostscript-8.70/base/watclib.mak	2010-01-10 23:25:03.490859364 +0100
@@ -115,7 +115,7 @@
 SYNC=winsync
 !endif
 
-PLATFORM=watclib_
+BUILD_PLATFORM=watclib_
 MAKEFILE=$(GLSRCDIR)\watclib.mak
 TOP_MAKEFILES=$(MAKEFILE) $(GLSRCDIR)\wccommon.mak
 PLATOPT=
diff -Nru ghostscript-8.70.orig/base/winlib.mak ghostscript-8.70/base/winlib.mak
--- ghostscript-8.70.orig/base/winlib.mak	2009-02-21 21:59:34.000000000 +0100
+++ ghostscript-8.70/base/winlib.mak	2010-01-10 23:25:12.830496306 +0100
@@ -28,8 +28,8 @@
 
 # Define the platform name.
 
-!ifndef PLATFORM
-PLATFORM=mswin32_
+!ifndef BUILD_PLATFORM
+BUILD_PLATFORM=mswin32_
 !endif
 
 # Define the auxiliary program dependency. We use this to 
