--- ../../../work.old/solaris8-sparc/build-isa-sparcv8/ghostscript-8.70.orig/Makefile.in	2009-05-22 01:46:45.000000000 +0200
+++ ghostscript-8.70/Makefile.in	2010-01-11 04:36:20.575145752 +0100
@@ -67,11 +67,11 @@
 # The following must be substituted using @datadir@ and @libdir@
 # to avoid adding RPM generation paths (CUPS STR #1112)
 gsdir = @datadir@/ghostscript
-gsdatadir = $(gsdir)/$(GS_DOT_VERSION)
+gsdatadir = $(gsdir)
 gssharedir = @libdir@/ghostscript/$(GS_DOT_VERSION)
 gsincludedir = @includedir@/ghostscript/
 
-docdir=$(gsdatadir)/doc
+docdir=$(gsdir)/doc/ghostscript
 exdir=$(gsdatadir)/examples
 GS_DOCDIR=$(docdir)
 
@@ -342,8 +342,8 @@
 RM=rm -f
 
 # ------ Dynamic loader options ------- #
-SOC_CFLAGS	=	@SOC_CFLAGS@
-SOC_LIBS	=	@SOC_LIBS@
+SOC_CFLAGS	=	-I/opt/csw/X11/include @SOC_CFLAGS@
+SOC_LIBS	=	-L/opt/csw/X11/lib @SOC_LIBS@
 SOC_LOADER	=	@SOC_LOADER@
 
 
