From fd8d878d5f1bdb7a49dbdfd220e6c7f3525743e1 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Tue, 17 May 2011 16:51:09 +0200
Subject: [PATCH] Do not return something in void context

---
 base/gdevdrop.c |    6 ++++--
 1 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/base/gdevdrop.c b/base/gdevdrop.c
index fc52ebe..50a315b 100644
--- a/base/gdevdrop.c
+++ b/base/gdevdrop.c
@@ -408,9 +408,11 @@ pack_planar_from_standard(gx_device_memory * dev, int y, int destx,
     const byte *sp = src;
     int x, plane;
 
-    if (pdepth == 1 && dev->color_info.num_components == 4)
-        return pack_planar_cmyk_1bit_from_standard(dev, y, destx, src, width,
+    if (pdepth == 1 && dev->color_info.num_components == 4) {
+        pack_planar_cmyk_1bit_from_standard(dev, y, destx, src, width,
                                                    depth, src_depth);
+        return;
+    }
 
     for (plane = 0; plane < dev->num_planes; plane++) {
         byte *dest = scan_line_base(dev, y + plane * dev->height);
-- 
1.7.3.2

