--- drupal-6.12/includes/file.inc.o	2009-05-21 23:10:08.590704147 -0500
+++ drupal-6.12/includes/file.inc	2009-05-21 23:09:48.867918382 -0500
@@ -961,7 +961,7 @@
  * @return A string containing the path to Drupal's 'files' directory.
  */
 function file_directory_path() {
-  return variable_get('file_directory_path', conf_path() .'/files');
+  return variable_get('file_directory_path', '/var/opt/csw/drupal/upload');
 }
 
 /**
