From d8e0acd341275bfc830fe5135fd91325b166bf19 Mon Sep 17 00:00:00 2001
From: Yann Rouillard <yann@pleiades.fr.eu.org>
Date: Mon, 13 Aug 2012 19:59:55 +0200
Subject: [PATCH] solaris10 has linux-like setenv

---
 misc.c    |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/misc.c b/misc.c
index 0ff7df8..2e79ae8 100644
--- a/misc.c
+++ b/misc.c
@@ -613,7 +613,7 @@ char *value;
    */
 # endif /* NEEDSETENV */
 #else /* USESETENV */
-# if defined(linux) || defined(__convex__) || (BSD >= 199103)
+# if defined(linux) || defined(__convex__) || (BSD >= 199103) || defined(__sun) || defined(__sun__)
   setenv(var, value, 1);
 # else
   setenv(var, value);
-- 
1.7.10.3

