From acfa6600510090092bd64b759400fbee81827f3b Mon Sep 17 00:00:00 2001
From: Ben Walton <bwalton@opencsw.org>
Date: Fri, 22 Jun 2012 04:15:19 +0200
Subject: [PATCH] Makefile: make options and tools configurable

Signed-off-by: Ben Walton <bwalton@opencsw.org>
---
 Makefile |   20 ++++++++++----------
 1 file changed, 10 insertions(+), 10 deletions(-)

diff --git a/Makefile b/Makefile
index 1351e33..da3695b 100644
--- a/Makefile
+++ b/Makefile
@@ -17,26 +17,26 @@ include version
 TARGET=httping
 
 DEBUG=yes
-WFLAGS=-Wall -W
-OFLAGS=-O2
+WFLAGS?=-Wall -W
+OFLAGS?=-O2
 CFLAGS+=$(WFLAGS) $(OFLAGS) -DVERSION=\"$(VERSION)\" -g
 
 PACKAGE=$(TARGET)-$(VERSION)
-PREFIX=/usr
+PREFIX?=/usr
 BINDIR=$(PREFIX)/bin
 MANDIR=$(PREFIX)/share/man
 DOCDIR=$(PREFIX)/share/doc/$(TARGET)
 
-INSTALL=install
+INSTALL?=install
 INSTALLDIR=$(INSTALL) -m 0755 -d
 INSTALLBIN=$(INSTALL) -m 0755
 INSTALLMAN=$(INSTALL) -m 0644
 INSTALLDOC=$(INSTALL) -m 0644
-STRIP=/usr/bin/strip
-RMDIR=/bin/rm -rf
-MKDIR=/bin/mkdir
-ARCHIVE=/bin/tar cf -
-COMPRESS=/bin/gzip -9
+STRIP?=/usr/bin/strip
+RMDIR?=/bin/rm -rf
+MKDIR?=/bin/mkdir
+ARCHIVE?=/bin/tar cf -
+COMPRESS?=/bin/gzip -9
 
 OBJS=mem.o http.o io.o str.o error.o utils.o main.o tcp.o res.o
 
@@ -63,7 +63,7 @@ endif
 all: $(TARGET)
 
 $(TARGET): $(OBJS)
-	$(CC) $(WFLAGS) $(OBJS) $(LDFLAGS) -o $(TARGET)
+	$(CC) $(CFLAGS) $(OBJS) $(LDFLAGS) -o $(TARGET)
 	#
 	# Oh, blatant plug: http://keetweej.vanheusden.com/wishlist.html
 
-- 
1.7.10.3

