From f252b0a2a7f87d34dbcdf8bcccde9e6069ac9729 Mon Sep 17 00:00:00 2001
From: Sebastian Kayser <skayser@opencsw.org>
Date: Tue, 12 Jan 2010 01:10:31 +0100
Subject: [PATCH] Makefile: make options and tools configurable

---
 Makefile |   18 +++++++++---------
 1 files changed, 9 insertions(+), 9 deletions(-)

diff --git a/Makefile b/Makefile
index 873c57e..0133741 100644
--- a/Makefile
+++ b/Makefile
@@ -16,26 +16,26 @@ include version
 
 TARGET=httping
 
-WFLAGS=-Wall -W
-OFLAGS=-O2
+WFLAGS?=-Wall -W
+OFLAGS?=-O2
 CFLAGS+=$(WFLAGS) $(OFLAGS) -DVERSION=\"$(VERSION)\"
 
 PACKAGE=$(TARGET)-$(VERSION)
-PREFIX=/usr
+PREFIX?=/usr
 BINDIR=$(PREFIX)/bin
 MANDIR=$(PREFIX)/share/man
 DOCDIR=$(PREFIX)/share/doc/$(TARGET)
 
-INSTALL=install
+INSTALL?=install
 INSTALLDIR=$(INSTALL) -m 0755 -d
 INSTALLBIN=$(INSTALL) -m 0755
 INSTALLMAN=$(INSTALL) -m 0644
 INSTALLDOC=$(INSTALL) -m 0644
-STRIP=/usr/bin/strip
-RMDIR=/bin/rm -rf
-MKDIR=/bin/mkdir
-ARCHIVE=/bin/tar cf -
-COMPRESS=/bin/gzip -9
+STRIP?=/usr/bin/strip
+RMDIR?=/bin/rm -rf
+MKDIR?=/bin/mkdir
+ARCHIVE?=/bin/tar cf -
+COMPRESS?=/bin/gzip -9
 
 OBJS=mem.o http.o io.o str.o error.o utils.o main.o tcp.o res.o
 
-- 
1.6.5.1

