From 93f4891fa7d19a152dcd0d10a4207933010b2fc9 Mon Sep 17 00:00:00 2001
From: Yann Rouillard <yann@pleiades.fr.eu.org>
Date: Sun, 31 Mar 2013 23:49:28 +0200
Subject: [PATCH] Makefile: make options and tools configurable

---
 Makefile | 20 ++++++++++----------
 1 file changed, 10 insertions(+), 10 deletions(-)

diff --git a/Makefile b/Makefile
index c27bf22..0133741 100644
--- a/Makefile
+++ b/Makefile
@@ -16,26 +16,26 @@ include version
 
 TARGET=httping
 
-WFLAGS=-Wall -W
-OFLAGS=-O2
+WFLAGS?=-Wall -W
+OFLAGS?=-O2
 CFLAGS+=$(WFLAGS) $(OFLAGS) -DVERSION=\"$(VERSION)\"
 
 PACKAGE=$(TARGET)-$(VERSION)
-PREFIX=/usr
+PREFIX?=/usr
 BINDIR=$(PREFIX)/bin
 MANDIR=$(PREFIX)/share/man
 DOCDIR=$(PREFIX)/share/doc/$(TARGET)
 
-INSTALL=install
+INSTALL?=install
 INSTALLDIR=$(INSTALL) -m 0755 -d
 INSTALLBIN=$(INSTALL) -m 0755
 INSTALLMAN=$(INSTALL) -m 0644
 INSTALLDOC=$(INSTALL) -m 0644
-STRIP=/usr/bin/strip
-RMDIR=/bin/rm -rf
-MKDIR=/bin/mkdir
-ARCHIVE=/bin/tar cf -
-COMPRESS=/bin/gzip -9
+STRIP?=/usr/bin/strip
+RMDIR?=/bin/rm -rf
+MKDIR?=/bin/mkdir
+ARCHIVE?=/bin/tar cf -
+COMPRESS?=/bin/gzip -9
 
 OBJS=mem.o http.o io.o str.o error.o utils.o main.o tcp.o res.o
 
@@ -62,7 +62,7 @@ endif
 all: $(TARGET)
 
 $(TARGET): $(OBJS)
-	$(CC) $(WFLAGS) $(OBJS) $(LDFLAGS) -o $(TARGET)
+	$(CC) $(CFLAGS) $(OBJS) $(LDFLAGS) -o $(TARGET)
 	#
 	# Oh, blatant plug: http://keetweej.vanheusden.com/wishlist.html
 
-- 
1.8.1.4

