From f0b61c050f2b244a33a7c19b8c2164d95725470d Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Fri, 19 Nov 2010 14:28:47 +0100
Subject: [PATCH 4/4] Fix ambinguity to str.insert

---
 zcat.cc |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/zcat.cc b/zcat.cc
index f813bbe..34a37f3 100644
--- a/zcat.cc
+++ b/zcat.cc
@@ -45,7 +45,7 @@ public:
       str[i] = '0';
       }
     if( first_digit_pos > 0 ) str[--first_digit_pos] = '1';
-    else str.insert( 0, 1, '1' );
+    else str.insert( (unsigned) 0, 1, '1' );
     }
 
   int sprint( uint8_t * const buf )
-- 
1.7.3

