From cb8af8d92f6e61c8ccb86ebc41ad75b7e121c1b1 Mon Sep 17 00:00:00 2001
From: Peter Felecan <pfelecan@opencsw.org>
Date: Mon, 8 Jul 2013 16:03:40 +0200
Subject: [PATCH] Use gcc in Makefile

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index d0ffbf6..8b877a6 100644
--- a/Makefile
+++ b/Makefile
@@ -47,7 +47,7 @@ SETARCH = env DYLD_NO_PIE=1
 # Fedora-Core-1
 #SETARCH = setarch i386
 
-#CC = gcc
+CC = gcc
 #CFLAGS = -g
 #LIBS =
 #LD = $(CC) -g
-- 
1.8.3.1

