From 08494b5743cb3266ff69d1e26e83f5e142a0ac65 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Tue, 29 Nov 2011 13:51:05 +0100
Subject: [PATCH] Do not use unnamed unions

---
 print-rpki-rtr.c |    8 ++++----
 1 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/print-rpki-rtr.c b/print-rpki-rtr.c
index 6b0ac52..99be576 100644
--- a/print-rpki-rtr.c
+++ b/print-rpki-rtr.c
@@ -48,7 +48,7 @@ typedef struct rpki_rtr_pdu_ {
     union {
 	u_char cache_nonce[2];	/* Cache Nonce */
 	u_char error_code[2];	/* Error code */
-    };
+    } ui;
     u_char length[4];
     u_char msg[0];		/* message body */
 } __attribute__((packed)) rpki_rtr_pdu;
@@ -203,7 +203,7 @@ rpki_rtr_pdu_print (const u_char *tptr, u_int indent)
     case RPKI_RTR_END_OF_DATA_PDU:
 	printf("%sCache-Nonce: 0x%04x, Serial: %u",
 	       indent_string(indent+2),
-	       EXTRACT_16BITS(pdu_header->cache_nonce),
+	       EXTRACT_16BITS(pdu_header->ui.cache_nonce),
 	       EXTRACT_32BITS(pdu_header->msg));
 	break;
 
@@ -221,7 +221,7 @@ rpki_rtr_pdu_print (const u_char *tptr, u_int indent)
     case RPKI_RTR_CACHE_RESPONSE_PDU:
 	printf("%sCache-Nonce: 0x%04x",
 	       indent_string(indent+2),
-	       EXTRACT_16BITS(pdu_header->cache_nonce));
+	       EXTRACT_16BITS(pdu_header->ui.cache_nonce));
 	break;
 
     case RPKI_RTR_IPV4_PREFIX_PDU:
@@ -262,7 +262,7 @@ rpki_rtr_pdu_print (const u_char *tptr, u_int indent)
 	    encapsulated_pdu_length = EXTRACT_32BITS(pdu->encapsulated_pdu_length);
 	    tlen = pdu_len;
 
-	    error_code = EXTRACT_16BITS(pdu->pdu_header.error_code);
+	    error_code = EXTRACT_16BITS(pdu->pdu_header.ui.error_code);
 	    printf("%sError code: %s (%u), Encapsulated PDU length: %u",
 		   indent_string(indent+2),
 		   tok2str(rpki_rtr_error_codes, "Unknown", error_code),
-- 
1.7.6.1

