From bb471680f4e946d252f480d0ca9ce485048cac7f Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Sun, 26 Feb 2012 22:42:53 +0100
Subject: [PATCH] Use expr instead of bashism

---
 tests/TESTrun.sh |    8 ++++----
 1 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/tests/TESTrun.sh b/tests/TESTrun.sh
index 656974d..1f5fee0 100755
--- a/tests/TESTrun.sh
+++ b/tests/TESTrun.sh
@@ -14,10 +14,10 @@ do
   then
       echo $i: passed.
       rm -f DIFF/$i.result
-      passed=$(($passed + 1))
+      passed=`expr $passed + 1`
   else
       echo $i: failed.
-      failed=$(($failed + 1))
+      failed=`expr $failed + 1`
   fi          
 done 
 
@@ -36,11 +36,11 @@ do
   then
       echo $name: passed.
       rm -f DIFF/$output.diff
-      passed=$(($passed + 1))
+      passed=`expr passed + 1`
       echo $passed >.passed
   else
       echo $name: failed.
-      failed=$(($failed + 1))
+      failed=`expr $failed + 1`
       echo $failed >.failed
   fi
 done 
-- 
1.7.9

