--- php-5.2.6/ext/recode/recode.c.orig	2009-02-25 11:18:18.179302185 -0600
+++ php-5.2.6/ext/recode/recode.c	2009-02-25 11:20:18.906816126 -0600
@@ -221,7 +221,7 @@
 /* }}} */
 
 #endif
-
+int error;
 /*
  * Local variables:
  * tab-width: 4
--- php-5.2.6/ext/recode/config.m4.orig	2009-02-25 11:24:11.727519777 -0600
+++ php-5.2.6/ext/recode/config.m4	2009-02-25 11:24:44.598312108 -0600
@@ -41,6 +41,7 @@
     LIBS="$LIBS -lrecode"
     AC_TRY_LINK(
     [
+int error;
 char *program_name;
     ],[
 recode_format_table();
