From 2a0b4df7c481f2f072d432e02389893516ed6007 Mon Sep 17 00:00:00 2001
From: Carsten Grzemba <cgrzemba@unstable10s.bo.opencsw.org>
Date: Tue, 10 Jan 2012 15:52:22 +0100
Subject: [PATCH] add user and var run dirsrv

---
 ldap/admin/src/initconfig.in |   20 ++++++++++++++++----
 1 files changed, 16 insertions(+), 4 deletions(-)

diff --git a/ldap/admin/src/initconfig.in b/ldap/admin/src/initconfig.in
index 7f9dd8f..f024586 100644
--- a/ldap/admin/src/initconfig.in
+++ b/ldap/admin/src/initconfig.in
@@ -1,9 +1,9 @@
-# This file is sourced by @package_name@ upon startup to set
+# This file is sourced by dirsrv upon startup to set
 # the default environment for all directory server instances.
 # To set instance specific defaults, use the file in the same
-# directory called @package_name@-instance where "instance"
+# directory called dirsrv-instance where "instance"
 # is the name of your directory server instance e.g.
-# @package_name@-localhost for the slapd-localhost instance.
+# dirsrv-localhost for the slapd-localhost instance.
 
 # In order to make more file descriptors available
 # to the directory server, first make sure the system
@@ -26,13 +26,15 @@
 # server needs to know where to find its keytab
 # file - uncomment the following line and set
 # the path and filename appropriately
-# KRB5_KTNAME=@instconfigdir@/myname.keytab ; export KRB5_KTNAME
+# KRB5_KTNAME=/etc/opt/csw/dirsrv/myname.keytab ; export KRB5_KTNAME
 
 # other environment settings can be added here too
 OS=`uname -s`
 # use the new mt slab memory allocator on Solaris
 # this requires Solaris 9 update 3 or later
 if [ "$OS" = "SunOS" -a -f /usr/lib/libumem.so ] ; then
+#   UMEM_DEBUG=default
+#   UMEM_LOGGING=transaction
     LD_PRELOAD=/usr/lib/libumem.so
     export LD_PRELOAD
 fi
@@ -41,6 +43,16 @@ if [ "$OS" = "SunOS" -a -f /usr/lib/64/libumem.so ] ; then
     export LD_PRELOAD_64
 fi
 
+USER=`/usr/xpg4/bin/id -u -n`
+export USER
+[ -d /var/run/dirsrv ] && mkdir -p /var/run/dirsrv
+if [ -f /etc/opt/csw/dirsrv/admin-serv/adm.conf ]; then
+  slapd-user=`grep SuiteSpotUserID /etc/opt/csw/dirsrv/admin-serv/adm.conf | cut -d: -f2 | tr -d [:space:]`
+  slapd-group=`grep SuiteSpotGroup /etc/opt/csw/dirsrv/admin-serv/adm.conf | cut -d: -f2 | tr -d [:space:]`
+  chown $slapd-user:$slapd-group /var/run/dirsrv
+fi
+chmod g+w /var/run/dirsrv
+
 # how many seconds to wait for the startpid file to show
 # up before we assume there is a problem and fail to start
 #STARTPID_TIME=10 ; export STARTPID_TIME
-- 
1.7.8.2

