From f4315b6b8516a8bfd9b09d2723b78c90e4581652 Mon Sep 17 00:00:00 2001
From: Carsten Grzemba <cgrzemba@opencsw.org>
Date: Thu, 12 Jan 2012 12:09:34 +0100
Subject: [PATCH] add i386 platform to Makefile.am and configure.ac

---
 Makefile.am  |    4 ++--
 configure.ac |   18 ++++++++++++++----
 2 files changed, 16 insertions(+), 6 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 4bf1966..6b01783 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -682,7 +682,7 @@ libslapd_la_SOURCES = ldap/servers/slapd/add.c \
 	$(libavl_a_SOURCES)
 
 libslapd_la_CPPFLAGS = $(PLUGIN_CPPFLAGS) @sasl_inc@ @db_inc@ @svrcore_inc@ @kerberos_inc@ @pcre_inc@
-if SOLARIS
+if SPARC
 libslapd_la_SOURCES += ldap/servers/slapd/slapi_counter_sunos_sparcv9.S
 endif
 
@@ -863,7 +863,7 @@ libcollation_plugin_la_SOURCES = ldap/servers/plugins/collation/collate.c \
         ldap/servers/plugins/collation/config.c \
         ldap/servers/plugins/collation/orfilter.c
 
-libcollation_plugin_la_CPPFLAGS = @icu_inc@ $(PLUGIN_CPPFLAGS)
+libcollation_plugin_la_CPPFLAGS = $(PLUGIN_CPPFLAGS) @icu_inc@
 libcollation_plugin_la_LIBADD = libslapd.la $(LDAPSDK_LINK) $(NSPR_LINK) $(ICU_LINK) $(LIBCSTD) $(LIBCRUN)
 libcollation_plugin_la_LDFLAGS = -avoid-version
 libcollation_plugin_la_LINK = $(CXXLINK) -avoid-version
diff --git a/configure.ac b/configure.ac
index 6d8e277..ad94787 100644
--- a/configure.ac
+++ b/configure.ac
@@ -460,20 +460,18 @@ case $host in
     # HPUX doesn't use /etc for this
     initdir=/init.d
     ;;
-  sparc-sun-solaris*)
+  *-*-solaris*)
     AC_DEFINE([XP_UNIX], [1], [UNIX])
     AC_DEFINE([SVR4], [1], [SVR4])
     AC_DEFINE([__svr4], [1], [SVR4])
     AC_DEFINE([__svr4__], [1], [SVR4])
     AC_DEFINE([_SVID_GETTOD], [1], [SVID_GETTOD])
     AC_DEFINE([SOLARIS], [1], [SOLARIS])
-    AC_DEFINE([CPU_sparc], [], [cpu type sparc])
     AC_DEFINE([OS_solaris], [1], [OS SOLARIS])
     AC_DEFINE([sunos5], [1], [SunOS5])
     AC_DEFINE([OSVERSION], [509], [OS version])
     AC_DEFINE([_REENTRANT], [1], [_REENTRANT])
     AC_DEFINE([NO_DOMAINNAME], [1], [no getdomainname])
-    AC_DEFINE([ATOMIC_64BIT_OPERATIONS], [1], [enabling atomic counter])
 dnl socket nsl and dl are required to link several programs and libdb
     LIBSOCKET=-lsocket
     AC_SUBST([LIBSOCKET], [$LIBSOCKET])
@@ -488,8 +486,19 @@ dnl Cstd and Crun are required to link any C++ related code
     AC_SUBST([LIBCRUN], [$LIBCRUN])
     platform="solaris"
     # assume 64 bit
-    perlexec='/usr/lib/sparcv9/dirsec/perl5x/bin/perl'
     initdir='$(sysconfdir)/init.d'
+    case $host in
+      i?86-*-solaris2.1[[0-9]]*)
+dnl I dont know why i386 need this explicit
+        AC_DEFINE([HAVE_GETPEERUCRED], [1], [have getpeerucred])
+        ;;
+      sparc-*-solaris*)
+dnl includes some assembler stuff in counter.o
+        AC_DEFINE([ATOMIC_64BIT_OPERATIONS], [1], [enabling atomic counter])
+        AC_DEFINE([CPU_sparc], [], [cpu type sparc])
+	TARGET='SPARC'
+        ;;
+    esac
     ;;
   *)
     platform=""
@@ -519,6 +528,7 @@ AC_SUBST(initconfigdir)
 
 AM_CONDITIONAL(HPUX,test "$platform" = "hpux")
 AM_CONDITIONAL(SOLARIS,test "$platform" = "solaris")
+AM_CONDITIONAL(SPARC, test x$TARGET = xSPARC)
 
 # Check for library dependencies
 m4_include(m4/nspr.m4)
-- 
1.7.8.2

