From 84b9164a4de6e6a31fe9b1fb239f09debd71b001 Mon Sep 17 00:00:00 2001
From: Carsten Grzemba <cgrzemba@opencsw.org>
Date: Mon, 27 Feb 2012 10:26:02 +0100
Subject: [PATCH] add i386 platform in configure.ac

---
 configure.ac |   18 ++++++++++++++----
 1 files changed, 14 insertions(+), 4 deletions(-)

diff --git a/configure.ac b/configure.ac
index f993e37..a773763 100644
--- a/configure.ac
+++ b/configure.ac
@@ -524,20 +524,18 @@ case $host in
     # HPUX doesn't use /etc for this
     initdir=/init.d
     ;;
-  sparc-sun-solaris*)
+  *-*-solaris*)
     AC_DEFINE([XP_UNIX], [1], [UNIX])
     AC_DEFINE([SVR4], [1], [SVR4])
     AC_DEFINE([__svr4], [1], [SVR4])
     AC_DEFINE([__svr4__], [1], [SVR4])
     AC_DEFINE([_SVID_GETTOD], [1], [SVID_GETTOD])
     AC_DEFINE([SOLARIS], [1], [SOLARIS])
-    AC_DEFINE([CPU_sparc], [], [cpu type sparc])
     AC_DEFINE([OS_solaris], [1], [OS SOLARIS])
     AC_DEFINE([sunos5], [1], [SunOS5])
     AC_DEFINE([OSVERSION], [509], [OS version])
     AC_DEFINE([_REENTRANT], [1], [_REENTRANT])
     AC_DEFINE([NO_DOMAINNAME], [1], [no getdomainname])
-    AC_DEFINE([ATOMIC_64BIT_OPERATIONS], [1], [enabling atomic counter])
 dnl socket nsl and dl are required to link several programs and libdb
     LIBSOCKET=-lsocket
     AC_SUBST([LIBSOCKET], [$LIBSOCKET])
@@ -552,8 +550,19 @@ dnl Cstd and Crun are required to link any C++ related code
     AC_SUBST([LIBCRUN], [$LIBCRUN])
     platform="solaris"
     # assume 64 bit
-    perlexec='/usr/lib/sparcv9/dirsec/perl5x/bin/perl'
     initdir='$(sysconfdir)/init.d'
+    case $host in
+      i?86-*-solaris2.1[[0-9]]*)
+dnl I dont know why i386 need this explicit
+        AC_DEFINE([HAVE_GETPEERUCRED], [1], [have getpeerucred])
+        ;;
+      sparc-*-solaris*)
+dnl includes some assembler stuff in counter.o
+        AC_DEFINE([ATOMIC_64BIT_OPERATIONS], [1], [enabling atomic counter])
+        AC_DEFINE([CPU_sparc], [], [cpu type sparc])
+       TARGET='SPARC'
+        ;;
+    esac
     ;;
   *)
     platform=""
@@ -584,6 +593,7 @@ AC_SUBST(initconfigdir)
 
 AM_CONDITIONAL([HPUX],[test "$platform" = "hpux"])
 AM_CONDITIONAL([SOLARIS],[test "$platform" = "solaris"])
+AM_CONDITIONAL(SPARC, [test "$TARGET" = "SPARC"])
 AM_CONDITIONAL([SYSTEMD],[test -n "$with_systemdsystemunitdir" -o -n "$with_systemdsystemconfdir"])
 
 # Check for library dependencies
-- 
1.7.9

