From e2a18924e644ee93ef31bed2a18a83bfbe78cf62 Mon Sep 17 00:00:00 2001
From: Carsten Grzemba <cgrzemba@opencsw.org>
Date: Tue, 10 Jan 2012 15:05:25 +0100
Subject: [PATCH 7/7] create var run dirsrv

---
 ldap/admin/src/dirsrv |   12 ++++++++++++
 1 files changed, 12 insertions(+), 0 deletions(-)

diff --git a/ldap/admin/src/dirsrv b/ldap/admin/src/dirsrv
index 31753b3..f024586 100644
--- a/ldap/admin/src/dirsrv
+++ b/ldap/admin/src/dirsrv
@@ -33,6 +33,8 @@ OS=`uname -s`
 # use the new mt slab memory allocator on Solaris
 # this requires Solaris 9 update 3 or later
 if [ "$OS" = "SunOS" -a -f /usr/lib/libumem.so ] ; then
+#   UMEM_DEBUG=default
+#   UMEM_LOGGING=transaction
     LD_PRELOAD=/usr/lib/libumem.so
     export LD_PRELOAD
 fi
@@ -41,6 +43,16 @@ if [ "$OS" = "SunOS" -a -f /usr/lib/64/libumem.so ] ; then
     export LD_PRELOAD_64
 fi
 
+USER=`/usr/xpg4/bin/id -u -n`
+export USER
+[ -d /var/run/dirsrv ] && mkdir -p /var/run/dirsrv
+if [ -f /etc/opt/csw/dirsrv/admin-serv/adm.conf ]; then
+  slapd-user=`grep SuiteSpotUserID /etc/opt/csw/dirsrv/admin-serv/adm.conf | cut -d: -f2 | tr -d [:space:]`
+  slapd-group=`grep SuiteSpotGroup /etc/opt/csw/dirsrv/admin-serv/adm.conf | cut -d: -f2 | tr -d [:space:]`
+  chown $slapd-user:$slapd-group /var/run/dirsrv
+fi
+chmod g+w /var/run/dirsrv
+
 # how many seconds to wait for the startpid file to show
 # up before we assume there is a problem and fail to start
 #STARTPID_TIME=10 ; export STARTPID_TIME
-- 
1.7.6.1

