From e81994a6f342f98d03c7008b1865bb8f29e6931b Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Thu, 18 Feb 2010 18:19:49 +0100
Subject: [PATCH 1/2] Do not do ldconfig

---
 Makefile.in |    3 +--
 1 files changed, 1 insertions(+), 2 deletions(-)

diff --git a/Makefile.in b/Makefile.in
index c2ba00d..e51513b 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -92,12 +92,11 @@ install : all install-info
 	$(INSTALL_DATA) ./lib$(libname).a "$(DESTDIR)$(libdir)/lib$(libname).a"
 	if [ -n "$(progname_shared)" ] ; then \
 	  $(INSTALL_PROGRAM) ./lib$(libname).so.$(pkgversion) "$(DESTDIR)$(libdir)/lib$(libname).so.$(pkgversion)" ; \
-	  if [ -e "$(DESTDIR)$(libdir)/lib$(libname).so.$(soversion)" ] ; then \
+	  if [ -f "$(DESTDIR)$(libdir)/lib$(libname).so.$(soversion)" ] ; then \
 	    run_ldconfig=no ; rm -f "$(DESTDIR)$(libdir)/lib$(libname).so.$(soversion)" ; \
 	  else run_ldconfig=yes ; \
 	  fi ; \
 	  cd "$(DESTDIR)$(libdir)" && ln -s lib$(libname).so.$(pkgversion) lib$(libname).so.$(soversion) ; \
-	  if [ $${run_ldconfig} = yes ] ; then $(LDCONFIG) "$(DESTDIR)$(libdir)" ; fi ; \
 	fi
 
 install-info :
-- 
1.6.6

