From 664c4b62afcf76d5e406806595bfca8df64d1299 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Fri, 7 Nov 2014 14:24:16 +0100
Subject: [PATCH] Use Sun Studio syntax for shared libs

---
 Makefile.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.in b/Makefile.in
index cabff9a..72e8621 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -21,7 +21,7 @@ lib$(libname).a : lzlib.o
 	$(AR) -rcs $@ $<
 
 lib$(libname).so.$(pkgversion) : lzlib_sh.o
-	$(CC) $(CFLAGS) $(LDFLAGS) -fpic -fPIC -shared -Wl,--soname=lib$(libname).so.$(soversion) -o $@ $<
+	$(CC) $(CFLAGS) $(LDFLAGS) -fpic -fPIC  -G -h lib$(libname).so.$(soversion) -o $@ $<
 
 $(progname) : $(objs) lib$(libname).a
 	$(CC) $(CFLAGS) $(LDFLAGS) -o $@ $(objs) lib$(libname).a
-- 
2.0.3

