From a9d51c5abb07d90c0f286af8bcf8842dc3ebf273 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Fri, 7 Nov 2014 14:29:05 +0100
Subject: [PATCH] sh does not support -e

---
 Makefile.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.in b/Makefile.in
index 72e8621..5372395 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -87,7 +87,7 @@ install-bin : all
 	fi
 	if [ -n "$(progname_shared)" ] ; then \
 	  $(INSTALL_PROGRAM) ./lib$(libname).so.$(pkgversion) "$(DESTDIR)$(libdir)/lib$(libname).so.$(pkgversion)" ; \
-	  if [ -e "$(DESTDIR)$(libdir)/lib$(libname).so.$(soversion)" ] ; then \
+	  if [ -f "$(DESTDIR)$(libdir)/lib$(libname).so.$(soversion)" ] ; then \
 	    run_ldconfig=no ; \
 	  else run_ldconfig=yes ; \
 	  fi ; \
-- 
2.0.3

