From 583bcd2e4276b290bd8cf29894292a1fe5b0057a Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Thu, 18 Feb 2010 18:20:56 +0100
Subject: [PATCH 2/2] Use ucb echo as it only understands -n

---
 testsuite/check.sh |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/testsuite/check.sh b/testsuite/check.sh
index e35876d..69b5c03 100755
--- a/testsuite/check.sh
+++ b/testsuite/check.sh
@@ -5,6 +5,9 @@
 # This script is free software: you have unlimited permission
 # to copy, distribute and modify it.
 
+PATH=/usr/ucb:$PATH
+export PATH
+
 LC_ALL=C
 export LC_ALL
 objdir=`pwd`
-- 
1.6.6

