From ccab3f0f516aa027a935e70e77ff0c0b0a6445f8 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Thu, 18 Feb 2010 18:08:08 +0100
Subject: [PATCH] Change linker flags for shared libs to be Sun Studio

---
 Makefile.in |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Makefile.in b/Makefile.in
index a68698a..c2ba00d 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -23,7 +23,7 @@ lib$(libname).a: $(lib_objs)
 	$(AR) -rcs lib$(libname).a $(lib_objs)
 
 lib$(libname).so.$(pkgversion) : $(sh_lib_objs)
-	$(CXX) -shared -Wl,--soname=lib$(libname).so.$(soversion) -o lib$(libname).so.$(pkgversion) $(sh_lib_objs)
+	$(CXX) $(LDFLAGS) -G -h lib$(libname).so.$(soversion) -o lib$(libname).so.$(pkgversion) $(sh_lib_objs)
 
 $(progname) : $(objs) lib$(libname).a
 	$(CXX) $(LDFLAGS) -o $(progname) $(objs) lib$(libname).a
-- 
1.6.6

