diff --speed-large-files --minimal -Nru openssh-5.0p1.orig/channels.c openssh-5.0p1/channels.c
--- openssh-5.2p1/channels.c	2008-04-02 17:43:57.000000000 -0400
+++ openssh-5.2p1.new/channels.c	2008-04-30 08:32:59.300403974 -0400
@@ -2906,6 +2906,9 @@
 				debug2("bind port %d: %.100s", port, strerror(errno));
 				close(sock);
 
+				if (errno == EADDRNOTAVAIL)
+					continue;
+
 				for (n = 0; n < num_socks; n++) {
 					close(socks[n]);
 				}
