From df22eb20fe2b75a3a39384c11fd57194a8977a27 Mon Sep 17 00:00:00 2001
From: =?utf-8?q?Maciej=20Blizi=C5=84ski?= <blizinski@google.com>
Date: Mon, 19 Oct 2009 13:22:37 +0100
Subject: [PATCH] Fix for usb-unix.c, from http://www.cups.org/str.php?L3332

---
 backend/usb-unix.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/backend/usb-unix.c b/backend/usb-unix.c
index c749253..8ce92b9 100644
--- a/backend/usb-unix.c
+++ b/backend/usb-unix.c
@@ -560,7 +560,7 @@ open_device(const char *uri,		/* I - Device URI */
  * 'side_cb()' - Handle side-channel requests...
  */
 
-static void
+static int				/* O - 0 on success, -1 on error */
 side_cb(int         print_fd,		/* I - Print file */
         int         device_fd,		/* I - Device file */
         int         snmp_fd,		/* I - SNMP socket (unused) */
-- 
1.6.3.2

