From 101afaf697b8cbd8d0194febbba3ea94d99086ab Mon Sep 17 00:00:00 2001
From: Maciej Blizinski <maciej@opencsw.org>
Date: Tue, 20 Oct 2009 15:05:58 +0200
Subject: [PATCH] Mapping GNUTLS_E_INVALID_SESSION to EIO in http_read_ssl

---
 cups/http.c |   12 +++++++++++-
 1 files changed, 11 insertions(+), 1 deletions(-)

diff --git a/cups/http.c b/cups/http.c
index 840e3fc..47b9c0a 100644
--- a/cups/http.c
+++ b/cups/http.c
@@ -2719,7 +2719,17 @@ http_read_ssl(http_t *http,		/* I - Connection to server */
   return (SSL_read((SSL *)(http->tls), buf, len));
 
 #  elif defined(HAVE_GNUTLS)
-  return (gnutls_record_recv(((http_tls_t *)(http->tls))->session, buf, len));
+  int bytes;
+  bytes = (gnutls_record_recv(((http_tls_t *)(http->tls))->session, buf, len));
+  if (bytes < 0) {
+    if (bytes == GNUTLS_E_INVALID_SESSION) {
+      errno = EIO; /* Not sure which value to map it to. */
+    } else if (bytes < 0 && !errno) {
+      /* Shouldn't happen! Need a warning or an error, or something similar. */
+      DEBUG_printf(("gnutls_record_recv() returned a negative number but errno=0."));
+    }
+  }
+  return bytes;
 
 #  elif defined(HAVE_CDSASSL)
   int		result;			/* Return value */
-- 
1.6.5

