--- /dev/null	2009-04-13 19:20:44.000000000 +0200
+++ docbook-xsl-1.74.3/Makefile.csw	2009-04-13 19:24:36.996214871 +0200
@@ -0,0 +1,43 @@
+DESTDIR ?= ... should be overridden by GAR ...
+
+SHELL = /opt/csw/bin/bash
+
+# directories that don't get installed by the generic install-xsl target
+# note the . (to work around find output)
+SS_EX_DIRS = doc docsrc tools tests .
+# directories that do get installed by install-xsl
+DIRS = $(subst ./,,$(shell gfind . -maxdepth 1 -type d))
+SS_DIRS  = $(addprefix dir-,$(filter-out $(SS_EX_DIRS),$(DIRS)))
+
+prefix = $(DESTDIR)/opt/csw
+bindir = $(prefix)/bin
+datadir = $(prefix)/share
+
+instdir = $(datadir)/sgml/docbook/xsl-stylesheets
+docdir = $(datadir)/doc/docbookxsl
+
+all: install
+
+install: install-xsl install-misc install-docs
+
+$(instdir):
+	gmkdir -p $(instdir)
+
+#so we don't get the example bin files, etc.  we can always install them
+# 'properly' to csw/bin/ if there is demand.
+dir-epub: $(instdir)
+	gmkdir -p $(instdir)/epub
+	cp epub/*xsl $(instdir)/epub
+
+dir-%:
+	cp -R $*/ $(instdir)/
+
+install-xsl: $(instdir) $(SS_DIRS)
+
+install-misc: $(instdir)
+	cp VERSION $(instdir)
+
+install-docs:
+	gmkdir -p $(docdir)
+	cp COPYING AUTHORS BUGS README TODO RELEASE-NOTES* NEWS* $(docdir)/
+	cp -R doc $(docdir)/
