From 1e2f9e4981200dff4884c21eb9add3c1a18ad01e Mon Sep 17 00:00:00 2001
From: Oliver Kiddle <opk@opencsw.org>
Date: Thu, 23 Jan 2014 21:46:32 +0100
Subject: [PATCH] adapt Makefile for opencsw

---
 Makefile.Solaris | 46 ++++++++++++----------------------------------
 1 file changed, 12 insertions(+), 34 deletions(-)

diff --git a/Makefile.Solaris b/Makefile.Solaris
index 527ec58..94c3c2a 100644
--- a/Makefile.Solaris
+++ b/Makefile.Solaris
@@ -26,11 +26,11 @@ FILTER = 1
 EXTNS = GIF TIFF XPM PNG PPM PNM PGM PCX BMP EIM JPEG SVG WMF ICO
 
 # Comment this line out if your system doesn't have getopt_long().
-#GETOPT_LONG = -DHAVE_GETOPT_LONG
+GETOPT_LONG = -DHAVE_GETOPT_LONG
 
 # This program will be run on the manual page after it is installed.
 # If you don't want to compress the manpage, change it to 'true'.
-COMPRESS_PROG = gzip -9f
+COMPRESS_PROG = true
 
 # Comment this line out if your system doesn't have lcms2 installed
 # (for minimal Color Management support)
@@ -51,13 +51,13 @@ EXIF = -DHAVE_EXIF
 # Do not edit below here!
 ######################################################################
 
-CC        = gcc
-CFLAGS    = -O2 -Wall -fomit-frame-pointer -finline-functions \
+#CC        = gcc
+#CFLAGS    = -O2 -Wall -fomit-frame-pointer -finline-functions \
 	    -fcaller-saves -ffast-math -fno-strength-reduce \
 	    -fthread-jumps #-march=pentium #-DSTAT_MACROS_BROKEN
 
-INCLUDES  := $(shell pkg-config --cflags gdk-2.0 imlib2)
-LIBS      := $(shell pkg-config --libs gdk-2.0 imlib2) -lX11 lXext
+INCLUDES  := -I/opt/csw/include $(shell pkg-config --cflags gdk-2.0 imlib2)
+LIBS      := $(filter-out -lintl,$(shell pkg-config --libs gdk-2.0 imlib2)) -lX11 -lXext
 
 # [as] thinks that this is not portable enough:
 # [lc] I use a virtual screen of 1600x1200, and the resolution is 1024x768,
@@ -140,33 +140,11 @@ distclean : clean
 
 install: $(PROGRAM)
 	@echo "Installing QIV..."
-	@if [ ! -d $(PREFIX)/bin ]; then \
-	  install -d -m 0755 $(PREFIX)/bin; \
-	  echo install -d -m 0755 $(PREFIX)/bin; \
-        fi
-	install -s -m 0755 $(PROGRAM) $(PREFIX)/bin
-	@if [ ! -d $(PREFIX)/man/man1 ]; then \
-	  echo install -d -m 0755 $(PREFIX)/man/man1; \
-	  install -d -m 0755 $(PREFIX)/man/man1; \
-	fi
-	install -m 0644 $(PROGRAM).1 $(PREFIX)/man/man1
-	$(COMPRESS_PROG) $(PREFIX)/man/man1/$(PROGRAM).1
-	install -D -m 0644 qiv.png $(PREFIX)/share/pixmaps/qiv.png
-	install -D -m 0644 qiv.desktop $(PREFIX)/share/applications/qiv.desktop
-	@if [ ! -e $(PREFIX)/share/pixmaps ]; then \
-	  echo install -d -m 0755 $(PREFIX)/share/pixmaps; \
-	  install -d -m 0755 $(PREFIX)/share/pixmaps; \
-	fi
-	install -m 0644 qiv.png $(PREFIX)/share/pixmaps/qiv.png
-	@if [ ! -e $(PREFIX)/share/applications ]; then \
-	  echo install -d -m 0755 $(PREFIX)/share/applications; \
-	  install -d -m 0755 $(PREFIX)/share/applications; \
-	fi
-	install -m 0644 qiv.desktop $(PREFIX)/share/applications/qiv.desktop
-	@if ./qiv -f ./intro.jpg ; \
-	then echo "-- Test Passed --" ; \
-	else echo "-- Test Failed --" ; \
-	fi
-	@echo "\nDont forget to look into the \"qiv-command\" file and install it!\n-> cp qiv-command.example $(PREFIX)/bin/qiv-command\n\n"
+	ginstall -D -s -m 0755 $(PROGRAM) $(DESTDIR)$(bindir)/$(PROGRAM)
+	ginstall -D -m 0644 $(PROGRAM).1 $(DESTDIR)$(mandir)/man1/$(PROGRAM).1
+	$(COMPRESS_PROG) $(DESTDIR)/man/man1/$(PROGRAM).1
+	ginstall -D -m 0644 qiv.png $(DESTDIR)$(datadir)/share/pixmaps/qiv.png
+	ginstall -D -m 0644 qiv.desktop $(DESTDIR)$(datadir)/applications/qiv.desktop
+	ginstall -D -m 0644 contrib/qiv-command.example $(DESTDIR)$(datadir)/doc/qiv/qiv-command.example
 
 # the end... ;-)
-- 
1.8.4.1

