From b8f575fd06bcb98a44d633e290358361543645f7 Mon Sep 17 00:00:00 2001
From: Juraj Lutter <wilbury@opencsw.org>
Date: Sun, 23 Oct 2011 13:59:56 +0200
Subject: [PATCH] Fix memcached_dtrace.d and add ``touch'' command.

---
 memcached_dtrace.d |   11 +++++++++++
 1 files changed, 11 insertions(+), 0 deletions(-)

diff --git a/memcached_dtrace.d b/memcached_dtrace.d
index eeffb6f..59a1a89 100644
--- a/memcached_dtrace.d
+++ b/memcached_dtrace.d
@@ -207,6 +207,17 @@ provider memcached {
    probe command__add(int connid, const char *key, int keylen, int size, int64_t casid);
 
    /**
+    * Fired for an touch-command.
+    * @param connid connection id
+    * @param key requested key
+    * @param keylen length of the key
+    * @param size the new size of the key's data (or signed int -1 if
+    *             not found)
+    * @param casid the casid for the item
+    */
+   probe command__touch(int connid, const char *key, int keylen, int size, int64_t casid);
+
+   /**
     * Fired for a set-command.
     * @param connid connection id
     * @param key requested key
-- 
1.7.6.1

