From 4164824ced3e17247d15bfeca890d127fc313d24 Mon Sep 17 00:00:00 2001
From: Jan Holzhueter <jh@opencsw.org>
Date: Tue, 28 Sep 2010 13:57:15 +0200
Subject: [PATCH] add setenv patch

---
 libtransmission/torrent.c |   18 ++++++++++++++++++
 ltmain.sh                 |    2 +-
 2 files changed, 19 insertions(+), 1 deletions(-)

diff --git a/libtransmission/torrent.c b/libtransmission/torrent.c
index ba9fc83..f6d45de 100644
--- a/libtransmission/torrent.c
+++ b/libtransmission/torrent.c
@@ -47,6 +47,24 @@
 #include "verify.h"
 #include "version.h"
 
+#ifdef __sun
+void setenv(char *n, char *v, int x) {
+  char buf[256];
+  snprintf(buf,sizeof(buf),"%s=%s",n,v);
+  putenv(buf);
+}
+
+void unsetenv(char *env_name) {
+  extern char **environ;
+  char **cc;
+  int l;
+  l=strlen(env_name);
+  for (cc=environ;*cc!=NULL;cc++) {
+    if (strncmp(env_name,*cc,l)==0 && ((*cc)[l]=='='||(*cc)[l]=='\0')) break;
+  } for (; *cc != NULL; cc++) *cc=cc[1];
+}
+#endif
+
 /***
 ****
 ***/
-- 
1.7.3

