From a5c877eb668a0b0334b535bcfacb59ec1075d698 Mon Sep 17 00:00:00 2001
From: Rafael Ostertag <raos@opencsw.org>
Date: Fri, 15 Nov 2013 13:52:12 +0100
Subject: [PATCH] utils-test.c: fix test

---
 libtransmission/utils-test.c | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/libtransmission/utils-test.c b/libtransmission/utils-test.c
index f163ce2..28be91f 100644
--- a/libtransmission/utils-test.c
+++ b/libtransmission/utils-test.c
@@ -451,7 +451,11 @@ test_truncd( void )
     check( !strcmp( buf, "3" ) );
 
     tr_snprintf( buf, sizeof( buf ), "%.2f", tr_truncd( nan, 2 ) );
-    check( strstr( buf, "nan" ) != NULL );
+#if defined(__sparc)
+    check( strstr( buf, "NaN" ) != NULL );
+#else
+    check( strstr( buf, "-NaN" ) != NULL );
+#endif
 
     return 0;
 }
-- 
1.8.4.1

