From 450c6c8c49e7992fbca39d4cf5f3920b62540e75 Mon Sep 17 00:00:00 2001
From: Jan Holzhueter <jh@build8s.(none)>
Date: Wed, 6 Jan 2010 14:26:25 +0100
Subject: [PATCH] describe your change

---
 gtk/icons/Makefile.in |   16 ++++++++--------
 1 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/gtk/icons/Makefile.in b/gtk/icons/Makefile.in
index 879e4e3..3c7c529 100644
--- a/gtk/icons/Makefile.in
+++ b/gtk/icons/Makefile.in
@@ -474,14 +474,14 @@ install-icons:
 		mkdir -p $(DESTDIR)$(datadir)/icons/$$THEME/$$SIZE/$$CONTEXT; \
 		$(INSTALL_DATA) $(srcdir)/$$icon $(DESTDIR)$(datadir)/icons/$$THEME/$$SIZE/$$CONTEXT/$$ICONFILE; \
 	done; \
-	for icon in $(private_icons); do \
-		THEME=`echo $$icon | cut -d_ -f1`; \
-		CONTEXT=`echo $$icon | cut -d_ -f2`; \
-		SIZE=`echo $$icon | cut -d_ -f3`; \
-		ICONFILE=`echo $$icon | cut -d_ -f4`; \
-		mkdir -p $(DESTDIR)$(pkgdatadir)/icons/$$THEME/$$SIZE/$$CONTEXT; \
-		$(INSTALL_DATA) $(srcdir)/$$icon $(DESTDIR)$(pkgdatadir)/icons/$$THEME/$$SIZE/$$CONTEXT/$$ICONFILE; \
-	done
+#	for icon in $(private_icons); do \
+#		THEME=`echo $$icon | cut -d_ -f1`; \
+#		CONTEXT=`echo $$icon | cut -d_ -f2`; \
+#		SIZE=`echo $$icon | cut -d_ -f3`; \
+#		ICONFILE=`echo $$icon | cut -d_ -f4`; \
+#		mkdir -p $(DESTDIR)$(pkgdatadir)/icons/$$THEME/$$SIZE/$$CONTEXT; \
+#		$(INSTALL_DATA) $(srcdir)/$$icon $(DESTDIR)$(pkgdatadir)/icons/$$THEME/$$SIZE/$$CONTEXT/$$ICONFILE; \
+#	done
 
 uninstall-icons:
 	-for icon in $(public_icons); do \
-- 
1.6.6

