From 84469352eccf90852d7d58241516619ad1545272 Mon Sep 17 00:00:00 2001
From: Jan Holzhueter <jh@build8s.(none)>
Date: Wed, 6 Jan 2010 15:28:43 +0100
Subject: [PATCH] describe your change

---
 gtk/Makefile.in |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/gtk/Makefile.in b/gtk/Makefile.in
index 81b4e46..0c145b8 100644
--- a/gtk/Makefile.in
+++ b/gtk/Makefile.in
@@ -88,7 +88,7 @@ am__v_lt_ = $(am__v_lt_$(AM_DEFAULT_VERBOSITY))
 am__v_lt_0 = --silent
 transmission_LINK = $(LIBTOOL) $(AM_V_lt) --tag=CC $(AM_LIBTOOLFLAGS) \
 	$(LIBTOOLFLAGS) --mode=link $(CCLD) $(AM_CFLAGS) $(CFLAGS) \
-	$(transmission_LDFLAGS) $(LDFLAGS) -o $@
+	$(transmission_LDFLAGS) $(LDFLAGS) /usr/openwin/lib/libXext.so -o $@
 DEFAULT_INCLUDES = -I.@am__isrc@
 depcomp = $(SHELL) $(top_srcdir)/depcomp
 am__depfiles_maybe = depfiles
-- 
1.6.6

