--- a/common/utils.h
+++ b/common/utils.h
@@ -21,8 +21,8 @@
 #define SCAN_PLUGIN_SOAP "bb_soap.so"
 #define SCAN_PLUGIN_SOAPHT "bb_soapht.so"
 
-#define HPLIP_PLUGIN_STATE  "/var/lib/hp/hplip.state"
-#define CUPS_TMP_DIR   getenv("TMPDIR") ? : getenv("HOME") ?:"/tmp"
+#define HPLIP_PLUGIN_STATE  "/var/opt/csw/lib/hp/hplip.state"
+#define CUPS_TMP_DIR   getenv("TMPDIR") ? getenv("TMPDIR") : getenv("HOME") ? getenv("HOME") :"/tmp"
 
 enum UTILS_CONF_RESULT
 {
