From 43958262312e06e5fde2e62815798e2c6e8f8929 Mon Sep 17 00:00:00 2001
From: THURNER rupert <rupert@opencsw.org>
Date: Mon, 1 Nov 2010 16:57:20 -0500
Subject: [PATCH] inttypes.h instead of stdint.h

---
 fast_encoder.cc |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/fast_encoder.cc b/fast_encoder.cc
index f4becde..bfefe17 100644
--- a/fast_encoder.cc
+++ b/fast_encoder.cc
@@ -23,7 +23,7 @@
 #include <cstring>
 #include <string>
 #include <vector>
-#include <stdint.h>
+#include <inttypes.h>
 
 #include "lzip.h"
 #include "encoder.h"
-- 
1.7.3

