From f7d9e739bf4085c69cdc94f90b7d67299492e581 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Fri, 23 May 2014 16:35:39 +0200
Subject: [PATCH] Drop unknown linker flags

---
 src/Makefile.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Makefile.in b/src/Makefile.in
index d6d9be7..bb3d5bb 100644
--- a/src/Makefile.in
+++ b/src/Makefile.in
@@ -191,7 +191,7 @@ sharedstatedir = @sharedstatedir@
 sysconfdir = @sysconfdir@
 target_alias = @target_alias@
 lib_LTLIBRARIES = libgssapi.la
-libgssapi_la_LDFLAGS = -version-info 2:0:0 -Wl,--version-script,$(srcdir)/symbol_versions
+libgssapi_la_LDFLAGS = -version-info 2:0:0 
 libgssapi_la_SOURCES = \
 	g_accept_sec_context.c g_acquire_cred.c g_compare_name.c \
 	g_context_time.c g_delete_sec_context.c g_dsp_name.c g_dsp_status.c \
-- 
1.8.4.1

