diff -ur logrotate-3.8.0/test/test logrotate-3.8.0-solfix/test/test
--- logrotate-3.8.0/test/test	Tue Jun 21 08:12:02 2011
+++ logrotate-3.8.0-solfix/test/test	Sun Jun 26 23:08:30 2011
@@ -17,7 +17,7 @@
     input=test-config.$1.in
     output=test-config.$1
     sed "s,&DIR&,$PWD,g" < $input > $output
-    config_crc=$(md5sum $output)
+    config_crc=$(gmd5sum $output)
 }
 
 createlog() {
@@ -118,7 +118,7 @@
 	    echo expected: \'$expected\'
 	    exit 2
 	fi
-	echo "$config_crc" | md5sum -c - 2>&1 > /dev/null
+	echo "$config_crc" | gmd5sum -c - 2>&1 > /dev/null
 	if [ $? != 0 ]; then
 		echo "config file $output has been altered: MD5 sum mismatch"
 		exit 3
diff -ur logrotate-3.8.0/test/test-config.11.in logrotate-3.8.0-solfix/test/test-config.11.in
--- logrotate-3.8.0/test/test-config.11.in	Tue Jun 21 08:12:02 2011
+++ logrotate-3.8.0-solfix/test/test-config.11.in	Sun Jun 26 23:02:28 2011
@@ -11,7 +11,7 @@
 
     postrotate
 	touch scriptout
-	echo $(cat scriptout) foo > foo
+	echo `cat scriptout` foo > foo
 	mv foo scriptout
     endscript
 }
diff -ur logrotate-3.8.0/test/test-config.3.in logrotate-3.8.0-solfix/test/test-config.3.in
--- logrotate-3.8.0/test/test-config.3.in	Tue Jun 21 08:12:02 2011
+++ logrotate-3.8.0-solfix/test/test-config.3.in	Sun Jun 26 23:03:29 2011
@@ -8,7 +8,7 @@
 
     postrotate
 	touch scriptout
-	echo $(cat scriptout) foo > foo
+	echo `cat scriptout` foo > foo
 	mv foo scriptout
     endscript
 }
diff -ur logrotate-3.8.0/test/test-config.4.in logrotate-3.8.0-solfix/test/test-config.4.in
--- logrotate-3.8.0/test/test-config.4.in	Tue Jun 21 08:12:02 2011
+++ logrotate-3.8.0-solfix/test/test-config.4.in	Sun Jun 26 23:03:39 2011
@@ -9,7 +9,7 @@
 
     postrotate
 	touch scriptout
-	echo $(cat scriptout) foo > foo
+	echo `cat scriptout` foo > foo
 	mv foo scriptout
     endscript
 }
diff -ur logrotate-3.8.0/test/test-config.5.in logrotate-3.8.0-solfix/test/test-config.5.in
--- logrotate-3.8.0/test/test-config.5.in	Tue Jun 21 08:12:02 2011
+++ logrotate-3.8.0-solfix/test/test-config.5.in	Sun Jun 26 23:03:51 2011
@@ -9,7 +9,7 @@
 
     postrotate
 	touch scriptout
-	echo $(cat scriptout) foo > foo
+	echo `cat scriptout` foo > foo
 	mv foo scriptout
     endscript
 }
diff -ur logrotate-3.8.0/test/test-config.6.in logrotate-3.8.0-solfix/test/test-config.6.in
--- logrotate-3.8.0/test/test-config.6.in	Tue Jun 21 08:12:02 2011
+++ logrotate-3.8.0-solfix/test/test-config.6.in	Sun Jun 26 23:04:09 2011
@@ -10,7 +10,7 @@
 
     postrotate
 	touch scriptout
-	echo $(cat scriptout) foo > foo
+	echo `cat scriptout` foo > foo
 	mv foo scriptout
     endscript
 }
diff -ur logrotate-3.8.0/test/test-config.7.in logrotate-3.8.0-solfix/test/test-config.7.in
--- logrotate-3.8.0/test/test-config.7.in	Tue Jun 21 08:12:02 2011
+++ logrotate-3.8.0-solfix/test/test-config.7.in	Sun Jun 26 23:04:23 2011
@@ -10,7 +10,7 @@
 
     postrotate
 	touch scriptout
-	echo $(cat scriptout) foo > foo
+	echo `cat scriptout` foo > foo
 	mv foo scriptout
     endscript
 }
diff -ur logrotate-3.8.0/test/test-config.8.in logrotate-3.8.0-solfix/test/test-config.8.in
--- logrotate-3.8.0/test/test-config.8.in	Tue Jun 21 08:12:02 2011
+++ logrotate-3.8.0-solfix/test/test-config.8.in	Sun Jun 26 23:04:36 2011
@@ -11,7 +11,7 @@
 
     postrotate
 	touch scriptout
-	echo $(cat scriptout) foo > foo
+	echo `cat scriptout` foo > foo
 	mv foo scriptout
     endscript
 }
diff -ur logrotate-3.8.0/test/test-config.9.in logrotate-3.8.0-solfix/test/test-config.9.in
--- logrotate-3.8.0/test/test-config.9.in	Tue Jun 21 08:12:02 2011
+++ logrotate-3.8.0-solfix/test/test-config.9.in	Sun Jun 26 23:04:47 2011
@@ -11,7 +11,7 @@
 
     postrotate
 	touch scriptout
-	echo $(cat scriptout) foo > foo
+	echo `cat scriptout` foo > foo
 	mv foo scriptout
     endscript
 }
