diff -ur logrotate-3.8.1/config.c logrotate-3.8.1-sol10fix/config.c
--- logrotate-3.8.1/config.c	2011-08-31 14:39:04.000000000 +0200
+++ logrotate-3.8.1-sol10fix/config.c	2011-09-04 01:56:47.797271345 +0200
@@ -190,7 +190,7 @@
 
 	chptr = start;
 
-	while( (len = mbrtowc(&pwc, chptr, strlen(chptr), NULL)) != 0 ) {
+	while( (len = mbrtowc(&pwc, chptr, strlen(chptr), NULL)) != 0  && strlen(chptr) != 0) {
 		if( len == (size_t)(-1) || len == (size_t)(-2) || !iswprint(pwc) || iswblank(pwc) ) {
 		    message(MESS_ERROR, "%s:%d bad %s path %s\n",
 			    configFile, lineNum, key, start);
