--- logrotate-3.8.0/config.c    Tue Jun 21 08:12:02 2011
+++ logrotate-3.8.0-solfix/config.c     Mon Jun 27 20:29:25 2011
@@ -1133,5 +1133,5 @@
 						}
 					}
-					else continue;
+
 #if 0
 					if (stat(newlog->oldDir, &sb)) {
@@ -1178,5 +1178,4 @@
 						}
 					}
-					else continue;
 
 					if (access(newlog->compress_prog, X_OK)) {
@@ -1209,5 +1208,4 @@
 						}
 					}
-					else continue;
 
 					if (access(newlog->uncompress_prog, X_OK)) {
@@ -1245,5 +1243,5 @@
 							goto error;
 						}
-					} else continue;
+					}
 
 					if (poptParseArgvString(options,
@@ -1278,5 +1276,5 @@
 							goto error;
 						}
-					} else continue;
+					}
 
 					message(MESS_DEBUG, "compress_ext is now %s\n",
