From 41f345e8d78e79f7aaabfbd4569f890cc2e2f188 Mon Sep 17 00:00:00 2001
From: Ralph Boehme <sloowfranklin@gmail.com>
Date: Mon, 6 May 2013 11:20:05 +0200
Subject: [PATCH] Add mkdtemp replacement function

---
 service/dconf-state.c | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/service/dconf-state.c b/service/dconf-state.c
index 0ed156e..59f7106 100644
--- a/service/dconf-state.c
+++ b/service/dconf-state.c
@@ -7,6 +7,17 @@
 #include <stdio.h>
 #include <errno.h>
 
+#include <sys/types.h>
+#include <sys/stat.h>
+
+static char *
+mkdtemp(char *template)
+{
+  if (!mktemp(template) || mkdir(template, 0700))
+    return NULL;
+  return template;
+}
+
 static void
 dconf_state_init_session (DConfState *state)
 {
-- 
1.8.1.4

