From bbf70e55a25bb4aa32f51bb22d2c0e8fbd0a39ce Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Wed, 18 Feb 2015 15:15:28 +0100
Subject: [PATCH] Do not chown during package building

---
 setup.sh | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/setup.sh b/setup.sh
index 201511b..288be33 100755
--- a/setup.sh
+++ b/setup.sh
@@ -778,7 +778,7 @@ do
 	   if [ "$enable_livestatus" = yes ]
 	   then
 	       if [ -z "$YES" ] ; then echo -n "(Compiling MK Livestatus..." ; fi
-	       compile_livestatus 2>&1 | propeller > $SRCDIR/livestatus.log
+	       compile_livestatus 
 	       if [ "${PIPESTATUS[0]}" = 0 ]
 	       then
 
@@ -1018,8 +1018,6 @@ EOF
            # WATO. Also create an empty and Apache-writable auth.serials
            serials_file=$DESTDIR${htpasswd_file%/*}/auth.serials &&
            touch "$serials_file" &&
-           (chown $wwwuser "$serials_file" || true) &&
-           (chown $wwwuser "$htpasswd_file" || true) &&
 	   create_sudo_configuration &&
            if [ "$enable_mkeventd" = yes ]
            then
-- 
2.2.1

