From 61afd99627dbf87c98ac3df4cbd336ed01642e92 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Thu, 6 Dec 2012 16:06:56 +0100
Subject: [PATCH] Solaris header fixes

---
 src/check_icmp.c | 4 ++--
 src/livecheck.c  | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/check_icmp.c b/src/check_icmp.c
index bb53fef..353cec5 100644
--- a/src/check_icmp.c
+++ b/src/check_icmp.c
@@ -65,11 +65,11 @@ typedef unsigned short u_short;
 #include <netdb.h>
 #include <sys/socket.h>
 #include <net/if.h>
-#include "/usr/include/netinet/ip_icmp.h"
-// #include <netinet/ip_icmp.h>
+// #include "/usr/include/netinet/ip_icmp.h"
 #include <netinet/in_systm.h>
 #include <netinet/in.h>
 #include <netinet/ip.h>
+#include <netinet/ip_icmp.h>
 #include <arpa/inet.h>
 #include <signal.h>
 #include <float.h>
diff --git a/src/livecheck.c b/src/livecheck.c
index 019613a..dae61cd 100644
--- a/src/livecheck.c
+++ b/src/livecheck.c
@@ -26,7 +26,7 @@
 #include <sys/types.h>
 #include <sys/wait.h>
 #include <sys/timeb.h>
-#include <netinet/ip.h>
+#include <netinet/in.h>
 #include <stdlib.h>
 #include <fcntl.h>
 #include <string.h>
-- 
1.8.0

