From d9ee41aa38e8009bff734ec540c41822a7d3998f Mon Sep 17 00:00:00 2001
From: Ben Walton <bwalton@opencsw.org>
Date: Tue, 15 Sep 2009 19:45:33 +0200
Subject: [PATCH 3/3] do no hardcode /bin/bash to override env check

---
 configure.in |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/configure.in b/configure.in
index 727da83..bf854b7 100644
--- a/configure.in
+++ b/configure.in
@@ -18,7 +18,6 @@ AC_CHECK_PROG([MKTEMP], [mktemp],, [mktemp])
 AC_ARG_VAR([FIND], [Name of the GNU `find' program.])
 AC_CHECK_PROG([FIND], [find],, [find] )
 
-BASH=/bin/bash
 AC_ARG_VAR([BASH], [Name and path of the GNU `bash' shell.])
 AC_PATH_PROG([BASH], [bash], [/bin/bash])
 
-- 
1.6.3.2

