diff --speed-large-files --minimal -Nru xmlto-0.0.22.orig/configure.in xmlto-0.0.22/configure.in
--- xmlto-0.0.22.orig/configure.in	2009-03-25 14:48:02.000000000 +0100
+++ xmlto-0.0.22/configure.in	2009-09-09 02:43:47.431319471 +0200
@@ -18,9 +18,8 @@
 AC_ARG_VAR([FIND], [Name of the GNU `find' program.])
 AC_CHECK_PROG([FIND], [find],, [find] )
 
-BASH=/bin/bash
 AC_ARG_VAR([BASH], [Name and path of the GNU `bash' shell.])
-AC_PATH_PROG([BASH], [bash], [/bin/bash])
+AC_PATH_PROG([BASH], [bash],, [bash])
 
 AC_ARG_VAR([GETOPT], [Name of the `getopt' program (requires longopt support).])
 AC_CHECK_PROG([GETOPT], [getopt],, [getopt])
