diff --speed-large-files --minimal -Nru ruby-1.8.5-p12.orig/ext/dbm/extconf.rb ruby-1.8.5-p12/ext/dbm/extconf.rb
--- ruby-1.8.5-p12.orig/ext/dbm/extconf.rb	2006-12-05 14:27:06.000000000 -0500
+++ ruby-1.8.5-p12/ext/dbm/extconf.rb	2007-01-02 12:50:19.122490000 -0500
@@ -53,9 +53,9 @@
   end
 end
 
-have_header("cdefs.h") 
-have_header("sys/cdefs.h") 
-if dbm_hdr and have_func(db_prefix("dbm_open"))
-  have_func(db_prefix("dbm_clearerr")) unless $dbm_conf_have_gdbm
+#have_header("cdefs.h") 
+#have_header("sys/cdefs.h") 
+#if dbm_hdr and have_func(db_prefix("dbm_open"))
+#  have_func(db_prefix("dbm_clearerr")) unless $dbm_conf_have_gdbm
   create_makefile("dbm")
-end
+#end
