diff -ru lyx-1.4.2.orig/src/support/debugstream.h lyx-1.4.2/src/support/debugstream.h
--- lyx-1.4.2.orig/src/support/debugstream.h	2004-05-20 02:35:30.000000000 -0700
+++ lyx-1.4.2/src/support/debugstream.h	2006-08-03 10:05:53.806955000 -0700
@@ -21,6 +21,13 @@
 # undef DEBUG
 #endif
 
+/*
+ * Solaris 8 x86 defines this
+ */
+#ifdef ERR
+#undef ERR
+#endif
+
 struct debug_trait {
 	enum type {
 		NONE   = 0,
