Index: orca/data_gatherers/orcallator/orcallator.se
===================================================================
--- orca/data_gatherers/orcallator/orcallator.se	(revision 530)
+++ orca-new/data_gatherers/orcallator/orcallator.se	(working copy)
@@ -1423,7 +1423,8 @@
   // Traverse the mount table to find mounted ufs/vxfs file systems.
   for (mnt$mnt.number$=0; mnt$mnt.number$ != -1; mnt$mnt.number$++) {
     tmp_mnt = mnt$mnt;
-    if (tmp_mnt.mnt_fstype == "ufs" || tmp_mnt.mnt_fstype == "vxfs") {
+    if (tmp_mnt.mnt_fstype == "ufs" || tmp_mnt.mnt_fstype == "vxfs" ||
+	tmp_mnt.mnt_fstype == "zfs") {
       // Skip locally mounted /cdrom partitions.
       if (tmp_mnt.mnt_mountp =~ "^/cdrom/") {
         continue;
