diff --speed-large-files --minimal -Nru pv-0.8.0.orig/autoconf/make/rules.mk pv-0.8.0/autoconf/make/rules.mk
--- pv-0.8.0.orig/autoconf/make/rules.mk	2004-02-12 14:36:46.000000000 -0800
+++ pv-0.8.0/autoconf/make/rules.mk	2004-02-17 14:35:25.986235000 -0800
@@ -11,20 +11,20 @@
 	sh $(srcdir)/autoconf/scripts/depend.sh \
 	   $(CC) $< $(<:%.c=%) $(srcdir) $(CFLAGS) $(CPPFLAGS) > $@
 
-doc/quickref.txt: doc/quickref.1
-	man $+ | sed 's/.//g' | cat -s > $@ || :
-	chmod 644 $@ || :
+#doc/quickref.txt: doc/quickref.1
+#	man $+ | sed 's/.//g' | cat -s > $@ || :
+#	chmod 644 $@ || :
 
-doc/$(package).info: doc/manual.texi
-	-makeinfo --no-split doc/manual.texi -o doc/$(package).info
-	-chmod 644 doc/$(package).info
+#doc/$(package).info: doc/manual.texi
+#	-makeinfo --no-split doc/manual.texi -o doc/$(package).info
+#	-chmod 644 doc/$(package).info
 
-doc/manual.html: doc/manual.texi
-	-texi2html -monolithic doc/manual.texi
-	-test -e manual.html || mv $(package).html manual.html
-	-perl $(srcdir)/autoconf/scripts/htmlmunge.pl < manual.html > $@
-	rm -f manual.html
-	-chmod 644 doc/manual.html
+#doc/manual.html: doc/manual.texi
+#	-texi2html -monolithic doc/manual.texi
+#	-test -e manual.html || mv $(package).html manual.html
+#	-perl $(srcdir)/autoconf/scripts/htmlmunge.pl < manual.html > $@
+#	rm -f manual.html
+#	-chmod 644 doc/manual.html
 
 #
 # NLS stuff
