From 319645e8b4311a582012ea7c00211fff8ace5df6 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Wed, 22 May 2013 10:51:50 +0200
Subject: [PATCH] Fix test in testsuite

---
 test_jbig2dec.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test_jbig2dec.py b/test_jbig2dec.py
index c6f143e..8ca3425 100755
--- a/test_jbig2dec.py
+++ b/test_jbig2dec.py
@@ -184,4 +184,4 @@ for filename, file_hash, decode_hash in KnownFileHash.known_hashes:
 # run the defined tests if we're called as a script
 if __name__ == "__main__":
     result = suite.run()
-    sys.exit(not result)
+    sys.exit(result)
-- 
1.8.1.4

