From adf28f82a857d088282db1f86bf5efd6e3827ef6 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Tue, 12 Jul 2011 15:37:41 +0200
Subject: [PATCH 02/10] Include stdarg.h for va_list

---
 libexplain/vfprintf.h |    1 +
 libexplain/vprintf.h  |    1 +
 2 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/libexplain/vfprintf.h b/libexplain/vfprintf.h
index d16fc49..9367b8c 100644
--- a/libexplain/vfprintf.h
+++ b/libexplain/vfprintf.h
@@ -32,6 +32,7 @@
 #include <libexplain/gcc_attributes.h>
 
 #include <stdio.h>
+#include <stdarg.h>
 
 #ifdef __cplusplus
 extern "C" {
diff --git a/libexplain/vprintf.h b/libexplain/vprintf.h
index 493d920..1196b40 100644
--- a/libexplain/vprintf.h
+++ b/libexplain/vprintf.h
@@ -32,6 +32,7 @@
 #include <libexplain/gcc_attributes.h>
 
 #include <stdio.h>
+#include <stdarg.h>
 
 #ifdef __cplusplus
 extern "C" {
-- 
1.7.6

