From ac97d893f81343129be31469d9142a6272bb4f8f Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Tue, 12 Jul 2011 15:39:20 +0200
Subject: [PATCH 04/10] Use fallback for AI_NUMERICSERV

---
 libexplain/buffer/errno/getaddrinfo.h |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/libexplain/buffer/errno/getaddrinfo.h b/libexplain/buffer/errno/getaddrinfo.h
index 140f3dd..b64c3a8 100644
--- a/libexplain/buffer/errno/getaddrinfo.h
+++ b/libexplain/buffer/errno/getaddrinfo.h
@@ -21,6 +21,10 @@
 
 #include <libexplain/string_buffer.h>
 
+#ifndef AI_NUMERICSERV
+# define AI_NUMERICSERV 0
+#endif
+
 struct addrinfo; /* forward */
 
 /**
-- 
1.7.6

