From 1188c9ce6e410f9086fa85315f6da9e14b91b2a2 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Tue, 12 Jul 2011 15:40:14 +0200
Subject: [PATCH 05/10] Solaris does not have O_NOFOLLOW

---
 libexplain/buffer/errno/open.c |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/libexplain/buffer/errno/open.c b/libexplain/buffer/errno/open.c
index 1b086f4..1ef9c61 100644
--- a/libexplain/buffer/errno/open.c
+++ b/libexplain/buffer/errno/open.c
@@ -199,8 +199,10 @@ explain_buffer_errno_open_explanation(explain_string_buffer_t *sb,
         final_component.must_be_a_st_mode = 1;
         final_component.st_mode = S_IFDIR;
     }
+#if defined(O_NOFOLLOW)
     if (flags & O_NOFOLLOW)
         final_component.follow_symlink = 0;
+#endif
 
     switch (errnum)
     {
@@ -347,6 +349,7 @@ explain_buffer_errno_open_explanation(explain_string_buffer_t *sb,
 
     case ELOOP:
     case EMLINK: /* BSD */
+#if defined(O_NOFOLLOW)
         if (flags & O_NOFOLLOW)
         {
             struct stat     st;
@@ -370,6 +373,7 @@ explain_buffer_errno_open_explanation(explain_string_buffer_t *sb,
                 break;
             }
         }
+#endif
         explain_buffer_eloop(sb, pathname, "pathname", &final_component);
         break;
 
-- 
1.7.6

