From 7d1479dcaa9835c0b8d111d7e89fc40077f48f9e Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Tue, 12 Jul 2011 15:44:45 +0200
Subject: [PATCH 09/10] Use necessary mnttab.h

---
 libexplain/buffer/mount_point.h |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/libexplain/buffer/mount_point.h b/libexplain/buffer/mount_point.h
index b504dc6..5b487bb 100644
--- a/libexplain/buffer/mount_point.h
+++ b/libexplain/buffer/mount_point.h
@@ -22,6 +22,9 @@
 
 #include <libexplain/ac/sys/stat.h>
 
+#include <stdio.h>
+#include <sys/mnttab.h>
+
 #include <libexplain/string_buffer.h>
 
 /**
-- 
1.7.6

