From 6be9f8dbd84b5e1c61d46c1348f172afface0318 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Thu, 8 Sep 2011 10:45:42 +0200
Subject: [PATCH] Features patch

---
 src/main.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/main.c b/src/main.c
index ff5e6af..9b704d1 100644
--- a/src/main.c
+++ b/src/main.c
@@ -824,7 +824,7 @@ print_version (void)
       int line_length = MAX_CHARS_PER_LINE;
       while ((line_length > 0) && (compiled_features[i] != NULL))
         {
-          if (printf ("%s ", compiled_features[i]))
+          if (printf ("%s ", compiled_features[i]) < 0)
             exit (3);
           line_length -= strlen (compiled_features[i]) + 2;
           i++;
-- 
1.7.6

