From cbac50c7c355cbff05fcbb2322934e475bd526a2 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Thu, 8 Sep 2011 11:07:39 +0200
Subject: [PATCH] Make sure FIONBIO is defined

---
 src/gnutls.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/src/gnutls.c b/src/gnutls.c
index dfff00c..40a04ef 100644
--- a/src/gnutls.c
+++ b/src/gnutls.c
@@ -48,6 +48,8 @@ as that of the covered work.  */
 #include "ptimer.h"
 #include "ssl.h"
 
+#include <sys/fcntl.h>
+
 #ifdef WIN32
 # include "w32sock.h"
 #endif
-- 
1.7.6

