diff -r 68bd7297137e lib/util/Makefile.in
--- a/lib/util/Makefile.in	Sat Mar 21 15:41:59 2015 -0600
+++ b/lib/util/Makefile.in	Sun Mar 22 07:08:53 2015 -0600
@@ -79,7 +79,7 @@
 
 # Regression tests
 TEST_PROGS = atofoo_test conf_test hltq_test parseln_test progname_test @COMPAT_TEST_PROGS@
-TEST_LIBS = @LIBS@ @LIBINTL@
+TEST_LIBS = @LIBS@
 TEST_LDFLAGS = @LDFLAGS@
 
 # User and group ids the installed files should be "owned" by
@@ -136,9 +136,9 @@
 libsudo_util.la: $(LTOBJS) @LT_LDDEP@
 	case "$(LT_LDFLAGS)" in \
 	*-no-install*) \
-	    $(LIBTOOL) --mode=link $(CC) -o $@ $(LDFLAGS) $(LT_LDFLAGS) $(LTOBJS) @LIBDL@ @LIBRT@;; \
+	    $(LIBTOOL) --mode=link $(CC) -o $@ $(LDFLAGS) $(LT_LDFLAGS) $(LTOBJS) @LIBINTL@ @LIBDL@ @LIBRT@;; \
 	*) \
-	    $(LIBTOOL) --mode=link $(CC) -o $@ $(LDFLAGS) $(SSP_LDFLAGS) $(LT_LDFLAGS) $(LTOBJS) -version-info $(SHLIB_VERSION) -rpath $(libexecdir)/sudo @LT_DEP_LIBS@ @LIBDL@ @LIBRT@;; \
+	    $(LIBTOOL) --mode=link $(CC) -o $@ $(LDFLAGS) $(SSP_LDFLAGS) $(LT_LDFLAGS) $(LTOBJS) -version-info $(SHLIB_VERSION) -rpath $(libexecdir)/sudo @LT_DEP_LIBS@ @LIBINTL@ @LIBDL@ @LIBRT@;; \
 	esac
 
 siglist.c: mksiglist
diff -r 68bd7297137e plugins/sudoers/Makefile.in
--- a/plugins/sudoers/Makefile.in	Sat Mar 21 15:41:59 2015 -0600
+++ b/plugins/sudoers/Makefile.in	Sun Mar 22 07:08:53 2015 -0600
@@ -51,7 +51,7 @@
 
 # Libraries
 LT_LIBS = $(top_builddir)/lib/util/libsudo_util.la
-LIBS = $(LT_LIBS) @LIBINTL@
+LIBS = $(LT_LIBS)
 NET_LIBS = @NET_LIBS@
 SUDOERS_LIBS = @SUDOERS_LIBS@ @AFS_LIBS@ @GETGROUPS_LIB@ $(LIBS) $(NET_LIBS) @ZLIB@ @LIBMD@
 REPLAY_LIBS = @REPLAY_LIBS@ @ZLIB@
diff -r 68bd7297137e src/Makefile.in
--- a/src/Makefile.in	Sat Mar 21 15:41:59 2015 -0600
+++ b/src/Makefile.in	Sun Mar 22 07:08:53 2015 -0600
@@ -37,7 +37,7 @@
 
 # Libraries
 LT_LIBS = $(top_builddir)/lib/util/libsudo_util.la
-LIBS = @LIBS@ @SUDO_LIBS@ @GETGROUPS_LIB@ @NET_LIBS@ @LIBINTL@ $(LT_LIBS)
+LIBS = @LIBS@ @SUDO_LIBS@ @GETGROUPS_LIB@ @NET_LIBS@ $(LT_LIBS)
 
 # C preprocessor flags
 CPPFLAGS = -I$(incdir) -I$(top_builddir) -I. -I$(srcdir) -I$(top_srcdir) @CPPFLAGS@
@@ -86,7 +86,7 @@
 RC_LINK=@RC_LINK@
 
 TEST_PROGS = check_ttyname
-TEST_LIBS = @LIBS@ @LIBINTL@ $(LT_LIBS)
+TEST_LIBS = @LIBS@ $(LT_LIBS)
 TEST_LDFLAGS = @LDFLAGS@
 
 # OS dependent defines
@@ -133,8 +133,8 @@
 sudo_noexec.la: sudo_noexec.lo
 	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) $(LT_LDFLAGS) $(SSP_LDFLAGS) -o $@ sudo_noexec.lo -module -avoid-version -rpath $(noexecdir) -shrext .so
 
-sesh: $(SESH_OBJS) @LIBINTL@ $(LT_LIBS)
-	$(LIBTOOL) --mode=link $(CC) -o $@ $(SESH_OBJS) $(LDFLAGS) $(PIE_LDFLAGS) $(SSP_LDFLAGS) @LIBINTL@ $(LIBS)
+sesh: $(SESH_OBJS) $(LT_LIBS)
+	$(LIBTOOL) --mode=link $(CC) -o $@ $(SESH_OBJS) $(LDFLAGS) $(PIE_LDFLAGS) $(SSP_LDFLAGS) $(LIBS)
 
 check_ttyname: $(CHECK_TTYNAME_OBJS) $(top_builddir)/lib/util/libsudo_util.la
 	$(LIBTOOL) --mode=link $(CC) -o $@ $(CHECK_TTYNAME_OBJS) $(TEST_LDFLAGS) $(PIE_LDFLAGS) $(SSP_LDFLAGS) $(TEST_LIBS)
