diff -Naur libX11-1.2.2.orig/include/X11/Xlib.h libX11-1.2.2.patched/include/X11/Xlib.h
--- libX11-1.2.2.orig/include/X11/Xlib.h	2009-07-09 05:55:54.000000000 +0200
+++ libX11-1.2.2.patched/include/X11/Xlib.h	2009-07-11 23:36:25.003094998 +0200
@@ -38,6 +38,14 @@
 
 #define XlibSpecificationRelease 6
 
+#if defined(__GNUC__) && (__GNUC__ >= 4)
+# define _X_SENTINEL(x) __attribute__ ((__sentinel__(x)))
+# define _X_ATTRIBUTE_PRINTF(x,y) __attribute__((__format__(__printf__,x,y)))
+#else
+# define _X_SENTINEL(x)
+# define _X_ATTRIBUTE_PRINTF(x,y)
+#endif /* GNUC >= 4 */
+
 #include <sys/types.h>
 
 #if defined(__SCO__) || defined(__UNIXWARE__)
