diff --speed-large-files --minimal -Nru MPlayer-1.0rc1.orig/configure MPlayer-1.0rc1/configure
--- MPlayer-1.0rc1.orig/configure	2006-10-22 18:32:31.000000000 -0400
+++ MPlayer-1.0rc1/configure	2007-07-23 09:00:09.361157000 -0400
@@ -1178,7 +1178,7 @@
 	    sun4c) proc=v7 ;; 
 	    sun4d) proc=v8 ;;
 	    sun4m) proc=v8 ;;
-	    sun4u) proc=v9 _vis='yes' _def_vis='#define HAVE_VIS = yes' ;;
+#building for v8	    sun4u) proc=v9 _vis='yes' _def_vis='#define HAVE_VIS = yes' ;;
 	    sun4v) proc=v9 ;;
 	    *) proc=v8 ;;
 	esac
