From 9c6745fa9e8feb7174cf38ba7ecbf0ac94cf0540 Mon Sep 17 00:00:00 2001
From: Jan Holzhueter <jh@opencsw.org>
Date: Sun, 25 Jan 2015 17:01:17 +0100
Subject: [PATCH] hardcode_libaugeas

---
 augeas.py | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/augeas.py b/augeas.py
index ca5e191..8efbf04 100644
--- a/augeas.py
+++ b/augeas.py
@@ -69,8 +69,11 @@ def _dlopen(*args):
     """Search for one of the libraries given as arguments and load it.
     Returns the library.
     """
-    libs = [l for l in [ ctypes.util.find_library(a) for a in args ] if l]
-    lib  = reduce(lambda x, y: x or ctypes.cdll.LoadLibrary(y), libs, None)
+    # ctypes find_libary does not work with python2.6. 2.7 uses crle with only returen default paths
+    # so hardcore the path as this is opencsw package anyway
+    #libs = [l for l in [ ctypes.util.find_library(a) for a in args ] if l]
+    #lib  = reduce(lambda x, y: x or ctypes.cdll.LoadLibrary(y), libs, None)
+    lib  = ctypes.cdll.LoadLibrary("/opt/csw/lib/libaugeas.so.0")
     if not lib:
         raise ImportError("Unable to import lib%s!" % args[0])
     return lib
-- 
2.2.1

