From d66bef54d6ba8c874f4e7c7abb3df27366d64121 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Fri, 29 Nov 2013 09:17:54 +0100
Subject: [PATCH] Do not use stdbool.h

---
 ascii.c | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/ascii.c b/ascii.c
index a2f8db0..231bcf5 100644
--- a/ascii.c
+++ b/ascii.c
@@ -20,7 +20,10 @@
 #include <stdio.h>
 #include <string.h>
 #include <stdlib.h>
-#include <stdbool.h>
+/* include <stdbool.h> */
+#define bool int
+#define false 0
+#define true 1
 #include <getopt.h>
 
 #ifdef __CYGWIN__
-- 
1.8.4.1

