From 28e394a55164dc3c6d28fb449b8624915f66f8c9 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Fri, 29 Nov 2013 09:22:07 +0100
Subject: [PATCH] Honour CFLAGS

---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 7b9d39e..0082595 100644
--- a/Makefile
+++ b/Makefile
@@ -2,9 +2,9 @@
 
 VERS=3.14
 
-CFLAGS = -O
+CFLAGS ?= -O
 
-PREFIX = /usr
+PREFIX ?= /usr
 
 all: ascii ascii.1
 
-- 
1.8.4.1

