From 8c97093a01763998f67272c479f59fa98d32c49c Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Tue, 7 Dec 2010 13:27:46 +0100
Subject: [PATCH 4/8] Some more Makefile tweaks

---
 src/Makefile |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/Makefile b/src/Makefile
index b1cc23f..af7a410 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -40,7 +40,7 @@ LIBRARY = $(LIBRARY_NAME).$(SO_SUFFIX).$(LIB_VER)
 MKSHLIB = $(CC) -shared
 LIB_OPTS = -Wl,-soname=$(LIBRARY)
 SHFLAGS = -fPIC
-SMCFLAGS = -DHAVE_VA_COPY -DVA_COPY=va_copy
+SMCFLAGS = -DHAVE_VA_COPY -DVA_COPY=va_copy $(CFLAGS)
 
 ifeq ($(OS_ARCH),Darwin)
   SO_SUFFIX = dylib
@@ -51,7 +51,7 @@ ifeq ($(OS_ARCH),Darwin)
   SMCFLAGS =
 endif
 
-CFLAGS = -g -DXP_UNIX -DVERSION=$(VERSION)
+CFLAGS += -DXP_UNIX -DVERSION=$(VERSION)
 
 ifndef PYTHON
   PYTHON = python
-- 
1.7.3.2

