From 7da2f65f28296c3755478f2060609a7247a65801 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Wed, 29 Sep 2010 12:49:48 +0200
Subject: [PATCH] Use inttypes also for Solaris

---
 include/GL/glew.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/include/GL/glew.h b/include/GL/glew.h
index 5105a20..9f82d40 100644
--- a/include/GL/glew.h
+++ b/include/GL/glew.h
@@ -200,7 +200,7 @@ typedef _W64 int ptrdiff_t;
 
 /* SGI MIPSPro doesn't like stdint.h in C++ mode */
 
-#if defined(__sgi) && !defined(__GNUC__)
+#if defined(__sun) || defined(__sgi) && !defined(__GNUC__)
 #include <inttypes.h>
 #else
 #include <stdint.h>
-- 
1.7.3

