--- a/config/config.mk.in
+++ b/config/config.mk.in
@@ -129,14 +129,14 @@ ifdef DEP_TRACKING
         # dependencies. If you're not using gcc, you may need to change
         # this to something suitable for your compiler or simply unset
         # the variable. See the link above for suggestions.
-	MAKEDEPEND = $(CXX) -MM $(CPPFLAGS) -o $*.d $<
+	MAKEDEPEND = $(CXX) -xM1 $(CPPFLAGS) -o $*.d $<
 
         # Dependency files post-process commands
-        POSTDEPEND = if test ! -d $(DEPDIR); then mkdir $(DEPDIR); fi; \
+        POSTDEPEND = $(shell if test ! -d $(DEPDIR); then mkdir $(DEPDIR); fi; \
 	        if test -e $*.d; then cp $*.d $(DEPDIR)/$*.d; \
 	        sed -e 's/^\#.*//' -e 's/^[^:]*: *//' -e 's/ *\\$$//' \
                     -e '/^$$/ d' -e 's/$$/ :/' < $*.d >> $(DEPDIR)/$*.d; \
-                $(RM) $*.d; fi
+                $(RM) $*.d; fi)
 
         # Compiler flags to generate dependency files at the same time 
         # as object files (for gcc)
