From 8dbe61abe2af6ce7633337a2d1005b8ddfcf5610 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Tue, 25 Oct 2011 17:55:17 +0200
Subject: [PATCH] Hack around missing __argv

---
 tools/compat.c |   14 +++++++++-----
 1 files changed, 9 insertions(+), 5 deletions(-)

diff --git a/tools/compat.c b/tools/compat.c
index 93ba77c..a708ac5 100644
--- a/tools/compat.c
+++ b/tools/compat.c
@@ -42,6 +42,7 @@
 #include <stdio.h>
 #include <string.h>
 #include <stdarg.h>
+#include <errno.h>
 
 static const char *
 calc_prog_name (void)
@@ -51,19 +52,22 @@ calc_prog_name (void)
 
 	if(!prepared)
 	{
-		const char* beg = strrchr(__argv[0], '\\');
-		const char* temp = strrchr(__argv[0], '/');
+		strcpy( prognamebuf, "" );
+/*
+		const char* beg = strrchr(argv[0], '\\');
+		const char* temp = strrchr(argv[0], '/');
 		beg = (beg > temp) ? beg : temp;
-		beg = (beg) ? beg + 1 : __argv[0];
+		beg = (beg) ? beg + 1 : argv[0];
 
-		temp = strrchr(__argv[0], '.');
-		temp = (temp > beg) ? temp : __argv[0] + strlen(__argv[0]);
+		temp = strrchr(argv[0], '.');
+		temp = (temp > beg) ? temp : argv[0] + strlen(argv[0]);
 
 		if((temp - beg) > 255)
 			temp = beg + 255;
 
 		strncpy(prognamebuf, beg, temp - beg);
 		prognamebuf[temp - beg] = 0;
+*/
 		prepared = 1;
 	}
 
-- 
1.7.6.1

