From d3532e3ea6e4a82c029ba2337610b4e726a7bb44 Mon Sep 17 00:00:00 2001
From: Juergen Arndt <ja@opencsw.org>
Date: Thu, 12 Jul 2012 11:54:27 +0200
Subject: [PATCH] asprintf for solaris 10

---
 configure.in    |   10 ++++++++++
 src/Makefile.in |    4 +++-
 2 files changed, 13 insertions(+), 1 deletion(-)

diff --git a/configure.in b/configure.in
index aa34e02..88eca45 100644
--- a/configure.in
+++ b/configure.in
@@ -260,6 +260,14 @@ if test "x$TARGET_OS" = "xSunOS" ; then
 fi
 AC_SUBST(TARGET_PLATFORM)
 
+# add asprintf for solaris
+if test "x$host_os" = "xsolaris2.10" ; then
+	ASPRINTF_SRC="asprintf.c"
+else
+	ASPRINTF_SRC=""
+fi
+AC_SUBST(ASPRINTF_SRC)
+
 AC_PATH_PROG(PERL,perl)
 AC_OUTPUT()
 
@@ -278,6 +286,8 @@ echo ""
 echo " General Options:"
 echo " -------------------------"
 
+AC_MSG_RESULT([ TARGET_PLATFORM:  $host])
+AC_MSG_RESULT([ TARGET_OS:  $host_os])
 AC_MSG_RESULT([ NSCA port:  $nsca_port])
 AC_MSG_RESULT([ NSCA user:  $nsca_user])
 AC_MSG_RESULT([ NSCA group: $nsca_grp])
diff --git a/src/Makefile.in b/src/Makefile.in
index ecdedc7..c294103 100644
--- a/src/Makefile.in
+++ b/src/Makefile.in
@@ -8,6 +8,8 @@ srcdir=@srcdir@
 
 # Source code directories
 SRC_INCLUDE=@srcdir@/../include
+# Solaris asprintf (empty for non solaris hosts)
+SOL10_ASPRINTF=@ASPRINTF_SRC@
 
 CC=@CC@
 CFLAGS=@CFLAGS@ @DEFS@
@@ -21,7 +23,7 @@ CP=@CP@
 all: nsca send_nsca
 
 nsca: $(srcdir)/nsca.c $(srcdir)/netutils.c $(srcdir)/utils.c $(SRC_INCLUDE)/netutils.h $(SRC_INCLUDE)/utils.h $(SRC_INCLUDE)/common.h $(SRC_INCLUDE)/config.h
-	$(CC) $(CFLAGS) -o $@ $(srcdir)/nsca.c $(srcdir)/netutils.c $(srcdir)/utils.c $(LDFLAGS) $(SOCKETLIBS) $(LIBWRAPLIBS)
+	$(CC) $(CFLAGS) -o $@ $(srcdir)/nsca.c $(srcdir)/netutils.c $(srcdir)/utils.c $(SOL10_ASPRINTF) $(LDFLAGS) $(SOCKETLIBS) $(LIBWRAPLIBS)
 
 send_nsca: $(srcdir)/send_nsca.c $(srcdir)/netutils.c $(srcdir)/utils.c $(SRC_INCLUDE)/netutils.h $(SRC_INCLUDE)/utils.h $(SRC_INCLUDE)/common.h $(SRC_INCLUDE)/config.h
 	$(CC) $(CFLAGS) -o $@ $(srcdir)/send_nsca.c $(srcdir)/netutils.c $(srcdir)/utils.c $(LDFLAGS) $(SOCKETLIBS)
-- 
1.7.10.3

