From b0b1ebe1c567fe17c1f44d71b0dba956849bb31d Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <daÂm@opencsw.org>
Date: Sun, 15 Apr 2012 18:58:16 +0200
Subject: [PATCH] Do not use unnamed union

---
 libGeoIP/GeoIPCity.h |    7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/libGeoIP/GeoIPCity.h b/libGeoIP/GeoIPCity.h
index f6f8fc8..d9de181 100644
--- a/libGeoIP/GeoIPCity.h
+++ b/libGeoIP/GeoIPCity.h
@@ -39,10 +39,9 @@ typedef struct GeoIPRecordTag {
 	char *postal_code;
 	float latitude;
 	float longitude;
-	union {
-	  int metro_code; /* metro_code is a alias for dma_code */
-	  int dma_code;
-        };
+	int dma_code;
+/* metro_code is a alias for dma_code */
+#define metro_code dma_code
 	int area_code;
 	int charset;
 	char *continent_code;
-- 
1.7.10

