--- protobuf-2.3.0/src/google/protobuf/descriptor_database.cc.orig	2010-02-20 23:39:07.835347049 +0100
+++ protobuf-2.3.0/src/google/protobuf/descriptor_database.cc	2010-02-20 23:43:55.328201640 +0100
@@ -35,6 +35,7 @@
 #include <google/protobuf/descriptor_database.h>
 
 #include <set>
+#include <map>
 
 #include <google/protobuf/descriptor.pb.h>
 #include <google/protobuf/wire_format_lite_inl.h>
