--- protobuf-2.3.0/src/google/protobuf/compiler/command_line_interface.h.orig	2010-02-20 19:04:19.471295716 +0100
+++ protobuf-2.3.0/src/google/protobuf/compiler/command_line_interface.h	2010-02-20 19:05:14.390216481 +0100
@@ -55,6 +55,8 @@
 
 namespace compiler {
 
+using std::map;
+
 class CodeGenerator;        // code_generator.h
 class OutputDirectory;      // code_generator.h
 class DiskSourceTree;       // importer.h
