--- protobuf-2.3.0/src/google/protobuf/descriptor_database.h.orig	2010-02-20 23:33:10.978002985 +0100
+++ protobuf-2.3.0/src/google/protobuf/descriptor_database.h	2010-02-20 23:33:21.265334100 +0100
@@ -233,7 +233,7 @@
 
     // Find the last entry in the by_symbol_ map whose key is less than or
     // equal to the given name.
-    typename map<string, Value>::iterator FindLastLessOrEqual(
+    typename std::map<string, Value>::iterator FindLastLessOrEqual(
         const string& name);
 
     // True if either the arguments are equal or super_symbol identifies a
