diff --speed-large-files --minimal -Nru lftp-3.5.10.orig/configure.ac lftp-3.5.10/configure.ac
--- lftp-3.5.10.orig/configure.ac	2007-03-26 03:26:29.000000000 -0400
+++ lftp-3.5.10/configure.ac	2007-04-10 17:46:47.802962000 -0400
@@ -105,6 +105,7 @@
 LFTP_CXX_TEST
 CXX_DYNAMIC_INITIALIZERS
 LFTP_CXX_BOOL
+LFTP_CXX__BOOL
 LFTP_CXX_ANSI_SCOPE
 
 AC_ISC_POSIX
diff --speed-large-files --minimal -Nru lftp-3.5.10.orig/m4/lftp.m4 lftp-3.5.10/m4/lftp.m4
--- lftp-3.5.10.orig/m4/lftp.m4	2004-02-12 08:39:32.000000000 -0500
+++ lftp-3.5.10/m4/lftp.m4	2007-04-10 17:48:05.431586000 -0400
@@ -207,6 +207,27 @@
 #endif ])
 ])
 
+dnl try to build and run a dummy program
+AC_DEFUN([LFTP_CXX__BOOL],
+[
+   AC_MSG_CHECKING(whether $CXX supports _Bool type)
+   AC_CACHE_VAL(lftp_cv_cxx__Bool,
+   [
+      AC_LANG_PUSH(C++)
+      AC_TRY_COMPILE([_Bool t=true;bool f=false;],[],
+	 [lftp_cv_cxx__Bool=yes],[lftp_cv_cxx__Bool=no])
+      AC_LANG_POP(C++)
+   ])
+   AC_MSG_RESULT($lftp_cv_cxx__Bool)
+   if test x$lftp_cv_cxx__Bool = xyes; then
+      AC_DEFINE(HAVE_CXX__BOOL, 1, [define if c++ compiler supports _Bool])
+   fi
+   AH_VERBATIM([OPT_CPP__BOOL], [
+#if defined(__cplusplus) && !defined(HAVE_CXX__BOOL)
+   typedef bool _Bool;
+#endif ])
+])
+
 dnl check if C++ compiler needs extra arguments to grok ANSI scoping rules
 AC_DEFUN([LFTP_CXX_ANSI_SCOPE],
 [
