diff -ru ntop-3.3.10.old/configureextra/SOLARIS10 ntop-3.3.10/configureextra/SOLARIS10
--- ntop-3.3.10.old/configureextra/SOLARIS10	2007-05-03 23:31:40.000000000 +0200
+++ ntop-3.3.10/configureextra/SOLARIS10	2009-12-24 02:31:19.464874160 +0100
@@ -5,9 +5,9 @@
 #
 
 echo "        Setting Solaris 10 specific flag values"
-LDFLAGS="-L/opt/sfw/lib -L/usr/local/lib -R/opt/sfw/lib -R/usr/local/lib ${LDFLAGS}"
+LDFLAGS="-L/opt/csw/lib -R/opt/csw/lib ${LDFLAGS}"
 LIBS="-lnsl -lsocket ${LIBS}"
-CPPFLAGS="${CPPFLAGS} -I/usr/local/include -I/opt/sfw/include"
+CPPFLAGS="${CPPFLAGS} -I/opt/csw/include"
 
 . configureextra/functions
 
@@ -28,4 +28,4 @@
 echo "- ln -s libpcap.so.0.X.Y libpcap.so"
 echo "- rm libpcap.a"
 echo ""
-echo "*************************************************"
\ No newline at end of file
+echo "*************************************************"
diff -ru ntop-3.3.10.old/configureextra/SOLARIS8 ntop-3.3.10/configureextra/SOLARIS8
--- ntop-3.3.10.old/configureextra/SOLARIS8	2005-09-29 10:42:23.000000000 +0200
+++ ntop-3.3.10/configureextra/SOLARIS8	2009-12-24 02:33:36.592681360 +0100
@@ -1,8 +1,9 @@
 #!/bin/sh
 
 echo "        Setting Solaris 8 specific flag values"
-LDFLAGS="-L/usr/local/lib"
+LDFLAGS="-L/opt/csw/lib -R/opt/csw/lib ${LDFLAGS}"
 LIBS="-lnsl -lsocket ${LIBS}"
+CPPFLAGS="${CPPFLAGS} -I/opt/csw/include"
 
 . configureextra/functions
 
diff -ru ntop-3.3.10.old/configureextra/SOLARIS9 ntop-3.3.10/configureextra/SOLARIS9
--- ntop-3.3.10.old/configureextra/SOLARIS9	2003-12-02 13:03:47.000000000 +0100
+++ ntop-3.3.10/configureextra/SOLARIS9	2009-12-24 02:33:21.611018077 +0100
@@ -1,9 +1,9 @@
 #!/bin/sh
 
 echo "        Setting Solaris 9 specific flag values"
-LDFLAGS="-L/opt/sfw/lib -L/usr/local/lib -R/opt/sfw/lib -R/usr/local/lib ${LDFLAGS}"
+LDFLAGS="-L/opt/csw/lib -R/opt/csw/lib ${LDFLAGS}"
 LIBS="-lnsl -lsocket ${LIBS}"
-CPPFLAGS="${CPPFLAGS} -I/usr/local/include -I/opt/sfw/include"
+CPPFLAGS="${CPPFLAGS} -I/opt/csw/include"
 
 . configureextra/functions
 
