--- ntop-3.3.10.old/configure.in	2009-05-06 16:55:22.000000000 +0200
+++ ntop-3.3.10/configure.in	2009-12-24 02:57:08.602866458 +0100
@@ -274,6 +274,15 @@
               [  --with-pcap-root=DIR        LBNL pcap located in DIR],
               PCAP_ROOT="$withval",
               PCAP_ROOT=)
+AC_ARG_WITH(  pcap-lib,
+              [  --with-pcap-lib=DIR          or libpcap located in DIR],
+              PCAP_LIB="$withval",
+              PCAP_LIB=)
+AC_ARG_WITH(  pcap-include,
+              [  --with-pcap-include=DIR      or pcap.h located in DIR],
+              PCAP_INCLUDE="$withval",
+              PCAP_INCLUDE=)
+
 AC_ARG_WITH(  gdbm-root,
               [  --with-gdbm-root=DIR        gdbm located in DIR],
               GDBM_DIRECTORY="$withval",
@@ -670,6 +679,7 @@
        INCS="${INCS} -I/usr/include/pcap"
        AC_MSG_RESULT([found in $PCAP_ROOT])
 else
+     NTOP_APPENDS([${PCAP_INCLUDE}], [${PCAP_LIB}], [pcap])
      AC_CHECK_LIB(pcap, pcap_lookupdev, , [
        AC_MSG_RESULT([             *** FATAL ERROR ***             ])
        AC_MSG_RESULT([ It looks that you don't have the libpcap distribution installed.])
@@ -1659,7 +1669,7 @@
    wget http://www.lua.org/ftp/$LUA_VERSION.tar.gz
 fi
 
-tar xvfz $LUA_VERSION.tar.gz
+gtar xvfz $LUA_VERSION.tar.gz
 cat $LUA_VERSION/src/Makefile | sed -e s,'MYCFLAGS=-DLUA_USE_POSIX',' MYCFLAGS="-fPIC -DLUA_USE_POSIX"',g > /tmp/lua.temp
 cat /tmp/lua.temp >  $LUA_VERSION/src/Makefile
 #rm -f /tmp/lua.temp
@@ -1968,7 +1978,7 @@
 else
    wget http://www.maxmind.com/download/geoip/api/c/GeoIP.tar.gz
 fi
-tar xvfz GeoIP.tar.gz
+gtar xvfz GeoIP.tar.gz
 GEO_DIR=`find $PWD -type d -name "GeoIP-*"`
 cd $GEO_DIR; ./configure --prefix=${prefix}; make; cd ..
 # OSX Fix
