From 6c03bcffdf2504e7ae757128c3e2335d3fe7cd77 Mon Sep 17 00:00:00 2001
From: Yann Rouillard <yann@pleiades.fr.eu.org>
Date: Sat, 20 Apr 2013 19:04:04 +0200
Subject: [PATCH] Fix building with fribidi2

---
 frm/common.c | 34 +++++++++++++----------
 1 files changed, 16 insertions(+), 11 deletions(-)

diff --git a/frm/common.c b/frm/common.c
index 4ec73c5..a32b10e 100644
--- a/frm/common.c
+++ b/frm/common.c
@@ -62,10 +62,10 @@ static char *output_charset = NULL;
 const char *
 get_charset ()
 {
-  char *tmp;
-  
   if (!output_charset)
     {
+      char *tmp;
+      const char *str = NULL;
       char locale[32];
       
       memset (locale, 0, sizeof (locale));
@@ -86,16 +86,16 @@ get_charset ()
 	  
 	  lang = strtok_r (locale, "_", &sp);
 	  terr = strtok_r (NULL, ".", &sp);
-	  output_charset = strtok_r (NULL, "@", &sp);
+	  str = strtok_r (NULL, "@", &sp);
 
-	  if (output_charset)
-	    output_charset = xstrdup (output_charset);
-	  else
-	    output_charset = mu_charset_lookup (lang, terr);
+	  if (!str)
+	    str = mu_charset_lookup (lang, terr);
 	}
       
-      if (!output_charset)
-	output_charset = "ASCII";
+      if (!str)
+        str = "ASCII";
+
+      output_charset = xstrdup (str);
     }
   return output_charset;
 }
@@ -104,6 +104,9 @@ get_charset ()
 /* BIDI support (will be moved to lib when it's ready) */
 #ifdef HAVE_LIBFRIBIDI
 
+#   include <wchar.h>
+#   define mu_fribidi_wcwidth(c) wcwidth((wchar_t)c)
+
 static int fb_charset_num = -1;
 FriBidiChar *logical;
 char *outstring;
@@ -170,7 +184,7 @@ puts_bidi (char *string)
 	      if (fb_charset_num != FRIBIDI_CHARSET_CAP_RTL)
 		{
 		  while (wid > 0 && idx < len)
-		    wid -= fribidi_wcwidth (visual[idx++]);
+		    wid -= mu_fribidi_wcwidth (visual[idx++]);
 		}
 	      else
 		{
-- 
1.8.1.4

