From b2c9549631b42638687e261eefb9bc8981032016 Mon Sep 17 00:00:00 2001
From: Yann Rouillard <yann@pleiades.fr.eu.org>
Date: Sun, 21 Apr 2013 00:57:25 +0200
Subject: [PATCH] Add missing libraries

---
 mail.local/Makefile.in         |   5 +
 1 files changed, 5 insertions(+), 0 deletions(-)

diff --git a/mail.local/Makefile.in b/mail.local/Makefile.in
index 3e2e2d8..00a86b0 100644
--- a/mail.local/Makefile.in
+++ b/mail.local/Makefile.in
@@ -374,6 +374,11 @@ mail_local_LDADD = \
  ../lib/libmuaux.la \
  ${MU_LIB_SIEVE}\
  ${MU_LIB_MBOX}\
+ ${MU_LIB_IMAP}\
+ ${MU_LIB_MAILDIR}\
+ ${MU_LIB_MH}\
+ ${MU_LIB_NNTP}\
+ ${MU_LIB_POP}\
  ${MU_LIB_AUTH}\
  @MU_AUTHLIBS@\
  ${MU_LIB_MAILUTILS} \
-- 
1.8.1.4

