From 722db871d1a24c3ceb7cc6a297dc2a3278534699 Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Mon, 17 Jan 2011 20:32:12 +0100
Subject: [PATCH] Do not use -Wall

---
 Makefile.in |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Makefile.in b/Makefile.in
index 33bdd06..272a5b1 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -232,7 +232,7 @@ top_srcdir = @top_srcdir@
 EXTRA_DIST = README TODO Copyright COPYING doc/xmlstarlet.txt
 WIN32_DIST = $(EXTRA_DIST) AUTHORS ChangeLog INSTALL NEWS xml.exe doc
 ACLOCAL_AMFLAGS = -I m4
-AM_CFLAGS = -Wall
+AM_CFLAGS =
 
 # testing
 egdir = $(srcdir)/examples
-- 
1.7.3.2

